{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "d2fce4db",
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio.Entrez import efetch\n",
    "from dicttoxml import dicttoxml\n",
    "from IPython.display import clear_output\n",
    "import json\n",
    "\n",
    "global entry_data\n",
    "\n",
    "def query_pubmed(pmid_file):\n",
    "    with open(pmid_file, \"r\") as f:\n",
    "        for pmid in f:\n",
    "            pmid = pmid.rstrip()\n",
    "            try:\n",
    "                handle = efetch(db=\"pubmed\", id=pmid, retmode=\"xml\", rettype=\"abstract\")\n",
    "            except:\n",
    "                yield pmid, None\n",
    "            yield pmid, handle.read()\n",
    "            \n",
    "def parse_pubmed_query(raw_text):\n",
    "    title, abstract, medline = \"None\", \"None\", \"None\"\n",
    "    raw_text = \"\".join(raw_text[0].decode(\"utf-8\"))\n",
    "    title_start_idx, title_end_idx = raw_text.find(\"<ArticleTitle>\") + len(\"<ArticleTitle>\"), raw_text.find(\"</ArticleTitle>\") \n",
    "    title = raw_text[title_start_idx:title_end_idx]\n",
    "    abstract_start_idx, abstract_end_idx = raw_text.find(\"<AbstractText>\") + \\\n",
    "        len(\"<AbstractText>\"), raw_text.find(\"</AbstractText>\") \n",
    "    if \"<ChemicalList>\" in raw_text:\n",
    "        medline_start_idx, medline_end_idx = raw_text.find(\"<ChemicalList>\") + len(\"<ChemicalList>\"), \\\n",
    "            raw_text.find(\"</ChemicalList>\")\n",
    "        medline = raw_text[medline_start_idx: medline_end_idx]\n",
    "    abstract = raw_text[abstract_start_idx:abstract_end_idx]\n",
    "    \n",
    "    print(title)\n",
    "    \n",
    "    return {\"title\": title, \"abstract\": abstract, \"medline\": medline}\n",
    "            \n",
    "def pubmed_query_to_train(pmid_file, train_size=10000):\n",
    "    global entry_data\n",
    "    entry_data = dict()\n",
    "    current_idx = 0\n",
    "    for pmid, *data in query_pubmed(pmid_file):\n",
    "        print(current_idx, \"/\", train_size)\n",
    "        try:\n",
    "            data = parse_pubmed_query(data)\n",
    "        except:\n",
    "            data = {\"title\": None, \"abstract\": None, \"medline\": None}\n",
    "        entry_data[pmid] = data\n",
    "        clear_output(wait=True)\n",
    "        current_idx += 1\n",
    "    json_data      = json.dumps(entry_data)\n",
    "        \n",
    "    with open(\"proteomics_output.json\", \"w+\", encoding='utf-8') as out:\n",
    "        out.write(json_data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "3d594cc9",
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'pubmed_query_to_train' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-33-384964d5ff4f>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m      1\u001b[0m \u001b[1;32mdef\u001b[0m \u001b[0mdownload_pubmed_data\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mfile_path\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      2\u001b[0m     \u001b[0mpubmed_query_to_train\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m'pmid-proteomics-set.txt'\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m----> 3\u001b[1;33m \u001b[0mdownload_pubmed_data\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m\"\"\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[1;32m<ipython-input-33-384964d5ff4f>\u001b[0m in \u001b[0;36mdownload_pubmed_data\u001b[1;34m(file_path)\u001b[0m\n\u001b[0;32m      1\u001b[0m \u001b[1;32mdef\u001b[0m \u001b[0mdownload_pubmed_data\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mfile_path\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m----> 2\u001b[1;33m     \u001b[0mpubmed_query_to_train\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m'pmid-proteomics-set.txt'\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m      3\u001b[0m \u001b[0mdownload_pubmed_data\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m\"\"\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mNameError\u001b[0m: name 'pubmed_query_to_train' is not defined"
     ]
    }
   ],
   "source": [
    "def download_pubmed_data(file_path):\n",
    "    pubmed_query_to_train('pmid-proteomics-set.txt')\n",
    "download_pubmed_data(\"\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "796b4cc6",
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "from unidecode import unidecode\n",
    "\n",
    "json_data = None\n",
    "\n",
    "with open(\"proteomics_output.json\", \"r\", encoding=\"utf-8\") as f:\n",
    "    json_data = json.loads(f.read(), strict=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "5125f9d1",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[nltk_data] Downloading package stopwords to\n",
      "[nltk_data]     C:\\Users\\joshu\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package stopwords is already up-to-date!\n",
      "100%|██████████| 10000/10000 [00:10<00:00, 945.05it/s]\n",
      "100%|██████████| 10000/10000 [00:00<00:00, 1999668.18it/s]\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "from sklearn.feature_extraction.text import TfidfVectorizer\n",
    "\n",
    "from nltk.stem import PorterStemmer\n",
    "from nltk.tokenize import sent_tokenize, word_tokenize\n",
    "\n",
    "from tqdm import tqdm\n",
    "import nltk\n",
    "from nltk.corpus import stopwords\n",
    "\n",
    "nltk.download(\"stopwords\")\n",
    "\n",
    "tokens = []\n",
    "sentences = []\n",
    "\n",
    "for pmid in tqdm(json_data.keys()):\n",
    "    document_abstract = json_data[pmid][\"abstract\"]\n",
    "    document_tokens   = word_tokenize(document_abstract)\n",
    "    tokens.extend(document_tokens)\n",
    "    \n",
    "for pmid in tqdm(json_data.keys()):\n",
    "    document_abstract = json_data[pmid][\"abstract\"]\n",
    "    sentences.append(document_abstract)\n",
    "\n",
    "porter_stemmer    = PorterStemmer()\n",
    "pre_tf_idf_tokens = []\n",
    "\n",
    "for word in tokens:\n",
    "    pre_tf_idf_tokens.append(porter_stemmer.stem(word))\n",
    "\n",
    "tf_idf_vec_smooth = TfidfVectorizer(use_idf=True,  \n",
    "                        smooth_idf=True,  \n",
    "                        ngram_range=(1,1), stop_words='english')\n",
    "\n",
    "X_unigram = tf_idf_vec_smooth.fit_transform(pre_tf_idf_tokens)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "91058aef",
   "metadata": {},
   "outputs": [],
   "source": [
    "tf_idf_vec_smooth_bigram = TfidfVectorizer(use_idf=True,\n",
    "                                  smooth_idf=True,\n",
    "                                  ngram_range=(1,2), stop_words='english')\n",
    "\n",
    "X_unigram_bigram = tf_idf_vec_smooth_bigram.fit_transform(pre_tf_idf_tokens)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "id": "359e20db",
   "metadata": {},
   "outputs": [],
   "source": [
    "from numpy import count_nonzero\n",
    "\n",
    "def calculate_sparsity(sparse_matrix):\n",
    "    sparsity = 1.0 - ( count_nonzero(sparse_matrix) / float(sparse_matrix.size) )\n",
    "    return sparsity"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "id": "ac716569",
   "metadata": {},
   "outputs": [],
   "source": [
    "from nltk.corpus import wordnet\n",
    "import string\n",
    "import re\n",
    "from nltk.stem import WordNetLemmatizer\n",
    "\n",
    "words = set(nltk.corpus.words.words())\n",
    "\n",
    "def is_english_token(token):\n",
    "    if not wordnet.synsets(token):\n",
    "        print(token)\n",
    "        \n",
    "        \n",
    "translator=str.maketrans('','',string.punctuation)\n",
    "lemmatizer = WordNetLemmatizer()\n",
    "\n",
    "def preprocess_sentence(sentence, translator):\n",
    "    sentence = sentence.lower()\n",
    "    if 'doctype' in sentence: #flagging badly encoded JSON documents\n",
    "        return \"\"\n",
    "    out = sentence.translate(translator)\n",
    "    out = re.sub(\"\\d+\\.?\\d+?\", \"\",  out)\n",
    "    out = re.sub(\"[^\\u0000-\\u05C0\\u2100-\\u214F]+\", \"\", out)\n",
    "    \n",
    "    #greek aware lemmatizer required for proper lemmatization\n",
    "    out = \" \".join([lemmatizer.lemmatize(token) for token in nltk.word_tokenize(out) if len(token) > 2])\n",
    "    \n",
    "    return out"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 129,
   "id": "5e5bc10f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['proteomics involves the application technology for the identication and quantication overall protein present content cell tissue organism supplement the other omics technology such genomic and transcriptomics expound the identity protein organism and cognize the structure and function particular protein proteomicsbased technology are utilized various capacity for different research setting such detection various diagnostic marker candidate for vaccine production understanding pathogenicity mechanism alteration expression pattern response different signal and interpretation functional protein pathway different disease proteomics practically intricate because includes the analysis and categorization overall protein signature genome mass spectrometry with lcmsms and malditoftof being widely used equipment the central among current proteomics however utilization proteomics facility including the software for equipment database and the requirement skilled personnel substantially increase the cost therefore limit their wider use especially the developing world furthermore the proteome highly dynamic because complex regulatory system that control the expression level protein this review effort describe the various proteomics approach the recent development and their application research and analysis',\n",
       " 'mass spectrometry msbased highthroughput proteomics the core technique for largescale protein characterization due the extreme complexity proteome sophisticated separation technique and advanced instrumentation have been developed extend coverage and enhance dynamic range and sensitivity this review discus the separation and prefractionation technique applied for largescale analysis both bottomup peptidelevel and topdown proteinlevel proteomics different approach for quantifying peptide intact protein including labelfree and stableisotopelabeling strategy are also discussed addition present brief overview different type mass analyzer and fragmentation technique well selected emerging technique',\n",
       " 'mass spectrometrybased proteomics ha been extensively used map bacterial proteome which ha led better understanding the molecular mechanism underlying bacterial infection and bacteriahost interaction quantitative proteomics using selected parallel reaction monitoring considered one the most sensitive and specific quantitative msbased approach and ha significantly advanced proteome study pathogenic bacteria here recent application targeted proteomics for bacteria identification biomarker discovery and the characterization bacterial virulence and antimicrobial resistance are reviewed among others result such study are expected further contribute improve the fight against the most common human pathogenic bacteria',\n",
       " 'spermatogenesis complex and tightly regulated process leading the continuous production male gamete the spermatozoon this developmental process requires the sequential and coordinated expression thousand gene including many that are testisspecific the molecular network underlying normal and pathological spermatogenesis have been widely investigated recent decade and many highthroughput expression study have studied gene and protein involved male fertility this review focus study that have attempted correlate transcription and translation during spermatogenesis comparing the testicular transcriptome and proteome also discus the recent development and use new transcriptomic approach that provide better proxy for the proteome from both qualitative and quantitative perspective finally provide illustration how testisderived transcriptomic and proteomic data can integrated address new question and how the proteomics informed transcriptomics technique combining rnaseq and msbased proteomics can contribute significantly the discovery new proteincoding gene new protein isoforms expressed during spermatogenesis',\n",
       " 'amyloidoses are spectrum disorder caused abnormal folding and extracellular deposition protein the deposit lead tissue damage and organ dysfunction particularly the heart kidney and nerve there are least different protein that can cause amyloidosis the clinical management depends entirely the type protein deposited and thus the underlying pathogenesis and often requires highrisk therapeutic intervention application mass spectrometrybased proteomic technology for analysis amyloid plaque ha transformed the way amyloidosis diagnosed and classified proteomic assay have been extensively used for clinical management patient with amyloidosis providing unprecedented diagnostic and biological information they have shed light the pathogenesis different amyloid type and have led identification numerous new amyloid type including alect2 amyloidosis which now recognized one the most common cause systemic amyloidosis north america',\n",
       " 'despite the explosive increase the use assisted reproductive technology art over the last year their success rate remain suboptimal proteomics rapidlyevolving technologydriven science that ha already been widely applied the exploration human reproduction and fertility providing useful insight into it physiology and leading the identification numerous protein that may potential biomarkers andor treatment target successful art pregnancy here present brief overview the technique used proteomic analysis and attempt comprehensive presentation recent data from mass spectrometrybased proteomic study human regarding all component art including the male and female gamete the derived zygote and embryo the endometrium and finally the art offspring both pre and postnatally',\n",
       " 'proteomics allows the simultaneous detection and identification thousand protein within sample here describe quantitative method compare protein expression and subcellular localization different cell line representative different stage colorectal cancer using stable isotope labeling with amino acid culture silac also describe biochemical fractionation approach separate different cellular compartment and the necessary step obtain specific proteomic profile each cell line this technique enables comprehensive proteomic analysis cancer cell line and the identification pathway that are deregulated different cancer cell line',\n",
       " 'nextgeneration sequencing allows the analysis genome including those representing disease state however the cause most disorder are multifactorial and systemslevel approach including the analysis proteome are required for more comprehensive understanding the proteome extremely multifaceted owing splicing and protein modification and this further amplified the interconnectivity protein into complex and signalling network that are highly divergent time and space proteome analysis heavily relies mass spectrometry msbased proteomics starting mature and deliver through combination development instrumentation sample preparation and computational analysis here describe this emerging next generation proteomics and highlight recent application',\n",
       " 'protein are the key player many cellular process their composition trafficking and interaction underlie the dynamic process life furthermore disease are frequently accompanied malfunction protein multiple level understanding how biological process are regulated the protein level critically important understanding the molecular basis for disease and often shed light disease prevention diagnosis and treatment with rapid advance mass spectrometry instrument and experimental methodology msbased proteomics ha become reliable and essential tool for elucidating biological process the protein level over the past decade have witnessed great expansion knowledge human disease with the application msbased proteomic technology which ha led many exciting discovery herein review the recent progress msbased proteomics biomedical research including that establishing diseaserelated proteome and interactomes also discus how this progress will benefit biomedical research and clinical diagnosis and treatment disease',\n",
       " 'given superior analytical feature proteomics well suited for the basic investigation and clinical diagnosis human disease modern enables detailed functional characterization the pathogenic biochemical process achieved accurate and comprehensive quantification protein and their regulatory chemical modification here describe how highaccuracy combination with highresolution chromatographic separation can leveraged meet these analytical requirement mechanismfocused manner review the quantification method capable producing accurate measurement protein abundance and posttranslational modification stoichiometry then discus how experimental design and chromatographic resolution can leveraged achieve comprehensive functional characterization biochemical process complex biological proteome finally describe current approach for quantitative analysis common functional protein modification reversible phosphorylation all current instrumentation and method highresolution chromatography and proteomics are poised for immediate translation into improved diagnostic strategy for pediatric and adult disease',\n",
       " 'the enormous diversity proteoforms produce tremendous complexity within cellular proteome facilitates intricate network molecular interaction and constitutes formidable analytical challenge for biomedical researcher currently quantitative wholeproteome profiling often relies nontargeted liquid chromatographymass spectrometry lcm which sample proteoforms broadly but can suffer from lower accuracy sensitivity and reproducibility compared with targeted lcm recent advance bottomup proteomics using targeted lcm have enabled previously unachievable identification and quantification target protein and posttranslational modification within complex sample consequently targeted lcm rapidly advancing biomedical research especially system biology research diverse area that include proteogenomics interactomics kinomics and biological pathway modeling with the recent development targeted lcm assay for nearly the entire human proteome targeted lcm positioned enable quantitative proteomic profiling unprecedented quality and accessibility support fundamental and clinical research here review recent application bottomup proteomics using targeted lcm for system biology research significance advance targeted proteomics are rapidly advancing system biology research recent application include systemslevel investigation focused posttranslational modification such phosphoproteomics protein conformation proteinprotein interaction kinomics proteogenomics and metabolic and signaling pathway notably absolute quantification metabolic and signaling pathway protein ha enabled accurate pathway modeling and engineering integration targeted proteomics with other technology such rnaseq ha facilitated diverse research such the identification hundred missing human protein gene and transcript that appear encode protein but direct experimental evidence wa lacking',\n",
       " 'singlecell proteomics mass spectrometry scopems recently introduced method quantify multiplexed singlecell proteome while this technique ha generated great excitement the underlying technology isobaric labeling and mass spectrometry have technical limitation with the potential affect data quality and biological interpretation these limitation are particularly relevant when carrier proteome sample added the amount singlecell proteome used enable peptide identification here perform controlled experiment with increasing carrier proteome amount and evaluate quantitative accuracy relates mass analyzer dynamic range multiplexing level and number ion sampled demonstrate that increase carrier proteome level requires concomitant increase the number ion sampled maintain quantitative accuracy lastly introduce singlecell proteomics companion scpcompanion software tool that enables rapid evaluation singlecell proteomic data and recommends instrument and data analysis parameter for improved data quality',\n",
       " 'organism exposed plethora environmental perturbation undergoes proteomic change which enable the characterization total protein much the proteomic information obtained from genomic data additional information the proteome such posttranslational modification proteinprotein interaction protein localization metabolic pathway and are deduced using proteomic tool which genomics and transcriptomics fail offer the proteomic analysis allows identification precise change protein which turn solve the complexity microbial population providing insight into the microbial metabolism cellular pathway and behavior microorganism new environment furthermore they provide clue for the exploitation their special feature for biotechnological application numerous technique for the analysis microbial proteome such electrophoretic chromatographic mass spectrometricbased method well quantitative proteomics are available which facilitate protein separation expression identification and quantification protein understanding the potential each the proteomic tool ha created significant impact diverse microbiological aspect and the same ha been discussed this review',\n",
       " 'mass spectrometry ha arguably become the core technology proteomics the application mass spectrometry based technique for the qualitative and quantitative analysis global proteome sample derived from complex mixture ha had big impact the understanding cellular function here give brief introduction principle mass spectrometry and instrumentation currently used proteomics experiment addition recent development the application mass spectrometry proteomics are summarised strategy allowing highthroughput identification protein from highly complex mixture include accurate mass measurement peptide derived from total proteome digest and multidimensional peptide separation coupled with mass spectrometry mass spectrometric analysis intact protein permit the characterisation protein isoforms recent development stable isotope labelling technique and chemical tagging allow the mass spectrometry based differential display and quantitation protein and newly established affinity procedure enable the targeted characterisation posttranslationally modified protein finally advance mass spectrometric imaging allow the gathering specific information the local molecular composition relative abundance and spatial distribution peptide and protein thin tissue section',\n",
       " 'human individual differ from one another almost all their gene due single nucleotide polymorphism snp when the maternal and the paternal genome become combined individual the two allele each gene represent arbitrary combination consequence individual show high variability protein expression furthermore within proteome the protein form network proteinprotein interaction these network differ between individual robustness against genetic orand environmental perturbation due polymorphism which differ type and composition between individual and modify the arrangement protein the proteomic network general conclusion the robustness human individual against disease may depend the structure and expression the proteinprotein interaction network that varies it functional efficiency between individual due networkpolymorphisms this article part special issue entitled year proteomics memory viatliano pallini guest editor luca bini juan calvete natacha turck denis hochstrasser and jeancharles sanchez',\n",
       " 'antibodybased proteomic approach play important role highthroughput multiplexed protein expression profiling health and disease these antibodybased technology will provide miniaturized setup capable the simultaneously profiling numerous protein specific sensitive and rapid manner targeting high well lowabundant protein even crude proteome such serum the generated protein expression pattern proteomic snapshot can then transformed into proteomic map detailed molecular fingerprint revealing the composition the target sample proteome molecular level using bioinformatics candidate biomarker signature can deciphered and evaluated for clinical applicability the approach will provide unique opportunity for disease diagnostics biomarker discovery patient stratification predicting disease recurrence and evidencebased therapy selection this review describe the current status the antibodybased proteomic approach focusing antibody array furthermore the current benefit and limitation the approach well set selected key application outlining the applicative potential will discussed',\n",
       " 'the complex interaction among protein and protein with small molecular weight protein ligand are overturned every time one the component the network missing for study purpose animal model lacking one protein are obtained experimental manipulation the genome the knocking out approach gene altered through the insertion artificial dna sequence which halt the transcriptiontranslation sequence event this review have compiled the research paper that analyze the effect knocking out individual gene the proteome various tissuesorgans throughout the body have gathered and organized all the available evidence and then compared the proteomic data order stress the contextspecificity the outcome every time two more organ were investigated the same mouse finally symmetrical approach the above surveyed whether there any obvious overlap among the effect different the same organ marking affection general pathway lacking specificity the gene targeting specific attention wa put the possible involvement cellular stress marker',\n",
       " '',\n",
       " '',\n",
       " 'the sequencing the human genome and that numerous pathogen ha opened the door for proteomics providing sequencebased framework for mining proteome result there intense interest applying proteomics foster better understanding disease process develop new biomarkers for diagnosis and early detection disease and accelerate drug development this interest creates numerous opportunity well challenge meet the need for high sensitivity and high throughput required for diseaserelated investigation',\n",
       " 'detailed investigation the neural crest cell proteome ha lingered behind that the transcriptome due the challenge obtaining sufficient starting material for subsequent proteomic investigation compounded the complexity protein abundance large number posttranslational modification and the stochastic nature proteomics approach little data far exists describing the true neural crest cell proteome however recent advance instrument sensitivity and recovery material during sample preparation have alleviated many these problem and make proteomics analysis underutilized tool study neural crest cell biology here present quantitative proteomics protocol for deep analysis both whole proteome and posttranslational modification and separate protocol for ultrasensitive proteomic profiling from submicrogram amount protein',\n",
       " 'there increasing realisation that human health status adulthood depends critically upon environmental condition pertaining around the time conception and during pregnancy poor maternal diet adverse environmental condition around the periconception period somehow induces the resultant embryo adapt predictively order survive this level stress for the whole it life however there mismatch between expectation and reality where the condition during later life are better than expected thing wrong and the adult suffers range illness including diabetes heart disease hypertension and stroke understanding the molecular signal that direct the early embryo adopt appropriate adaptation suit it future life would extremely valuable however although appears ideal task for proteomic application there are technical ethical and practical limitation what can achieved with the current framework proteomic technology here review what ha been achieved date explain some the experimental problem and suggest some strategy for taking this field forward',\n",
       " 'lactobacillus are mainly used for the manufacture fermented dairy sourdough meat and vegetable food used probiotic under optimal processing condition lactobacillus strain contribute food functionality through their enzyme portfolio and the release metabolite extensive genomic diversity analysis wa conducted elucidate the core feature the genus lactobacillus and provide better comprehension niche adaptation the strain however proteomics indispensable omics science elucidate the proteome diversity and the mechanism regulation and adaptation lactobacillus strain this review focus the novel and comprehensive knowledge functional proteomics and metaproteomics lactobacillus specie large list proteomic case study different lactobacillus specie provided illustrate the adaptability the main metabolic pathway carbohydrate transport and metabolism pyruvate metabolism proteolytic system amino acid metabolism and protein synthesis various life condition these investigation have highlighted that lactobacillus modulate the level complex panel protein growthsurvive different ecological niche addition the general regulation and stress response specific metabolic pathway can switched and off modifying the behavior the strain',\n",
       " 'for the last century have relied model organism help understand fundamental biological process now with advancement genome sequencing assembly and annotation nonmodel organism may studied with the same advanced bioanalytical toolkit model organism proteomics one such technique which classically relies predicted protein sequence catalog and measure complex proteome across tissue and biofluids applying proteomics nonmodel organism can advance and accelerate biomimicry study biomedical advancement veterinary medicine agricultural research behavioral ecology and food safety this postmodel organism era can study almost any specie meaning that many nonmodel organism are fact important emerging model organism herein specifically focus eukaryotic organism and discus the step generate sequence database analyze proteomic data with without database and interpret result well future research opportunity proteomics more accessible than ever before and will continue rapidly advance the coming year enabling critical research and discovery nonmodel organism that were hitherto impossible',\n",
       " '',\n",
       " 'over the last forty year global aquaculture presented growth rate per annum with amazing production million tonne and contribution aquatic animal food for human consumption order meet the world health requirement fish protein continuous growth production still expected for decade come aquaculture though very competitive market and global awareness regarding the use scientific knowledge and emerging technology obtain better farmed organism through sustainable production ha enhanced the importance proteomics seafood biology research proteomics powerful comparative tool ha therefore been increasingly used over the last decade address different question aquaculture regarding welfare nutrition health quality and safety this paper will give overview these biological question and the role proteomics their investigation outlining the advantage disadvantage and future challenge brief description the proteomics technical approach will presented special focus will the latest trend related the aquaculture production fish with defined nutritional health quality property for functional food and the integration proteomics technique addressing this challenging issue',\n",
       " 'this review provides introduction for the nonspecialist proteomics and particular the major approach available for global protein identification and quantification proteomics technology offer considerable opportunity for improved biological understanding and biomarker discovery the central platform for proteomics tandem mass spectrometry but number other technology resource and expertise are absolutely required perform meaningful experiment these include protein separation science and protein biochemistry general genomics and bioinformatics there are range workflow available for protein peptide separation prior tandem and subsequent bioinformatics analysis achieve protein identification the predominant approach are electrophoresis 2de and subsequent liquid chromatographyms lcm and gelcms beyond protein identification there are number wellestablished option available for protein quantification difference gel electrophoresis dige following 2de one option but msbased method most commonly itraqisobaric tag for relative and absolute quantification silacstable isotope labeling amino acid are now the preferred option sample preparation critical performing good experiment and subcellular fractionation can additionally provide protein localization information compared with whole cell lysates differential detergent solubilization another valid option with biological fluid possible remove the most abundant protein immunodepletion sample enrichment also used extensively certain analysis and most commonly phosphoproteomics with the initial purification phosphopeptides proteomics produce considerable datasets and resource facilitate the necessary extended analysis this data are improving all the time beyond the opportunity afforded proteomics there are definite challenge achieving full proteomic coverage proteome are highly complex and identifying and quantifying low abundance protein significant issue additionally the analysis poorly soluble protein such membrane protein and multiprotein complex difficult however without doubt that proteomics ha already provided significant insight into biological function and this will continue the technology continues improve also anticipate that the promise proteomics term biomarker discovery will increasingly realized',\n",
       " 'polycystic ovary syndrome pcos complex endocrine disorder heterogeneous etiology proteomics technique have been used for elucidating the physiopathology pcos yet the protein identified far were rarely the same across tissue and study the present review discus the current challenge the application proteomics the study pcos welldefined research design and appropriate selection study population sample and proteomic platform are essential clinical proteomics furthermore the finding derived from proteomic approach should validated complementary technique and the reproducibility the result ha ideally confirmed different study only when meeting these requirement the protein identified proteomic technique should considered candidate for future study aiming define specific molecular phenotype pcos and their possible role the metabolic and hormonal abnormality characteristic this syndrome',\n",
       " 'proteome are highly dynamic and can respond rapidly environmental and cellular signal within cell protein often form distinct pool with different function and property however quantitative proteomics study common measure averaged value for protein that not reflect variation that may occur between different protein isoforms different subcellular compartment cell different cell cycle stage and here review experimental approach that can used enhance the signal from specific pool protein that may otherwise obscured through averaging across protein population this signal enhancement can help reveal function associated with specific protein pool providing insight into the regulation cellular process review different strategy for proteomic signal enhancement with focus the analysis protein pool different subcellular location describe how msbased proteome analysis can combined with general physicochemical cell fractionation procedure that can applied many cultured cell line',\n",
       " 'proteomic approach are increasingly being used complement genetic and rna interferencebased study gene function caenorhabditis elegans several strategy isolate protein complex from whole worm and individual differentiated cell type have been described vivo labelling method have also been developed quantitatively ass proteomewide change depending genetic composition developmental stage here review proteomic approach that are becoming part the essential toolbox for study gene function elegans and highlight specific example where their application ha led important new insight',\n",
       " 'the inability make broad minimally biased measurement cell proteome stand major bottleneck for understanding how gene expression translates into cellular phenotype unlike sequencing for nucleic acid there dominant method for making singlecell proteomic measurement instead method typically focus either absolute quantification small number protein highly multiplexed protein measurement advance microfluidics and output encoding have led major improvement both aspect here review the most recent progress that ha enabled hundred protein measurement and ultrahighsensitivity quantification also highlight emerging technology such singlecell mass spectrometry that may enable unbiased measurement cellular proteome',\n",
       " 'more than year ago wa recognized that platelet are key mediator hemostasis nowadays established that platelet participate additional physiological process and contribute the genesis and progression cardiovascular disease recent data indicate that the platelet proteome defined the complete set expressed protein comprises protein and highly similar between different healthy individual owing their anucleate nature platelet have limited protein synthesis implication patient experiencing platelet disorder platelet dysfunction almost completely attributable alteration protein expression and dynamic difference posttranslational modification modern platelet proteomics approach can reveal quantitative change the abundance thousand protein posttranslational modification proteinprotein interaction and protein localization while requiring only small blood donation the range few milliliter consequently platelet proteomics will represent invaluable tool for characterizing the fundamental process that affect platelet homeostasis and thus determine the role platelet health and disease this article provide critical overview the achievement the current possibility and the future perspective platelet proteomics study patient experiencing cardiovascular inflammatory and bleeding disorder',\n",
       " 'structural proteomics one the powerful research area the postgenomic era elucidating structurefunction relationship uncharacterized gene product based the protein structure proposes biochemical and cellular function unannotated protein and thereby identifies potential drug design and protein engineering target recently number pioneering group structural proteomics research have achieved proof structural proteomic theory predicting the structure hypothetical protein that successfully identified the biological function those protein the pioneering group made use number technique including nmr spectroscopy which ha been applied successfully structural proteomics study over the past year addition advance hardware design data acquisition method sample preparation and automation data analysis have been developed and successfully applied highthroughput structure determination technique these effort ensure that nmr spectroscopy will become important methodology for performing structural proteomics research genomic scale nmrbased structural proteomics together with xray crystallography will provide comprehensive structural database predict the basic biological function hypothetical protein identified the genome project',\n",
       " '',\n",
       " 'pluripotent stem cell are unique cell type with promising potential regenerative and personalized medicine yet the difficulty understand and coax their seemingly stochastic differentiation and spontaneous selfrenewal have largely limited their clinical application call ha been made numerous researcher for better characterization surface protein these cell search biomarkers that can dictate developmental stage and lineage specification and can help formulate mechanistic insight stemcell fate choice the past two decade proteomics ha gained significant recognition profiling surface protein high throughput this review will summarize the impact these study stemcell biology and discus the used proteomic technique systematic comparison all the technique and their result also attempted here help reveal pro con and the complementarity the existing method this awareness should assist selecting suitable strategy for stemcell related research and shed light technical improvement that can explored the future',\n",
       " 'environmental biotechnology relies the exploration novel biological system and thorough understanding the underlying molecular mechanism nextgeneration proteomics based the latest generation mass analyzer currently allows the recording complete proteome from any microorganism interpreting these data can straightforward the genome the organism established relatively easy perform through proteogenomics approach draft sequence can obtained however nextgeneration proteomics face new interesting challenge when the organism distantly related previously characterized organism when mixture organism have analyzed new mass spectrometer and innovative bioinformatics tool are reshaping the possibility homologybased proteomics proteogenomics and metaproteomics for the characterization biological system novel time and costeffective screening strategy are also possible with this methodology exemplified whole proteome thermal profiling and subpopulation proteomics the complexity environmental sample allows for unique development approach and concept',\n",
       " 'nowadays proteomic study longer focus only identifying many protein possible given sample but aiming for accurate quantification them especially clinical proteomics the investigation variable protein expression profile can yield useful information pathological pathway biomarkers and drug target related particular disease over the time many quantitative proteomic approach have been established allowing researcher the field proteomics refer comprehensive toolbox different methodology this review will give overview different method quantitative proteomics with focus labelfree proteomics and it use clinical proteomics',\n",
       " 'this review present the use proteomics advance knowledge the field environmental biotechnology including study bacterial physiology metabolism and ecology bacteria are widely applied environmental biotechnology for their ability catalyze dehalogenation methanogenesis denitrification and sulfate reduction among others their tolerance radiation and toxic compound also importance proteomics ha important role helping uncover the pathway behind these cellular process environmental sample are often highly complex which make proteome study this field especially challenging some these challenge are the lack genome sequence for the vast majority environmental bacteria difficulty isolating bacteria and protein from certain environment and the presence complex microbial community despite these challenge proteomics offer unique dynamic view into cellular function present example environmental proteomics model organism and then discus metaproteomics microbial community proteomics which ha the potential provide insight into the function community without isolating organism finally the environmental proteomics literature summarized pertains the specific application area wastewater treatment metabolic engineering microbial ecology and environmental stress response',\n",
       " 'the proteome the cell the frontier being too complex for proteomic analysis organelle provide step organelle compartmentalize the cell enabling proteome physiology and metabolism analysis time and space protein complex separated electrophoresis have been identified the next natural level characterize the organelle compartmentalized membrane and soluble proteome mass spectrometry work mitochondrion and chloroplast ha shown where are the characterization complex proteome understand the network endogenous and extrinsic factor which regulate growth and development adaptation and evolution',\n",
       " 'the current omicsage research major development have been made technology that attempt survey the entire repertoire gene transcript protein and metabolite present within cell while genomics ha led dramatic increase our understanding such thing disease morphology and how organism respond medication critical obtain information the proteome level since protein carry out most the function within the cell the primary tool for obtaining proteomewide information protein within the cell mass spectrometry while ha historically been associated with the protein identification development over the past couple decade have made robust technology for protein quantitation well identifying quantitative change proteome complicated it dynamic nature and the inability any technique guarantee complete coverage every protein within proteome sample fortunately the combined development sample preparation and method have made capable quantitatively comparing many thousand protein obtained from cell and organism',\n",
       " 'endospore are metabolically dormant multilayered cellular structure formed grampositive bacteria belonging the genus bacillus clostridium and related organism their external layer are composed protein which part play role the resistance behaviour spore varied chemical and environmental assault thus protein analysis major interest spore biology spore proteomic study have been carried out previously but these study have focused the soluble coat protein fraction using gelbased technique protein identification and analysis were performed mass spectrometrydriven proteomics ha opened new avenue resolve particular the insoluble part the spore layer proteome mass spectrometrybased qualitative and quantitative proteomics method expand the knowledge about both the actual composition and the amount protein their various layer the technique can also used study the integrity the layer well spore biology general this notion explored concisely this minireview',\n",
       " 'prokaryotic and eukaryotic microorganism make vital contribution biogeochemical cycle decomposing virtually all natural compound and thereby exert lasting effect biosphere and climate the rapidly growing number metagenomic sequence together with revolutionary advance bioinformatics and protein analysis have opened completely new horizon investigate the molecular basis such complex process proteomics ha contributed substantially our understanding individual organism the cellular level offer excellent possibility probe many protein function and response simultaneously however ha not yet been widely applied microbial ecology although most protein have intrinsic metabolic function which can used relate microbial activity the identity defined organism multispecies community albeit still it infancy environmental proteomics enables simple protein cataloging comparative and semiquantitative proteomics analysis protein localization discovery posttranslational modification and even determination aminoacid sequence and genotype strainresolved proteogenomics this review trace the historical development environmental proteomics and summarizes milestone publication the field conclusion briefly discus current limitation microbial community proteomics but also the potential emerging technology shape the future metaproteome analysis',\n",
       " 'the study the proteome proteomics which includes the dynamic protein expression regulation interaction and it function ha played le prominent role evolutionary and ecological investigation comparison with the study the genome and transcriptome there are however number argument suggesting that this situation should change first the proteome closer the phenotype than the genome the transcriptome and such may more directly responsive natural selection and thus closely linked adaptation second there evidence low correlation between protein and transcript expression level across gene many different organism finally there have been some recent important technological improvement proteomics method that make them feasible practical and useful address wide range evolutionary question even nonmodel organism the different proteomic method their limitation and problem when interpreting empirical data are described and discussed addition the proteomic literature pertaining evolutionary ecology reviewed with example and potential application proteomics variety evolutionary context are outlined new proteomic research trend such the study posttranslational modification and proteinprotein interaction well the combined use the different omics approach are discussed relation the development more functional and integrated perspective needed for achieving more comprehensive knowledge evolutionary change',\n",
       " 'few technological development have created much excitement and skepticism proteomics over their potential change clinical diagnostic and prognostic procedure proteomics concern itself with the characterization and function all cellular protein the ultimate determinant cellular function such represents the end result all mechanism gene regulation and thus offer tremendous potential for characterizing biology much the same way what ha occurred with the genome the scientific community coming grip with the fact that the proteome although enormously complex finite conceivable that will learn the identity all possible protein including all posttranslational modification the rate protein discovery continues accelerate large part because improvement mass spectrometrybased technology coupled with improved genomic database and bioinformatic tool addition there reason believe that proteomics the verge moving from methodology that requires repeated proteome discovery one that can more systematically profile proteome this review discus current proteomicbased technology and the effort scientist move them into the clinic for use patient treated with radiotherapy and other modality',\n",
       " 'the development mass spectrometry technology ha brought the ability gather massive amount data characterising the proteome complex mixture major focus proteomics leverage this datagathering capability conduct comparative analysis biofluids from healthy and diseaseaffected patient for the identification highly specific biomarkers andor the development msbased diagnostic platform much effort ha gone into optimising the biofluid proteome coverage that can obtained using these technology leaving proteomics poised make important impact disease diagnostics the future',\n",
       " 'proteomic approach are useful for the identification functional protein these have been enhanced not only the development proteomic technique but also concert with genome sequencing this chapter database and web site relating plant proteomics are reviewed and recent technology relating data collection and annotation are surveyed',\n",
       " 'the comparison proteome between genetically heterogeneous bacterial strain may offer valuable insight into physiological diversity and function particularly where such variation aid the survival and virulence clinicallyrelevant strain however report such comparison frequently fail account for underlying genetic variance consequence the current knowledge regarding bacterial physiological diversity the protein level may incomplete inaccurate address this greater consideration must given the impact genetic heterogeneity proteome comparison this may possible through the use panproteomics analytical concept that permit the ability qualitatively and quantitatively compare the proteome genetically heterogeneous organism limited example this emerging technology highlight currently unmet analytical challenge this article define panproteomics where it value lie microbiology and discus the technical consideration critical it successful execution and potential future application',\n",
       " 'despite all the dental information acquired over century and the importance proteome research the crosslink between these two area only emerged around midnineties proteomic tool can help dentistry the identification risk factor early diagnosis prevention and systematic control that will promote the evolution treatment all dentistry specialty this review mainly focus the evolution dentistry different specialty based proteomic research and how these tool can improve knowledge dentistry the subject covered are overview proteomics dentistry specific information different field dentistry dental structure restorative dentistry endodontics periodontics oral pathology oral surgery and orthodontics and future direction there are many new proteomic technology that have never been used dentistry study and some dentistry area that have never been explored proteomic tool expected that greater integration these area will help understand what still unknown oral health and disease',\n",
       " 'proteomics allows the assessment cellular process unprecedented scale providing comprehensive quantitative and qualitative overview the protein content cell consequently proteomics ha been employed investigate multitude bacterial process ranging from the analysis environmental community identification virulence factor the proteomeguided optimization production strain proteomics ha short become indispensable tool for the global analysis bacterial physiology nonetheless challenge exist especially the accurate prediction phenotypic consequence based any given proteome composition this review will give overview current highlight the area microbial proteomics discus some current challenge and present new development that may help overcoming them',\n",
       " 'technical development the field proteomics are poised generate advance our understanding protein structure function and organization complex signaling and regulatory network improvement mass spectrometry instrumentation the implementation protein array and the development robust informatics software are providing sensitive highthroughput technology for largescale identification and quantitation protein expression protein modification subcellular localization protein function and proteinprotein interaction these advance have significant implication for understanding how cellular proteome are regulated health and disease',\n",
       " '',\n",
       " 'the emergence shotgun proteomics ha facilitated the numerous biological discovery made proteomic study however comprehensive proteomic analysis remains challenging and shotgun proteomics continually changing field this review detail the recent development shotgun proteomics and describes emerging technology that will influence shotgun proteomics going forward addition proteomic study integral membrane protein remain challenging due the hydrophobic nature integral membrane protein and their general low abundance level however there have been many strategy developed for enriching isolating and separating membrane protein for proteomic analysis that have moved this field forward summary while shotgun proteomics widely used and mature technology the continued pace improvement mass spectrometry and proteomic technology and method indicate that future study will have even greater impact biological discovery',\n",
       " 'over the past decade substantial advance have been made both the early diagnosis and accurate prognosis many cancer because the impressive development novel proteomic strategy however remains difficult standardize proteomic approach addition the heterogeneity protein distinct tissue result incomplete population the whole proteome which inevitably limit it clinical practice one the most complex proteome the human body the plasma proteome contains secreted protein originating from multiple organ and tissue making favorable matrix for comprehensive biomarker discovery here will discus the role plasma proteome profiling cancer biomarker discovery and validation and highlight both the inherent advantage and disadvantage although several hurdle lay ahead further advance this technology will greatly increase our understanding cancer biology reveal new biomarkers and biomarker panel and open new avenue for more efficient early diagnosis and surveillance cancer leading toward personalized medicine',\n",
       " 'the mycobacterium tuberculosis bacillus ha number unique feature that make particularly effective human pathogen although genomic analysis ha added our current understanding the molecular basis which tuberculosis damage it host proteomics may better suited describe the dynamic interaction between mycobacterial and host system that underpin this disease the tuberculosis proteome ha been investigated using proteomics for over decade with increasingly sophisticated mass spectrometry technology and sensitive method for comparative proteomic profiling deeper coverage the tuberculosis proteome ha led the identification hundred putative virulence determinant well unsurpassed coverage posttranslational modification proteomics therefore uniquely poised contribute our understanding this pathogen which may ultimately lead better management the disease',\n",
       " 'currently relatively little known regarding the protein production mammalian embryo unlike the genome the proteome itself dynamic reflecting both internal and external environmental stimulus until now the lack sensitivity ha remained stumbling block for the global introduction proteomics into the field mammalian embryology however new development mass spectrometry have been revolutionary utilizing protein profiling and peptide sequencing elucidate underlying biological process the sensitivity these platform have allowed for the development new protocol that are capable profiling the proteome individual mammalian oocyte and embryo this information fundamental unravelling the complexity embryo physiology including the dialogue between the developing embryo and it maternal environment such proteomic approach are also assisting the optimization art technique including oocyte cryopreservation and vitro maturation embryo selection for transfer another area art that should benefit this era proteomics currently mammalian embryo are selected for transfer based morphological grading system although great value analysis morphology alone can not determine the embryo physiological state chromosomal complement subsequently there need identify culture those embryo with the highest implantation potential proteomic analysis the embryonic secretome protein produced the embryo and secreted into the surrounding medium followed the identification specific protein critical for implantation may lead the development noninvasive viability assay assist the selection embryo for transfer',\n",
       " 'serum the body fluid most often used biomarker discovery albumin the most abundant serum protein contributes approximately the serum protein content with additional dozen abundant protein dominating the rest the serum proteome profile this challenging protein mixture proteomics the abundant protein must depleted allow for detection the lowabundant protein the primary biomarker target current serum depletion approach for proteomics are costly and relatively complex couple with protein digestion demonstrate simple affordable serum depletion methodology that within few minute processing result two captured serum fraction albumindepleted and albuminrich which are digested iin situi believe our method useful addition the biomarker sample preparation toolbox',\n",
       " 'this review the continuation three previously published article jorrin maldonado castillejo plant proteome analysis update proteomics rossignol peltier mock matros maldonado jorrin plant proteome analysis update proteomics canovas dumasgaudot recorbet jorrin mock rossignol plant proteome analysis proteomics and aim update the contribution proteomics plant research between and september reviewing most the paper which number approximately that appeared the plant proteomics field during that period most the paper published deal with the proteome arabidopsis thaliana and rice oryza sativa and focus profiling organ tissue cell subcellular proteome and studying developmental process and response biotic and abiotic stress using differential expression strategy although the platform based 2de still the most commonly used the use gelfree and secondgeneration quantitative proteomic technique ha increased proteomic data are beginning validated using complementary omics classical biochemical cellular biology technique addition appropriate experimental design and statistical analysis are being carried out accordance with the required minimal information about proteomic experiment miape standard result the coverage the plant cell proteome and the plant biology knowledge increasing compared human and yeast system however plant biology research ha yet exploit fully the potential proteomics particular it application ptms and interactomics',\n",
       " 'mass spectrometry ha become major tool the study proteome the analysis proteolytic peptide and their fragment ion this technique enables the identification and quantitation the precursor protein mixture however deducing chemical structure and then protein sequence from masstocharge ratio challenging computational task software tool incorporating powerful algorithm and statistical method improved our ability process the large quantity proteomics data repository spectral data make both data analysis and experimental design more efficient new approach quantitative and statistical proteomics make possible greater coverage the proteome the identification more posttranslational modification and greater sensitivity the quantitation targeted protein',\n",
       " 'proteolytic processing pervasive and irreversible posttranslational modification that expands the protein universe generating new proteoforms protein isoforms unlike signal peptide prodomain removal proteasegenerated proteoforms can rarely predicted from gene sequence positional proteomic technique that enrich for cterminal peptide from proteome are indispensable for comprehensive understanding protein function biological environment since protease cleavage frequently result altered protein activity and localization protease often process other protease and protease inhibitor which perturbs proteolytic network and potentiates the initial cleavage event affect other molecular network and cellular process physiological and pathological condition this review aimed researcher with keen interest state the art system level positional proteomic approach that enable the study complex proteaseprotease proteaseinhibitor and proteasesubstrate crosstalk and network allow the identification proteolytic signature candidate disease biomarkers and iii are expected fill the human proteome project missing protein gap predict that these methodology will integral part emerging precision medicine initiative that aim customize healthcare converting reactive medicine into personalized and proactive approach improving clinical care and maximizing patient health and wellbeing while decreasing health cost eliminating ineffective therapy trialanderror prescribing and adverse drug effect such initiative require quantitative and functional proteome profiling and dynamic disease biomarkers addition current pharmacogenomics approach with protease the pathogenic center many disease highthroughput protein terminus identification technique such tail terminal amine isotopic labeling substrate and cofradic combined fractional diagonal chromatography will fundamental for individual and comprehensive assessment health and disease',\n",
       " '',\n",
       " 'major aim presentday proteomics study change protein expression level global level ideally monitoring all protein present cell tissue mass spectrometry wellrespected technology proteomics that widely used for the identification protein more recently methodology have been introduced showing that mass spectrometry can also used for protein quantification this article review various mass spectrometrybased technology quantitative proteomics highlighting several interesting application area ranging from cell biology clinical application',\n",
       " 'with the completion genome sequence the model plant such rice oryza sativa and arabidopsis thaliana ha come into plant functional genomics era which becomes hard base the appearance and development plant proteomics this review focus the background proteomics concept proteomics and key technique proteomics the key technique proteomics include separation such 2de twodimensional electrophoresis rphplc reverse phase high performance liquid chromatography and seldi surface enhanced laser desorption ionization protein chip mass spectrometry such malditofms matrix assisted laser desorptionionizationtime flightmass spectrometry and esimsms electrospray ionization mass spectrometrymass spectrometry database related proteomics quantitative proteome tap tandem affinity purification and yeast twohybrid system challenge and prospect proteomic technique are discussed',\n",
       " '',\n",
       " 'virus continue pose some the greatest threat human and animal health and food security worldwide therefore new approach are required increase our understanding virushost cell interaction and subsequently design more effective therapeutic countermeasure quantitative proteomics based stable isotope labeling amino acid cell culture silac coupled lcmsms and bioinformatic analysis providing excellent resource for studying host cell proteome and can readily applied for the study virus infection here review this approach and discus how virushost cell interaction can best studied what realistically feasible and the potential limitation for example subcellular fractionation can reduce sample complexity for lcmsms increase data return and provide information regarding protein trafficking between different cellular compartment the key successful quantitative proteomics combine good experimental design and appropriate sample preparation with statistical analysis and validation the data through the use independent technique and functional analysis the annotation the human genome and the increasing availability biological reagent such antibody provide the optimum parameter for studying virus that infect human human cell line silacbased quantitative proteomics can also used study the interactome viral protein with the host cell coupling proteomic study with global transcriptomic and rna depletion experiment will provide great insight into the complexity the infection process and potentially reveal new antiviral target',\n",
       " 'the analysis brain function normal aging and neurodegenerative psychiatric and neurological disease ha long been subject interest and ha historically been investigated through descriptive analysis macroscopic microscopic observation now possible characterize brain cell such neuron and glial cell even their subcellular component the molecular level this ability enables researcher more closely examine brain cell specific molecular pathway elucidate distinct brain function furthermore the analysis neuronal maintenance and diseasecausing effect central component neurological investigation which include proteomic approach proteomics allows the identification thousand protein through descriptive and comparative analysis and can provide detailed overview distinct cellular state such analysis often require the isolation individual cell type subcellular component investigate specific question this review provides overview the currently applied stateoftheart prefractionation strategy this field',\n",
       " 'despite the fact that data deposition not generalised fact yet the field proteomics several mass spectrometry based proteomics repository are publicly available for the scientific community the main existing resource are the global proteome machine database gpmdb peptideatlas the proteomics identification database pride tranche and ncbi peptidome this review the capability each these will described paying special attention four key property data type stored applicable data submission strategy supported format and available data mining and visualization tool additionally the data content from model organism will enumerated for each resource there are other valuable smaller andor more specialized repository but they will not covered this review finally the concept behind the proteomexchange consortium collaborative effort among the main resource the field will introduced',\n",
       " 'recent decade mass spectrometry ha moved more than ever before into the front line proteincentered research after being established the qualitative level the more challenging question quantification protein and peptide using mass spectrometry ha become focus for further development this chapter discus and review actual strategy and problem the method for the quantitative analysis peptide protein and finally proteome mass spectrometry the common theme the difference and the potential pitfall the main approach are presented order provide survey the emerging field quantitative mass spectrometrybased proteomics',\n",
       " 'shotgun proteomic method involving isobaric tagging peptide enable highthroughput proteomic analysis itraq reagent allow simultaneous identification and quantitation protein four different sample using tandem mass spectrometry this article provide brief description proteome analysis using itraq reagent and review the current application these reagent proteomic study also compare different aspect protein identification including protein sequence coverage and proteome coverage obtained using itraq reagent with those using other shotgun proteomic technique briefly discus the issue isotope purity correction measured peak area during protein quantitation using itraq reagent finally conclude with some the current challenge msbased proteomic analysis that are limiting protein identification obtained different shotgun proteomic method',\n",
       " 'after the successful completion the human genome project the next major challenge wa understand when and where the encoded protein were expressed and generate map the complex interconnected pathway network and molecular system the human proteome that taken together control the working all cell tissue organ and organism proteomics will fundamental for such study this review summarizes the key discovery that laid down the foundation for proteomics now know and describes key recent technological advance that will undoubtedly contribute achieving the initial goal the human proteome organization identifying and characterizing least one protein product and representative posttranslational modification single amino acid polymorphism and splice variant isoforms from the human proteincoding gene within the next year successful unraveling the human proteome will undoubtedly improve our understanding human biology the cellular level and lay the foundation for improved diagnostic prognostic therapeutic and preventive medical outcome enter the era personalized medicine',\n",
       " 'advance the separation microbial subpopulation and proteomics technology have paved the way for the global molecular characterization microbial cell that share common functional characteristic quantitative characterization the dynamic microbial proteome enables unprecedented view the adaptive response microbe environmental stimulus during interaction with other specie host cell however the intrinsic complexity such data requires sophisticated visualization method for the display mining interpretation and efficient exploitation these data resource this review discus how new approach data visualization such streamgraphs network graph voronoi treemaps are being used the field provide new insight into the functional complexity microbial cell population and multispecies consortium',\n",
       " '',\n",
       " '',\n",
       " 'the success massspectrometrybased proteomics method for analyzing protein biological sample accompanied challenge owning demand for increased throughput these challenge arise from the vast volume data generated proteomics experiment combined with the heterogeneity data format processing method software tool and database that are involved the translation spectral data into relevant and actionable information for scientist informatics aim provide answer these challenge transferring existing solution from information management proteomics andor generating novel computational method for automation proteomics data processing',\n",
       " '',\n",
       " 'proteomics rapidly evolving postgenomic science utilizing advanced technology protein separation identification quantitation and heavily relying bioinformatics proteomic research pediatrics important and most the success thus far are seen research that utilize sample that require le invasive procedure and focus prevailing childhood disease such acute lymphoblastic leukaemia and neuroblastoma recent advance proteomics are helping elucidate platelet process that are relevant bleeding and clotting disorder well other important role platelet such angiogenesis and inflammation nevertheless most platelet proteome data obtained date are derived from the adult population and the potential platelet proteomic application child ha not yet been explored happens all research field there are additional challenge studying child such procuring sufficient biological sample and access le common disease cohort compared adult furthermore many the prevalent plateletmediated disease adult such coronary heart disease and atherosclerotic lesion are believed have origin during childhood hence platelet proteomic research child may reveal some important information how platelet play role the pathogenesis disease this article refer the current knowledge from platelet proteomic research strategy adult and address the specific concern the study pediatric sample',\n",
       " 'the comprehensive study proteome ha become important part attempt uncover the systemic property biological system proteomics provides data quality which increasingly fulfills strict requirement system biology for quantitative and qualitative information notably proteomics can generate rich datasets that describe dynamic change proteome the other hand largescale modeling requires the development mathematic tool that are adequate for the processing largely uncertain biological data this review recent development that pave the way for the integration proteomics into system biology are discussed these development include the standardization data acquisition and presentation the increased comprehensiveness proteomics study description functional status localization and dynamic protein and advanced modeling approach',\n",
       " 'cyanobacteria are photosynthetic bacteria notable for their ability produce hydrogen and variety interesting secondary metabolite result the growing number completed cyanobacterial genome project the development postgenomics analysis for this important group ha been accelerating dna microarrays and classical twodimensional gel electrophoresis 2de were the first technology applied such analysis many other system shotgun proteomics employing multidimensional liquid chromatography and tandem mass spectrometry ha proven powerful tool however this approach ha been relatively underutilized cyanobacteria this study ass progress cyanobacterial shotgun proteomics date and add new perspective developing protocol for the shotgun proteomic analysis the filamentous cyanobacterium anabaena variabilis atcc model for fixation using approach for enhanced protein extraction protein were identified which more than double the previous result obtained using 2de notably the improved extraction method and shotgun approach resulted significantly higher representation basic and hydrophobic protein the use protein bioinformatics tool further mine these shotgun data illustrated through the application psortb for localization the grand average hydropathy gravy index for hydrophobicity lipop for lipoprotein and the exponentially modified protein abundance index empai for abundance the result are compared with the most wellstudied cyanobacterium synechocystis pcc some general issue shotgun proteome identification and quantification are then addressed',\n",
       " 'the molecular level living cell are enormously complicated complex adaptive system which intertwined genomic transcriptomic proteomic and metabolic network all play crucial role the same time cell are spatially heterogeneous system which subcellular compartmentalization different function ubiquitous and requires efficient crosscompartmental communication dynamic redistribution multitudinous protein different subcellular location response cellular functional state increasingly recognized crucial characteristic cellular function that seems least important overall change protein abundance characterization the subcellular spatial dynamic protein distribution major challenge for proteomics and recent result with mcf7 breast cancer cell suggest that this may particular importance for cancer cell',\n",
       " 'the concept personalized medicine includes novel protein biomarkers that are expected improve the early detection diagnosis and therapy monitoring malignant disease tissue biofluids cell line and xenograft model are the common source biomarker candidate that require verification clinical value independent patient cohort targeted proteomics based selected reaction monitoring data extraction from dataindependent acquisition based digital map now represents promising mass spectrometry alternative immunochemical method date ha been successfully used high number study answering clinical question solid malignancy breast colorectal prostate ovarian endometrial pancreatic hepatocellular lung bladder and others play important role functional proteomic experiment that include studying the role posttranslational modification cancer progression this review summarizes verified biomarker candidate successfully quantified targeted proteomics this field and directs the reader who plan design their own hypothesisdriven experiment appropriate source method and knowledge',\n",
       " 'cardiovascular disease the leading cause mortality and morbidity the industrialized world total worldwide death due this disease are currently estimated million per year and this number expected increase over the next several decade address this epidemic major effort ha begun develop new cardiovascular disease marker through the use proteomic analysis the global study protein this review discus strategy recent technological advance and other issue plasmaserum biomarker discovery for cardiovascular disease emphasis lie the need for standardizing specimen collection method for reducing plasma proteome complexity subproteomes selection appropriate technology platform and strategy evaluate candidate multiplexed immune assay the overall goal this effort identify serum biomarkers for diagnosis therapeutic monitoring and risk stratification cardiovascular disease',\n",
       " 'approximately the gene mammal are expressed the nervous system differential expression gene distinct pattern necessary for the generation the large variety neuronal phenotype proteomic analysis brain compartment may useful understand the complexity investigate disorder the central nervous system and search for corresponding early marker now proteomics ha mainly studied the identity and level the abundant human rat and mouse brain protein well change their level and the modification that result from various neurological disorder like alzheimers disease and down syndrome human and animal model those disease the protein for which altered level these disorder have been observed exert mainly neurotransmission guidance and signaltransduction function are involved detoxification metabolism and conformational change some those protein may potential drug target further improvement proteomics technology increase sensitivity and efficiency detection certain protein class necessary for more detailed analysis the brain proteome this review description the proteomics technology applied the investigation the brain the major finding that resulted from their application and the potential and limitation the current technology are discussed',\n",
       " 'the global analysis protein composition modification and dynamic are important goal cell biology mass spectrometry msbased proteomics ha matured into attractive technology for this purpose particularly high resolution method have been extremely successful for quantitative analysis cellular and organellar proteome rapid advance all area the proteomic workflow including sample preparation and computational analysis should make the technology more easily available broad community and turn into staple methodology for cell biologist',\n",
       " 'saliva extraordinary fluid term research and diagnostic possibility it composition electrolyte hormone and especially it proteome contains information about feeding status nutritional requirement and adaptation diet and environment and also about health status animal easy collect noninvasive and routine basis without any need for special training therefore the analysis salivary proteome going emerge into field high interest with the future goal maintain and improve livestock productivity and welfare moreover the comprehensive analysis and identification salivary protein and peptide whole and glandular saliva necessary prerequisite identify animal disease biomarkers and powerful tool better understand animal physiology this review focus the different approach used study the salivary proteome farm animal respect the physiology nutrition and food perception relation food choice the potential animal saliva source disease biomarkers will also pointed out special emphasis laid the ruminating triad cattle goat and sheep well swine major specie animal production western and southern europe',\n",
       " 'proteome analysis utilizing highthroughput proteomics approach involves studying protein that whole organism specific tissue cellular compartment express under certain condition intrinsic difficulty these study well the enormous volume data they typically produce make the proteome analysis and interpretation very difficult with any highthroughput approach proteomics experiment should carefully designed analyzed and verified addition computational standardsexperimental standardssimple and complex mixture known proteinsfor highthroughput proteomics have developed and utilized this article discus such experimental standard and their implementation',\n",
       " 'not much more than year ago handful visionary scientist around the world suggested sequence and analyze not only the human genome but also many genome possible order compare dna well deduce protein sequence that mean they expected get idea about the organization life however after now having now sequenced the human genome and least identified around gene coding region are still left with the fundamental question how gene are regulated and what the rationale genetic regulatory network the basic knowledge and methodology elucidate functional regulatory network cell and organism the protein level had been around for much longer than dnabased discovery tool this wa mainly due the fact that protein have fulfill universal function nature and unlike dna polynucleotides protein differ not only their amino acid sequence they come nearly all shape and size and have all kind physical well chemical property they can highly water soluble serum and milk protein nearly insoluble any solvent keratin and some other structural protein addition structure function well the respective stability protein inside and outside biological system are individual feature any given polypeptide one hand the individuality protein allows adaptation any life form the environment and the other still real challenge for biotech rampd and production the present review actually the first approach evaluate and judge the achievement made applied proteome analysis and proteomics over the last year',\n",
       " 'understanding protein interaction within the complexity living cell challenging but technique coupling affinity purification and mass spectrometry have enabled important progress made the past year identification proteinprotein interaction becoming easier the quantification the interaction dynamic the next frontier several quantitative mass spectrometric approach have been developed address this issue that vary their strength and weakness while isotopic labeling approach continue contribute the identification regulated interaction technique that not require labeling are becoming increasingly used the field here describe the major type labelfree quantification used interaction proteomics and discus the relative merit data dependent and data independent acquisition approach labelfree quantification this article part special issue entitled from protein structure clinical application',\n",
       " 'the systematic analysis biological process requires understanding the quantitative expression pattern protein their interacting partner and their subcellular localization this information wa formerly difficult accrue the relative quantification protein relied antibodybased method and other approach with low throughput the advent soft ionization technique mass spectrometry plus advance separation technology ha aligned protein system biology with messenger rna dna and microarray technology provide data system opposed singular protein entity another aspect quantitative proteomics that increase it importance for the coming few year the significant technical development underway both for high pressure liquid chromatography and mass spectrum device hence robustness reproducibility and mass accuracy are still improving with every new generation instrument nonetheless the method employed require validation and comparison design fit for purpose experiment advanced protein analysis this review considers the newly developed systematic protein investigation method and their value from the standpoint that relative absolute protein quantification required rigueur biomedical research',\n",
       " 'recent interest proteomics ha been fueled the completion multiple genome project and ignited the common need biologist rapidly and comprehensively evaluate complex sample protein global level shotgun proteomics refers the direct analysis complex protein mixture rapidly generate global profile the protein complement within the mixture this approach ha been facilitated the use multidimensional protein identification technology mudpit which incorporates multidimensional highpressure liquid chromatography lclc tandem mass spectrometry msms and databasesearching algorithm this review will focus the most recent advance methodology for shotgun proteomics and address the limitation the application each real biological sample',\n",
       " 'with the growing amount experimental data produced proteomics experiment and the requirementsrecommendations journal the proteomics field publicly make available data described paper need for longterm storage proteomics data public repository arises for such upload one need proteomics data standardized format therefore desirable that the proprietary vendor software will integrate the future such export functionality using the standard format for proteomics result defined the hupopsi group currently not all search engine and analysis tool support these standard format the meantime there need provide userfriendly freetouse conversion tool that can convert the data into such standard format order support wetlab scientist creating proteomics data file ready for upload into the public repository procon such conversion tool written java for conversion proteomics identification data into standard format mzidentml and pride xml allows the conversion sequest™comet out file search result from the popular and often used proteomediscoverer® xversions software and search result stored the lims system proteinscape® and into mzidentml and pride xml this article part special issue entitled computational proteomics',\n",
       " 'membrane protein are involved central process such cell signaling cellcell interaction and communication ion and metabolite transport and general play crucial role cell homeostasis cancer and cancer metastasis have been correlated protein expression level and dysfunction with membrane protein playing important role and are thus used drug target and potential biomarkers for prognostic diagnostic purpose despite the critical biological significance membrane protein proteomic analysis ha been challenging task due their hydrophobicity this review recent advance the proteomic analysis membrane protein are presented focusing membrane fraction enrichment technique combined with labeled labelfree shotgun proteomics approach for the identification potential cancer biomarkers',\n",
       " 'the realm clinical chemistry the field clinical proteomics that the application proteomic method for understanding mechanism and enabling diagnosis prediction measurement activity and treatment response disease first and foremost discovery and research tool that feed assay development downstream putative new assay candidate generated proteomics discovery project compete with wellestablished assay with known indication welldescribed performance and known value specific clinical setting careful attention the many option available the design execution and interpretation clinical proteomics study thus necessary for translation into clinical practice here review and discus important option associated with clinical proteomics endeavor stretching from the planning phase the final use clinical chemistry',\n",
       " 'stable isotope labeling amino acid cell culture silac ha become very popular quantitative proteomic method since wa firstly introduced matthias mann group metabolic labeling strategy which isotopelabeled amino acid are metabolically incorporated vivo into protein during translation after natural light heavy amino acid incorporation differentially labeled sample are mixed immediately after cell lysis and before any further processing which minimizes quantitative error caused handling different sample parallel this unit describe protocol for basic duplex silac triplex silac for use nondividing cell such neuron and for measuring amount newly synthesized protein john wiley amp son inc',\n",
       " 'the ultimate objective clinical proteomics the successful discovery validation and translation biomarkers together with new therapeutic target into medical practice new highly developed technology proteomics and their use understanding tumor biology have significant clinical potential the diagnosis prognosis and treatment disease area such new labeling technology and advancement bioinformatics system are now used successfully detect diseaseassociated biomarkers together with therapeutic target complex biological specimen including biofluids cell lysates and tissue biopsy recent improvement sample preparation specifically focused fractionation and enrichment are enabling the analysis lowabundance protein together with many type posttranslational modification targeted proteomic diagnostics will play significant role the development personalized molecular medicine process that will vital modernizing healthcare structure',\n",
       " 'mass spectrometry provides key tool for the analysis protein new type mass spectrometer that provide enhanced capability discover protein identity and perform improved proteomic experiment are discussed handling the complex mixture peptide and protein generated from protein complex and whole cell requires multidimensional separation several form separation are discussed application mass spectrometrybased approach for contemporary proteomic analysis are described',\n",
       " 'the eukaryotic cell division cycle ha been studied the molecular level for over year most fruitfully model organism the past year development mass spectrometrybased proteomics have been applied the study protein interaction and posttranslational modification involving key cell cycle regulator such cyclindependent kinase and the anaphasepromoting complex well effector such centrosome the kinetochore and dna replication fork addition innovation chemical biology functional proteomics and bioinformatics have been employed study the cell cycle the proteome level this review survey the contribution proteomics cell cycle research the near future should see the application more quantitative proteomic approach probe the dynamic aspect the molecular system that underlie the cell cycle model organism and human cell',\n",
       " 'despite the increasing popularity gelfree proteomic strategy twodimensional gel electrophoresis 2de still the most widely used approach topdown proteomic study for all sort biological model order achieve meaningful biological insight using 2de approach importance must given not only ensure proper experimental design experimental practice and 2de technical performance but also valid approach for data acquisition processing and analysis this paper review and illustrates several different aspect data analysis within the context gelbased proteomics summarizing the current state research within this field particular focus given discussing the usefulness available multivariate analysis tool both for data visualization and feature selection purpose visual example are given using real gelbased proteomic dataset basis',\n",
       " 'clinical biomarkers identified shotgun proteomics require protein body fluid tissue enzymatically digested before being separated and sequenced liquid chromatographytandem mass spectrometry how well peptide signal can resolved and detected largely dependent the quality sample preparation conventional approach such ingel insolution and filterbased digestion despite their extensive implementation the community become le appealing due their unsatisfying proteinpeptide recovery rate lengthy sample processing andor lowcosteffectiveness suspension trapping ha recently been demonstrated ultrafast approach for proteomic analysis here for the first time extend it application human salivary proteome analysis particular present simple selfassembled glass fiber filter device which can packed with minimal difficulty extremely costeffective and maintains the same performance commercial filter proofofprinciple analyzed the whole saliva from healthy individual well cohort subject oral squamous cell carcinoma oscc patient and nonoscc subject labelfree quantification revealed surprisingly low interindividual variability and several known marker our study provides the first evidence easytouse and lowcost device for clinical proteomics well for general proteomic sample preparation',\n",
       " 'autophagy evolutionarily conserved intracellular degradation process autophagy closely involved human health and disease recent year mass spectrometrybased proteomic method have become important and powerful tool for autophagy study these type technique have been especially helpful reveal the range degradation substrate autophagy through largescale unbiased analysis cellular proteome present variety mass spectrometrybased proteomics method have been successfully applied autophagy research',\n",
       " 'arraying technology have shown the way smaller sample volume more efficient analysis and higher throughput proteomics field which ha grown significance the last five year this review outline recent development protein array and their application proteomics and discus the requirement current limitation and the potential and future perspective the technology',\n",
       " 'mass spectrometry based proteomics generally seek identify and characterize protein molecule with high accuracy and throughput recent speed and quality improvement the independent step integrated platform have removed many limitation the robust implementation top down proteomics tdp for protein below kda improved intact protein separation coupled highperformance instrument have increased the quality and number protein and proteoform identification date tdp application have shown protein identification expanding average more proteoforms for each protein detected the near future increased fractionation power new mass spectrometer and improvement proteoform scoring will combine accelerate the application and impact tdp this century biomedical problem',\n",
       " 'technical advance several frontier have expanded the applicability existing method structural biology and helped close the resolution gap between them result are now poised integrate structural information gathered multiple level the biological hierarchy from atom cell into common framework the goal comprehensive description the multitude interaction between molecular entity which turn prerequisite for the discovery general structural principle that underlie all cellular process',\n",
       " 'this review will examine the current situation with labelfree quantitative shotgunoriented proteomics technology and discus the advantage and limitation associated with it capability capturing and quantifying large portion proteome microorganism such approach allows comparison between physiological genetic state organism the protein level painting proteomic data onto genome databased metabolic map enhancement the utility genomic data and finally surveying nongenome sequenced microorganism taking advantage available inferred protein data order gain new insight into straindependent metabolic physiological capacity the technology essentially powerful addition system biology with capacity used ask hypothesisdriven topdown question for more empirical bottomup exploration',\n",
       " 'proteomics the study the protein complement genome and employ number newly emerging tool one such tool chemical proteomics which branch proteomics devoted the exploration protein function using both vitro and vivo chemical probe chemical proteomics aim define protein function and mechanism the level directly observed proteinligand interaction whereas chemical genomics aim define the biological role protein using chemical knockout and observing phenotypic change chemical proteomics therefore traditional mechanistic biochemistry performed systemsbased manner using either activity affinitybased probe that target protein related chemical reactivity binding site shapeproperties respectively system are group protein related metabolic pathway regulatory pathway binding the same ligand study can based two main type proteome sample pooled protein mixture protein isolated protein given system and studied parallel single protein sample although the field chemical proteomics originated with the use covalent labeling strategy such isotopecoded affinity tagging expanding include chemical probe that bind protein noncovalently and include more method for observing proteinligand interaction this review present emerging role for nuclear magnetic resonance spectroscopy chemical proteomics both vitro and vivo application include functional proteomics using cofactor fingerprinting assign protein gene family gene familybased structural characterization proteinligand complex gene familyfocused design drug lead and chemical proteomic probe using nuclear magnetic resonance solve and study proteinligand interaction vivo',\n",
       " '',\n",
       " 'the last few decade have seen the rise widelyavailable proteomics tool from new data acquisition device such maldims and 2de new database searching software these new product have paved the way for high throughput microbial proteomics htmp these tool are enabling researcher gain new insight into microbial metabolism and are opening new area study such proteinprotein interaction interactomics discovery computer software key part these emerging field this current review considers software tool for identifying the proteome such mascot pdquest online database proteome such swissprot proteome web the proteomics facility the pathogen functional genomics resource center and software tool for applying proteomic data such psiblast vespa these tool allow for research network biology protein identification functional annotation target identificationvalidation protein expression protein structural analysis metabolic pathway engineering and drug discovery',\n",
       " 'stable isotope labeling amino acid cell culture silac powerful approach for highthroughput quantitative proteomics silac allows highly accurate protein quantitation through metabolic encoding whole cell proteome using stable isotope labeled amino acid since it introduction silac ha become increasingly popular this chapter review the methodology and application silac with emphasis three research area dynamic posttranslational modification proteinprotein interaction and protein turnover',\n",
       " 'crucial part successful system biology experiment assay that provides reliable quantitative measurement for each the component the system being studied for proteomics key part such study must deliver accurate quantification all the component the system for each tested perturbation without any gap the data this will require new approach proteomics that based emerging targeted quantitative mass spectrometry technique the peptideatlas project comprises growing publicly accessible database peptide identified many tandem mass spectrometry proteomics study and software tool that allow the building peptideatlas well it use the research community here describe the peptideatlas project it content and component and show how together they provide unique platform select and validate mass spectrometry target thereby allowing the next revolution proteomics',\n",
       " '',\n",
       " 'growing rice important socioeconomic activity rice proteomics ha achieved tremendous progress establishing technique proteome almost all tissue organ and organelle during the past one decade year have compiled these progress time time over this period the present compilation discus proteomics research rice published between 1st april and july progress continues mainly towards protein cataloging deep into the proteome with highconfident protein assignment and some functional significance than ever before identifying previously unreportedlowabundance protein quantifying relativeabsolute value protein iii assigning protein response bioticabiotic stress protein localization into organelle validating previous proteome and eliminating falsepositive protein and discovering potential biomarkers for tissue organ organelle and for screening transgenic plant and foodsafety evaluation the notable achievement global mapping phosphorylation site and identifying several novel secreted protein into the extracellular space are worth appreciating our everincreasing knowledge the rice proteomics beginning impact the biology not only rice but also crop and plant these major achievement will discussed this review keeping mind newcomer young and established scientist proteomics and plant',\n",
       " 'proteomics new methodological and conceptual approach the study live system which aim map the whole proteom the protein complement genome aspires describe quantitatively and qualitatively all protein present the given moment the cell tissue the organism the principal tool proteomics are separation technique based electrophoresis and chromatography which are used fractionate complex protein mixture and namely the mass spectrometry used for the identification individual protein proteomics represents very young and rapid developing specialization which ha already proved it valuable potential for the study various physiological and pathlogical molecular mechanism and for the direct use the diagnostics serious disease',\n",
       " '',\n",
       " 'organellar proteomics aim describe the full complement protein subcellular structure and organelle when compared with wholecell wholetissue proteome the more focused result from subcellular proteomic study have yielded relatively simpler datasets from which biologically relevant information can more easily extracted every proteomic study the quality and purity the biological sample investigated the utmost importance for successful analysis organellar proteomics one the most crucial step sample preparation the initial subcellular fractionation procedure which the enriched preparation the soughtafter organelle obtained nearly all available organellar proteomic study the method choice relies one several round densitybased gradient centrifugation although this method ha been recognized for decade yielding relatively pure preparation organelle recent technological advance protein separation and identification can now reveal even minute amount contamination which turn can greatly complicate data interpretation the scope this review focus recently published innovative complementary alternative method perform subcellular fractionation which can further refine the way which sample preparation accomplished organellar proteomics',\n",
       " 'proteomics ha evolved substantially since it early day some year ago this minireview aim provide overview general methodology and more recent development mass spectrometric approach used for relative and absolute quantitation protein enhancement sensitivity the mass spectrometer well improved sample preparation and protein fractionation method are resulting more comprehensive analysis proteome also document some upcoming trend for quantitative proteomics such the use labelfree quantification method hopefully microbiologist will continue explore proteomics tool their research understand the adaptation microorganism their ever changing environment encourage them incorporate some the described new development mass spectrometry facilitate their analysis and improve the general knowledge the fascinating world microorganism',\n",
       " 'the proteome the protein compliment the genome and the result genetic expression ribosomal synthesis and proteolytic degradation protein participate most major cell process and their function highly regulated posttranslational modification such phosphorylation and glycosylation result neurotoxicantinduced change protein level function regulation could have negative impact neuronal viability the molecular level direct oxidative covalent modification individual protein various chemical drug likely lead perturbation tertiary structure and loss function the proteome and the functional determinant it individual protein component are therefore likely target neurotoxicant action and resulting characteristic disruption could critically involved corresponding mechanism neurotoxicity clearly investigating change the proteome can provide important clue for deciphering mechanism toxicant action and therefore proteomics the study the proteome currently and will likely remain significant experimental approach for mechanistic research neurotoxicology the purpose this review discus proteomics tool for neurotoxicological investigation variety classic proteomic technique liquid chromatography lctandem mass spectroscopy twodimensional gel image analysis well more recently developed approach twohybrid system antibody array protein chip isotopecoded affinity tag icat are available determine protein level identify component multiprotein complex and detect posttranslational change proteomics therefore offer comprehensive overview cell protein and the case neurotoxicant exposure can provide quantitative data regarding change corresponding expression level andor posttranslational modification that might associated with neuron injury',\n",
       " 'adsorption plasma protein onto the membrane surface during the hemodialysis session represents key feature membrane used for chronic dialysis therapy this issue proteomicsclinical application han originally describe how using proteomic technology the adsorptive property two membrane made from the same biomaterial polyamix may have different flux characteristic low flux and high flux the former having smaller pore size total differentially expressed protein were identified eluate obtained after vivo hemodialysis protein concentrated more lowflux membrane predominantly protein with molecular weight kda and high flux most represented protein with molecular weight kda use bioinformatics tool shed light the involvement adsorbed protein important biological pathway such the coagulation cascade and the complement system again with some difference between the two membrane the study indicates that flux characteristic biomaterial used for hemodialysis membrane strongly influence it adsorptive property and that proteomic application may provide information relevant renal replacement therapy',\n",
       " 'the advent highthroughput proteomic technology for global detection and quantitation protein creates new opportunity and challenge for those seeking gain greater understanding the cellular machinery here recent advance highresolution capillary liquid chromatography coupled fourier transform ion cyclotron resonance mass spectrometry are reviewed along with it potential application highthroughput proteomics these technological advance combined with quantitative stable isotope labeling methodology provide powerful tool for expanding our understanding biology the system level',\n",
       " 'for some year now scientist have been spending lot effort developing method analyse and compare complex protein sample one the goal such global analysis what known proteome discover specific protein markersor fingerprint protein markersfrom various type affected biological sample considering the battery technology currently available mass spectrometry constitutes essential tool proteomics describe here the type instrumentation that currently dedicated proteomics research also describe the major experimental workflow that are typically used proteomics today with focus those incorporating major analysis tool',\n",
       " 'the proteomic characterization protein and protein complex from cell and cell organelle the next challenge for investigation the cell after isolation the cell compartment three step have performed the laboratory yield information about the protein present the protein mixture must separated into single specie broken down into peptide and finally identified mass spectrometry most scientist engaged proteomics separate protein electrophoresis for characterization and identification proteome mass spectrometry peptide the method choice combine electrophoresis and mass spectrometry sample preparation ingel digestion ha been developed many procedure are available for ingel digestion which inspired review ingel digestion approach',\n",
       " 'the present review report the principle fundamental and some application twodimensional difference gel electrophoresis for analytical proteomics based plant proteome analysis also emphasizing some advantage dige over page technique some fluorescent protein labeling reagent method protein labeling model dige experiment and some limitation this technique are presented and discussed term dige plant proteome finally some practical application this technique are pointed out emphasizing it potentiality plant proteomics',\n",
       " '',\n",
       " 'mass spectrometry technique widely employed for the identification and characterization protein the role bioinformatics fundamental for the elaboration mass spectrometry data due the amount data that this technique can produce process data efficiently new software package and algorithm are continuously being developed improve protein identification and characterization term highthroughput and statistical accuracy however many limitation exist concerning bioinformatics spectral data elaboration this review aim critically cover the recent and future development new bioinformatics approach mass spectrometry data analysis for proteomics study',\n",
       " 'antibodybased microarrays rapidly evolving technology that ha gone from the first proofofconcept study more demanding proteome profiling application during the last year miniaturized microarrays can printed with large number antibody harbouring predetermined specificity capable targeting high well lowabundant analytes complex nonfractionated proteome consequently the resolution such proteome profiling effort correlate directly the number antibody included which today key limiting factor overcome this bottleneck and able perform indepth global proteome survey propose interface affinity proteomics with msbased readout outlined this technical perspective briefly have defined range peptide motif each motif being present different protein this manner antibody binding different motif commonly distributed among different protein would potentially target protein cluster individual molecule around the nonredundant human proteome notably these motifspecific antibody would directly applicable any proteome specie independent manner and not biased towards abundant protein certain protein class the biological sample digested exposed these immobilized antibody whereby motifcontaining peptide are specifically captured enriched and subsequently detected and identified using',\n",
       " '',\n",
       " 'proteomics the study the proteome the collection all the protein expressed from the genome all isoforms polymorphism and posttranslational modification rapidly developing field which there are numerous new and often expensive technology making imperative use the most appropriate technology for the biological system and hypothesis being addressed this review provides some guideline approaching broadbased proteomics project including strategy refining hypothesis choosing model and proteomic approach with emphasis aspect sample complexity including abundance and protein characteristic and separation technology and their respective strength and weakness finally issue related quantification mass spectrometry and informatics strategy are discussed the goal this review therefore twofold the first section provides brief outline proteomic technology specifically with respect their application broadbased proteomic approach and the second part provides more detail about the application these technology typical scenario dealing with physiological and pathological process proteomics it best the integration carefully planned research and complementary technique with the advantage powerful discovery technology that ha the potential make substantial contribution the understanding disease and disease process',\n",
       " 'proteomic study have increased our understanding the molecular pathway affected psychiatric disorder mass spectrometry and twodimensional gel electrophoresis analysis postmortem brain sample from psychiatric patient have revealed effect synaptic cytoskeletal antioxidant and mitochondrial protein network multiplex immunoassay profiling study have found alteration hormone growth factor transport and inflammationrelated protein serum and plasma from living firstonset patient despite these advance there are still difficulty translating these finding into platform for improved treatment patient and for discovery new drug with better efficacy and side effect profile this review describes how the next phase proteomic investigation psychiatry should include stringent replication study for validation biomarker candidate and functional followup study which can used test the impact physiological function all biomarker candidate should now tested series with traditional and emerging cell biological approach this should include investigation the effect posttranslational modification protein dynamic and network analysis using targeted proteomic approach most importantly there still urgent need for development diseaserelevant cellular model for improved translation proteomic finding into mean developing novel drug treatment for patient with these lifealtering disorder',\n",
       " 'living organism are comprised various system different level organ tissue and cell each system carry out it diverse function response environmental and genetic perturbation utilizing biological network which nodal component such dna mrna protein and metabolite closely interact with each other system biology investigates such system producing comprehensive global data that represent different level biological information the dna mrna protein metabolite level and integrating this data into network model that generate coherent hypothesis for given biological situation this review present system biology framework called the integrative proteomics data analysis pipeline ipdap which generates mechanistic hypothesis from network model reconstructed integrating diverse type proteomic data generated mass spectrometrybased proteomic analysis the devised framework includes serial set computational and network analysis tool here demonstrate it functionality applying these tool several conceptual example',\n",
       " 'proteomics the study the function all expressed protein tremendous progress ha been made the past few year generating largescale data set for proteinprotein interaction organelle composition protein activity pattern and protein profile cancer patient but further technological improvement organization international proteomics project and open access result are needed for proteomics fulfil it potential',\n",
       " '',\n",
       " 'labelfree quantification lfq ha emerged viable option for quantitative lcmsmsbased proteomic analysis for use the scale hundred sample such are encountered clinical analysis notably sample preparation sample loading hplc separation and mass spectrometric performance must highly reproducible for this approach effective the following protocol describe the key step the method related sample preparation and analysis for lcmsmsbased labelfree quantitation using standard datadependent acquisition',\n",
       " 'the technology platform for proteome analysis have advanced considerably over the last few year due these improvement the number study the analysis the proteomepeptidome with the aim defining biomarkers ha escalated this review will summarise the technical aspect that relate the proteomics field targeting the discovery biomarkers for disease diagnosis will describe the course from biomarker discovery potential biomarkers those that are clinically important also present several example successful proteomic study that have defined biomarker pattern for clinical application focussed urine material source and capillary electrophoresis coupled mass spectrometry technology finally current challenge and consideration for future study will discussed',\n",
       " 'the emerging scientific field proteomics encompasses the identification characterization and quantification the protein content proteome whole cell tissue body fluid the potential for proteomic technology identify and quantify novel protein the plasma that can function biomarkers the presence severity clinical disease state hold great promise for clinical use however there are many challenge translating plasma proteomics from bench bedside and relatively few plasma biomarkers have successfully transitioned from proteomic discovery routine clinical use key barrier this translation include the need for orthogonal biomarkers uncorrelated with existing marker the complexity the proteome biological sample the presence high abundance protein such albumin biological sample that hinder detection low abundance protein false positive association that occur with analysis high dimensional datasets and the limited understanding the effect growth development and age the normal plasma proteome strategy overcome these challenge are discussed',\n",
       " '',\n",
       " 'functional proteomics constitutes emerging research area the proteomic field focused two major target the elucidation biological function unknown protein and the definition cellular mechanism the molecular level understanding protein function well unravelling molecular mechanism within the cell then depending the identification the interacting protein partner the association unknown protein with partner belonging specific protein complex involved particular mechanism would fact strongly suggestive it biological function furthermore detailed description the cellular signalling pathway might greatly benefit from the elucidation proteinprotein interaction the cell isolation functional protein complex essentially rely affinitybased procedure the protein interest and it specific partner can fished out from the cellular extract using suitable ligand bait taking advantage the specific binding property the ligand molecule immobilised agarosesepharose support alternative strategy essentially relying immunoprecipitation technique have been introduced allow purification protein complex formed vivo within the cell the gene coding for the bait tagged with epitope against which good antibody exist flag cmyc etc transfected into the appropriate cell line and expressed the cognate host the cell extract are immunoprecipitated with antitag monoclonal antibody using suitable experimental condition avoid dissociation the complex both case protein component specifically recognised the bait and retained the agarose bead can then eluted and fractionated sdspage the protein band detected the gel are situ enzymatically digested and the resulting peptide mixture analysed capillary lcmsms technique leading the identification the protein interactors',\n",
       " '',\n",
       " 'revolutionary advance biological mass spectrometry have provided basic tool make possible comprehensive proteomic analysis traditionally twodimensional gel electrophoresis ha been used separation method coupled with facilitate analysis complex protein mixture despite the utility this method the many challenge comprehensive proteomic analysis ha motivated the development gelfree msbased strategy obtain information not accessible using twodimensional gel separation these advanced strategy have enabled researcher dig deeper into complex proteome gaining insight into the composition quantitative response covalent modification and macromolecular interaction protein that collectively drive cellular function this review describes the current state gelfree high throughput proteomic strategy using including the separation approach commonly used for complex mixture analysis strategy for largescale quantitative analysis iii analysis posttranslational modification and recent advance and future direction the use these strategy make new discovery the proteome level into the effect disease other cellular perturbation discussed variety context providing information the potential these tool electromagnetic field research',\n",
       " 'modern science proteomic analysis inseparable from other field systemic biology possessing huge resource quantitative proteomics operates colossal information molecular mechanism life advance proteomics help researcher solve complex problem cell signaling posttranslational modification structure and functional homology protein molecular diagnostics etc more than various method have been developed proteomics for quantitative analysis protein although each method unique and ha certain advantage and disadvantage all these use various isotope label tag this review will consider the most popular and effective method employing both chemical modification protein and also metabolic and enzymatic method isotope labeling',\n",
       " '',\n",
       " 'technology development and the application proteomics are emerging area among chemical engineer and others who presented the american institute chemical engineer aiche annual meeting overall the centennial meeting offered broad current perspective the discipline chemical engineering enters it second century biomedical and biochemical engineering continue grow important facet the discipline within these the value and applicability proteomics were demonstrated number interesting presentation this year the recent past the aiche annual meeting wa held conjunction with the american electrophoresis society annual meeting american electrophoresis society presenter offered further academic and industrial viewpoint the stilldeveloping role proteomics and proteomic technology biological and clinical analysis',\n",
       " '',\n",
       " 'proteome analysis provides way identify protein related the quality trait beer number protein specie beer and wort have been identified twodimensional gel electrophoresis combined with enzyme digestion such trypsin followed mass spectrometry analysis andor liquid chromatography massmass spectrometry addition low molecular weight polypeptide beer have been identified the combination nonenzyme digestion and mass analysis these data set various molecular weight polypeptide proteome provide platform for analyzing protein function beer several novel protein related beer quality trait such foam stability and haze formation have been identified analyzing these proteome some the protein have been applied the development efficient protein dna marker for trait selection malting barley breeding this chapter recent proteome study beer and wort are reviewed and the method and protocol beer and wort proteome analysis are described',\n",
       " 'while nearly comprehensive proteome coverage can achieved from bulk tissue cultured cell the data usually lack spatial resolution result tissue based proteomics average protein abundance across multiple cell type andor localization with proteomics platform lacking sensitivity and throughput undertake deep singlecell proteome study order resolve spatial cell type dependent protein expression gradient within tissue proteome analysis ha been combined with sorting technique enrich for certain cell population however the spatial resolution and context lost after cell sorting here report optimized method for the proteomic analysis neuron isolated from postmortem human brain laser capture microdissection lcm tested combination sample collection method lysis buffer and digestion method maximize the number identification and quantitative performance identifying protein from μmsup2sup thick cerebellar molecular layer with excellent reproducibility demonstrate the ability our workflow resolve cell type specific proteome within human brain tissue isolated set individual betz and purkinje cell both neuronal cell type are involved motor coordination and were found express highly specific proteome depth protein',\n",
       " 'numerous workflow exist for largescale bottomup proteomics many which achieve exceptional proteome depth herein evaluated the performance several commonly used sample preparation technique for proteomic characterization hela lysates unfractionated insolution digest sdspage coupled with ingel digestion geleluted liquid fraction entrapment electrophoresis gelfree technology scx stagetips and highlowph reversed phase fractionation hph hph fractionation wa found superior term proteome depth protein detected and fractionation efficiency compared other technique scx stagetip fractionation required minimal sample handling and wa also substantial improvement over sdspage separation and gelfree technology sequence coverage the hela proteome increased when combining all workflow however total protein detected improved only slightly summary hph fractionation and scx stagetips are robust technique and highly suited for complex proteome analysis',\n",
       " 'this article describes the origin working practice and various development project the human proteome organisation proteomics standard initiative hupo psi specifically our work reporting requirement data exchange format and controlled vocabulary term also offer our view the two functional genomics project which the psi play role fuge and fugo discussing their impact our process and laying out the benefit see accruing both the psi and biomedical science whole result their widespread acceptance',\n",
       " 'biomarker discovery the translational research ha heavily relied labeled and labelfree quantitative bottomup proteomics here describe new approach biomarker study that utilizes highthroughput topdown proteomics and the first offer whole protein characterization and relative quantitation within the same experiment using yeast model report procedure for labelfree approach quantify the relative abundance intact protein ranging from kda two different state this chapter describe the integrated methodology for the largescale profiling and quantitation the intact proteome liquid chromatographymass spectrometry lcm without the need for metabolic chemical labeling this recent advance for quantitative topdown proteomics best implemented with robust and highly controlled sample preparation workflow before data acquisition highresolution mass spectrometer and the application hierarchical linear statistical model account for the multiple level variance contained quantitative proteomic comparison sample for basic and clinical research',\n",
       " '',\n",
       " '',\n",
       " 'the term proteomics encompasses the largescale detection and analysis protein and their posttranslational modification driven major improvement mass spectrometric instrumentation methodology and data analysis the proteomics field ha burgeoned recent year now provides range sensitive and quantitative approach for measuring protein structure and dynamic that promise revolutionize our understanding cell biology and molecular mechanism both human cell and model organism the proteomics specification time and space prospect network unique eufunded project that brings together leading european research group spanning from instrumentation biomedicine collaborative five year initiative develop new method and application for the functional analysis cellular protein this special issue molecular and cellular proteomics present research paper reporting major recent progress the prospect group including improvement the resolution and sensitivity the orbitrap family mass spectrometer systematic detection protein using highly characterized antibody collection and new method for absolute well relative quantification protein level manuscript this issue exemplify approach for performing quantitative measurement cell proteome and for studying their dynamic response perturbation both during normal cellular response and disease mechanism here present perspective how the proteomics field moving beyond simply identifying protein with high sensitivity toward providing powerful and versatile set assay system for characterizing proteome dynamic and thereby creating new third generation proteomics strategy that offer indispensible tool for cell biology and molecular medicine',\n",
       " 'the emergence msbased proteomic platform prominent technology utilized biochemical and biomedical research ha increased the need for highquality measurement address this need national institute standard and technology nist reference material yeast protein extract introduced proteomics quality control material for benchmarking the preanalytical and analytical performance proteomicsbased experimental workflow yeast protein extract based upon the wellcharacterized eukaryote saccharomyces cerevisiae and can utilized the design and optimization proteomicsbased methodology from sample preparation data analysis demonstrate it utility proteomics quality control material coupled lcmsms measurement with the nist quality control msqc performance metric quantitatively ass the lcmsms instrumentation parameter that influence measurement accuracy repeatability and reproducibility due the complexity the yeast proteome also demonstrate how nist along with the nist msqc performance metric can used the evaluation and optimization proteomicsbased sample preparation method',\n",
       " 'deep proteomic analysis mammalian cell line would yield inventory the building block the most commonly used system biological research mass spectrometrybased proteomics can identify and quantify protein global and unbiased manner and can highlight the cellular process that are altered between such system analyzed human cell line using ltqorbitrap family mass spectrometer with high field orbitrap mass analyzer with improved resolution and sequencing speed identified total protein and average protein each cell line this very high proteome coverage enabled analysis broad range process and function despite the distinct origin the cell line our quantitative result showed surprisingly high similarity term expressed protein nevertheless this global similarity the proteome did not imply equal expression level individual protein across the cell line found significant difference expression level for estimated twothird them the variability cellular expression level wa similar for low and high abundance protein and even many the most highly expressed protein with household role showed significant difference between cell metabolic pathway which have high redundancy exhibited variable expression whereas basic cellular function such the basal transcription machinery varied much le harness knowledge these cell line proteome for the construction broad coverage supersilac quantification standard together with the accompanying paper schaab mcp pmid these data can used obtain reference expression profile for protein interest both within and across cell line proteome',\n",
       " 'proteomics one the most important functional genomics research the postgenomic era which closely related medical biology chemistry physic information science and modern technology through review research progress some important proteomics proteomics special issue published find problem explore the possible application and outlook the development prospect proteomics the special issue consists review and original paper mainly involving the following aspect proteomics about different specie such human mammal prokaryote and actinobacterial proteomics methodology and technique including tandem mass spectrometry analysis film urimem preservation urine protein quantitative proteomic analysis and meta analysis iii function and application proteome such spider latrodectus tredecimguttatus toxin proteome protein phosphorylation proteome oocyte and early embryo proteome liver fibrosis proteome drugresistant mycobacterium tuberculosis proteome etc',\n",
       " 'epithelial lining fluid elf form thin fluid layer that cover the mucosa the alveolus the small airway and the large airway since constitutes the first barrier between the lung and the outer world interesting target for proteomics study that focus lung disease bronchoscopic microprobe bmp sampling elf us small probe with absorptive tip that are introduced bronchoscopically contrast other method used far for the collection biofluids from the lung bronchoalveolar lavage fluid induced sputum this technique ha the advantage that elf not diluted and contains high concentration biomolecules addition the investigated location the tracheobronchial tree well defined and there contamination with oropharyngeal bacteria saliva despite occasional blood contamination the probe scratching the mucosa the airway the proteomic analysis microprobesampled elf open new possibility for research lung disease our work focus particularly the induction and progression cigarette smokeinduced chronic obstructive pulmonary disease copd this chapter describe the practical aspect sampling elf followed detailed description proteomics analysis lcmsms after protein separation sdspage and ingel digestion example apply this proteomic platform the identification and quantification protein elf from copd patient and healthy subject',\n",
       " 'reference material are vital benchmarking the reproducibility clinical test and essential for monitoring laboratory performance for clinical proteomics the reference material utilized for mass spectrometric analysis the human proteome would ideally contain enough protein suitably representative the human proteome well exhibit stable protein composition different batch sample regeneration previously the clinical proteomic tumor analysis consortium cptac utilized pdxderived comparative reference compref material for the longitudinal assessment proteomic performance however inherent drawback pdxderived material including extended time needed grow tumor and high level expertise needed have resulted effort identify new source compref material this study examined the utility using panel seven cancer cell line nci7 cell line panel reference material for mass spectrometric analysis human proteome our result showed that not only the nci7 material suitable for benchmarking laboratory sample preparation method but also nci7 sample generation highly reproducible both the global and phosphoprotein level addition the predicted genomic and experimental coverage the nci7 proteome suggests the nci7 material may also have application universal standard proteomic reference',\n",
       " 'the aim this work wa the functional and proteomic analysis mutant bgl isolated from batch culture escherichia coli strain maintained room temperature without addition nutrient for year when the mutant wa evaluated competition experiment coculture with the wildtype exhibited the growth advantage stationary phase gasp phenotype proteome the gasp mutant and it parental strain were compared using 2de coupled with approach several differentially expressed protein were detected and many them were successful identified mass spectrometry identified expressionchanging protein were grouped into three functional category metabolism protein synthesis chaperone and stress responsive protein among them the prevalence wa ascribable the metabolism group for the gasp mutant and chaperone and stress responsive protein group for the parental strain',\n",
       " 'average human cell type express around different protein coding gene synthesizing all the different molecular form the protein product proteoforms found cell typical shotgun bottom proteomic approach the protein are enzymatically cleaved producing several different peptide that are analyzed with liquid chromatographytandem mass spectrometry lcmsms one the major consequence this high sample complexity that coelution peptide can not avoided moreover low abundant peptide are difficult identify they have lower chance being selected for fragmentation due ionsuppression effect and the semistochastic nature the precursor selection datadependent shotgun proteomic analysis where peptide are selected for fragmentation analysis onebyone they elute from the column the current study explore simple novel approach that ha the potential counter some the effect coelution peptide and improves the number peptide identification bottomup proteomic analysis this method peptide from hela cell digest were eluted from the reverse phase column using three different elution solvent acetonitrile methanol and acetone three replicate reversed phase lcmsms shotgun proteomic analysis result were compared with three technical replicates using the same solvent which common practice proteomic analysis total see increase unique protein and unique peptide identification from the combined analysis using different elution solvent when compared the combined identification from the three replicates the same solvent addition the overlap unique peptide identification common all three lcm analysis our approach only compared the replicates using the same solvent the method presented here thus provides easy implement method significantly reduce the effect coelution and ion suppression peptide and improve protein coverage shotgun proteomics data are available via proteomexchange with identifier pxd',\n",
       " 'saliva identified functional equivalent serum reflecting the physiological state the body well hormonal emotional nutritional and metabolic alteration the application mass spectrometry based approach ha allowed thorough characterization the saliva proteome and led the discovery putative biomarkers several salivary biomarkers have been recently explored potentially useful screening tool patient diagnosed with metabolic disorder this review provide overview saliva proteomics study with focus diabetes and explore the evidence for the utility well identified marker for the diagnosis and monitoring the disease emerging approach salivary diagnostics that may significantly advance the field diabetes research are also highlighted',\n",
       " '',\n",
       " 'the proteome the functional entity the cell and perturbation cellular system almost always cause change the proteome these change are molecular fingerprint allowing characterization and greater understanding the effect the perturbation the cell whole monitoring these change ha therefore given great insight into cellular response stress and disease state and analytical platform comprehensively analyze the proteome are thus extremely important tool biological research mass spectrometry ha evolved the most relevant technology characterize proteome comprehensive way however due lack throughput capacity mass spectrometrybased proteomics researcher frequently use measurement mrna level approximate proteome change growing evidence substantial difference between mrna and protein level well recent improvement mass spectrometrybased proteomics are heralding increased use mass spectrometry for comprehensive proteome mapping here describe the use multiplexed quantitative proteomics using isobaric labeling with tandem mass tag tmt for the simultaneous quantitative analysis five cancer cell proteome biological duplicate one mass spectrometry experiment',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'although major advance have been made during the past year our understanding the genetic and genomic consequence polyploidy our knowledge polyploidy and the proteome it infancy one our goal stimulate additional study particularly broadscale proteomic analysis polyploid and their progenitor although may too early generalize regarding the extent which transcriptomic data are predictive the proteome polyploid clear that the proteome doe not always reflect the transcriptome despite limited data important observation the proteome polyploid are emerging some case proteomic profile show qualitatively andor quantitatively nonadditive pattern and proteomic novelty ha been observed allopolyploids generally combine the parental contribution but there evidence parental dominance one contributing genome some allopolyploids autopolyploids are typically qualitatively identical but quantitatively different from their parent there also evidence parental legacy the proteomic level proteome clearly provide insight into the consequence genomic merger and doubling beyond what obtained from genomic andor transcriptomic data translating proteomic change polyploid difference morphology and physiology remains the holy grail polyploidythis daunting task linking genotype proteome phenotype should emerge focus polyploidy research the next decade this article part special issue entitled plant proteomicsa bridge between fundamental process and crop production edited hanspeter mock',\n",
       " 'understanding biological system and their variation upon stimulus requires knowledge their composition primarily including information organization and dynamic proteome the total protein approach tpa label and standardfree method for absolute protein quantitation protein using largescale proteomic data the method relies the assumption that the total signal from all identified protein the dataset reflectsin biochemical sensethe total protein and the signal from single protein corresponds it abundance the studied sample the method offer easy way quantify thousand protein per sample related method the proteomic ruler enables conversion the protein abundance data calculated tpa compute number protein copy per cell tpa and the proteomic ruler are powerful tool for studying dynamic cell architecture',\n",
       " 'research proteomics the next step after genomics understanding life process the molecular level the largest sense proteomics encompasses knowledge the structure function and expression all protein the biochemical biological context all organism since that impossible goal achieve least our lifetime appropriate set more realistic achievable goal for the field now primarily for reason feasibility scientist have tended concentrate accumulating information about the nature protein and their absolute and relative level expression cell the primary tool for this have been gel electrophoresis and mass spectrometry although these data have been useful and will continue the information inherent the broader definition proteomics must also obtained the true promise the growing field realized acquiring this knowledge the challenge for researcher proteomics and the mean support these endeavor need provided attempt ha been made present the major issue confronting the field proteomics and two clear message come through this report the first that the mandate proteomics and should much broader than frequently recognized the second that proteomics much more complicated than sequencing genome this will require new technology but highly likely that many these will developed looking back year from now the question will have done the job wisely wastefully this report summarizes the presentation made symposium the national academy science february',\n",
       " 'advance mass spectrometry have transformed the scope and impact protein characterization effort identifying hundred protein from rather simple biological matrix such yeast wa daunting task just few decade ago now expression more than half the estimated human protein coding gene can confirmed record time and from minute sample quantity access proteomic information such unprecedented depth ha been fueled stride every stage the shotgun proteomics workflowfrom sample processing data analysisand promise revolutionize our understanding the cause and consequence proteome variation',\n",
       " 'the application mass spectrometry the analysis proteome ha enabled the highthroughput identification and abundance measurement hundred thousand protein per experiment however the formidable informatics challenge associated with analyzing data ha required wide variety data file format encode the complex data type associated with workflow these format encompass the encoding input instruction for instrument output product the instrument and several level information and result used and produced the informatics analysis tool brief overview the most common file format use today presented here along with discussion related topic',\n",
       " 'designing experiment for quantitative proteomic analysis not trivial task one the key factor influencing the success such study the number biological replicates included the analysis this along with the measured variation will determine the statistical power the analysis presented simple yet powerful analysis determine the appropriate sample size required for reliable and reproducible result based the total variation technical and biological this approach can also applied retrospectively for the interpretation result take into account both significance value and quantitative difference fold change the result',\n",
       " 'the proteome determined rate transcription translation and protein turnover definition stem cell population therefore requires stem cell proteome signature however the limit the number primary cell available ha restricted extensive proteomic analysis present mass spectrometric method using isobaric covalent modification peptide for relative quantification itraq which wa employed compare the proteome approximately million longterm reconstituting hematopoietic stem cell linscakit lsk and nonlongterm reconstituting progenitor cell linscakit lsk respectively extensive 2dimensional liquid chromatography peptide separation prior mass spectrometry enabled enhanced proteome coverage with relative quantification protein the change the proteome were not seen the transcriptome hypoxiarelated change protein controlling metabolism and oxidative protection were observed indicating that lsk cell are adapted for anaerobic environment this approach can define proteomic change primary sample thereby characterizing the molecular signature stem cell and their progeny',\n",
       " 'the past decade mass spectrometrybased proteomics ha greatly contributed shaping our knowledge about saccharomyces cerevisiae mitochondrion from the initial identification novel essential component purified protein complex the actual characterization the mitochondrial proteome the specific analysis mitochondrial subcompartment proteome and the study regulatory mechanism that govern mitochondrial homeostasis here provide overview relevant mitochondrial proteome study and furthermore discus future possibility how proteomics will further improve our existing understanding mitochondrion although mitochondrion were the first organelle that have been intensively studied using proteomics indeed the recent progress and development more powerful and sensitive method instrumentation and data analysis strategy indicate that are only beginning exploit the full potential mitochondrial proteomics and it possibility decipher cell biology beside mere quantitative inventory under different condition this will for instance include studying the role multiple posttranslational modification mitochondrial homeostasis well the systemwide mapping proteinprotein complex and proteinlipid interaction',\n",
       " 'new development proteomics enable scientist examine hundred thousand protein parallel quantitative proteomics allows the comparison different proteome cell tissue body fluid with each other analyzing and especially organizing these data set often herculean task pathway analysis software tool aim take over this task based present knowledge company promise that their algorithm help understand the significance scientist data but the benefit remains questionable and fundamental systematic evaluation the potential such tool ha not been performed until now here tested the commercial ingenuity pathway analysis tool well the freely available software string using welldefined study design regard the applicability and value their result for proteome study wa our goal cover wide range scientific issue simulating different established pathway including mitochondrial apoptosis tau phosphorylation and insulin app and wntsignaling next general assessment and comparison the pathway analysis tool provide recommendation for user well for software developer improve the added value pathway study implementation proteomic pipeline',\n",
       " 'the usage strong cation exchange scx chromatography proteomics limited it poor resolution and nonspecific hydrophobic interaction with peptide which lead peptide overlap across fraction and change peptide retention respectively the application high concentration salt scx also restricted it use online scxrp the present research first exploited the chromatographic ability online scxrp combination acid salt and gradient three relatively independent mode eluting peptide from scx column acn wa added elution buffer for eliminating hydrophobic interaction between scx matrix and peptide and the concentration volatile salt wa reduced acidsaltph gradient showed superior resolution and sensitivity well uniform distribution across fraction consequently leading significant improvement peptide and protein identification unique peptide and protein were identified acidsaltph fractionation while unique peptide and protein were identified salt elution that and more protein and unique peptide respectively identified the former fraction overlap wa also significantly minimized acidsaltph approach furthermore acidsaltph elution showed more identification for acidic peptide and hydrophilic peptide',\n",
       " 'chloroplast the photosynthetic machinery convert photoenergy atp and nadph which power the production carbohydrate from atmospheric cosub2sub and hsub2subo also serf major production site multivariate prodefense molecule and coordinate with other organelle for cell defense chloroplast harbor total cellular protein out which are membrane resident and are difficult solubilize while proteome profiling ha illuminated vast area biological protein space great deal effort must invested understand the proteomic landscape the chloroplast which play central role photosynthesis energy metabolism and stressadaptation therefore characterization chloroplast proteome would not only provide the foundation for future investigation expression and function chloroplast protein but would open new avenue for modulation plant productivity through synchronizing chloroplastic key component this review summarize the progress that ha been made build new understanding the chloroplast proteome and implication chloroplast dynamicsing generate metabolic energy and modulating stress adaptation',\n",
       " 'proteomics ha shown great promise key tool the analysis living system providing highthroughput view the expression and translation gene into protein their abundance and their modification yet the rapid advance the field have led emphasis the development novel methodology the cost establishing clear quality control metric and acceptable operational parameter for many these method since proteomics ultimately analytical science however essential for the future the field invest quality control and quality assurance strategy for existing approach well with the support the european science foundation frontier functional genomics programme workshop wa therefore held cambridge from until november that focused quality control proteomics report this workshop provided here',\n",
       " 'proteomics mass spectrometrybased discipline that aim analyze proteome and their function many proteomic study require welldeveloped protein database for reference however most proteome are not wellannotated aside from model organism technique like sixframe translation initio gene prediction and est database can aid maximizing the amount protein identified proteomics experiment however each these ha it downfall proteogenomics term used describe the union proteomics genomics and transcriptomics assist the identification peptide which would help build better annotated proteome database here current proteomic and proteogenomic method will reviewed and example comparative proteomics method using lake trout liver sample will described',\n",
       " 'multiparameter optimization lcmsms shotgun proteomics experiment wa performed without any hardware software modification the commercial instrument under the optimized experimental condition with cmlong separation column and lcm run including optimized gradient protein group and peptide were identified single run and protein group and peptide triplicate analysis the human cell line for approximately coverage the expressed proteome the major step enabling such performance included optimization the cell lysis and protein extraction digestion even insoluble cell debris tailoring the gradient profile and choosing the optimal dynamic exclusion window datadependent msms well the optimal scan window',\n",
       " 'have confidence result acquired during biological mass spectrometry experiment systematic approach quality control vital importance nonetheless until now only scattered initiative have been undertaken this end and these individual effort have often not been complementary address this issue the human proteome organizationproteomics standard initiative ha established new working group quality control it meeting the spring the goal this working group provide unifying framework for quality control data the initial focus will providing communitydriven standardized file format for quality control for this purpose the previously proposed qcml format will adapted support variety use case for both proteomics and metabolomics application and will established official psi format important consideration avoid enforcing restrictive requirement quality control but instead provide the basic technical necessity required support extensive quality control for any type mass spectrometrybased workflow want emphasize that this open community effort and seek participation from all scientist with interest this field',\n",
       " '',\n",
       " 'gel electrophoresis enables resolution intact protein complex mixture and thus useful for comparative proteomic analysis particularly posttranslationally modified proteoforms that might not distinguished shotgun proteomic analysis peptide gel electrophoresis multistep procedure that can require samplespecific optimization present comprehensive protocol that effective for electrophoretic analysis protein from leishmania promastigotes and may also employed for leishmania amastigotes and for trypanosomes',\n",
       " '',\n",
       " 'single eukaryotic gene can give rise variety protein form proteoforms result genetic variation and multilevel regulation gene expression addition alternative splicing increasing line evidence show that alternative translation contributes the overall complexity proteome identifying the repertoire protein and micropeptides expressed alternative selection nearcognate translation initiation site and different reading frame however remains challenging with contemporary proteomics msenabled identification proteoforms expected benefit from transcriptome and translatome data the creation customized and samplespecific protein sequence database here focus contemporary integrative omics approach that complement proteomics with dna andor rnaoriented technology elucidate the mechanism translational control together these technology enable map the translation initiation landscape and more comprehensively define the inventory proteoforms raised upon alternative translation thus assisting the reannotation genome',\n",
       " '',\n",
       " 'mass spectrometrybased proteomics greatly benefited from recent improvement instrument performance and the development bioinformatics solution facilitating the highthroughput quantification protein complex biological sample addition quantification approach using stable isotope labeling labelfree quantification ha emerged the method choice for many laboratory over the last year dataindependent acquisition approach have gained increasing popularity the integration ion mobility separation into commercial instrument enabled researcher achieve deep proteome coverage from limiting sample amount additionally ion mobility provides new dimension separation for the quantitative assessment complex proteome facilitating precise labelfree quantification even highly complex sample the present work provides thorough overview the combination ion mobility and dataindependent acquisitionbased labelfree quantification lcm and it application biomedical research',\n",
       " 'formalinfixed and paraffinembedded ffpetissue archive are potential treasure trove the search for clinically interesting specimen however while the ffpetreatment provides excellent conservation the threedimensional structure the tissue and prevents degradation over decade also introduces numerous nonspecific and irreversible protein modification this study have evaluated several published workflow for ffpetissue fitforpurpose proteomics technology demonstrate that many protein modification and crosslinks remain after treatment and conclude that the proteomics ffpetissue value but clearcut limitation must kept mind the analysis abundant protein ffpe straightforward but confident identification lowlevel protein andor biologically relevant modification seriously hampered the ffpetreatment peptide assignment should only performed highquality spectrum even this the cost lower number protein id yergey and coorssen stated data quality considered the primary criterion and thus emphasize that the standard analytical chemistry must apply throughout any proteomic analysis',\n",
       " 'there are likely more than potential phosphorylation site cellular proteome this dynamic phosphorylation under tight control variety kinase and phosphatase recent year significant progress ha been made the largescale analysis these vitro and vivo protein phosphorylation event much this data have been deposited public depository which have described far about phosphorylation site about human protein however the amount phosphoproteomics data generated now growing tremendously rate about site per reported study the major advance made global phosphoproteomics analysis originate from technological advance whereby the largest contribution are from more selective phosphopeptide affinity enrichment technology and from faster and more sensitive mass spectrometer',\n",
       " 'this planet home countless specie some more wellknown than the others while have developed many technique able interrogate some the omics proteomics becoming recognized very important part the puzzle given how important the protein functional part the cell within human health the proteome fairly wellestablished with numerous reagent being available decipher cellular pathway recent research advancement have assisted characterizing the proteome some model nonhuman specie however many other specie are only just touching the surface this review considers three main reagent classesantibodies aptamers and nanobodiesas mean continuing investigate the proteome nonmodel specie without the complication understanding the full protein signature specie consideration ease production potential application and the necessity for producing new reagent depending homology are presented',\n",
       " 'mass spectrometry frequently used quantitative proteomics detect differentially regulated protein very important but unfortunately oftentimes neglected part detecting differential protein the statistical analysis data from proteomics experiment are usually highdimensional and hence require profound statistical method especially important already correctly design proteomic experiment before conducted the laboratory only this can ensure that the statistical analysis capable detecting truly differential protein afterward this chapter thus cover aspect both statistical planning well the actual analysis quantitative proteomic experiment',\n",
       " 'the highly complex and speciesselective mechanism fertilization central theme developmental biology gametogenesis sperm activation and eggsperm recognition are fundamental biological process warranting detailed study into the molecular composition gamete biological ha been instrumental for the comprehensive itemizing gamete proteome the protein constellation sperm cell and it subcellular structure ha been established for variety animal specie spermatogenesis and the crucial activation sperm cell prerequisite successful fertilization and physiological adaptation external stressor wa investigated using proteomics well the underlying mechanism male infertility with respect proteomewide alteration this review outline recent achievement sperm proteomics and exemplifies the usefulness gelbased survey outlining the comparative analysis abnormal spermatozoon globozoospermia besides labelfree technique and cellbased labeling methodology highresolution fluorescence 2de ha been shown highly suitable proteomic biomarker discovery tool sperm protein research the appropriateness novel protein marker for improving our understanding normal spermatogenesis and sperm activation versus the molecular pathogenesis male infertility will discussed new biomarker candidate might useful improve diagnostic prognostic and therapeutic aspect infertility',\n",
       " 'microalgae present promising feedstock produce renewable fuel and chemical intermediate part due high storage carbon flux capacity triacylglycerides storage carbohydrate upon nutrient deprivation however the mechanism governing deprivationmediated carbon partitioning remain fully elucidated limiting targeted strain engineering strategy algal biocatalyst though genomic and transcriptomic analysis offer key insight into these mechanism active posttranscriptional regulatory mechanism ubiquitous many microalgae necessitate proteomic and posttranslational phospho and nitrosoproteomic analysis more completely evaluate algal responsiveness nutrient deprivation herein describe method for isolating total algal protein and conducting proteomic phosphoproteomic and nitrosoproteomic analysis focus method deployed for the chlorophyte chlorella vulgaris model oleaginous alga with high flux renewable fuel and chemical precursor',\n",
       " 'animal venom are renowned for their toxicity biochemical complexity and source compound with potential application medicine agriculture and industry polypeptide underlie much the pharmacology animal venom and elucidating these arsenal polypeptide toxinsknown the venom proteome venomeis important step venom research proteomics used for the identification venom toxin determination their primary structure including posttranslational modification well investigation into the physiology underlying their production and delivery advance proteomics and adjacent technology ha led recent upsurge publication reporting venom proteome improved mass spectrometer better proteomic workflow and the integration nextgeneration sequencing venomgland transcriptomes and venomous animal genome allow quicker and more accurate profiling venom proteome with greatly reduced starting material technology such imaging mass spectrometry are revealing additional insight into the mechanism location and kinetics venom toxin production however these numerous new development may overwhelming for researcher designing venom proteome study here the field venom proteomics reviewed and some practical solution for simplifying mass spectrometry workflow study animal venom are offered',\n",
       " 'critical analysis proteome provides basis for understanding the operation complex biochemical system personalized approach therapy take into account biological uniqueness each patient genome transcriptome and proteome level and priority area molecular medicine the identification proteoforms which have dramatic impact the phenotype disease fundamental task personal molecular profiling considerable progress proteomic approach presented new avenue for accurate specific and highperformance protein analysis thus the identification new efficient biomarkers can expected based study aberrant proteoforms associated with various disease',\n",
       " 'the minimization preanalytical variable sample preparation imperative for successful discoverydriven and translational research involving largescale biomolecular profiling here demonstrate novel technique using high hydrostatic pressure addition several chaotropes and solvent maximize efficiency both cell lysis and enzymatic digestion while minimizing the time manual involvement sample processing and preanalytical variability introduced prior mass spectrometrybased proteomic analysis the digestion technique were evaluated and optimized for insolution ingel and onmembrane application using protein standard and cell lysates the lysis technique were evaluated using human hepg2 cell our result demonstrate that the use elevated pressure and organic solvent can achieve superior protein recovery organelle complex and especially membraneassociated protein meanwhile obtaining more than fold increase throughput with improved reproducibility this study introduces the concept ultrahighperformance sample preparation platform for targeted characterization proteome subset biological system',\n",
       " '',\n",
       " 'advanced proteomic research effort involving area such system biology biomarker discovery are enabled the use high level informatics tool that allow the effective analysis large quantity differing type data originating from various study performing such analysis large scale not feasible without computational platform that performs data processing and management task such platform must able provide highthroughput operation while having sufficient flexibility accommodate evolving data analysis tool and methodology the proteomics research information storage and management system prism provides platform that serf the need the accurate mass and time tag approach developed pacific northwest national laboratory prism incorporates diverse set analysis tool and allows wide range operation incorporated using state machine that accessible independent distributed computational node the system ha scaled well data volume ha increased over several year while allowing adaptability for incorporating new and improved data analysis tool for more effective proteomics research',\n",
       " 'polyacrylamide gel electrophoresis ha been the traditional workhorse proteomics allowing for the resolution several thousand protein single gel difference gel electrophoresis emerging technology that allows for accurate quantification with statistical confidence while controlling for nonbiologic variation and also increase the dynamic range and sensitivity traditional polyacrylamide gel electrophoresis with inclusion internal standard formed from equal amount every sample experiment difference gel electrophoresis technology also allows for repetitive measurement and multivariable analysis quantitatively analyzed one coordinated experiment yielding statisticallysignificant change protein expression related many disease state this technique promise important tool clinical proteomics and the study the mechanism disease investigating diagnostic biomarkers and pinpointing novel therapeutic target',\n",
       " 'capillary zone electrophoresis cze emerging useful tool proteomic analysis interest arises from dramatic improvement performance that result from improvement the background electrolyte used for the separation the incorporation advanced sample injection method the development robust and sensitive electrospray interface and the coupling with orbitrap mass spectrometer with high resolution and sensitivity the combination these technology produce performance that rapidly approaching the performance uplcbased method for microgram sample and exceeds the performance uplcbased method for mid low nanogram sample these system now produce over peptide id single min analysis the hela proteome',\n",
       " 'although very attractive for noninvasive specimen collection saliva ha not yet been considered relevant bodily fluid for the diagnosis and prognosis disease the functional role specific salivary peptide and protein have also not yet been studied detail recent proteomic analysis human whole saliva ha shown that salivary biomarkers could contribute the detection local and systemic disease provided the standardization proper sampling procedure exists recently interesting and novel function for different family specific secretory peptide and protein have been demonstrated which could basis for the design peptidomimetics with relevant biotechnological application this review focus the most recent advance analysing salivary protein and their potential application biotechnology',\n",
       " 'newly reorganized hupo cardiovascular initiative wa announced the hupo cardiovascular initiative workshop geneva the new initiative now part the biology and diseasedriven component the hupo human proteome project bdhpp here report the recent achievement and future direction the initiative and offer perspective the present challenge cardiovascular proteomics and it integration with the cardiovascular biology community large',\n",
       " 'peptide intensity from mass spectrum are increasingly used for relative quantitation protein complex sample however numerous issue inherent the mass spectrometry workflow turn quantitative proteomic data analysis into crucial challenge and others have shown that modeling the peptide level outperforms classical summarizationbased approach which typically also discard lot protein the data preprocessing step peptidebased linear regression model however still suffer from unbalanced datasets due missing peptide intensity outlying peptide intensity and overfitting here further improve upon peptidebased model three modular extension ridge regression improved variance estimation borrowing information across protein with empirical bayes and mestimation with huber weight illustrate our method the cptac spikein study and study comparing wildtype and argp knockout francisella tularensis proteome show that the fold change estimate our robust approach are more precise and more accurate than those from stateoftheart summarizationbased method and peptidebased regression model which lead improved sensitivity and specificity also demonstrate that ionization competition effect come already into play very low spikein concentration and confirm that analysis with peptidebased regression method peptide intensity value aggregated charge state and modification status maxquants peptidestxt file are slightly superior analysis raw peptide intensity value maxquants evidencetxt file',\n",
       " 'characterizing the proteome composition organelle and subcellular region living cell can facilitate the understanding cellular organization well protein interactome network proximity labelingbased method coupled with mass spectrometry offer highthroughput approach for systematic analysis spatially restricted proteome proximity labeling utilizes enzyme that generate reactive radical covalently tag neighboring protein with biotin the biotinylated endogenous protein can then isolated for further analysis analyze proteinprotein interaction identify component that localize discrete subcellular compartment spatial expression achieved fusing the enzyme specific protein signal peptide that target particular subcellular region although these technology have only been introduced recently they have already provided deep insight into wide range biological process here describe and compare current method proximity labeling well their application each method ha it own unique feature the goal this review describe how different proximity labeling method can used answer different biological question wire dev biol doi wdev for further resource related this article please visit the wire website',\n",
       " 'direct consequence the high diversity the aggressive blood cancer acute myeloid leukemia aml proteomic sample from patient are strongly heterogeneous rendering their accurate relative quantification challenging the present study investigated the benefit using supersilac mix aml derived cell line internal standard for quantitative shotgun study the molm nb4 thp1 and ociaml3 cell line were selected for their complementarity with regard clinical cytogenetic and molecular risk factor used for prognostication aml patient the resulting present high coverage the aml proteome compared single cell line allied with high technical reproducibility thus enabling it use for aml patient comparison this wa confirmed comparing the protein regulation between the five cell line and applying the patient material hence were able reproduce specific functional regulation known related disease progression and molecular genetic abnormality the proteomics data have been deposited the proteomexchange consortium httpproteomecentralproteomexchangeorg via the pride partner repository with the dataset identifier pxd',\n",
       " 'the proteomics work reported smith represents giant step forward characterizing the cerebrospinal fluid csf proteome mouse model human disease whereas prior study were limited analysis csf pool smith proteomics base their conclusion data derived from individual mouse thereby capturing fuller range the biological diversity present these result underscore how far proteomics ha come the past few year developing into modern tool with the capacity remove bottleneck the study neuropsychiatric disease past effort with mass spectrometry have been hampered limitation access csf sample and small volume when available these barrier have been overcome with newer platform and advance sample preparation are far closer than before producing the production clinically useful proteomic data for biomarker discovery and for deriving insight into pathogenesis that can lead more effective treatment for many disease',\n",
       " 'the dawn new proteomics era just over decade ago allowed for largescale protein profiling study that have been applied the identification distinctive molecular cell signature proteomics provides powerful approach for identifying and studying these multiple molecular marker vast array biological system whether focusing basic biological research diagnosis therapeutic system biology this continuously expanding field that relies the combination different methodology and current advance both technological and analytical which have led explosion protein signature and biomarker candidate but how are these biological marker obtained and most importantly what can learn from them herein briefly overview the currently available approach for obtaining relevant information the proteome level while noting the current and future role both traditional and modern proteomics moreover provide some consideration how the development powerful and robust bioinformatics tool will greatly benefit highthroughput proteomics such strategy are the utmost importance the rapidly emerging field immunoproteomics which may play key role the identification antigen with diagnostic andor therapeutic potential and the development new vaccine finally consider the present limitation the discovery new signature and biomarkers and speculate how such hurdle may overcome while also offering prospect for the next few year what could one the most significant strategy translational medicine research',\n",
       " '',\n",
       " 'membrane protein are situated the interface bacterial cell and it environment and are therefore involved vital physiological process such nutrient exchange signal transduction and virulence due their distinct biophysical property especially hydrophobicity they are difficult subject study classical proteomics technology have relied multidimensional separation protein gel which largely limited the choice detergent and made the development specialized enrichment protocol for membrane protein necessary shotgun proteomic approach based the digestion whole proteome and subsequent analysis peptide lcm ha largely circumvented these problem due it compatibility with potent detergent here briefly present and discus the major development bacterial membrane proteomics and argue that recent development biochemical sample preparation and high resolution mass spectrometry have the potential comprehensively identify and quantify membrane protein without the need for specific enrichment procedure prior lcm analysis',\n",
       " 'protein isiacylation commonly known palmitoylation widespread reversible lipid modification which play critical role regulating protein localization activity stability and complex formation the deregulation protein isiacylation contributes many disease such cancer and neurodegenerative disorder the past decade ha witnessed substantial progress proteomic analysis protein isiacylation which significantly advanced our understanding isiacylation biology this review summarized the technique for the enrichment isiacylated protein peptide critically reviewed proteomic study protein isiacylation eight different level and proposed major challenge for the isiacylproteomics field summary proteomescale analysis protein isiacylation come age and will play increasingly important role discovering new disease mechanism biomarkers and therapeutic target',\n",
       " 'the oxidative stress response characterized various effect range biologic molecule when examined the protein level both expression level and protein modification are altered oxidative stress while these effect have been studied the past classic biochemical method the recent onset proteomics method ha allowed the oxidative stress response studied much wider scale the input proteomics the study oxidative stress response and the evidence oxidative stress component biologic phenomenon reviewed this paper',\n",
       " 'massspectrometrybased shotgun proteomics ha become widespread technology for analyzing complex protein mixture here describe new module integrated into patternlab for proteomics that allows the pinpointing differentially expressed domain this accomplished inferring functional domain through our cloud service using hmmer3 and pfam remotely and then mapping the quantitation value into domain for downstream analysis all spotting which functional domain are changing when comparing biological state serf complementary approach facilitate the understanding system biology exemplify the new module use reanalyzing previously published mudpit dataset cryptococcus gattii cultivated under irondepleted and replete condition show how the differential analysis functional domain can facilitate the interpretation proteomic data providing further valuable insight',\n",
       " 'mass spectrometry assay demonstrate that hsp inhibitor alter the expression approximately onequarter the assayable proteome mammalian cell these change are extraordinarily robust and reproducible making proteomics profiling the gold standard for validating the effect new hsp inhibitor cultured cell proteomics assay can also suggest novel hypothesis regarding drug mechanism assist investigator adopting this approach this chapter provides detailed protocol for conducting simple proteomics assay hsp inhibition the protocol present robust labelfree approach that utilizes prefractionation protein sample sdspage thereby providing reasonably good penetration into the proteome while addressing common issue with sample quality the actual programming and operation liquid chromatographytandem mass spectrometer not covered but expectation for achievable performance are discussed are alternative approach common challenge and software for data analysis',\n",
       " 'ever since antibiotic were used help humanity battle infectious disease microorganism straight away fought back antibiotic resistance mechanism indeed provide microbe with possibility bypass and survive the action antibiotic drug several method have been employed identify these microbial resistance mechanism ongoing effort reduce the steadily increasing number treatment failure due multidrugresistant microbe proteomics ha evolved important tool for this area research following rapid advance whole genome sequencing proteomic technology have been widely used investigate microbial gene expression this review highlight the contribution proteomics identifying microbial drug resistance mechanism summarizes different proteomic study bacteria resistant different antibiotic drug the review further includes overview the methodology used well list key protein identified thus providing the reader not only summary research already done but also direction for future research this article part special issue entitled trend microbial proteomics',\n",
       " 'the search for protein biomarkers ha been highly pursued topic the proteomics community the last decade this relentless search due the constant need for validated biomarkers that could facilitate disease risk stratification disease diagnosis prognosis monitoring well drug development which ultimately would improve our quality life the recent development proteomic technology including the advancement mass spectrometer with high sensitivity and speed ha greatly advanced the discovery potential biomarkers one the bottleneck lie the development wellestablished verification assay screen the biomarker candidate identified the discovery stage recently absolute quantitation using multiplereaction monitoring mass spectrometry mrmms combination with isotopelabeled internal standard ha been extensively investigated tool for highthroughput protein biomarker verification this review describe and discus recent development and application mrmms method for biomarker verification',\n",
       " 'most recently approved anticancer drug the fda are targeted therapeutic agent and this represents important trend for future anticancer therapy unlike conventional chemotherapy that rarely considers individual difference crucial for targeted therapy identify the beneficial subgroup patient for the treatment currently genomics and transcriptomics are the major omic analytics used study drug response prediction however proteomic profiling excels both it advantage directly detecting instantaneous dynamic the whole proteome which contains most current diagnostic marker and therapeutic target moreover proteomic profiling improves understanding the mechanism for drug resistance and help finding optimal combination therapy this article review the recent success application proteomic analytics predicting the response targeted anticancer therapeutic and discus the potential avenue and pitfall proteomic platform and technique used most the field',\n",
       " 'proteolytic processing ubiquitous and irreversible posttranslational modification involving limited and highly specific hydrolysis peptide and isopeptide bond protein protease cleavage generates shorter protein chain displaying neon and terminus often with new modified biological activity within the past decade degradomics and terminomics have emerged significant proteomics subfields dedicated characterizing proteolysis product well natural protein and terminus here provide overview contemporary proteomicsbased method including specific quantitation data analysis and curation consideration and highlight exciting new and emerging application within these field enabling vivo analysis proteolytic event',\n",
       " 'formalinfixed paraffinembedded ffpe tissue specimen comprise potentially valuable resource for both prospective and retrospective biomarker discovery unlocking the proteomic profile clinicopathological ffpe tissue critically essential step for annotating clinical finding and predicting biomarkers for ultimate disease prognosis and therapeutic followup',\n",
       " 'the diversity biological sample and dynamic range analytes being analyzed can prove analytical challenge and particularly prevalent proteomic study maximizing the peak capacity the workflow employed can extend the dynamic range and increase identification rate the focus this chapter present mean achieving this for various analytical technique such liquid chromatography mass spectrometry and ion mobility combination these method can used part dataindependent acquisition strategy thereby limiting issue such chimericy when analyzing region extreme analyte density',\n",
       " '',\n",
       " 'the longterm challenge proteomics enormous define the identity quantity structure and function complete complement protein and characterize how these property vary different cellular context one critical step tackling this goal the generation set clone that express representative each protein proteome useful format followed the analysis these set genomewide basis such study enable genetic biochemical and cell biological technology applied systematic level leading the assignment biochemical activity the construction protein array the identification interaction and the localization protein within cellular compartment',\n",
       " 'proteomics the global study protein expression and characteristic ha recently emerged key component the field molecular analysis the dynamic nature protein from ion channel chaperone present challenge yet the understanding these molecule provides rich source information when applying proteomic analysis directly human tissue sample additional difficulty arise the following article present overview the current proteomic tool used the analysis tissue beginning with conventional method such western blot analysis and polyacrylamide gel electrophoresis the most current highthroughput technique being used today are also reviewed these include protein array reversephase protein lysate array matrixassisted laser desorptionionization surfaceenhanced laser desorptionionization and layered expression scanning addition bioinformatics well issue regarding tissue preservation and microdissection obtain pure cell population are included finally future direction the tissue proteomics field are discussed',\n",
       " 'genomic and transcriptomic study generate rich source molecular information however neither static genome nor the presence messenger rna provides the most direct insight into the functional state microbial cell any given time rather the ensemble protein proteome that the primary workhorse for most biological process that are concurrent and coordinately active living cell currently massspectrometrybased technology provide unprecedented information into the composition the proteome shedding light the numerous complex biological process actively shaping observed phenotype herein detail protocol for the exploration intracellular proteome through the largescale unbiased identification protein and their relative abundance using liquid chromatography coupled tandem mass spectrometry msms general this protocol applicable both microbial and eukaryotic system',\n",
       " 'hplc ha emerged valuable tool for separating protein address the analysis complex proteome and quantitative change protein therein developed multidimensional mdlcbased approach followed large gel sdspage here present novel strategy that allows for simultaneously identifying and quantifying differentially regulated protein following three separation and fractionation step this mdlc platform integrates advantage dual protein labelling using both fluorescence and isotopecoded tag for subsequent detection and quantification abundance ratio protein',\n",
       " 'the generation proteomic data becoming ever more high throughput both the technology and experimental design used generate and analyze data are becoming increasingly complex the need for method which such data can accurately described stored and exchanged between experimenter and data repository ha been recognized work the proteome standard initiative the human proteome organization ha laid the foundation for the development standard which experimental design can described and data exchange facilitated the minimum information about proteomic experiment data model describes both the scope and purpose proteomics experiment and encompasses the development more specific interchange format such the mzdata model mass spectrometry the extensible markup languagemi data interchange format which allows exchange molecular interaction data ha already been published and major database within this field are supplying data downloads this format',\n",
       " 'the quest decipher protein alteration cancer ha spanned well over half century the vast dynamic range protein abundance coupled with plethora isoforms and disease heterogeneity have been formidable challenge progress cancer proteomics ha substantially paralleled technological development advance analytical technique and the implementation strategy decomplex the proteome into manageable component have allowed protein across wide dynamic range explored the massive amount data that can currently collected through proteomics allow the nearcomplete definition cancer subproteomes which reveals the alteration signalling and developmental pathway this allows the discovery predictive biomarkers and the annotation the cancer genome based proteomic finding there remains considerable need for infrastructure development and the organized collaborative effort efficiently mine the cancer proteome',\n",
       " 'extracellular vesicle ev are lipid bilayer enclosed particle which present almost all type biofluids and contain specific protein lipid and rna increasing evidence ha demonstrated the tremendous clinical potential ev diagnostic and therapeutic tool especially biofluids since they can detected without invasive surgery with the advanced mass spectrometry possible decipher the protein content ev under different physiological and pathological condition therefore msbased proteomic study have grown rapidly the past decade for biomarker discovery this review focus the study that isolate ev from different biofluids and contain msbased proteomic analysis literature published the past decade were selected and summarized with emphasis isolation method ev and analysis strategy with the aim give overview msbased proteomic study and provide reference for future research',\n",
       " 'cell proteome are complex given they consist several thousand protein twodimensional electrophoresis 2de unique not only for it ability simultaneously separate thousand protein but also for detecting post and cotranslational modification which can not predicted from genome sequence this review will describe the protocol applied prepare gel properly and analyse and summarise the major challenge for successful proteome analysis using 2de the ability analyse very alkaline hydrophobic andor low high protein with high resolution and the ability detect minor component challenge involving sample preparation and solubilisation prior the first dimension iefipg step will studied depth sample preparation crucial 2de study and greatly influence other stage the technique the aim this review not only describe the challenge and limitation 2de but also suggest the avenue the evolution the potential and the future 2de proteomics',\n",
       " 'genome sequencing and system biology revolutionized life science proteomics emerged fundamental technique this novel research area this review aim summarize the contribution arabidopsis thaliana model organism for plant and the increasing impact proteome research technique for proteomics based 2de and especially gelfree shotgun lcmsms platform have improved significantly during the last decade proteomics ha proven complementary other omics technique such transcriptomics and metabolomics arabidopsis thaliana one the first model organism worldwide served several the most comprehensive study for enhance genome annotation profiling organelle tissue cell subcellular proteome well developmental process and response biotic and abiotic stress using differential relative and absolute quantitative strategy consequently insight into plant proteome dynamic and cell function are rapidly increasing proteomicstoolbox developed for system biology research arabidopsis will introduced',\n",
       " 'the hostpathogen interaction represents complex and dynamic biological system the outcome this interaction dependent the microbial pathogen property establish infection and the ability the host control infection although bacterial pathogen have evolved variety strategy subvert host defense function several general mechanism have been shown shared among these pathogen result host effector that are critical for pathogen entry survival and replication inside the host cell have become new paradigm for antimicrobial targeting this review focus the potential utility proteomics approach defining the hostpathogen interaction from the host perspective',\n",
       " 'blood serum sample are the major source for clinical proteomics approach which aim identify diagnostically relevant treatmentresponse related protein but the presence very highabundance protein and the enormous dynamic range protein distribution hinders whole serum analysis innovative tool overcome these limitation utilizes combinatorial hexapeptide ligand library proteominer here demonstrate that proteominer can used for comparative and quantitative analysis complex proteome spiked serum sample with increasing amount microg microg whole coli lysate processed with proteominer and performed quantitative analysis 2dgels found that the concentration the spiked bacteria proteome reflected the maintained proportional spot intensity wa not altered proteominer treatment therefore conclude that the proteominer technology can used for quantitative analysis low abundant protein complex biological sample',\n",
       " 'posttranslational modification ptms protein represent fascinating extension the dynamic complexity living cell proteome the result enzymatically catalyzed spontaneous chemical reaction ptms form fourth tier the gene transcript protein cascade and contribute not only protein biological function but also challenge their analysis there have been tremendous advance proteomics during the last decade identification and mapping ptms protein have improved dramatically mainly due constant increase the sensitivity speed accuracy and resolution mass spectrometry however also becoming increasingly evident that simple gelfree shotgun profiling unlikely suffice for comprehensive detection and characterization protein andor protein modification present low amount here review current approach for enriching and separating posttranslationally modified protein and their msindependent detection first discus general approach for proteome separation fractionation and enrichment then consider the commonest form ptms phosphorylation glycosylation and glycation lipidation methylation acetylation deamidation ubiquitination and various redox modification and the best available method for detecting and purifying protein carrying these ptms this article part special issue entitled posttranslational protein modification biology and medicine',\n",
       " 'recent year knowledge about immunerelated disorder ha substantially increased especially the field central nervous system cns disorder recent innovation proteinrelated microarray technology have enabled the analysis interaction between numerous sample and target antibody directed against ion channel receptor and other synaptic protein have been identified and their causative role different disorder have been identified knowledge about immunological disorder likely expand further more antibody target are discovered therefore protein microarrays may become established tool for routine diagnostic procedure the future the identification relevant target protein requires the development new strategy handle and process vast quantity data that these data can evaluated and correlated with relevant clinical issue such disease progression clinical manifestation and prognostic factor this review will mainly focus new protein array technology which allow the processing large number sample and their various application with deeper insight into their potential use diagnostic tool neurodegenerative disease and other disease this article part special issue entitled biomarkers proteomic challenge',\n",
       " 'newcomer the omics era proteomics broad instrumentintensive research area that ha advanced rapidly since it inception le than year ago although the wetbench aspect proteomics have undergone renaissance with the improvement protein and peptide separation technique including various improvement twodimensional gel electrophoresis and gelfree offgel protein focusing ha been the seminal advance that have led the ascension this field recent improvement sensitivity mass accuracy and fragmentation have led achievement previously only dreamed including wholeproteome identification and quantification and extensive mapping specific ptms posttranslational modification with such capability present one might conclude that proteomics ha already reached it zenith however capability indicates that the envisioned goal have not yet been achieved the present review focus what perceive the area requiring more attention achieve the improvement workflow and instrumentation that will bridge the gap between capability and achievement for least most proteome and ptms additionally essential that extend our ability understand protein structure interaction and localization towards these end briefly focus selected method and research area where anticipate the next wave proteomic advance',\n",
       " '',\n",
       " 'proteome the ensemble all protein expressed cell tissue are typically analysed mass spectrometry recent technical and computational advance have greatly increased the fraction proteome that can identified and quantified single study current mass spectrometrybased proteomic strategy have the potential reproducibly accurately quantitatively and comprehensively measure any protein whole proteome from cell and tissue different state achieving these goal will require complete proteome map and analytical strategy that use these map prior information and will greatly enhance the impact proteomics biological and clinical research',\n",
       " 'current approach within affinitybased proteomics are driven both the accessibility and availability antigen and capture reagent and suitable multiplexed technology onto which these are implemented combining planar microarrays and other multiparallel system with set reagent possibility discover new and unpredicted proteindisease association either via directed hypothesisdriven via undirected hypothesisgenerating target selection can created the following stage the discovery made during these screening phase have verified for potential clinical relevance based both technical and biological aspect the use affinity tool throughout discovery and verification ha the potential streamline the introduction new marker transition into clinically required assay format appears straightforward this article summarize some the current building block within array and affinitybased proteomic profiling with focus body fluid giving perspective how current and upcoming development this bioscience could enable path pursuit for biomarker discovery',\n",
       " 'posttranslational modification ptms are vital cellular control mechanism which affect protein property including folding conformation activity and consequently their function result they play key role various disease condition including cancer and diabetes proteomics rapidly growing field ha witnessed tremendous advancement during the last decade which ha led the generation prodigious quantity data for various organism proteome ptms being biologically and chemically dynamic process pose greater challenge for it study amidst these complexity connecting the modification with physiological and cellular cascade event are still very challenging advancement proteomic technology such mass spectrometry and microarray provides platform study ptms and help decipher role some the very essential biological phenomenon enhance our understanding various ptms different organism and simplify the analysis complex ptm data many database software and tool have been developed these ptm database and tool contain crucial information and provide valuable resource the research community this article intends provide comprehensive overview various ptm database software tool and analyze critical information available from these resource study ptms various biological organism',\n",
       " 'helminth can interact with their host many different way including through the secretion soluble molecule such lipid glycans and protein and extracellular vesicle ev the field helminth secreted ev ha significantly advanced recent year mainly due the molecular characterisation proteome and research highlighting the potential ev and their constituent molecule the diagnosis and control parasitic infection despite these advancement the lack appropriate isolation and purification method impeding the discovery suitable biomarkers for the differentiation helminth population the present review offer our viewpoint the different proteomic technique and approach that have been developed well solution common pitfall and challenge that could applied advance the study helminth ev',\n",
       " 'leishmaniasis infectious disease caused leishmania that widespread country the differentiation leishmania from procyclic metacyclic promastigote ha occurred along with morphological and biochemical change proteome scale aim here identify the proteome two successive developmental form procyclic and metacyclic promastigotes from leishmania major isolates using swathms quantitative proteomics technique isolated protein from procyclic and metacyclic lysate were digested fractionated and subjected swathms protein significantly different abundance were analyzed using gene ontology and proteinprotein interaction network ppin our study showed that protein were changed abundance between the two consecutive developmental stage differentially expressed protein were classified into nine class analysis significant modulation translation antioxidant and stressrelated defense energy metabolism structural and motilityrelated protein were detected between procyclic and metacyclic stage found that elongation factor2 and various structural constituent ribosome were downregulated during metacyclogenesis while motility related protein including adpribosylation factor3 paraflegellar rod protein2c and tubulin alphachain were regulated according network analysis enol ha been introduced main hubbottleneck protein and ef1b hsp and gdh have been determined seed protein our result show that significant protein abundance are crucial feature metacyclogenesis major the protein function analysis illustrated that synthetic pathway involved protein were downregulated metacyclic which the main feature this stage parasite growth cycle while upregulation motility and energy metabolism related protein consistent with infective feature metacyclic stage based our result suppose that differentially expressed protein possibly play critical role major differentiation addition our finding demonstrated the possibility swathms viable technique faster detect new stagespecific protein leishmania and further study are required for the validation the result',\n",
       " 'proteomic analysis are critically important for system biology because important aspect related the structure function and control biological system are only amenable direct protein measurement ha become apparent that the current proteomics technology are unlikely allow routine quantitative measurement whole proteome have therefore suggested and largely implemented twostep strategy for quantitative proteome analysis first step the discovery phase the proteome observable mass spectrometry extensively analyzed the resulting proteome catalog can then used select peptide specific only one protein socalled proteotypic peptide ptps represents the basis realize sensitive robust and reproducible measurement based targeted mass spectrometry these ptps subsequent scoring phase this extra view describe the need for such proteome catalog and their multiple benefit for catalyzing the shift towards targeted quantitative proteomic analysis and beyond use the insulin signaling cascade representative example illustrate the limitation currently used proteomics approach for the specific analysis individual pathway component and describe how the recently published drosophila proteome catalog already helped overcome many these limitation',\n",
       " 'colorectal cancer crc one the most prevalent malignancy worldwide biomarkers that can facilitate better clinical management crc are high demand improve patient outcome and reduce mortality this regard proteomic analysis hold promising prospect the hunt novel biomarkers for crc and understanding the mechanism underlying tumorigenesis this review aim provide overview the current progress proteomic research focusing discovery and validation diagnostic biomarkers for crc will summarize the contribution proteomic strategy recent discovery protein biomarkers for crc and also briefly discus the potential and challenge different proteomic approach biomarker discovery and translational application',\n",
       " 'after the first introduction the concept proteome more than year ago largescale study protein expression localization activity and interaction have gained exponential increase interest leading extensive research and technology development proteomics expansively applied many area ranging from basic research various disease and malignant tumor diagnostic and biomarker discovery therapeutic application several proteomics approach have been developed for protein separation and identification and for the characterization protein function and structure twodimensional gel electrophoresis chromatography capillary electrophoresis and mass spectrometry have become the most used proteomics method these technique are also under constant development this review provides overview the main technique and their combination used proteomics the emphasis made description advantage and disadvantage each technique navigate selection the best application for solving specific problem',\n",
       " 'advance proteomic technology have led the creation largescale proteome database that can used elucidate invaluable information the dynamic the metabolic signaling and regulatory network and aid understanding physiological change particular proteomics can have practical application for example through the identification protein that may potential target for the biotechnology industry and through the extension our understanding the physiological action these protein this review describe proteomic approach for the discovery target that have potential biotechnological application these target include promoter chaperone soluble fusion partner anchoring motif and excretion fusion partner addition discus the potential application proteomic technique for the design future bioprocesses and the optimization existing one successful application proteomic information have proven have enormous value for both scientific and practical application',\n",
       " 'proteomics ha it origin twodimensional gel electrophoresis 2de technique developed more than twenty year ago 2de ha highresolution capacity and wa initially used primarily for separating and characterizing protein complex mixture 2de remains important tool for protein identification but now normally coupled with mass spectrometry technique which ha advanced considerably recent year the recent completion human genome project ha produced large dna database which can utilized through bioinformatics and the next challenge for scientist uncover the entire proteome particular organism the integration genomic and proteomic data will help elucidate the function protein the pathogenesis disease and the ageing process and could lead the discovery novel drug target protein and biomarkers disease this review describes recent advance proteomic technology and discus the potential application proteomics biomedical research',\n",
       " 'the integration small kinase inhibitor and monoclonal antibody into oncological practice ha opened new paradigm for treating cancer patient protein are the direct target the new generation targeted therapeutic many which are kinaseenzymatic inhibitor there increasing interest developing technology capable monitoring posttranslational change the human proteome for the identification new predictive prognostic and therapeutic biomarkers well known that the vast majority the activationdeactivation these drug target driven phosphorylation this review provides description the main proteomic platform planar and bead array reverse phase protein microarray phosphoflow aqua and mass spectrometry that have successfully been used for measuring change phosphorylation level drug target and downstream substrate using clinical specimen major emphasis wa given the strength and weakness the different platform and the major barrier that are associated with the analysis the phosphoproteome finally number example application the abovementioned technology the clinical setting are reported',\n",
       " 'proteomics wa initially viewed promising new scientific discipline study complex disorder such polygenic infectious and environmentrelated disease however the first attempt understand monogenic disease such cystic fibrosis proteomicsbased approach have proved quite rewarding the impairment unique protein the transmembrane conductance regulator doe not completely explain the complex and variable clinical phenotype the great advance our knowledge about the molecular and cellular consequence such impairment have not been sufficient translated into effective treatment and patient are still dying due chronic progressive lung dysfunction the progression proteomics application will certainly unravel new protein that could useful biomarkers either elucidate basic mechanism and better monitor the disease progression promote the development novel therapeutic strategy against this review will summarize the recent technological advance proteomics and the first result it application address the most important issue the field',\n",
       " 'the foundation proteomics are study gene product and their regulatory role within cell paradoxically the only evidence that sperm cell make new protein through mitochondrial protein synthesis yet despite this spermatozoon are the perfect candidate for mass spectrometry and hence proteomic analysis these enterprising cell use plethora posttranslational modification order gain functionality following their production within the testis using combination twodimensional polyacrylamide gel electrophoresis 2dpage and more recently liquid chromatographymass spectrometry lcmsms recent advance sperm cell biology through the use proteomics making unparalleled progress the protein inventory list being generated have shed light transmembrane protein kinase and chaperone never previously recognized addition the ability isolate either phosphopeptides glycopeptides and quantify the difference between cell two different population make proteomic analysis spermatozoon real chance finally answer some age old question',\n",
       " '',\n",
       " '',\n",
       " 'human saliva harbour protein clinical relevance and about blood protein are also present saliva this highlight that saliva can used for clinical application just urine blood however the translation salivary biomarker discovery into clinical setting hampered the dynamic and complexity the salivary proteome this review focus the current status technological development and achievement relating approach for unravelling the human salivary proteome discus the dynamic the salivary proteome well the importance sample preparation and processing technique and their influence downstream protein application posttranslational modification salivary proteome and protein protein interaction addition describe possible enrichment strategy for discerning posttranslational modification salivary protein the potential utility selectedreactionmonitoring technique for biomarker discovery and validation limitation proteomics and the biomarker challenge and future perspective summary provide recommendation for practical saliva sampling processing and storage condition increase the quality future study emerging field saliva clinical proteomics propose that the advent technology allowing sensitive and high throughput proteomewide analysis coupled wellcontrolled study design will allow saliva enter clinical practice alternative bloodbased method due it simplistic nature sampling noninvasiveness easy collection and multiple collection untrained professional and costeffective advantage',\n",
       " 'the emergence advanced liquid chromatography mass spectrometry technology for characterizing very complex mixture protein ha greatly propelled the field proteomics the goal which the simultaneous examination all the protein expressed organism this research area represents paradigm shift molecular biology attempting provide topdown qualitative and quantitative view all the protein including their modification and interaction that are essential for organism life cycle rather than targeting particular protein family this level global protein information about organism such bacterium can combined with genomic and metabolomic data enable system biology approach for understanding how these organism live and function',\n",
       " 'the 7th central and eastern european proteomic conference ceepc considered the bedrock proteomics central and eastern europe wa held october the max planck institute for chemical ecology jena germany established ceepc now represents cradle proteomic interaction and around central and eastern europe without limitation border and linking international institution worldwide it mission remains contribute all approach proteomics including clinical quantitative and structural proteomics and with view identifying potential target for therapeutic intervention the 7th ceepc excelled stimulating exchange proteomic knowledge and imbibing local hospitality offered jena with it limestone hill and exotic charm',\n",
       " 'goal system biology develop integrated picture how the myriad component biological system work together produce response environmental input achieving this goal requires assembling list the component part cellular regulatory system and understanding how the connection between these component enable information processing work toward these end number method have matured parallel the compilation cellular part list ha been accelerated the advent omics technology which enable simultaneous characterization large collection biomolecules particular type omics technology that useful for understanding proteinprotein interaction network proteomics which can give information about number dimension the state the cell protein quantification protein abundance within the cell characterization the posttranslational modification state the proteome through phosphopeptide enrichment and identification proteinprotein interaction through coimmunoprecipitation mathematical model can useful analyzing proteomic data',\n",
       " 'the proteome most likely contain clinically useful disease biomarkers are those human body fluid three recent largescale proteomic analysis tear urine and seminal plasma using the latest mass spectrometric technology will provide useful datasets for biomarker discovery',\n",
       " 'with the completion the human genome project hugo during recent year gene function protein abundance and expression pattern tissue and cell type have emerged central area for the scientific community mapped human proteome will extend the value the genome sequence and largescale effort aiming elucidating protein localization abundance and function are invaluable for biomarker and drug discovery this research area termed proteomics more demanding than any genome sequencing effort and perform this wide scale highly diverse task therefore the proteomics field employ range method examine different aspect proteomics including protein localization proteinprotein interaction posttranslational modification and alteration protein composition differential expression tissue and body fluid here some the most commonly used method including chromatographic separation together with mass spectrometry and number affinity proteomics concept are discussed and exemplified',\n",
       " 'this chapter describes the technology free flow electrophoresis ffe and protocol separate human plasma for proteome analysis ffe highly versatile technology applied the field proteomics because it continuous processing sample and high resolution separation most kind charged chargeable particle including ion protein peptide organelle and whole cell ffe carried out aqueous medium without inducing any solid matrix such acrylamide that simplifies complex sample for the downstream analysis two ffe protocol are described separate human plasma protein under native and denaturing condition plasma separated under native condition wa pooled into acidic alkaline and albumin fraction that were furthered for gelbased analysis under denaturing condition plasma protein were separated into fraction each fraction can supplied for insolution digestion and further lcmsms analysis from single ffe fraction different protein protein family have been identified demonstrating ffe high efficient separation tool for human plasma proteome study',\n",
       " 'proteome are intricate typically thousand protein interact through physical association and posttranslational modification ptms give rise the emergent function cell understanding these function requires one study proteome system rather than collection individual protein molecule the abstraction the interacting proteome protein network ha recently gained much attention network are effective representation that lose specific molecular detail but provide the ability see the proteome whole mostly two aspect the proteome have been represented network model proteomewide physical proteinproteinbinding interaction organized into protein interaction network pin and proteomewide ptm relation organized into protein signaling network psns mass spectrometry technique have been shown essential reveal both these aspect proteomewide scale technique such affinity purification followed have been used elucidate proteinprotein interaction and msbased quantitative phosphoproteomics critical understand the structure and dynamic signaling through the proteome here review the current stateoftheart msbased analytical pipeline for the purpose characterize proteomescale network',\n",
       " 'the component that enable cell and organism fulfill plethora chemical and physical reaction including their ability metabolize replicate repair and communicate with their environment are mostly based the functioning highly complex cellular machine which are large extent composed protein with the development technique compatible with the analysis minute amount biological material ha become more and more feasible dissect the composition and modification these protein machinery indeed new purification method protein complex followed analysis together with the genomic sequencing various organism and particular crop specie now provide unforeseen insight understand biological process molecular level here review the current state the art vivo protein complex isolation and their msbased analytical characterization emphasizing the tandem affinity purification approach',\n",
       " 'spider venom are important source novel molecule with different pharmacological property recent technological development proteomics especially mass spectrometry have greatly promoted the systematic analysis spider venom the enormous diversity venom component between spider specie and the lack complete genome sequence and the limited database protein and peptide sequence make spider venom profiling challenging task and special consideration for technical strategy are required this review highlight recently used method for spider venom profiling general spider venom profiling can achieved two part proteome profiling the component with molecular weight above kda and peptidome profiling the component with molecular weight kda under through the use different method venom proteome are rich various enzyme hemocyanins toxinlike protein and many unknown protein peptidomes are dominated peptide with mass kda with three five disulfide bond although there are some similarity peptide superfamily type venom from different spider specie the venom profile each specie unique the linkage the peptidomic data with that the cdna approach discussed briefly future challenge and perspective are also highlighted this review',\n",
       " 'over the last decade microfabricated bioanalytical platform have gained enormous interest due their potential revolutionize biological analytics their popularity based several key property such high flexibility design low sample consumption rapid analysis time and minimization manual handling step which are interest for proteomics analysis ideal totally integrated chipbased microfluidic device could allow rapid automated workflow starting from cell cultivation and ending with msbased proteome analysis reducing eliminating sample handling and transfer step and increasing the throughput analysis these workflow would dramatically improve the reliability reproducibility and throughput proteomic investigation while these complete device not exist for routine use yet many improvement have been made the translation proteomic sample handling and separation step into microfluidic format this review will focus recent development and strategy enable and integrate proteomic workflow into microfluidic device',\n",
       " 'two the primary response cell when perturbed are modulation the dynamic it constituent gene expression and protein abundance restore steadystate homeostasis obtain detailed model the restoration this balance necessary examine the kinetics transcription and translation thus going beyond establishing mere abundance level transcript and protein this review discus proteomic approach that utilize genomic tagging and metabolic labeling reveal turnover kinetics for cellular protein highthroughput manner novel metabolic and multiplexed labeling technique coupled mass spectrometry combination with nextgeneration sequencing approach provide tool for studying the principle cellular adaptation and dynamic unprecedented detail',\n",
       " 'the definition proteome cell and animal particular stage facilitates understanding protein function this issue cell chemical biology elliott report elegant approach bioorthogonal labeling and enrichment proteome from stochastic orthogonal recoding translation with this method low abundance proteome can identified multicellular system',\n",
       " 'proteinprotein interaction ppis drive all biologic system the subcellular and extracellular level change the specificity and affinity these interaction can lead cellular malfunction and disease consequently the binding interface between interacting protein partner are important drug target for the next generation therapy that block such interaction unfortunately proteinprotein contact point have proven very difficult pharmacological target because they are hidden within complex interface for the vast majority characterized binary ppis the specific amino acid sequence their close contact region remains unknown there ha been important need for experimental technology that can rapidly reveal the functionally important contact point native protein complex solution this review experimental technique employing mass spectrometry explore protein interaction binding site are discussed hydrogendeuterium exchange hydroxyl radical footprinting crosslinking and the newest technology protein painting are compared and contrasted',\n",
       " 'the success clinical proteome analysis should assessed based the clinical impact following implementation finding although there have been several technological advancement mass spectrometry the last year these have not resulted similar advancement clinical proteomics addition application proteomic biomarkers clinical diagnostics and practical improvement the disease management extremely rare this review discus the relevant issue associated with identification robust biomarkers clinical value urine appears ideal source biomarkers for theoretical methodological and practical reason therefore this review focused the search for biomarkers urine within the last decade urine can used for noninvasive assessment variety disease including those affecting the urogenital tract and also other pathology such cardiovascular disease appendicitis also discus the importance data validation essential step translating biomarkers into the clinical practice furthermore examine several example apparently successful proteomic biomarker discovery study and their implication for disease diagnosis prognosis and therapy evaluation also discus some current challenge this field and reflect future research prospect this article part special issue entitled biomarkers proteomic challenge',\n",
       " 'mitochondrion are cellular site diverse redox biology including ro production ironsulfur biogenesis and secondary metabolism which all rely proteogenic reactive cysteine residue mass spectrometrybased proteomic method monitor the reactivity and functionality cysteine residue across complex proteome have greatly expanded over the past decade here describe mitochondrial isolation procedure coupled with cysteinereactive labeling that affords identification and characterization functional mitochondrial cysteine residue that were heretofore inaccessible wholecell proteomic analysis',\n",
       " 'isobaric tandem mass tag tmt labeling widely used proteomics because it high multiplexing capacity and deep proteome coverage recently expanded plex tmt method ha been introduced which further increase the throughput proteomic study this manuscript present optimized protocol for plex tmtbased deepproteome profiling including protein sample preparation enzymatic digestion tmt labeling reaction twodimensional reversephase liquid chromatography lclc fractionation tandem mass spectrometry msms and computational data processing the crucial quality control step and improvement the process specific for the plex tmt analysis are highlighted this multiplexed process offer powerful tool for profiling variety complex sample such cell tissue and clinical specimen more than protein and posttranslational modification such phosphorylation methylation acetylation and ubiquitination highly complex biological sample from different sample can quantified single experiment providing potent tool for basic and clinical research',\n",
       " 'proteomics analogous with genomics the analysis the protein complement present cell organ organism any given time while the genome provides information about the theoretical status the cellular protein the proteome describes the actual content which ultimately determines the phenotype the broad application proteomic technology basic science and clinical medicine ha the potential accelerate our understanding the molecular mechanism underlying disease and may facilitate the discovery new drug target and diagnostic disease marker proteomics rapidly developing and changing scientific discipline and the last have seen major advance the underlying technique well expansion into new application core technology for the separation protein andor peptide are one and twodimensional gel electrophoresis and one and twodimensional liquid chromatography and these are coupled almost exclusively with mass spectrometry proteomic study have shown that the most effective analysis even simple biological sample requires subfractionation andor enrichment before protein identification mass spectrometry selection the appropriate technology combination technology match the biological question essential for maximum coverage the selected subproteome and ensure both the full interpretation and the downstream utility the data this review describe the current technology for proteome fractionation and separation biological sample based our lab workflow for biomarker discovery and validation',\n",
       " 'biomarker discovery and proteomics have become synonymous with mass spectrometry recent year although this conflation injustice the many essential biomolecular technique widely used biomarkerdiscovery platform underscore the power and potential contemporary mass spectrometry numerous novel and powerful technology have been developed around mass spectrometry proteomics and biomarker discovery over the past year globally study complex proteome plasma however very few largescale longitudinal study have been carried out using these platform establish the analytical variability relative true biological variability the purpose this review not cover exhaustively the application mass spectrometry biomarker discovery but rather discus the analytical method and strategy that have been developed for mass spectrometrybased biomarkerdiscovery platform and place them the context the many challenge and opportunity yet addressed',\n",
       " 'the human proteome organisation proteomics standard initiative hupopsi wa established with the aim defining community standard for data representation proteomics and facilitating data comparison exchange and verification over the last year significant advance have been made with common data standard now published and implemented the field both mass spectrometry and molecular interaction the meeting further advanced this work with the mass spectrometry group finalising approach capturing the output from recent development the field such quantitative proteomics and srm the molecular interaction group focused improving the integration data from multiple resource both group united with guest work track organized the hupo technologystandards committee formulate proposal for data submission from the hupo human proteome project and start initiative collect standard experimental protocol',\n",
       " 'many disease autoimmune disease cancer humoral immune response thought involved therefore identifying relevant antigen that trigger the humoral immune response the heart current approach for discovering biomarkers and designing therapeutic strategy although immunoblotting and protein array have been used for antigen profiling several disease with the rapid progress mass spectrometry and separation technology msbased proteomic method have become promising tool for identifying diseaseassociated antigen this review introduces msbased antigen profiling method immunoproteomics and immune complexome analysis and present application example these technique the analysis biological sample',\n",
       " 'adaptation the environment during development influence the lifelong survival animal while brainwide proteomic change are expected underlie such experiencedriven physiological and behavioral flexibility comprehensive overview the nature and extent the proteomic regulation following environmental challenge during development currently lacking this study the brain proteome larval zebrafish identified and determined how altered exposure natural and physical environmental challenge namely prolonged exposure strong water current comprehensive larval zebrafish brain proteome presented here furthermore protein that are regulated the exposure environmental challenge are identified which cover multiple function including neuronal plasticity the stress response axonal growth and guidance spatial learning and energy metabolism these represent candidate protein that may play crucial role for the adaption environmental challenge during development',\n",
       " 'since the formulation the concept proteomics plethora proteomic technology have been developed order study proteome tissue cell and organelle the powerful new technology enabled proteomic approach have lead the application these method exponentially increasing variety biological question for highly complex protein mixture continuous technical optimization allows for everincreasing sensitivity proteomic technique this review brief overview currently available proteomic technique and their application given followed more detailed description advantage and technical challenge twodimensional electrophoresis 2de some solution circumvent currently encountered technical difficulty for 2de analysis are proposed',\n",
       " 'with the advent highresolution mass spectrometry together with sophisticated data analysis and interpretation algorithm determination protein synthesis and degradation rate protein turnover proteomewide scale employing stable isotopelabelled amino acid ha become feasible these dynamic data provide deeper understanding protein homeostasis and stress response mechanism microorganism than wellestablished steady state proteomics approach this article summarize the technological challenge and solution both the biochemistrymass spectrometry and bioinformatics level for turnover proteomics with focus chromatographic technique although the number available case study for corynebacterium glutamicum and related actinobacteria still very limited our review illustrates the potential protein turnover study for improved understanding question the area biotechnology and biomedicine here new insight from investigation growth phase transition and different stress dynamic including iron acid and heat stress for pathogenic but also for industrial actinobacteria are presented finally will comment the advantage integrated software solution for biologist and briefly discus the remaining technical challenge and upcoming possibility for protein turnover analysis',\n",
       " '',\n",
       " 'stateoftheart proteomics workflow involve multiple interdependent step sample preparation proteinpeptide separation mass spectrometry and data analysis while improvement any these step can increase the depth and breadth analysis advance mass spectrometry have catalysed many the most important development discus common class mass analyser and how these analyser are put together produce some the most popular mass spectrometry platform the capability these platform determine how they can used variety common proteomic strategy and turn what type biological question can addressed moving forward powerful new hybrid mass spectrometer and application emerging type tandem mass spectrometry promise that our ability analyse complex mixture protein will continue advance',\n",
       " 'the use proteomic library designed express the complete set protein from organism ha resulted the identification many rna modification enzyme whose function wa previously unknown here describe generalized procedure for the biochemical analysis yeast proteomic library for identification nucleic acidmodifying enzyme use the yeast morf moveable open reading frame library gelperin the source protein activity and the known yeast trna methyltransferase trm4 test case the procedure outlined this chapter can applied any proteomic expression library from any organism many which will become increasingly available the number sequenced genome increase and genomic cloning technique improve',\n",
       " '',\n",
       " 'proteome research proteolysis important procedure prior the mass spectrometric identification protein the typical time conventional insolution proteolysis long several hour half day enhance proteolysis efficiency variety microchip bioreactors have been developed for the rapid digestion and identification protein the past decade this review mainly focus the recent advance and the key strategy microchip bioreactors protein digestion the subject covered include microchip proteolysis system the immobilization protease microchannels the application microchip bioreactors highly efficient proteolysis and future prospect expected that microchip bioreactors will become powerful tool protein analysis and will find wide range application highthroughput protein identification',\n",
       " 'recent advance mass spectrometry msbased quantitative proteomics now allow the identification and quantification deep proteome and posttranslational modification ptms relatively short time therefore the last few year this technology ha proven successful the circadian field characterize temporal oscillation the proteome and more recently ptms cellular system and tissue this chapter describe robust and simple protocol based the easyphos workflow enable preparation large number proteome and phosphoproteomes from mouse tissue for msbased quantitative analysis additionally discus computational method analyze proteome and phosphoproteome time series determine circadian oscillation',\n",
       " 'the early detection distinct disease crucial for successful treatment and depends sensitive well specific diagnosis last year tremendous attempt were undertaken identify new biomarker applying proteomics but relevant candidate ha been identified for clinical application although proteomics enabling quantitative and qualitative analysis protein within complex mixture could not significantly contribute this field therefore different proteomics approach have been established focusing the direct analysis cell population involved pathogenic process identify candidate biomarkers even for vitro diagnosis here will outline approach applying cell and tissue based proteome analysis the first decisive step the pipeline for the discovery new diagnostic biomarkers will show example for analysing precursor lesion the pancreatic ductal adenocarcinoma pdac nephron glomerulus and fibrotic and nonfibrotic liver tissue this article provides also overview about currently available technique the field cell enrichment and quantitative proteome analysis lowest amount sample',\n",
       " 'mapping the human proteome ha advanced significantly recent year and will provide knowledge base accelerate our understanding how protein and protein network can affect human health and disease however providing solution human health challenge will likely fail insight are exclusively based study human sample and human proteome recent year ha become evident that human health depends integrated understanding the many specie that make human life possible these include the commensal microorganism that are essential human life pathogen and food specie well the classic model organism that enable study biological mechanism the human proteome organization hupo initiative multiorganism proteome imop work support proteome research undertaken nonhuman specie that remain widely understudied compared with the progress human proteome research this perspective argues the need for further research multiple specie that impact human life also present update recent progress model organism microbiota and food specie address the emerging problem antibiotic resistance and outline how imop activity could lead more inclusive approach for the human proteome project hpp better support proteome research aimed improving human health and furthering knowledge human biology',\n",
       " 'numerous gelfree proteomics technique have been reported over the past few year introducing move from protein peptide bit information qualitative and quantitative proteome study many shotgun proteomics technique randomly sample thousand peptide qualitative and quantitative manner but overlook the vast majority protein modification that are often crucial for proper protein structure and function peptidebased proteomic approach have thus been developed profile diverse set modification including but not all limited phosphorylation glycosylation and ubiquitination typical here that each modification need specific tailormade analytical procedure this minireview discus how one technique diagonal reversephase chromatography applied study two different type protein modification protein processing and protein nglycosylation additionally discus activitybased proteome study which purinebinding protein were profiled diagonal chromatography',\n",
       " 'shell matrix protein smps are occluded within molluscan shell and are fundamental the biological control over mineralization while many study have been performed adult smps those larval stage remain largely undescribed therefore this study aimed characterize the larval shell proteome the blue mussel for the first time and compare adult mussel shell proteome following development method for cleaning larval shell tissue contaminant smps were identified using shotgun proteomics twentyone protein were independently identified all sample indicating that they form subset the core larval shell proteome these included the blue mussel shell protein peroxidase domaincontaining sequence laminin domaincontaining sequence zip domaincontaining sequence and ferricchelate reductase 1like sequence additional smp domain identified were fibronectin type iii bptikunitz chitinbinding type thyroglobulin and efhand while key predictable molluscan shell matrix function are identified sequence remain unknown uncharacterized indicating that this shell proteome unique mussel larva specifically comparison with adult mytilid reveals that nine domain are exclusive the larval shell proteome and only four domain are conserved among specie and developmental stage thus strong speciesspecific and ontogenetic variation exists shell proteome composition',\n",
       " 'analysis blood sample from patient suffering from autoimmune disease remains mainstay the clinic for initial diagnosis prognostication and clinical decision making particular testing for the presence serum autoantibody ha proved one the most useful confirmatory assay for many different disease recent genomic and transcript profiling study have implicated certain cytokine surface receptor signaling pathway and cell type the pathogenesis inflammatory disease the next obvious step delve into the much more complex level that follows the genome and transcriptomethe expressed proteome this review focus several proteomics technology being applied andor developed our laboratory for the study autoimmunity cancer and cardiovascular disease all which are known associated with defect immunity and inflammation the finding other participant the recent human immunology conference hosted the dana foundation and the new york academy science may amp are included particular major pitfall the study the human proteome are pointed out and important area for immediate investigation move the field forward rapidly possible are proposed',\n",
       " 'protein and proteome are dynamic and complex the accurate identification and measurement their property such abundance location and turnover are challenging task even though highthroughput proteomics ha significantly evolved the technique still lack fully quantitative and reproducible quality mass spectrometrybased targeted proteomic strategy called selective reaction monitoring srm ha emerged recent year important multiplex platform precisely quantify set protein multiple sample this ha several advantage term sensitivity reproducibility and sample consumption compared classical method including those based antibody here present detailed protocol for quantitation panel protein from cell line extract using the srm targeted proteomics approach',\n",
       " 'cytochrome enzyme comprise family highly homologous protein these protein play pivotal role oxidative drug metabolism and are important target drug discovery research proteomics today valuable tool for the analysis protein the past qualitative analysis the proteome wa the main focus research but the last few year interest the mathematical modelling protein network ha been growing and ha the demand quantitative proteome analysis thorough understanding cytochrome dependent metabolism crucial for drug discovery thus not astounding that cytochrome enzyme are target for quantitative proteomics research this article review the technique available for quantitative proteome analysis and what extent these technique have been used for the quantification cytochrome enzyme and give brief outlook the technique that have promising potential for the analysis these protein the future',\n",
       " 'the present review cover modern aspect combinatorial peptide ligand library cpll used analyze the lowabundance proteome association with mass spectrometry first the capturing property bait different length from single amino acid hexapeptides are described show that plateau rapidly reached above tetrapeptide length thus confirming the validity having adopted hexapeptides for the considered application the mechanism interaction with protein from very complex proteome and the ability decrease the dynamic concentration range demonstrated with the help mass spectrometry analysis example are given how treatment with cplls dramatically improves the detectability peptide mass spectrometry analysis permitting detection very large number protein compared with control untreated sample the use complementary library discussed with the aim discover additional lowabundance specie that escaped the first library discussion the possibility discover extremely rare gene product and the quantitative aspect the technology when associated with mass spectrometry also provided some insight the application for hidden lowabundance biomarkers are also presented',\n",
       " 'proteomics ha been proposed one the key technology the postgenomic era far however the comprehensive analysis cellular proteome ha been challenge because the dynamic nature and complexity the multitude protein cell and tissue various approach have been established for the analysis protein cell given state and mass spectrometry ha proven efficient and versatile tool msbased proteomics approach have significantly improved beyond the initial identification protein comprehensive characterization and quantification proteome and their posttranslational modification ptms despite these advance there still ongoing development new technology profile and analyze cellular proteome more completely and efficiently this review focus msbased technique describe basic approach for msbased profiling cellular proteome and analysis method identify protein complex mixture and discus the different approach for quantitative proteome analysis finally briefly discus novel development for the analysis ptms altered level ptm sometimes the absence protein expression change are often linked cellular response and disease state and the comprehensive analysis cellular proteome would not complete without the identification and quantification the extent ptms protein',\n",
       " '',\n",
       " 'defining protein composition key step understanding the function both healthy and diseased biological system there currently little consensus between existing published proteome tissue such the aorta cartilage and organ such skin lack agreement both the number and identity protein may due issue protein extraction sensitivityspecificity detection and the use disparate tissuecell source here developed method combining bioinformatics and systematic review screen gtm article from the web science for evidence protein healthy human skin the resulting manchester proteome wwwmanchesterproteomemanchesteracuk collates existing evidence which characterises skin protein unique our database and evidenced both mass spectrometry and immunebased technique this approach circumvents the limitation individual proteomics study and can applied other specie organ cell diseasestates accurate tissue proteome will aid development engineered construct and offer insight into disease treatment highlighting difference proteomic composition',\n",
       " '',\n",
       " 'sample preparation and fractionation technology are one the most crucial process proteomic analysis and biomarker discovery solubilized sample chromatographic electrophoretic proteomic technology are also available for separation cellular protein component there are however considerable limitation currently available proteomic technology none them allows for the analysis the entire proteome simple step because the large number peptide and because the wide concentration dynamic range the proteome clinical blood sample the result any undertaken experiment depend the condition the starting material therefore proper experimental design and pertinent sample preparation essential obtain meaningful result particularly comparative clinical proteomics which one looking for minor difference between experimental diseased and control nondiseased sample this review discus problem associated with general and specialized strategy sample preparation and fractionation dealing with sample that are solution suspension frozen tissue state formalinpreserved tissue archival sample and illustrates how sample processing might influence detection with mass spectrometric technique strategy that dramatically improve the potential for cancer biomarker discovery minimally invasive bloodcollected human sample are also presented',\n",
       " 'the definition safety and efficacy foodemployed bacteria well probiotic strain continuous often unattended challenge proteomic technique such 2de dige and lclcmsms are suitable and powerful tool reveal new aspect positive and negative known and unknown strain that can employed food making and nutraceutical supplement for human health unfortunately these technique are not used extensively should wise the present report describes the most significant result obtained our research group year study subproteomes bacteria chiefly lactic acid bacteria production desired and undesired metabolite difference between strain belonging same specie but isolated from different ecological niche the effect cryoprotectants survival lyophilization well the adhesive capability strain were elucidated analysis cytosolic membraneenriched surface and extracellular proteome the present review open window yet largely underexplored field and highlight the huge potential subproteome investigation for more rational choice microbial strain food starter probiotic and for production nutraceuticals these analysis will hopefully contribute manufacturing safer and healthier food and food supplement the near future this article part special issue entitled hupo',\n",
       " 'the purpose this review analyze investigation devoted characteristic protein variability and diversity their posttranslational modification healthy human the numerous research have demonstrated that proteomic profile ha considerable both intra and interindividual variability and quite often normal variability some protein can comparable change observed pathological process result obtained our research group have confirmed high intraindividual variability serum lowmolecular subproteome healthy volunteer certified special medial committee protein characterized high variability normal condition haptoglobin mgml lysozyme mgml creactive protein mgml should excluded from the list potential biomarkers the contrary protein and peptide characterized insignificant dispersion healthy population such albumincoefficient variation transferrincv c3c complementcv alpha1 acid glycoproteincv alpha2macroglobulincv transthyretin fragmentcv and betachain alpha2hsglycoproteincv can provide with important information about state health thus investigation plasticity proteomic profile healthy human will help correct reference interval used clinical proteomics',\n",
       " 'have developed trifunctional cleavable fluorescence turnon linker for chemoproteomic application this novel linker which became highly fluorescent only upon cleavage the azo bond wa successfully used for situ proteome profilingtarget identification and study newly synthesised proteome',\n",
       " 'the primary component human hair shaftkeratin and keratinassociated protein kaps together with their crosslinked networksare the underlying reason for it rigid structure therefore requisite overcome the obstacle hair insolubility and establish reliable protocol for the proteome analysis this accessible specimen the present study employed alkalinebased method for the efficient isolation hair protein and subsequently examined them using gelbased proteomics the introduction two proteomic protocol namely the conventional and modified protocol have resulted the detection more than protein spot the twodimensional gel electrophoresis 2de when compared the modified protocol deemed improve overall reproducibility whilst offering quick overview the total protein distribution hair the development this highperformance protocol hoped provide new approach for hair analysis which could possibly lead the discovery biomarkers for hair health and disease the future',\n",
       " '',\n",
       " 'the proteomic approach have considerably evolved over the past two decade this opened the door for larger scale and deeper exploration cellular physiology like for other living organism using the tool proteomics ha undoubtedly improved knowledge about the foodborne pathogen listeria monocytogenes among the different technology and approach permanently evolving the field proteomics the 2de analytical separation method choice resolve thousand protein simultaneously single gel allowing their quantification the study their posttranslational modification and the understanding their biological function this 2de remains perfectly complementary technique the new highthroughput technique such shotgun proteomics approach moreover order gain analysis depth and improve knowledge about the target action and the function protein relation their subcellular location necessary explore more specifically the different subcellular proteome thus the subproteomic analysis became essential and dramatically increased these last year particularly protein secreted into the extracellular milieu named exoproteome cell envelope protein cell wall and membrane protein which are involved the interaction with the surrounding environment here the extraction and separation monocytogenes subproteomes are described based cell fractionation and 2de technique this chapter give workflow obtain the exoproteome the intracellular proteome the cell wall and membrane proteome the grampositive bacterium monocytogenes the different step 2de technology composed first dimension based the separation protein according their charge equilibration step then second dimension based the separation protein according their mass and finally the staining protein the gel are detailed emerging technology extract the exoproteome the cell surface proteome after enzymatic shaving and analyze them shotgun method are also discussed briefly',\n",
       " 'rapid improvement instrumentation and data analysis make mass spectrometrybased proteomics the method choice for global characterization proteome and discovery proteinbased biomarkers the contrary tissue biopsy body fluidseg blood wound fluid urine and salivaare noninvasive and easy collect and process however they are very complex and present high dynamic range protein concentration rendering direct shotgun proteomics analysis inefficient for identification lowabundance protein these specimen sample prefractionation immunoaffinity depletion highly abundant protein and enrichment posttranslational modification ptm are common strategy for proteome simplification body fluid combinatorial peptide ligand library cpll relatively deplete highabundance protein binding equimolar amount protein specie the sample and provide elegant speciesindependent alternative immunoaffinitybased approach cleaving target protein protease catalyze irreversible ptm whereby uncontrolled proteolysis associated with many disease thus proteolytic event represent powerful indicator for disease progression and their specific identification body fluid hold great promise for establishment novel biomarkers quantitative nterminal enrichment strategy such terminal amine isotopic labeling substrate tail detect proteasegenerated neontermini with high specificity and increase coverage lowabundance protein inherent proteome simplification this chapter describe protocol that combine the cpll technology with itraqbased tail systematically characterize the basic nterminome body fluid proteome and it alteration disease condition that have successfully applied explore the wound fluid degradome multiple time point after skin injury',\n",
       " '',\n",
       " 'mass spectrometry and cryoelectron tomography together enable the determination the absolute and relative abundance protein and their localization laying the groundwork for comprehensive system analysis cell',\n",
       " 'the development microarraybased technology for characterising tumour both the genomic and proteomic level ha had significant impact the field oncology gene expression profiling various human tumour tissue ha led the identification expression pattern related disease outcome and drug resistance well the discovery new therapeutic target and insight into disease pathogenesis protein microarray technology such reversephase protein array provide the unique opportunity profile tissue and ass the activity signalling pathway within isolated cell population this technology can used identify patient likely benefit from specific treatment modality and also monitor therapeutic response sample obtained during and after treatment routine application genomic and proteomic microarray technology clinical practice will require significant effort standardise the technique control and reference standard and analytical tool used extensive independent validation using large statisticallypowered datasets will also necessary inclusion concomitant genomic and proteomicbased molecular profiling technique into clinical trial protocol will bring closer the reality patienttailored therapy',\n",
       " 'the early report and our previous work confirmed the existence the toxic proteinaceous component the body the tredecimguttatus newborn and adult spider for revealing the difference the protein expression profile and toxicity base the spider different developmental stage the spiderling and adult spider protein were comparatively analyzed using proteomic strategy total and protein were identified from the spiderlings and adult spider respectively with protein being identified from both them although some similarity between the spiderling and adult spider proteome exist there are obvious difference between the two proteome size complexity molecular weight distribution acidbase property and hydropathicity etc gene ontology analysis demonstrates that comparing based the percentage protein the spiderling and adult spider protein have generally similar distribution profile with respect the subcellular localization molecular function and biological process however there are still some difference between these two set protein some classification the three category for the adult spider latrotoxins together with other toxin and toxinlike protein etc constitute their toxicity basis whereas the toxicity the spiderlings depends mainly the synergistic action atypical latrotoxins and toxinlike protein most which are different from those the adult spider demonstrating that the spider different developmental stage have largely different toxicity mechanism',\n",
       " '',\n",
       " 'stem cell represent not only valuable research tool cell biology but they also bring promise for cellbased therapy the future enhance our knowledge the molecular basis stem cell selfrenewal and differentiation various experimental approach have been applied this review will discus recent advance the field proteomic research that have initiated the characterization stem cell proteome',\n",
       " 'mass spectrometry ha become essential for efficient and accurate quantification protein and proteome and thus key technology throughout all bioscience however validated method are still scarce for meat quality research application the objective this work wa develop and compare two targeted proteomic method namely selected reaction monitoring srm and sequential window acquisition all theoretical spectrum swath for the quantification bovine muscle protein that may indicator meat color both method require evaluation spectrum from proteotypic and quantotypic peptide and here report our evaluation which peptide and parameter are best suited for robust quantification these protein observed that the srm approach provides better reproducibility linearity and sensitivity than swath and therefore ideal for targeted quantification lowabundance protein while the swath approach provides more timeefficient method for targeted protein quantification highabundance protein and additionally support the search for novel biomarkers',\n",
       " 'rapid development genomic and proteomic methodology ha provided wealth data for deciphering the biomolecular circuitry living cell the main area computational research proteome outlined this review are understanding the system it feature and parameter help plan the experiment data integration help produce more reliable data set visualization and other form data representation simplify interpretation modeling the functional regulation and system biology with falsepositive rate reaching even the more reliable data set handling the experimental error remains one the most challenging task integrative approach incorporating result various genome and proteomewide experiment allow for minimizing the error and bring with them significant predictive power',\n",
       " 'the continued fast pace fungal genome sequence generation ha enabled proteomic analysis wide variety organism that span the breadth the kingdom fungi there some phylogenetic bias the current catalog fungi with reasonable dna sequence database genomic est that could analyzed global proteomic level however the rapid development next generation sequencing platform ha lowered the cost genome sequencing such that the near future having genome sequence will longer time cost bottleneck for downstream proteomic and transcriptomic analysis high throughput nongelbased proteomics offer snapshot protein present given sample single point time there are number variation the general method and technology for identifying peptide given sample present method that can serve baseline for proteomic study fungi',\n",
       " 'labelfree peptide quantification promising approach for the largescale characterization proteome dynamic low cost here describe method for absolute labelfree quantification using untargeted approach for peptide fragmentation referred mse show that spiked external standard provide sufficient accuracy for the quantification protein complex sample resulting similar protein quantification result spectral counting advantage labelfree quantification also work for small number sample whereas spectral counting requires large datasets result similar robustness the sensitivity protein identification increase significantly when ion mobility separation included addition the standard lcm setup the analysis workflow ion mobility decrease sample complexity and serf additional separation criterion align parent ion with it product ion after mse fragmentation drawback quantification high abundance protein becomes inaccurate because detector saturation describe here suitable workflow achieve good sensitivity for protein quantification and give initial guidance data interpretation achieve good identification and quantification accuracy the protein amount loaded onto the column should not exceed dynamic range window order magnitude robust quantification can obtained with complex sample comprising protein',\n",
       " 'proteomic technology have matured level enabling accurate and reproducible quantitation peptide and protein from complex biological matrix analysis sample diverse assembled protein complex whole cell lysates subcellular proteome from cell culture and direct analysis animal and human tissue and fluid demonstrate the incredible versatility the fundamental nature the technique that form the basis most proteomic application today mass spectrometry determining the mass biomolecules and their fragment related product with high accuracy can convey highly specific assay for detection and identification importantly ion current representative these specifically identified analytes can accurately quantified with the correct application smart isobaric tagging chemistry heavy and light isotopically derivatised sample standard careful application workflow compare unlabelled sample socalled labelfree and targeted selected reaction monitoring experiment term exploring biology myriad protein change and modification are being increasingly probed and quantified including diverse chemical change from relatively decisive modification such protein splicing and truncation more transient dynamic modification such phosphorylation acetylation and ubiquitination proteomic workflow can complex beast and several key consideration ensure effective application have been outlined the recent literature the past year ha witnessed the publication several excellent review that thoroughly describe the fundamental principle underlying the state the art this review further elaborates specific critical issue introduced these publication and raise other important unaddressed consideration and new development that directly impact the effectiveness proteomic technology particular for but not necessarily exclusive peptidecentric experiment these factor are discussed both term qualitative analysis including dynamic range and sampling issue and development improve the translation peptide fragmentation data into peptide and protein identity well quantitative analysis including data normalisation and the utility ontology functional annotation the effect modified peptide and considered experimental design facilitate the use robust statistical method',\n",
       " 'two recent study have provided the most complete screen for protein complex yeast date which partner were identified for approximately half the proteome comparison show that these two datasets are complementary addition one the analysis point modular organization the cellular protein network these data will prove useful defining principle and trend that arise when combining largescale datasets different nature and deriving property protein machine cellular system',\n",
       " '',\n",
       " 'the rapid development analytical instrumentation and methodical approach the course the last two decade ha significantly extended the possibility studying protein living system proteomic analysis provides ever deeper insight into the molecular nature biological process term qualitative and quantitative change protein composition connection with the physiological and pathological state the organism thus proteomic analysis contributes better understanding these process and becomes tool for the development and validation diagnostic and therapeutic approach thanks recent achievement the attention cancer specialist more and more focused human proteome research this brief review explain the principle widely used proteomic technique gel electrophoresis liquid chromatography mass spectrometry analysis protein array technology and show example their application oncology namely hematooncological disease',\n",
       " '',\n",
       " 'formaldehyde fixation widely used for longterm maintenance tissue however due formaldehydeinduced crosslinks fixed tissue protein are difficult extract which hamper mass spectrometry proteomic analysis recent year have seen the use different combination high temperature and solubilizing agent usually derived from antigen retrieval technique unravel formaldehydefixed paraffinembedded tissue proteome however achieve protein extraction yield similar those freshfrozen tissue hightemperature heating necessary such harsh extraction condition can affect sensitive amino acid and posttranslational modification resulting the loss important information while still not resulting protein yield comparable those freshfrozen tissue herein the objective evaluate cleavable protein crosslinkers fixative that allow tissue preservation and efficient protein extraction from fixed tissue for proteomics under mild condition with this goal mind disuccinimidyl tartrate dst and dithiobissuccinimidylpropionate dsp are investigated cleavable fixating reagent these compound crosslink protein reacting with amino group leading amide bond formation and can cleaved with sodium metaperiodate cisdiols dst reducing agent disulfide bond dsp respectively result show that cleavable protein crosslinking with dst and dsp allows tissue fixation with morphology preservation comparable that formaldehyde addition cleavage dsp improves protein recovery from fixed tissue factor and increase the number identified protein approximately under mild extraction condition compared with those formaldehydefixed paraffinembedded tissue major advantage dsp the introduction welldefined protein modification that can taken into account during database searching contrast dsp fixation dst fixation followed cleavage with sodium metaperiodate although effective result side reaction that prevent effective protein extraction and interfere with protein identification protein crosslinkers that can cleaved under mild condition and result defined modification such dsp are thus viable alternative formaldehyde tissue fixative facilitate protein analysis from paraffinembedded fixed tissue',\n",
       " 'the field molecular evolution started with the alignment few protein sequence the early among the first result found the genetic equidistance result ha turned out the most unexpected directly inspired the hoc universal molecular clock hypothesis that turn inspired the neutral theory unfortunately however what only maximum distance phenomenon wa mistakenly transformed into mutation rate phenomenon and became known such previous work studied small set selected protein have performed proteome wide study different set proteome involving total specie all set showed that within each set specie the least complex specie approximately equidistant average proteome wide identity the two more complex one thus the genetic equidistance result universal phenomenon maximum distance there reality constant albeit stepwise discontinuous increase complexity during evolution the rate which what the original molecular clock hypothesis really about these result provide additional line evidence for the recently proposed maximum genetic diversity mgd hypothesis',\n",
       " 'protein microarrays technology with great promise for highthroughput proteomics designing highperformance protein microarrays for global proteome analysis ha however turned out challenging this end major effort are under way design novel array format capable harboring the tremendous range probe required target complex proteome composed more than analytes adopting nanotechnology the first generation miniaturized nanoarrays ha recently emerged which open new avenue for global proteome analysis and disease proteomics this review describes the progress and key issue designing miniaturized protein array',\n",
       " 'kidney function rarely studied the context blood kidney and urine system kidney can considered black box while plasma and urine proteome closely represent the protein composition the input and output the kidney this idea provides new approach for studying organ function with proteomic methodology because it distinctive input plasma and output urine reasonable predict that the kidney will the first organ whose function are further elucidated proteomic method the near future',\n",
       " 'the use extra sd separation different buffer system provide technique for deconvoluting gel spot made several protein colignon proteomics this technique keep the quantitative analysis the protein amount and combine with strongly improved identification process mass spectrometry removing identification ambiguity most case some favorable case posttranslational variant can separated this procedure this versatile and easy use technique anticipated very valuable addition the toolbox used gelbased proteomics',\n",
       " '',\n",
       " '',\n",
       " 'hupo engages scientific and educational activity encourage the spread proteomics technology and disseminate knowledge pertaining the human proteome and that model organism this article part special issue entitled year proteomics memory viatliano pallini guest editor luca bini juan calvete natacha turck denis hochstrasser and jeancharles sanchez',\n",
       " '',\n",
       " 'analysis cerebrospinal fluid csf offer key insight into the status the cns characterization murine csf proteome can provide valuable resource for studying cns injury and disease animal model however the small volume csf mouse ha thus far limited individual mouse proteome characterization through nonterminal csf extraction cbl6 mouse and highresolution 2dlc msms analysis individual murine sample report the most comprehensive proteome characterization individual murine csf date identified total unique protein including protein from three individual csf sample that have been previously identified brain tissue our method and analysis provide mechanism for individual murine csf proteome analysis the data are available the proteomexchange with identifier pxd httpproteomecentralproteomexchangeorgdatasetpxd',\n",
       " 'advance both targeted and unbiased msbased proteomics are now mature stage for comprehensively and reproducibly characterizing large part the cancer proteome these development combined with the extensive genomic characterization several cancer type largescale initiative such the international cancer genome consortium and cancer genome atlas project have paved the way for proteogenomic analysis omics datasets and integration method the advance serve the basis for the national cancer institute clinical proteomic tumor analysis consortium and this article highlight it current work and future step the area proteogenomics',\n",
       " 'dataindependent acquisition method that acquire fragment ion from virtually any peptide sample have expanded the benefit lowthroughput targeted proteomics proteomewide analysis while these method have increased the reproducibility peptide quantification across multiple sample their sensitivity still limited and the quantification complete proteome remains challenge here present dia dia method that combine signal from identical peptide with different charge state resulting improved signaltonoise additional number fragment and therefore higher number identified and quantified peptide complex sample',\n",
       " 'highresolution mass spectrometry msbased proteomics ha progressed tremendously over the year for model organism like yeast can now quantify complete proteome just few hour development discussed this perspective will soon enable complete proteome analysis mammalian cell well with profound impact biology and biomedicine',\n",
       " 'the secretome full set secreted protein ha been studied multiple fungal genome elucidate the potential role those protein collection involved number metabolic process from host infection wood degradation being aminoacid composition key factor recognize secretory protein secretool comprises group web tool that enable secretome prediction out aminoacid sequence file complete fungal proteome one step secretool freely available the web httpgenomicscicbioguneessecretoolsecretoolphp',\n",
       " 'data analysis essential derive meaningful conclusion from proteomic data this chapter describes way performing common data visualization and differential analysis task gelbased proteomic datasets using freely available statistical software package workflow followed illustrated using synthetic dataset example',\n",
       " 'protein phosphorylation constitutes major type posttranslational modification that mobilizes high number gene especially plant involved many crucial cell function and largely participates the complexity the proteome for several biological and technical reason the characterization phosphorylation site requires complex procedure this review the different approach presently available select phosphoprotein and phosphopeptides are described special emphasis then given the numerous strategy that have emerged for the analysis phosphorylation site various technique mass spectrometry finally the few attempt proposed for the quantification phosphorylation event are presented another part the result the effort made the plant area analyze the phosphoproteome are compared those other biological system these overview are put together delineate according the objective pursued the different strategy possible and the corresponding challenge',\n",
       " 'proteomic method are acquiring greater importance archaeology and palaeontology due the longevity protein skeletal remains there are also developing interest forensic application offering the potential shed light postmortem interval and age death estimation however our understanding intra and interskeletal proteome variation currently severely limited here evaluated the proteome obtained from five distinct subsamples different skeletal element from buried pig carcass ascertain the extent variation within and between individual found that reproducibility data depends the skeletal element used for sampling and that intrabone difference exceed those observed between the same skeletal element sampled from different individual interestingly the abundance several serum protein appeared correlate with biological age with relative concentration alpha1 antitrypsin and chromogranina increasing and those fetuina decreasing also observed surprising level divergence data from different lcmsms run aliquot similar sample analyzed month apart adding constraint the comparison result such method across different study',\n",
       " 'formalinfixed paraffinembedded ffpe tissue ha recently gained interest alternative freshfrozen tissue for retrospective protein biomarker discovery however during the formalin fixation protein undergo degradation and crosslinking making conventional protein analysis technology challenging crosslinking even more challenging when quantitative proteome analysis ffpe tissue planned the use conventional protein labeling technology ffpe tissue ha turned out problematic the lysine residue labeling target are frequently blocked the formalin treatment have established qualitative and quantitative proteomics analysis technique for ffpe tissue that combine labelfree proteomic analysis with optimized protein extraction and separation condition',\n",
       " 'the general scope project determine the protein molecule that comprise the cell within the human body framed focusing protein primary structure expressed specific cell type this concept for cellbased version the human proteome project cbhpp crafted manner analogous the human genome project while recognizing that cell provide primary context which define proteome several activity flow from this articulation the hpp which enables the definition clear milestone and deliverable the cbhpp highlight major gap our knowledge regarding cell heterogeneity and protein isoforms and call for development technology that capable defining all human cell type and their proteome the main activity will involve mapping and sorting cell type combined with the application beyond the stateofthe art protein mass spectrometry',\n",
       " 'filteraided sample preparation fasp versatile and efficient way processing protein extract for bottomup proteomic analysis the method repurposes centrifugal ultrafiltration concentrators for removal detergent protein cleavage and isolation pure peptide fraction fasp can used for protein cleavage with different proteinase either with single enzyme mode successive multienzyme digestion medfasp the fasp method are useful for processing sample ranging their size from submicrogram several milligram amount total protein they also allow peptide fractionation and isolation and quantitation total rna and dna acid content this chapter describes principle limitation and application fasp additionally detailed fasp and medfasp protocol are provided',\n",
       " 'the proper integration multiple source data and the unbalance between annotated and unannotated protein represent two the main issue the automated function prediction afp problem most supervised and semisupervised learning algorithm for afp proposed literature not jointly consider these item with negative impact both sensitivity and precision performance due the unbalance between annotated and unannotated protein that characterize the majority functional class and the specific and complementary information content embedded each available source data propose unipred unbalanceaware network integration and prediction protein function algorithm that properly combine different biomolecular network and predicts protein function using parametric semisupervised neural model the algorithm explicitly take into account the unbalance between unannotated and annotated protein both construct the integrated network and predict protein annotation for each functional class fullgenome and ontologywide experiment with three eukaryotic model organism show that the proposed method compare favorably with stateoftheart learning algorithm for afp',\n",
       " 'hollow fiber flow fieldflow fractionation flfff ha been demonstrated tool for prefractionating proteome difference molecular mass where the resulting protein fraction are subsequently digested and analyzed shotgun proteomics using twodimensional liquid chromatographyelectrospray ionizationtandem mass spectrometry 2dlcesimsms flfff separation device capable fractionating protein cell hydrodynamic radius and protein fraction can readily collected intact condition aqueous buffer solution this study flfff wa applied fractionate the proteome corynebacterium glutamicum well known soil bacterium that ha been widely used bioindustry due it remarkable ability secrete high amount glutamic acid the collected flfff fraction different interval were enzymatically digested for protein identification 2dlcesimsms experiment showed improvement protein identification when flfff prefractionation wa applied due decrease the ionization suppression effect and the exclusion effect spectral congestion prefractionation glutamicum proteome allowed find additional protein 2dlcesimsms that were not found direct shotgun analysis without prefractionation total protein were found overall with protein commonly found from experiment with and without prefractionation',\n",
       " 'mass spectrometry specifically the analysis complex peptide mixture liquid chromatography and tandem mass spectrometry shotgun proteomics ha been the centre proteomics research for the past decade overcome some the fundamental limitation the approach including it limited sensitivity and high degree redundancy new proteomic workflow are being developed among these targeting method which specific peptide are selectively isolated identified and quantified are particularly promising here summarize recent incremental advance shotgun proteomic method and outline emerging targeted workflow the development the targetdriven approach with their ability detect and quantify identical nonredundant set protein multiple repeat analysis will crucially important for the application proteomics biomarker discovery and validation and system biology research',\n",
       " 'protein identification msms important technique proteome study the open mass spectrometry search algorithm omssa opensource search engine that can used identify msms spectrum acquired these experiment here present software tool termed omssapercolator which interface omssa with percolator postsearch machine learning method for rescoring database search result demonstrate that outperforms the standard omssa scoring scheme and provides reliable significant measurement omssapercolator programmed using java and can readily used standalone tool integrated into existing data analysis pipeline omssapercolator freely available and can downloaded httpsourceforgenetprojectsomssapercolator',\n",
       " 'lactobacillus acidophilus ncfm wellknown probiotic bacterium extensively studied for it beneficial health effect exoproteome protein exported into culture medium and surface proteome protein attached slayer this probiotic were identified using 2de followed maldi tof find protein potentially involved bacteriahost interaction the exo and surface proteome included and different protein from and successfully identified spot respectively twentytwo protein were shared between the two proteome both contained the major surface layer protein that participates host interaction well several wellknown and putative moonlighting protein the exoproteome contained nine classicallysecreted containing signal sequence and ten nonclassicallysecreted protein while the surface proteome contained four classicallysecreted and eight nonclassically secreted protein identification exo and surface proteome contributes describing potential proteinmediated probiotichost interaction',\n",
       " 'the discovery candidate biomarkers from biological material coupled with the development detection method hold both incredible clinical potential well significant challenge however the proteomic technique still provide the low dynamic range protein detection lower abundance this review describes the current development potential method enhance the detection and quantification proteome study also includes the bioinformatics tool that are helpfully used for data mining protein ontology therefore believe that this review provided many proteomic approach which would very potent and useful for proteome study and for further diagnostic and therapeutic application',\n",
       " 'analysis the cellular proteome can help elucidate the molecular mechanism underlying disease due the development technology that permit the largescale identification and quantification the protein present complex biological systemsthe knowledge gained from proteomic approach can potentially lead better understanding the pathogenic mechanism underlying disease allowing for the identification novel diagnostic and prognostic disease marker and hopefully therapeutic target however the cardiac mitral valve represents very challenging sample for proteomic analysis because the low cellularity proteoglycan and collagenenriched extracellular matrix this make challenging extract protein for global proteomic analysis this work describes protocol that compatible with subsequent protein analysis such quantitative proteomics and immunoblotting this can allow for the correlation data concerning protein expression with data quantitative mrna expression and nonquantitative immunohistochemical analysis indeed these approach when performed together will lead more comprehensive understanding the molecular mechanism underlying disease from mrna posttranslational protein modification thus this method can relevant researcher interested the study cardiac valve physiopathology',\n",
       " 'traditionally proteomic study have been carried out whole tissue organ enabling the profiling thousand protein within single lcm analysis disadvantage this approach that proteome generated from whole tissue are average that represents blend cell type and distinct anatomical region which can obscure important biological phenomenon laser capture microdissection lcm elegant method that allows tissue feature interest small single cell identified and isolated for downstream analysis herein describe approach that utilizes immobilized enzyme reactor imer coupled directly nanolcmsms for highly sensitive automated quantitative proteomic analysis the microscopic tissue specimen generated lcm',\n",
       " 'alongside antibody and other specific proteinbinding molecule have special place proteomics affinity reagent toolbox application for determining protein location quantitative distribution and function affinity proteomics the realisation that the range research antibody available while apparently vast nevertheless still very incomplete and frequently uncertain quality ha stimulated project with objective raising comprehensive proteomewide set protein binder with progress automation and throughput remarkable number recent publication refer the practical possibility selecting binder every protein encoded the genome here review the requirement pipeline production protein binder for the human proteome including target prioritisation antigen design next generation method database and the approach taken ongoing project europe and the usa while the task generating affinity reagent for all human protein complex and demanding the benefit wellcharacterised and qualitycontrolled panproteome binder resource for biomedical research industry and life science general would enormous and justify the effort given the technical personnel and financial resource needed fulfil this aim expansion current effort may best addressed through largescale international collaboration',\n",
       " 'this article highlight the emerging field chemical proteomics powerful technology for the study post and cotranslational modification protein genome mapping and the study protein posttranslational modification have revealed the astounding chemical complexity present the proteome even the simplest organism the identification and characterisation the modification present specific protein such complex mixture ha become central challenge for postgenomic functional study cell and system biology the chemical proteomic approach this problem proteinmodifying enzyme and bioorthogonal chemoselective elaboration are exploited deliver chemical tag specific modified residue enabling new advance our understanding protein modification',\n",
       " 'proteomic visualization serf complement proteomic identification recent year chemical biologist have made rapid progress developing new method tag and image defined set protein these researcher have requisitioned cellular machinery place small reactive analogue into biomolecules the analogue ha been labeled subsequently using selective ligation reaction many group have demonstrated the efficacy the coppercatalyzed strainpromoted azidealkyne ligation both enable rapid and precise labeling complex biological mixture this review provides overview the method which have been optimized tag and fluorophorelabel biomolecules for imaging subset the proteome bacterial and mammalian cell with the approach described herein should possible image cell they undergo change over time',\n",
       " 'mass spectrometry msbased proteomics ha enabled the identification and quantification thousand protein from complex proteome single experiment however it performance for masslimited proteome sample single cell and tissue sample from laser capture microdissection still not satisfying the development novel proteomic methodology with better overall sensitivity vital during the last several year substantial technical progress ha been achieved for the preparation and liquidphase separationms characterization masslimited proteome sample this review summarize recent technological progress sample preparation liquid chromatography lcm capillary zone electrophoresis czems and instrumentation for bottomup proteomics trace biological sample highlight some exciting application the novel technique for singlecell proteomics and provide very brief perspective about the field the end',\n",
       " 'present here new method for automatic metaanalysis proteomic article using assessment frequency individual protein name the text the list all possible human protein name including synonym wa retrieved from uniprot knowledgebase the retrieved name were searched fulltexts peerreviewed publication from electronic version proteomics journal and from pubmedcentral the automatic mode have confirmed the earlier list protein petrak proteomics most frequently reported differentially expressed deps human tissue have also verified that the most recurrent protein were reported proteomic paper regardless tissue experimental goal some extent experimental method employed frequently reported deps were annexins peroxiredoxins alphaenolase triosephosphate isomerase and hsp besides serum albumin cathepsin and vimentin were observed with relatively high frequency the deps were reported paper related oncological cardiovascular and neuronal disease and were involved such biological process inflammation cell regulation immune responce and signal transduction conclude that automatic metaanalysis proteomic paper enabled extraction frequently reported protein that are most likely the differentially expressed one',\n",
       " 'the ability investigate the proteome formalinfixed paraffinembedded ffpe tissue can considered major recent achievement the field clinical proteomics however gelbased approach the investigation ffpe tissue proteome have lagged behind mainly because insufficient quality fulllength protein extract here the dige technology wa investigated for applicability ffpe protein for internal reproducibility among replicate ffpe extract and for comparability between ffpe and freshfrozen tissue profile the dige pattern obtained upon labeling and electrophoresis replicate ffpe tissue extract were highly reproducible with satisfactory resolution and complexity moreover the implementation dige enabled highlight and characterize the consistent difference found the ffpe profile compared with freshfrozen profile represented acidic shift directly correlated the protein value and reduction spot signal intensity directly correlated molecular weight and percentage lysine residue being constantly and reproducibly present all ffpe tissue extract replicates similar extent these modification not appear hinder the comparative analysis ffpe tissue extract dige opening the way it application for the differential proteomic investigation archival tissue repository',\n",
       " 'the dentinenamel junction dej the border where two different mineralized structure enamel and dentin meet the proteinrich dej together with the inner enamel region mature teeth known exhibit higher fracture toughness and crack growth resistance than bulk phase enamel however explanation for this behavior ha been hampered the lack compositional information for the dej and the adjacent enamel organic matrix eom studied proteome the dej and eom healthy human molar and compared them with dentin and enamel proteome from the same teeth these tissue were cut out tooth section laser capture microdissection protein were extracted and cleaved trypsin then processed liquid chromatography coupled tandem mass spectrometry analyze the proteome profile these tissue this study identified protein dej and eom the protein identified have variety function including calcium ionbinding formation extracellular matrix formation cytoskeleton cytoskeletal protein binding cell adhesion and transport collagen were identified the most dominant protein tissuespecific protein such ameloblastin and amelogenin were also detected our finding reveal new insight into proteomics dej and eom highly mineralized tissue that are obviously difficult analyze',\n",
       " 'mass spectrometrybased proteomics the method choice for analyzing sub proteome from virtually any source with cell grown culture being the most frequently investigated sample using hela cell describe strategy for the optimization protocol for whole proteome analysis cover cell lysis protein precipitation and digestion comparing the result obtained using different parameter and offering various possibility for the optimization proteomic workflow which can used for virtually any type animal cell grown culture',\n",
       " 'recent progress life science technology and the availability much information gene obtained genome analysis ha enabled analyze the change protein large scale set protein are called proteome and proteomics the scientific field proteome analysis including differential post translational modification and interaction analysis various proteomic technique particularly twodimensional gel electrophoresis 2de mass spectrometry protein chip method and surface plasmon resonance spr are very useful for acquiring proteome cell tissue and body fluid and for analyzing interaction between protein and other biofactors including protein proteomic approach also useful for determining biomarkers disease and key protein involved various stage metabolism such differentiation cell cycle and apoptosis environmental pollutant including endocrine disruptors inhibit activity various organ wild animal and human proteomic approach could very useful tool for elucidating the mechanism damage caused environmental pollutant this review describe the application proteomic approach the field environmental medicine',\n",
       " 'protein directly control almost all cellular process and researcher many biological area routinely use mass spectrometry for the characterization protein amongst growing list available quantitative proteomic technique stable isotope labeling amino acid culture silac remains one the most simple accurate and robust technique for cultured cellular system silac enables strict quantitative peptide measurement thus removing false positive and facilitates largescale kinetics entire proteome this chapter describe optimized labeling strategy and experimental design for silac workflow for characterizing the component downstream cell death stimulus',\n",
       " 'advance proteomics are contributing the understanding pathophysiology cancer cancer diagnosis and anticancer drug discovery laser capture microdissection lcm provides ideal method for extraction cell from specimen which the exact morphology both the captured cell and the surrounding tissue are preserved differentially expressed protein tumor tissue are found comparing the protein expression pattern generated using seldi surfaceenhanced laser desorptionionizationbased protein chip technology proteomic technology have been used for the study cancer various organ continued refinement technique and method determine the abundance and status protein vivo hold great promise for future study cancer and development personalized cancer therapy',\n",
       " 'widely known that the progress proteomics mostly depends the development more advanced and sensitive protein separation technology immobilized metal affinity chromatography imac useful protein fractionation method used enrich metal associated protein imac represents affinity separation approach based the interaction between protein and metal ion immobilized solid support changing various metal ion and other experimental condition such and elution composition imac can selectively isolate metalbinding protein fraction for further specific proteomic analysis the combination imac with other protein analytical technology ha been successfully applied characterizing metalloproteome and posttranslational modification the future newly developed imac technique integrated with other proteomic method will significantly contribute the further development proteomics',\n",
       " 'recent year standardization and quality control have become important key point industry drug discovery and for developing medical product quality control academic proteomics minor problem nowadays where standard data format and public repository for data sharing exist this article discussed how standard format and repository already support the documentation quality control criterion protein identification and quantification and what ha improved the future stated that the proteomics community represented group like the proteomics standard initiative will have define minimum document regarding quality control and extend existing standard with additional quality control criterion enabling substantial and standardized quality control process',\n",
       " '',\n",
       " 'during the last decade have witnessed outstanding advance proteomics led mostly great technological improvement mass spectrometry field allowing highthroughput production highquality data used for massive protein identification and quantification from practical viewpoint these advance have been mainly exploited research project involving model organism with abundant genomic and proteomic information available public database however there growing number organism high interest different discipline such ecological biotechnological and evolutionary research yet poorly represented these database important advance massive parallel sequencing technology and easy accessibility this technology many research laboratory have made nowadays possible produce customized genomic and proteomic database any organism along this line the use proteogenomic approach combining the same analysis the data obtained from different omic level ha emerged very useful and powerful strategy run shotgun proteomic experiment specially focused nonmodel organism this chapter provide detailed procedure undertake shotgun quantitative proteomic experiment following either labelfree isobaric labeling approach nonmodel organism emphasizing also few key aspect related experimental design and data analysis',\n",
       " '',\n",
       " 'the quantification large proteome across multiple sample ha become the major focus proteomics addition the advantage ingel digestion the extensive time and sample handling required have precluded the use this type method for large quantitative screen therefore adaptation the ingel digestion method termed shortgelc proposed faster and more reproducible sample preparation method for quantitative approach the proposed methodology wa compared with two wellestablished procedure for sample preparation gelcms and the classic liquid digestion followed lcm using membrane proteinenriched sample the result show that the shortgelc approach substantially reduces the amount sample handling and the overall time required for analysis compared with the gelbased method without compromising the overall result the protein identification level furthermore the shortgelc approach combination with the swath acquisition method lead the best quantitative result more protein were quantified and the reproducibility wa improved finally this method performed well even challenging sample enriched membrane protein',\n",
       " 'over the past decade development high resolution mass spectrometry have enabled the high throughput analysis intact protein from complex proteome leading the identification thousand proteoforms several previous report topdown proteomics tdp relied hybrid ion trapfourier transform mass spectrometer combined with datadependent acquisition strategy further reduce tdp practice use quadrupoleorbitrap instrument coupled with software for proteoformdependent data acquisition identify and characterize nearly proteoforms false discovery rate from human fibroblast combining isolation window with short transient improve specificity and signaltonoise for proteoforms kda demonstrate improving proteome coverage capturing proteoforms the kda range three different data acquisition strategy were compared and resulted the identification many proteoforms not observed replicate datadependent experiment notably the data set reported with updated metric and tool including new viewer and assignment permanent proteoform record identifier for inclusion highly characterized proteoforms those with cscores repository curated the consortium for topdown proteomics',\n",
       " 'while lcmsbased proteomics with high nanogram microgram total protein ha become routine the analysis sample derived from low cell number challenged factor such sample loss difficulty encountered with the manual manipulation small liquid volume digital microfluidics dmf emerging technique for miniaturized and automated droplet manipulation which ha been proposed promising tool for proteomic sample preparation however proteome analysis sample prepared onchip dmf ha previously been unfeasible due incompatibility with downstream lcm instrumentation overcome these limitation here developed protocol for bottomup lcm based proteomics sample preparation little mammalian cell commercially available digital microfluidics device this end developed effective cell lysis condition optimized for dmf well detergentbuffer system compatible with downstream proteolytic digestion dmf chip and subsequent lcm analysis major step wa the introduction the singlepot solidphaseenhanced sample preparation sp3 approach onchip which allowed the removal salt and antifouling polymeric detergent thus rendering sample preparation dmf compatible with lcmsbased proteome analysis application dmfsp3 the proteome analysis jurkat cell led the identification protein from approximately cell and protein from approximately cell orbitrap mass spectrometer emphasizing the high compatibility dmfsp3 with low protein input and minute volume handled dmf taken together demonstrate the first sample preparation workflow for proteomics dmf chip device reported far allowing the sensitive analysis limited biological material',\n",
       " 'the unique peptide finder httpunipeptugentbepeptidefinder interactive web application quickly hunt for tryptic peptide that are unique particular specie genus any other taxon biodiversity within the target taxon represented set proteome selected from monthly updated list complete and nonredundant uniprot proteome supplemented with proprietary proteome loaded into persistent local browser storage the software computes and visualizes pan and core peptidomes union and intersection tryptic peptide occurring the selected proteome addition also computes and display unique peptidomes the set all tryptic peptide that occur all selected proteome but not any uniprot record not assigned the target taxon result the unique peptide can serve robust biomarkers for the target taxon for example targeted metaproteomics study computation are extremely fast since they are underpinned the unipept database the lowest common ancestor algorithm implemented unipept and modern web technology that facilitate inbrowser data storage and parallel processing',\n",
       " 'shotgun proteomic analysis are increasingly becoming method choice for complex sample the development effective method for fractionating peptide reduce the complexity the sample before mass analysis key point this strategy the offgel technology ha recently become tool choice proteomic analysis peptide level this offgel electrophoresis oge approach allows the insolution separation peptide from various biological source isoelectric focusing highly resolved fraction wa also demonstrated that oge technology filtering tool for pibased validation peptide identification peptide oge compatible with itraq labeling oge finding valuable application quantitative proteomics well the aim this study explain new 2doge approach that improves the proteomic coverage complex mixture such colorectal cell line lysates and which compatible with itraq labeling',\n",
       " 'normalization fundamental step data processing account for the sampletosample variation observed biological sample however data structure affected normalization this paper show how and what extent the correlation structure affected the application different normalization procedure also discus the consequence for data analysis and interpretation including principal component analysis partial leastsquares discrimination and the inference metabolitemetabolite association network',\n",
       " 'the goal quantitative proteomics determine the identity and relative quantity each protein present two more complex protein sample here describe novel approach quantitative proteomics based highly accurate algorithm for the automated quantification chromatographically fractionated isotopecoded affinitytagged peptide and maldi quadrupole timeofflight tandem mass spectrometry for their identification the method capable detecting and selectively identifying those protein within complex mixture that show difference relative abundance demonstrate the effectiveness and the versatility this approach the analysis standard protein mixture protein expression profiling human prostate cancer cell line model and identification the specific component the multiprotein transcriptional machinery cerevisiae',\n",
       " 'isotope analysis collagen extracted from ancient bone have been routinely used for dietary and chronological inference worldwide for decade these method involve the decalcification biomineralized tissue with acid often followed process remove exogenous contaminant and then gelatinization what often described the collagen fraction however little known about the relative content collagen the many other noncollagenous protein ncps potentially present some these ncps have great longevity ancient bone and some for example fetuina are useful for obtaining better taxonomic information than collagen this study us orbitrap elite lcmsms characterize the proteome the acidsoluble and basesoluble fraction which are usually discarded and the gelatinized collagen fraction obtained from both stable isotope and radiocarbon method applied several ancient bovine bone the result showed that all fraction tested contain numerous ncps but the basesoluble fraction for both method contains the greatest number ncps with the highest relative abundance this study confirms that not only the waste fraction obtained from the collagen extraction procedure stable isotope radiocarbon dating method yield plentiful resource ncps that currently being overlooked but that they also provide proteome complex those obtained from standard proteomics method',\n",
       " 'microglia the resident brain immune cell can exhibit broad range activation phenotype which have been implicated multitude central nervous system disorder current widely studied microglial cell line are mainly derived from neonatal rodent brain that can limit their relevance homeostatic function and diseaserelated neuroimmune response the adult brain recently adult mouse brainderived microglial cell line ha been established however comprehensive proteome dataset remains lacking here optimization method for sensitive and rapid quantitative proteomic analysis microglia described that involves suspension trapping strap for efficient and reproducible protein extraction from limited number microglial cell expected from adult mouse brain using gradient uplc column with modified data dependent acquisition method hybrid quadrupoleorbitrap mass spectrometer total protein have been identified where which are quantifiable labelfree quantitation with median and average coefficient variation and respectively this dataset highlight the high depth proteome coverage and related quantitation precision the adultderived microglial proteome including protein associated with several key pathway related immune response data are available via proteomexchange with identifier pxd',\n",
       " 'with recent technological development protein biomarker discovery directly from blood have regained interest due elevated feasibility mass spectrometry msbased proteomics can now characterize human plasma proteome greater extent than ha ever been possible before such deep proteome coverage come however with important limitation term analysis time which critical factor the case clinical study consequence compromise still need made balance the proteome coverage with realistic analysis time frame clinical research the analysis sufficient number sample compulsory empower statistically robust candidate biomarker finding have therefore recently developed scalable automated proteomic pipeline asapsup2sup enable the proteomic analysis large number plasma and cerebrospinal fluid csf sample from dozen thousand sample with the latter number being currently processed week distinct characteristic asapsup2sup relies the possibility prepare sample highly automated way mostly using well plate describe herein sample preparation procedure for human plasma that includes internal standard spiking abundant protein removal buffer exchange reduction alkylation tryptic digestion isobaric labeling pooling and sample purification other key element the pipeline study design sample tracking liquid chromatography tandem msms data processing and data analysis are also highlighted',\n",
       " 'the biennial siena proteome conference can regarded the cradle the massively expanding field proteomics the very term proteome wa introduced the 1st siena meeting marc wilkins and some the most important proteomics advancement were first reported siena the intervening year have seen proteomics develop into one the most powerful approach biology providing new insight into how cell tissue and even whole organism function the molecular level the 8th siena meeting from genome proteome integration and proteome completion wa attended over scientist there wa one nobel lecture eight plenary lecture invited talk and poster covering virtual proteome completion quantitative and functional proteomics genetics and evolution oncology model system gel electrophoresis mass spectrometry method and validation biomarkers usual the meeting wa particularly fruitful stimulating wideranging consideration future perspective',\n",
       " 'the multiplexing capability isobaric mass tagbased protein quantification such tandem mass tag isobaric tag for relative and absolute quantitation have dramatically increased the scope mass spectrometrybased proteomics study not only doe the technology allow for the simultaneous quantification multiple sample single injection but it seamless compatibility with extensive sample prefractionation method allows for comprehensive study complex proteome however reporter ionbased quantification ha often been criticized for limited quantification accuracy due interference from coeluting peptide and peptide fragment this study investigate the extent this problem and propose effective and easytoimplement remedy that relies spiking 6protein calibration mixture the sample evaluated our ratio adjustment approach using two large scale tmt plex data set derived from human cancer and noncancer cell line well coli cell grown two different condition furthermore analyzed complex 2proteome artificial sample mixture and investigated the precision tmt and precursor ion intensitybased label free quantification studying the protein set identified both method found that differentially abundant protein were assigned dramatically higher statistical significance when quantified using tmt data are available via proteomexchange with identifier pxd',\n",
       " 'highresolution mass spectrometry indispensable for the molecularlevel analysis biological and environmental sample with great intra and intermolecular complexity here summarize development fourier transform mass spectrometry ftms the flagship highresolution technique accomplished our laboratory particularly describe the recent and envisioned progress structural analysis isolated large protein and their simple mixture with focus monoclonal antibody via topdown middledown and extended bottomup mass spectrometry complex protein mixture and proteome via extended bottomup proteomics and iii crude oil fraction and similar complex molecular mixture despite the unequivocal success molecular structural analysis the demonstrated result clearly indicate that the compromise between acquisition speed throughput and achievable resolution level inhibits further advance application the area related life environmental and material science further advance beyond stateoftheart ftms capability these area present the technique superresolution mass spectrometry that ha been pioneered our laboratory',\n",
       " 'here explore application system using disposable solidphase extraction spe cartridge and very short lcmsms gradient that allows for rapid analysis le than min analysis time the setup consists autosampler harboring two set stage tip that function precolumns and short analytical column running min gradient this system combine efficiently with several proteomics workflow such offline prefractionation method including gel electrophoresis and strongcation exchange chromatography also enables the analysis interactomes obtained affinity purification with analysis time approximately typical shotgun proteomics experiment involving scx fraction aspn digested cell lysate detected over protein group with analysis time le than this innovative fast system reduces proteome analysis time while maintaining sufficient proteomic detail this ha particular relevance for the use proteomics within clinical environment where large sample number and fast turnover time are essential',\n",
       " 'proteomic screening complex biological sample becomes increasing importance clinical research and diagnosis expected that the meager number approx human gene give rise more than functional entity the protein level thus the proteome provides much richer source information than the genome for describing the state health disease the human organism especially the composition body fluid comprise rich source information possible change the status health disease particular organ and consequence the whole organism here describe the application capillary electrophoresis coupled online electrosprayionization esitimeofflight tofmass spectrometer the analysis human urine for the identification biomarkers for complication after allogeneic hematopoietic stem cell transplantation',\n",
       " 'pichia pastoris widely used eukaryotic host for production recombinant protein performed proteogenomic analysis using high resolution fourier transform characterize the proteome the strain our analysis resulted identification unique peptide mapping unique protein group which corresponds the predicted gene addition our proteogenomic analysis led the discovery novel gene and correction predicted gene model the strategy used here demonstrates the utility high resolution msderived peptide sequence data cover near complete proteome organism given the popularity pastoris protein expression host this proteome map should provide list contaminant derived from the host assist optimization heterologous protein production all data have been deposited the proteomexchange with identifier pxd httpproteomecentralproteomexchangeorgdatasetpxd',\n",
       " 'the purpose this study determine the behavioral and proteomic consequence shockinduced stress zebrafish danio rerio vertebrate model here describe the behavioral effect exposure predictable and unpredictable electric shock together with quantitative tandem mass tag isobaric labeling workflow detect altered protein candidate response shock exposure behavioral result demonstrate hyperactivity response electric shock and suppression activity stimulus predicting shock the basis the quantitative change protein abundance following shock exposure eight protein were significantly upregulated hadhb hspa8 hspa5 actb1 mych4 atp2a1 zgc and zgc these protein contribute crucially catalytic activity stress response cation transport and motor activity this behavioral proteomic driven study clearly showed that besides the rapid induction heat shock protein other catalytic enzyme and cation transporter were rapidly elevated mechanism counteract oxidative stress condition resulting from elevated fearanxiety level',\n",
       " 'analysis dilute protein sample challenging task for scientific and industrial lab all over the world although there are different method available that allow for protein enrichment from various biological source all them have serious limitation apart from their advantage order perform highly reproducible and sensitive protein analysis lowest concentrated sample optimized method enrich protein affinity bead strataclean recently this chapter describes the general protocol this strategy thereby discussing the power well the limit this technique for qualitative and quantitative proteomic study moreover additional application and protocol variant will discussed expanding the number compatible and downstream processing technique compared the originally published method hence evaluated the reduction time for sample preparation use preprimed affinity bead and shorter incubation duration well the influence high concentration salt urea the sample buffer',\n",
       " 'identification and characterization chinese hamster ovary cho host cell protein hcp impurity proteomic technique can aid bioprocess design and lead more efficient development and improved biopharmaceutical manufacturing operation recovery extracellular cho hcp for proteomic analysis particularly challenging due the relatively low protein concentration and complex composition medium this article report the development optimized protocol that improve proteome capture for cho hcp eleven precipitation protocol were screened for protein recovery and optimized for subset precipitant design experiment doe approach because total protein recovery doe not fully replicate proteomics experiment detect nonprotein agent that may interfere with proteomic method subset precipitation condition were compared twodimensional electrophoresis and liquid chromatography coupled with mass spectrometry with optimized recovery shown differ between the two proteomic method this work demonstrates broadly applicable method that can applied initial step optimize sample preparation any sample type for proteomic analysis and present optimized precipitation protocol for extracellular cho hcp recovery which can vary appreciably between gelbased and shotgun proteomic method',\n",
       " 'twodimensional remains one the main experimental approach proteome analysis however comigration protein lead several limitation lack accuracy protein identification impaired comparative quantification and ptm detection have optimized third additional step ingel separation alleviate comigration associated drawback spot resolution strikingly improved following this simple and rapid method and the positive impact protein and peptide identification from msms data the analysis relative change protein abundance and the detection ptm described',\n",
       " '',\n",
       " 'the trace proteome braulio aperitif alcohol beverage named after mountain the val stelvio italy ha been investigated via capture with combinatorial peptide ligand library cpll proteominer this aperitif made with infusion mountain herb and berry among which four are officially indicated the label achillea moschata juniper juniperus communis subsp alpina berry absinthe artemisia absinthium and gentian gentiana alpina root via capture with cplls and were able identify unique gene product among which the pr5 parasite resistance allergen jun kda specie from juniperus rigida due the paucity data these alpine herb wa difficult attribute these protein the specific plant extract presumably present this beverage however most the specie identified indeed belong alpine herb and plant living habitat between and elevation most them are enzyme spanning range from kda hoped that such proteomic signature should help tracking counterfeited product sold the market',\n",
       " 'after the successful completion the human genome project the human proteome organization ha recently officially launched global human proteome project hpp which designed map the entire human protein set given the lack proteinlevel evidence for about the estimated proteincoding gene systematic global effort will necessary achieve this goal with respect protein abundance distribution subcellular localization interaction with other biomolecules and function specific time point general experimental strategy hpp research group will use the three working pillar for hpp mass spectrometry antibody capture and bioinformatics tool and knowledge base the hpp participant will take advantage the output and crossanalyses from the ongoing human proteome organization initiative and chromosomecentric protein mapping strategy termed chpp with which many national team are currently engaged addition numerous biologically driven and diseaseoriented project will stimulated and facilitated the hpp timely planning with proper governance hpp will deliver protein part list reagent and tool for protein study and analysis and stronger basis for personalized medicine the human proteome organization urge each national research funding agency and the scientific community large identify their preferred pathway participate aspect this highly promising project hpp consortium funders and investigator',\n",
       " 'recently the term proteomics wa proposed define the largescale study the protein expressed genome typically proteomic study rely number technique such 2dimensional gel electrophoresis and mass spectrometry that have been used over the year the current technology have serious drawback term financial and skilled labor requirement automation sensitivity and robustness furthermore the current technology often fail identify large subset protein such hydrophobic protein posttranslationally modified protein and protein low abundance therefore clear that proteomics field under development and will require novel technology for become reliable largescale tool for the study biological process miniaturization and automation could address some the current proteomic limitation recently the development microfabricated system for the analysis protein ha been reported this paper review the recent development enclosed microfluidic system often termed lab chip and the development array device for the characterization protein',\n",
       " 'streptococcus pyogenes secretes various protein the extracellular environment during infection these protein interact with human macromolecule and contribute pathogenesis describe proteomic approach routinely used our laboratory characterize culture supernatant protein using smallformat twodimensional gel electrophoresis protein are collected after overnight growth the bacteria broth medium compound that inhibit isoelectric focusing such salt are removed enzymatic treatment and precipitation with trichloroacetic acid and acetone following resuspension denaturing solution the protein are separated isoelectric focusing using 7cm immobilized strip with gradient subsequently protein are further separated with sodium dodecyl sulfatepolyacrylamide gel electrophoresis sdspage and stained with sypro ruby the smallgel format requires le time reagent and smaller culture volume compared with largeformat approach while still resolving and detecting large proportion the exoprotein fraction',\n",
       " 'the accelerated growth protein database offer great possibility for the study protein function using sequence similarity and conservation however the huge number sequence deposited these database requires new way analyzing and organizing the data necessary group the many very similar sequence creating cluster with automated derived annotation useful understand their function evolution and level experimental evidence developed algorithm called fastaherder2 which can cluster any protein database putting together very similar protein sequence based nearfulllength similarity andor high threshold sequence identity compressed reference proteome along with the swissprot database which could compress the clustering algorithm wa benchmarked using orthobench and compared with fasta herder previous version the algorithm showing that fastaherder2 can cluster set protein yielding high compression with lower error rate than it predecessor illustrate the use fastaherder2 detect biologically relevant functional feature protein family with our approach seek promote modern view and usage the protein sequence database more appropriate the postgenomic era',\n",
       " 'pancreatic cancer one the most lethal malignancy and diseasespecific biomarkers are desperately needed for better diagnosis prognosis monitoring treatment efficacy and for accelerating the development novel targeted therapeutic being advanced stage manifestation and proximal fluid contact with cancer tissue the ascitic fluid present itself promising rich source biomarkers herein present comprehensive proteomic analysis pancreatic ascitic fluid fractionate the complex ascites proteome adopted multidimensional chromatographic approach that included sizeexclusion ionexchange and lectinaffinity chromatographic technique our detailed proteomic analysis with highresolution orbitrap® mass spectrometer resulted the identification protein followed rigorous filtering criterion that consisted pcspecific information obtained from three publicly available database oncomine protein atlas and unigene segregate putative biomarker candidate for future validation since these protein are membranous and extracellular origin most are glycosylated and many them are overexpressed cancer tissue the probability these protein entering the peripheral blood circulation high their validation serological biomarkers the future highly warranted',\n",
       " 'the physiological adaptation stationary growth pseudomonas putida model organism for the degradation aromatic compound wa investigated proteomewide labelfree quantificationthe data unveiled that entrance the stationary phase did not involve abrupt switch within the putida proteome but rather ongoing adaptation that started already during the midexponential growth phase the proteomic adaptation involved clear increase amino acid degradation capability and loss transcriptional well translational capacity the final entrance the stationary phase wa accompanied increased oxidative stress protection although the stress and stationary sigma factor rpos increased abundance already during midexponential growth the result show that important consider significant sample variation when exponentially growing culture are studied alone compared across proteomic transcriptomic literature all data have been deposited the proteomexchange with identifier pxd httpproteomecentralproteomexchangeorgdatasetpxd',\n",
       " '',\n",
       " 'blood serum one the easiest accessible source biomarkers and it proteome present significant parcel immune system protein these protein can provide not only biological explanation but also diagnostic and drug response answer independently the type disease condition question shotgun mass spectrometry ha profoundly contributed proteome analysis and presently considered indispensible tool the field biomarker discovery addition the multiplexing potential isotopic labeling technique such itraq can increase statistical relevance and accuracy proteomic data through the simultaneous analysis different biological sample here describe complete protocol using itraq shotgun proteomics workflow along with data analysis step customized for the challenge associated with the serum proteome',\n",
       " 'this paper demonstrate rapid and reproducible lcmsms workflow for fast quantitative proteomic research have optimized the lcmsms condition including digestion and gradient condition sample loading amount and parameter setting result were able obtain twice many protein identification compared with the lcmsms condition before optimization more than protein group and peptide were identified le than without any fractionation this workflow wa then applied the analysis the mln treateduntreated human umbilical vein endothelial cell huvec sample with labelfree quantification these experiment total protein showed statistically significant expression change after the mln treatment functional analysis showed that these protein are associated with cell death and survival gene expression cell cycle and dna replication recombination and repair',\n",
       " 'the application neural stem cell nsc research neurodegenerative disease ha led promising clinical trial currently nsc therapy most promising for parkinson disease conducted behavioral test and immunoassay for the profiling model rat ass the therapeutic effect nsc treatment further using multiple sample comparison workflow combined with olabeled proteome mixture compared the differentially expressed protein from control and nsctreated rat the result were analyzed bioinformatically and verified western blot based our initial finding believe that the proteomic approach valuable tool evaluating the therapeutic effect nsc transplantation neurodegenerative disorder',\n",
       " '',\n",
       " 'the liver play important role metabolism and elimination xenobiotics including drug determination concentration protein involved uptake distribution metabolism and excretion xenobiotics required understand and predict elimination mechanism this tissue this work have fractionated homogenate snapfrozen human liver differential centrifugation and performed quantitative mass spectrometrybased proteomic analysis each fraction concentration protein were calculated the total protein approach total protein were identified least five peptide and were quantified all fraction found that the xenobiotics transporter the canalicular and basolateral membrane were differentially enriched the subcellular fraction and that phase and metabolizing enzyme the cytochrome and the udpglucuronyl transferase have complex subcellular distribution these finding show that there simple way scale the data from measurement arbitrarily selected membrane fraction using single scaling factor for all the protein interest this study also provides the first absolute quantitative subcellular catalog human liver protein obtained from frozen tissue specimen our data provide quantitative insight into the subcellular distribution protein and can used guide for development fractionation procedure',\n",
       " 'developed new method consisting the proteomic reactor coupled with step fractionation for the analysis lowabundance protein from minute amount sample these new reactor were implemented using both sax and scx material the fraction from the sax reactor provided higher peptide and protein identification than scx reactor and conventional solution digestion interestingly the physical characteristic molecular weight missed cleavage site and grand average hydrophobicity gravy index and number acid and basic amino acid the peptide obtained from the sax and scx proteomic reactor are drastically different furthermore nearly half the peptide observed from the fractionation from the sax reactor are low abundance while only lowabundance protein are observed with conventional insolution digestion following lcmsms analysis',\n",
       " '',\n",
       " 'the plasma membrane serf the point contact between cell and the outside environment such change the proteome are important component understanding cellular response diverse array stimulus however intricate sample handling enrich proteome traditional method both technically challenging and time consuming here describe simplified method for decreasing the representation other membranecontaining organelle such the endoplasmic reticulum plastid and mitochondrion from crude microsomal membrane isolation the decrease other organellar proteome result increase both the total number protein and the number spectrum identified from these protein representing the proteome therefore this strategy represents simple and rapid method for enriching protein from arabidopsis cell culture for proteomic analysis',\n",
       " 'proteomic analysis are becoming more widely used archeology not only due the greater preservation protein ancient specimen than dna but also because they can offer different information particularly relating compositional preservation and potentially mean estimate biological and geological age however remains unclear what extent different burial environment impact these aspect proteome decay teeth have date been much le studied than bone but are ideal explore how protein decay with time due the negligible turnover that occurs dentine relative bone investigated the proteome variability and deamidation level different section molar teeth from archeological bovine mandible well their mandibular bone obtained greater yield protein from the crown the teeth but did not find difference between the different molar analyzed within each mandible also obtained the best variety protein from wellpreserved mandible that wa not the youngest one term chronological age showing the influence the preservation condition the final proteomic outcome intriguingly also noticed increase abundance level fetuina biologically younger mandible reported previously but the opposite trend tooth dentine interestingly observed higher glutamine deamidation level teeth from the geologically oldest mandible despite being the biologically youngest specimen showing that the archeological age strongly impact the level deamidations observed much more than biological aging this indicates that the glutamine deamidation ratio selected peptide may act good predictor the relative geochronological age archeological specimen',\n",
       " '',\n",
       " 'although plethora human protein are ubiquitously expressed several protein with high pharmaceutical relevance show tissue celltype specific expression pattern science across all discipline ranging from developmental biology personalized medicine would benefit from detailed knowledge about this socalled human proteome two recent publication nature use largescale proteomics create first draft the human proteome which are freely accessible online this letter analyze the proteomic data with regard the expression three different cytokine receptor the tumor necrosis factor tnfα receptor tnfrsf1a and tnfrsf1b and the interleukin6 receptor il6r therapeutic inhibition these protein highly effective high number inflammatory disease and tnfα blocking agent alone were sold for almost billion find that the known expression pattern the three receptor not reflected the current draft the human proteome the proteomics data fail detect protein expression several cell type and tissue which are known express these cytokine receptor thus our result suggest that the current draft the human proteome are far from complete and that the data have used with caution especially term personalized medicine',\n",
       " 'proteome analysis suffer from the large complexity even small proteome additionally many protein sample few highly abundant protein are hindering detailed proteomic study since they mask low abundant protein recently new technology ha emerged which reduces dynamic range protein concentration within given sample using combinatorial hexapeptide ligand library cplls this technique ha been widely used the microbial animal and human field and now going enter plant research can useful tool for fractionation protein sample and might help get deeper insight into specific plant proteome this review describe the cpll protein fractionation summarize it possible application the plant field and discus the limitation this method',\n",
       " '',\n",
       " 'several line evidence suggest that hnrnps hnrnps a2b1 and play important role proliferation although the functional overlap among member hnrnps remains largely unknown this study have employed rnai knockdown and proteomic approach investigate the biological function hnrnps depletion hnrnp but not produced significant inhibition cellular proliferation jurkat cell analysis the proteome the cell depleted for hnrnp ha identified total differentially expressed protein the depleted cell network analysis the protein altered the cell depleted for hnrnp revealed that the biological process likely affected these protein are related cell cycle cytoskeleton rearrangement and transcription regulation indeed have confirmed that the level rhoa and crkl wa selectively reduced the cell depleted hnrnp but not the cell depleted for hnrnp therefore suggest that the reduced proliferation observed the cell depleted hnrnp may result from it effect cell adhesion process the jurkat cell',\n",
       " 'new quantitative cysteinylpeptide enrichment technology qcet wa developed achieve higher efficiency greater dynamic range and higher throughput quantitative proteomics that use stableisotope labeling technique combined with highresolution liquid chromatography lcmass spectrometry this approach involves labeling tryptic peptide highefficiency enrichment cysteinecontaining peptide and confident protein identification and quantification using the accurate mass and time tag strategy proteome profiling naïve and vitrodifferentiated human mammary epithelial cell using qcet resulted the identification and quantification protein single lcfourier transform ion cyclotron resonance analysis advantage this technology include the following simple highly efficient method for enriching cysteinylpeptides highthroughput strategy suitable for extensive proteome analysis and improved labeling efficiency for better quantitative measurement this technology enhances both the functional analysis biological system and the detection potential clinical biomarkers',\n",
       " 'secretomic analysis requires removal serum protein from cellculture medium evaluate the protein washed from cell prepared bovine serumsupplemented medium pb and serumfreemedium sfm were the washing solution bradford assay wa used for total protein concentration and gel and lcmsms assign the protein human bovine origin for both wash solution all bovine protein had been removed the third wash without compromising the number living cell further wash reduced the number living cell especially when using pb proteomic analysis wash supernatant showed that sfm induced greater lysis dead cell three wash were sufficient minimize the effect cell viability while still removing serum protein washing sfm resulted contamination the wash supernatant with lysed dead cell protein washed cell were incubated sfm and exposed ionizing radiation analysis the supernatant showed increase human cytoplasmic plasma membrane and nuclear protein following irradiation secreted protein were also detected but smaller quantity the significance these finding extend vitro study bystander phenomenon since the protein lysed dead cell may participate driving bystander response',\n",
       " 'protein ubiquitination play important role the regulation many cellular process including protein degradation cell cycle regulation apoptosis and dna repair study the ubiquitin proteome have established immunoaffinity purification method for the proteomic analysis endogenously ubiquitinated protein complex strong specific enrichment ubiquitinated factor wa achieved using the fk2 antibody bound protein gbeaded agarose which recognizes monoubiquitinated and polyubiquitinated conjugate mass spectrometric analysis two fk2 immunoprecipitations ip resulted the identification fk2specific protein both experiment the isolation ubiquitinated and ubiquitinationrelated protein wa confirmed pathway analysis using ingenuity pathway analysis and gene ontologyannotation enrichment additionally comparing the protein that specifically came down the fk2 with database ubiquitinated protein showed that high percentage protein our enriched fraction wa indeed ubiquitinated finally assessment proteinprotein interaction revealed that significantly more fk2specific protein were residing protein complex than random protein set this method which capable isolating both endogenously ubiquitinated protein and their interacting protein can widely used for unraveling ubiquitinmediated protein regulation various cell system and tissue when comparing different cellular state',\n",
       " 'indepth analysis cancer cell proteome are needed elucidate oncogenic pathomechanisms well identify potential drug target and diagnostic biomarkers however method for quantitative proteomic characterization patientderived tumor and particular their cellular subpopulation are largely lacking here describe experimental setup that allows quantitative analysis proteome cancer cell subpopulation derived from either liquid solid tumor this achieved combining cellular enrichment strategy with quantitative supersilacbased mass spectrometry followed bioinformatic data analysis enrich specific cellular subset liquid tumor are first immunophenotyped flow cytometry followed facssorting for solid tumor lasercapture microdissection used purify specific cellular subpopulation second step protein are extracted from the purified cell and subsequently combined with tumorspecific silaclabeled spikein standard that enables protein quantification the resulting protein mixture subjected either gel electrophoresis filter aided sample preparation fasp followed tryptic digestion finally tryptic peptide are analyzed using hybrid quadrupoleorbitrap mass spectrometer and the data obtained are processed with bioinformatic software suite including maxquant mean the workflow presented here protein can identified and quantified patientderived sample and the resulting protein expression profile can compared among patient identify diagnostic proteomic signature potential drug target',\n",
       " '',\n",
       " 'the study the proteome human cell tissue and body fluid big challenge and several highly sophisticated workflow approach are pursued achieve comprehensive information possible initially proteome analysis wa exclusively based the gelbased workflow employing twodimensional electrophoresis protein extract followed mass spectrometry the tryptic peptide digest protein spot meanwhile several additional proteomics workflow are applied which are mostly based separation and analysis tryptic peptide without separating the protein mixture however direct information quantitative and qualitative change protein expression can only obtained method operating the protein level other method can replace twodimensional electrophoresis this review compile the different technique highresolution twodimensional electrophoresis and their further development increase the degree reliance the method',\n",
       " 'bouins solution ha been used for over century common fixative several pathology laboratory worldwide therefore considerable number bouinfixed paraffinembedded bfpe tumor sample various origin are available hospital repository powerful information mine for clinical investigation date however such archived tissue have not been subjected systematic study aimed evaluate their potential use proteomics this report investigated whether archival bfpe tissue specimen could exploited for proteomic study upon application protein extraction and proteomic analysis method previously optimized for formalinfixed sample result gastric bfpe protein extract exhibited poor suitability for 2dpage analysis whereas over unique protein could successfully detected when extract were subjected sdspage followed lcmsms gelcmsms among these several known marker for gastric cancer and normal gastric functionality were identified indicative biological and clinical significance proteomic data mined from bfpe tissue quantitative and qualitative comparison ffpe and bfpe tissue proteome wa also performed and result are reported conclusion demonstrated that bfpe specimen can analyzed mean proteomic approach such gelcmsms although considerable molecular bias and technical constraint exist bfpe tissue archive can fruitfully exploited for gathering proteomic data from particularly precious sample',\n",
       " 'the filteraided sample preparation fasp method allows gelfree processing biological sample solubilized with detergent for proteomic analysis mass spectrometry fasp detergent are removed ultrafiltration and after protein digestion peptide are separated from undigested material here compare the effectiveness different filtration device for analysis proteome and glycoproteomes show that microcon and vivacon filtration unit with nominal molecular weight cutoff and and respectively are equally suitable for fasp whereas microcon unit are most appropriate for mapping nglycosylation site the use filter with these relatively large cutoff facilitates depletion detergent',\n",
       " 'have presented complex graph matrix representation characterize proteomics map obtained from 2dgel electrophoresis this method each bubble 2dgel proteomics map represented complex number with component which are charge and mass then graph with complex weight constructed connecting the vertex the relative order abundance this yield adjacency matrix and distance matrix the proteomics graph with complex weight have computed the spectrum eigenvectors and other property complex graph and the euclidiangraph distance obtained from the complex graph the leading eigenvalue and eigenvectors and likewise the smallest eigenvalue and eigenvectors and the entire graph spectral pattern the complex matrix derived from them yield novel weighted biodescriptors that characterize proteomics map with information charge and mass protein have also applied these eigenvector and eigenvalue map contrast the normal cell and cell exposed four peroxisome proliferators namely clofibrate diethylhexyl phthalate dehp perfluorodecanoic acid pfda and perfluoroctanoic acid pfoa our complex eigenspectra show that the proteomic response induced dehp differs from the corresponding response other three chemical consistent with their chemical structure and property',\n",
       " '',\n",
       " 'the complexity mammalian proteome challenge bottomup proteomics for comprehensive proteome analysis multidimensional separation strategy are necessary online twodimensional liquid chromatographytandem mass spectrometry 2dlcmsms combining strong cation exchange scx the first dimension with reversedphase chromatography the second dimension provides powerful approach analyze complex proteome although the combination scx with chromatography provides good orthogonality only moderate separation achieved the first dimension for peptide with two three positive charge the aim this study wa improve the performance online scxrpmsms applying displacement chromatography the first separation dimension compared gradient chromatography mode gcm displacement chromatography mode dcm wa expected improve the separation 2peptides and 3peptides thus reducing complexity and increasing ionization and detectability the result show that dcm provided separation 2peptides and 3peptides remarkably sharp zone with low degree coelution thus providing fraction with significantly higher purity compared gcm particular 2peptides were separated over several fraction which wa not possible achieve gcm the better separation dcm resulted higher reproducibility and significantly higher identification rate for both peptide and protein including fold increase for 2peptides the higher number identified peptide dcm resulted significantly higher protein sequence coverage and considerably higher number unique peptide per protein compared conventionally used saltbased gcm dcm increased the performance online scxrpmsms and enabled comprehensive proteome profiling the low microgram range',\n",
       " 'the summer distinguished global representative proteome science gathered discus the futuristic vision the chromosomecentric human proteome project chpp cochairs paik omenn hosted archakov institute biomedical chemistry russia that wa broadcast the annual federation european biochemical society congress petersburg russia july technology breakthrough presented included new ultrasensitive tribrid massspectrometer from thermo and somamersslow offrate modified aptamers somalogic usa new type protein capture reagent professor archakovs group introduced the rectangle concept proteome size product proteome width and depth the discussion proteome width culminated with the introduction digital biomarkerslowcopied aberrant protein that differ from their typical form ptms alternative splicing single amino acid polymorphism the aberrant proteoforms complement wholegenome proteomic survey were presented ultimate goal for the proteomic community',\n",
       " 'the goal this study wa develop tool specifically designed identify iterative polyketide synthases ipkss from predicted fungal proteome fungibased pks prediction model specifically for fungal ipkss wa developed using profile hidden markov model phmms based two essential ipks domain the βketoacyl synthase domain and acyltransferase domain derived from fungal ipkss this fungibased pks prediction model wa initially tested the wellannotated proteome fusarium graminearum identifying ipkss that matched previous prediction and gene disruption study these fungibased phmms were subsequently applied the predicted fungal proteome alternaria brassicicola fusarium oxysporum fsp lycopersici verticillium alboatrum and verticillium dahliae the ipkss predicted were compared against those predicted the currently available mixedkingdom pks model that include both bacterial and fungal sequence these mixedkingdom model have been proven previously others better predicting true ipkss from nonipkss compared with other available model pfam and tigrfam the fungibased model wa found perform significantly better fungal proteome than the mixedkingdom pks model accuracy sensitivity specificity and precision addition the model wa capable predicting the reducing nature fungal ipkss comparison the bit score obtained from two separate reducing and nonreducing phmms for each domain which wa confirmed phylogenetic analysis the domain biological confirmation the prediction wa obtained polymerase chain reaction pcr amplification the and domain predicted ipkss from dahliae using domainspecific primer and genomic dna followed sequencing the pcr product expected that the fungibased pks model will prove useful tool for the identification and annotation fungal pkss from predicted proteome',\n",
       " 'describe gelassisted proteomic sample preparation method for analysis solubilized protein extract intact cell are copolymerized with acrylamide facilitating denaturation reduction quantitative cysteine alkylation and matrix formation gelaided sample preparation ha been optimized highly flexible scalable and allow reproducible sample generation from cell milligram protein extract this methodology fast sensitive easytouse wide range sample type and accessible nonspecialists',\n",
       " '',\n",
       " 'cervicalvaginal fluid cvf cover the lower part the female reproductive system and function the homeostasis and immunity the surrounding tissue contrast the cvf proteome both nonpregnant and pregnant woman the cvf peptidome ha not been reported date the current study identified protein cvf which protein were not previously identified cvf proteome the presence the tissuespecific protein oviductal glycoprotein and tubulin polymerizationpromoting protein family member strongly suggests that the tissue the upper female reproductive tract contribute the protein composition cvf the tremendous catalytic potential cvf wa highlighted the identification protease and the detection phdependent trypsinlike proteolytic activity over endogenous peptide were detected the cvf peptidome and peptide are predicted have antimicrobial activity the detailed proteomic and peptidomic analysis cvf will further aid the delineation pathway related reproduction immunity and host defense and assist developing new biomarkers for malignant and other disease the female reproductive tract data are available via proteomexchange with identifier pxd cvf peptidome and pdx cvf proteome',\n",
       " 'the hupo brain proteome project hupo bpp held it workshop vancouver canada september the focus the autumn workshop wa new insight into the proteomic profile alzheimers disease schizophrenia al and multiple sclerosis',\n",
       " 'many quantitative proteomics method rely protein and peptide labeling with stable isotope have recently found that the introduction into organism via vivo metabolic labeling affect protein expression level well metabolic pathway and behavioral phenotype here present further evidence for stable isotope effect based the plasma proteome analysis ¹nlabeled mouse compared plasma proteome ¹nlabeled and unlabeled mouse quantitative found number protein level difference some which were verified immunochemically addition observed divergent chromatographic retention time and peak full width half maximum fwhm between ¹nlabeled and tryptic peptide our data point toward systemic effect the introduction heavy isotope vivo',\n",
       " 'experiment probe for proteinprotein interaction are the focus functional proteomic study thus proteomic data repository are increasingly likely contain large crosssection such information here use the global proteome machine database gpmdb which the largest curated and publicly available proteomic data repository derived from tandem mass spectrometry develop silico protein interaction analysis tool using human histone protein for method development positively identified interaction partner from each histone protein family that form the histone octameric complex moreover this method applied the subunit the human proteasome identified all the subunit the core particle furthermore applied this approach human integrin αiib and integrin major receptor involved the activation platelet identified protein including protein network for integrin and platelet activation addition protein interacting with integrin obtained using this method were validated comparing them those identified formaldehydesupported coimmunoprecipitation experiment proteinprotein interaction database and the literature our result demonstrate that silico protein interaction analysis novel tool for identifying knowncandidate proteinprotein interaction and protein with shared function protein network',\n",
       " '',\n",
       " '',\n",
       " 'more sensitive and convenient coomassie brilliant blue cbb staining method for visualizing protein wa developed compared with the modification include the supplement methanol into the fixing solution increase additional sensitization step and cbb raised from the improved method can detect protein nanogram level the improved method more sensitive than blue silver and more convenient than the silver protocol mass spectrometry result confirmed that suitable for subsequent proteomic research',\n",
       " 'extensive genomic characterization multispecies acid mine drainage microbial consortium combined with laboratory cultivation ha enabled the application quantitative proteomic analysis the community level this study quantitative proteomic comparison were used functionally characterize laboratorycultivated acidophilic community sustained condition the distribution all protein identified for individual organism indicated bias for either high low and suggests phspecific niche partitioning for low abundance bacteria and archaea although the proteome the dominant bacterium leptospirillum group wa largely unaffected treatment analysis functional category indicated protein involved amino acid and nucleotide metabolism well cell membraneenvelope biogenesis were overrepresented high comparison specific protein abundance indicates higher condition favor leptospirillum group iii whereas low condition promote the growth certain archaea thus quantitative proteomic comparison revealed distinct difference community composition and metabolic function individual organism during different treatment proteomic analysis revealed other aspect community function different number phage protein were identified across biological replicates indicating stochastic spatial heterogeneity phage outbreak additionally proteomic data were used identify previously unknown genotypic variant leptospirillum group indication selection for specific leptospirillum group population laboratory community our result confirm the importance and related geochemical factor finetuning acidophilic microbial community structure and function the specie and strain level and demonstrate the broad utility proteomics laboratory community study',\n",
       " 'astrocyte are the most abundant cell the cns but their function remains largely unknown characterization the wholecell proteome and secretome astrocyte would facilitate the study their function various neurodegenerative disease and astrocyteneuron communication build reference proteome established c8d1a astrocyte proteome depth unique protein group using novel strategy that combined twostep digestion filteraided sample preparation stagetipbased high fractionation and highresolution nearly unique protein group were identified from conditioned medium astrocyte culture constituting the largest astrocyte secretome that ha been reported highconfidence wholecell proteome and secretomes are valuable resource studying astrocyte function labelfree quantitation and bioinformatics analysis all data have been deposited the proteomexchange with identifier pxd httpproteomecentralproteomexchangeorgdatasetpxd',\n",
       " 'multiplexed quantification with isobaric chemical tag tmt itraq provides robust and efficient mean comparatively examine proteome dynamic between several biological state using mass spectrometer the quantitative nature isobaric tag necessitates strict validation the observed ion signal the chosen detector before differential pattern are extracted between biological state present indepth analysis isobaric tag data acquired current generation orbitrap hardware illustrate pitfall acquisition setting that can negatively impact result establish for the first time the presence notch region observed value the reporter ion distribution from isobariclabeled peptide mixture acquired these instrument determine that this notch present published data across wide range instrument the same different type and isolated the orbitrap mass analyzer demonstrate that the impact the notch can minimized using manipulation orbitrap scan parameter and oncolumn injection amount lastly using mixture synthetic standard peptide investigated the impact identification rate and quantification precision together these data highlight important phenomenon that negatively impact peptide identification and quantification the orbitrap analyzer well outlining guideline follow ensure minimization msinduced artifact isobaric tag experiment resulting from the notch',\n",
       " 'the aim the study were identify differentially regulated protein cerebrospinal fluid csf between multiple sclerosis patient and nonms control examine the effect matching the csf sample either total protein amount volume and compare four protein normalization strategy for csf protein quantification csf from patient and control consisting other noninflammatory neurological disease and non neurological spinal anesthetic subject were analyzed using labelfree proteomics quantifying almost protein total protein were significantly regulated where protein had pvalue auc value hierarchical clustering indicated that there were two main group patient those with increased level inflammatory response protein and decreased level protein involved neuronal tissue development and those with normal protein level for both these protein group the main subgroup control clustering with the patient showing increased inflammation and decreased neuronal tissue development were patient suffering from chronic fatigue our data indicate that the preferable way quantify protein csf first match the sample total protein amount and then normalize the data based the median intensity preferably from the cnsenriched protein',\n",
       " '',\n",
       " 'induced pluripotent stem cell ipscs are somatic cell that have been reprogrammed pluripotent state via introduction defined transcription factor ipscs are valuable resource for regenerative medicine but whether ipscs are identical embryonic stem cell escs remains unclear this study performed comparative proteomic analysis human somatic cell human newborn foreskin fibroblast hffs human ipscs hipscs derived from hffs and human escs hescs reprogrammed hffs pluripotent state using core transcription factor oct4 sox2 klf4 and cmyc the proteome hipscs induced core transcription factor wa relatively similar that hescs however several protein including dutpase gapdh and fuse binding protein were differentially expressed between hescs and hipscs implying that hipscs are not identical hescs the proteomic level the proteome ipscs induced introducing transcription factor were also analyzed our proteomic profile provide valuable insight into the factor that contribute the similarity and difference between hescs and hipscs and the mechanism reprogramming',\n",
       " 'major challenge towards comprehensive analysis biological system the integration data from different omics source and their interpretation functional level here address this issue analysing transcriptomic and proteomic datasets from mouse brain tissue embryonic day and observe high concordance between transcript and their corresponding protein when they were compared the level expression ratio between embryonic stage absolute expression value show marginal correlation show example that poor concordance between protein and transcript expression part explained the fact that single gene give rise multiple transcript and protein variant the integration transcriptomic and proteomic data therefore requires proper handling such ambiguity closer inspection such case our datasets suggests that comparing gene expression exon level instead gene level could improve the comparability address the biological relevance difference expression profile literaturedata mining and analysis gene ontology term are widely used show here that this can complemented the inspection physical property gene transcript and protein',\n",
       " 'major improvement proteomic technique recent year have led increase their application all biological field including plant science for all proteomic approach protein extraction and sample preparation are utmost importance for optimal result however extraction protein from plant tissue represents great challenge plant tissue usually contain relatively low amount protein and high concentration protease and compound that potentially can limit tissue disintegration and interfere with subsequent protein separation and identification effective protein extraction protocol must also adaptable the great variation the set secondary metabolite and potentially contaminating compound that occurs between tissue leaf root fruit seed and stem and between specie here present two basic protein extraction protocol that have successfully been used with diverse plant tissue including recalcitrant tissue the first method based phenol extraction coupled with ammonium acetate precipitation and the second based trichloroacetic acid tca precipitation both extraction protocol can completed within',\n",
       " 'oxaliplatin widely used treat colorectal cancer both palliative and adjuvant setting also being tested for use treating hematological esophageal biliary tract pancreatic gastric and hepatocellular cancer despite it routine clinical use little known about the response induces cancer cell therefore the wholecell proteomics study wa conducted characterize the cellular response induced oxaliplatin chemosensitive ccrfcem cell were treated with oxaliplatin 5×icsubsub for min halftime caspase activation the proteome untreated cell were then compared highresolution mass spectrometry revealing protein expressed over biological replicates among these protein were significantly downregulated and significantly upregulated least two replicates agreement with the dnadamaging effect platinum drug protein involved dna damage response were present both the upregulated and downregulated group the downregulated protein were divided into three subgroup centrosomal protein rna processing and iii ribosomal protein which indicates nucleolar and ribosomal stress conclusion our data supported further validation experiment indicate the initial cellular response oxaliplatin the activation dna damage response which turn parallel trigger nucleolar and ribosomal stress',\n",
       " 'the clinical application human bone marrow stromal cell hbmscs largely depends their capacity expand vitro have conducted comprehensive comparative proteomic analysis cultureexpanded hbmscs obtained from different human donor the data reveal extensive donortodonor proteomic heterogeneity processing and databasesearching the tandem data resulted most comprehensive date proteomic dataset for hbmsc total protein including transcription and translation regulator kinase receptor protein and cytokine were confidently identified the protein identified are mainly nuclear and the share protein assigned more than one subcellular location constitutes the identified proteome bioinformatics tool ipa david and panther were used annotate protein with respect cellular location function and other physicochemical characteristic also compared the proteomic profile hbmscs recently compiled datasets for human and mouse pluripotent stem cell the result show the extent similarity between the three cell population and also identified protein expressed uniformly all line hbmscs but not reported the proteomic datasets the two pluripotent stem cell overall the proteomic database reported this paper can serve reference map for extensive evaluation hbmsc explain their biology well identify possible marker candidate for further evaluation',\n",
       " '',\n",
       " '',\n",
       " 'shotgun proteomics experiment require the collection thousand tandem mass spectrum these set data will continue grow new instrument become available that can scan even higher rate such data contain substantial amount redundancy with spectrum from particular peptide being acquired many time during single lcmsms experiment this article present ms2grouper algorithm that detects spectral duplication ass group related spectrum and replaces these group with synthetic representative spectrum error detecting spectral similarity are corrected using paraclique criterionspectra are only assessed group they are part clique least three completely interrelated spectrum are subsequently added such clique being similar all but one the clique member greedy algorithm construct representative spectrum for each group iteratively removing the tallest peak from the spectral collection and matching peak the other spectrum this strategy shown effective reducing spectral count lcmsms datasets from protein standard mixture and proteome reducing database search time without concomitant reduction identified peptide',\n",
       " 'bufalin exhibited antiproliferation and antimigration effect human lung cancer cell search it targetrelated protein protein expression profile bftreated and control cell were compared using two quantitative proteomic method itraqbased and labelfree proteomic analysis total protein were identified itraqbased analysis while protein were identified labelfree analysis the number common identified protein both method wa protein application fold for upregulated and fold for downregulated cutoff value and differentially expressed protein were found itraqbased and labelfree analysis respectively the number common differentially expressed protein both method wa protein result bioinformational analysis using metacoretm showed that the two proteomic method were complementary and both suggested the involvement oxidative stress and regulation gene expression the effect and fibronectinrelated pathway wa suggested important pathway affected western blotting assay result confirmed bfinduced change level fibronectin and other related protein overexpression fibronectin plasmid transfection ameliorated antimigration effect result the present study provided information about possible targetrelated protein and signal network',\n",
       " '',\n",
       " 'the hupo brain proteome project hupo bpp held it workshop honolulu hawaii during the january the island became the center the open workshop the scientific community',\n",
       " 'the tremendous complexity the serum and plasma proteome present extreme analytical challenge comprehensive analysis due the wide dynamic range protein concentration therefore robust sample preparation method remain one the important step the proteome characterization workflow present the result new column for the specific depletion highabundant protein from human serum and plasma and the subsequent reversedphase fractionation the flowthrough protein analysis tryptic peptide wa accomplished with microfluidic hplcchipms system result indicate that highabundant protein depletion combined with fractionation plasma showed improved dynamic range for proteomic analysis and enabled the identification lowabundant plasma protein',\n",
       " 'quantitation change protein abundance key understanding the alteration that biological system undergo and discovering novel biomarkers currently hplcmsms can used quantify change protein expression level ong and mann nat chem biol nevertheless quantitative analysis protein mixture hplcmsms still hampered the wide range protein expression level the high dynamic range protein concentration and the lack reliable quantitation algorithm dascenzo brief funct genomic proteomic lin proteome re matthiesen proteome re nucleic acid re this context describe two different sample 4protmix and 8protmix suitable for relative protein quantitation using itraq using the 4protmix relative protein change fold were measured the 8protmix allowed the quantitation the relative protein change mixture protein within the range two order magnitude concentration and tenfold difference relative abundance propose that the two sample are suited test the itraq quantitative proteomic workflow analyzed the itraq sample with ltq orbitrap using higher energy collisioninduced dissociation fragmentation olsen nat method and quantified with proteome discoverer thermo fisher scientific believe that the presented protein mixture will useful ass the performance the itraqbased quantitation proteomic strategy any laboratory',\n",
       " '',\n",
       " 'the use isobaric tag such itraq allows the relative and absolute quantification hundred protein single experiment for eight different sample more classical technique such 2de can offer complimentary approach for the analysis complex protein sample this study the proteome secreted and cytosolic protein genetically closely related strain mycobacterium tuberculosis were analyzed analysis gel afforded spot with variation protein abundance between strain these were identified msms meanwhile rigorous statistical analysis itraq data allowed the identification and quantification and protein the secreted and cytosolic fraction respectively interestingly several difference protein level were observed between the closely related strain and seven protein related cell wall and cell process were more abundant while enzyme related metabolic pathway glta2 succ gnd1 eno presented lower level the strain protein involved iron and sulfur acquisition bfrb viub and ssec2 were more abundant and general itraq afforded rapid identification fine difference between protein level such those presented between closely related strain this provides platform from which the relevance these difference can assessed further using complimentary proteomic and biological modeling method',\n",
       " 'the extracellular matrix ecm and secreted vesicle are unique structure outside cell that carry out dynamic biological function ecm created most cell type and responsible for the threedimensional structure the tissue organ which they are originated many cell also produce secrete specialized vesicle into the ecm which are thought influence the extracellular environment ecm not physical structure connect cell tissue organ the protein ecm and secreted vesicle are critical cell function differentiation motility and celltocell interaction although number major structural protein ecm and secreted vesicle have long been known appreciation the role lessabundant noncollagenous protein ha just begun emerge this chapter outline series method used isolate and enrich ecm constituent and secreted vesicle from boneforming osteoblast cell enabling comprehensive profile their proteome obtained mass spectrometry these method can easily adapted study ecm and secreted vesicle other cell type primary cell culture derived from animal model tissue specimen',\n",
       " 'availability complete genome sequence allows indepth comparison singleresidue and oligopeptide composition the corresponding proteome have used principal component analysis pca study the landscape compositional motif across more than genus from all three superkingdoms unexpectedly the first two principal component clearly differentiate archaea eubacteria and eukaryota from each other particular contrast compositional pattern typical the three superkingdoms and characterize difference between specie and phylum well among pattern shared all compositional proteomic signature these speciesspecific pattern may even extend subset the entire proteome such protein pertaining individual yeast chromosome identify factor that affect compositional signature such living habitat and detect strong eukaryotic preference for homopeptides and palindromic tripeptides further detect oligopeptides that are either universally over underabundant across the whole proteomic landscape well oligopeptides whose over underabundance phylum speciesspecific finally report that specie composition signature preserve evolutionary memory providing new method compare phylogenetic relationship among specie that avoids problem sequence alignment and ortholog detection',\n",
       " 'better understand the involvement retinal müller glial rmg cell retinal disease phenotyped primary porcine rmgs dependence cultivation time using different quantitative proteomic strategy wellestablished lcmsmsbased quantification method wa employed stable isotope labeling amino acid cell culture silac and directly compared labelfree quantification based total peak intensity using two different program maxquant and progenesis lcm the overall number detected protein were largely similar overlap protein only total protein were significantly altered between the different culture condition however among these only protein were shared between the three analytical strategy hence the majority altered protein only reached significance threshold one the applied analysis with larger overlap between the two approach among the shared differentially abundant protein were known rmg marker well new protein associated with glial cell transition however protein correlated cellular transition and dedifferentiation were also found among the protein only significant one two the applied strategy consequently the application different quantification and analytical strategy could increase the analytical depth proteomic phenotyping',\n",
       " 'combination 2de and lcmsms approach wa used identify the differentially expressed proteome pigmented sweet orange citrus sinensis moro comparison with common cultivar cadenera ripening time the comparison the protein pattern moro and cadenera showed differential expressed protein spot fiftyfive differentially expressed protein were identified protein were classified according their putative function and known biosynthetic pathway most the protein related sugar metabolism were overexpressed moro while those related stress response were overexpressed cadenera the abundance protein belonging unknownunnamed and hypothetical class could associated the incomplete data available the citrus genome the relative abundance secondary metabolism and oxidative process protein substantiated the key role the anthocyanin pathway moro which characterized strong pigmentation ripening time the potential role protein differential expression between moro and cadenera fruit wa discussed and proteomic result were compared with the known variation transcript the same fruit the latter analysis highlighted many discrepancy confirming the necessity associate both proteomic and transcriptomic approach order achieve more complete characterization the biological system',\n",
       " 'novel method such massspectrometry enable view the proteome cell unprecedented detail recently these effort have culminated quantitative measurement the number copy per cell for most expressed protein organism ranging from bacteria mammalian cell here estimate the expected total number protein per unit cell volume using known parameter related the composition cell such the fraction cell mass that protein and the average protein length using simple argument estimate range million protein per cubic micron 1fl bacteria yeast and mammalian cell interestingly find that measured value that are reported for fission yeast and mammalian cell are often about time lower discus this apparent discrepancy and how use the estimate benchmark recalibrate proteomewide quantitative census revisit assumption about cell composition',\n",
       " 'recent advance peptidebased bottomup quantitative proteomics and bioinformatics have opened unprecedented opportunity for extensive investigation cellular proteome and their dynamic here discus two approach currently used investigate the global dynamic phosphorylation based the isolation phosphorylated protein peptide evaluate the accuracy these methodology grasp the global dynamic phosphorylation and raise awareness ambiguity inherent these analysis conclude that further development targeted approach should prevent inaccurate conclusion about the nature biological regulation and particular kinasesubstrate network',\n",
       " 'the lipoaspirate fluid laf emerging potentially valuable source regenerative medicine particular our group recently demonstrated that able exert osteoinductive property vitro this original observation stimulated the investigation the proteomic component laf mean lcesiltqorbitrapms topdownbottomup integrated approach which represents the object the present study topdown analysis required the optimization sample pretreatment procedure enable the correct investigation the intact proteome bottomup analysis have been directly applied untreated sample after monodimensional sdspage separation the analysis the acidsoluble fraction laf topdown approach allowed demonstrating the presence albumin and hemoglobin fragment and lvvhemorphin7 thymosin and peptide ubiquitin and acylcoa binding protein adipogenesis regulatory factor perilipin1 fragment and sa6 along with their ptms part the bottomup proteomic profile wa reproducibly found both tested sample the bottomup approach allowed demonstrating the presence protein listed among the component adipose tissue andor comprised within the ascs intracellular content and secreted proteome our data provide first glance the laf molecular profile which consistent with it tissue environment laf appeared contain bioactive protein peptide and paracrine factor suggesting it potential translational exploitation',\n",
       " '',\n",
       " 'clinical and diagnostic proteomics essential develop comprehensive and robust system for proteome analysis although multidimensional liquid chromatographytandem mass spectrometry lcmsms system have been recently developed powerful tool especially for identification protein complex these system still some drawback their application clinical research that requires analysis large number human sample therefore this study have constructed technically simple and high throughput protein profiling system comprising twodimensional 2dlcmsms system which integrates both strong cation exchange scx chromatography and microlcmsms system with microflowing reversedphase chromatography using the microlcmsms system the second dimensional chromatography scx separation ha been optimized offline first dimensional peptide fractionation evaluate the performance the constructed 2dlcmsms system the result detection and identification protein were compared using digest mixture authentic protein with those obtained using onedimensional microlcmsms system the number peptide fragment detected and the coverage protein sequence were found more than double through the use our newly built 2dlcmsms system furthermore this multidimensional protein profiling system ha been applied plasma proteome order examine it feasibility for clinical proteomics the experimental result revealed the identification protein from one serum sample depleted hsa and igg which corresponds only microl plasma and the total analysis run time wa le than half day indicating fairly high possibility practicing clinical proteomics high throughput manner',\n",
       " 'more than century research sea anemone venom ha shown that they contain diversity biologically active protein and peptide however recent omics study have revealed that much the venom proteome remains unexplored used for the first time combination proteomic and transcriptomic technique obtain holistic overview the venom arsenal the wellstudied sea anemone stichodactyla haddoni purely searchbased approach identify putative toxin transcriptome from tentacle regenerating after venom extraction identified unique toxinlike transcript grouped into family however proteomic analysis venom revealed that these toxin family are likely false positive contrast the combination transcriptomic and proteomic data enabled positive identification family putative toxin which have homology known protein peptide our data highlight the importance using proteomics milked venom correctly identify venom proteinspeptides both known and novel while minimizing false positive identification from nontoxin homologues identified transcriptomes venomproducing tissue this work lay the foundation for uncovering the role individual toxin sea anemone venom and how they contribute the envenomation prey predator and competitor',\n",
       " 'silico proteomics complement computational genomics characterizing genome evolution here examine cluster pattern archaeal and bacterial proteome using compositional property protein sequence contrast the traditionally used sequence alignment procedure application standard principal component analysis the multidimensional data identified cluster pattern two type cluster pattern exist bacterial proteome proteome type have one major cluster with few isolated point space revealing underlying largely homogeneous compositional structure type proteome two cluster protein distribution were discernible the two cluster differ size and were separated from each other although the boundary wa somewhat fuzzy protein falling the major cluster were labeled typical and protein the minor cluster were called atypical the atypical protein were mapped cluster orthologous group specie distribution cog map with respect atypical protein illuminated the biological relationship extreme diversity among the archaeal member and diversity among bacteria relation their niche amino acid that were overrepresented the atypical protein had higher biosynthetic cost compared typical ribosomal protein however archaea and bacteria economize preferring the le costly amino acid others closely related chemical structure further overrepresentation serine atypical protein archaeal member suggests reexamining these proteome for the presence serinethreonine phosphatase and kinase archaea our computational procedure can serve useful addition the existing tool for carrying out silico proteomics',\n",
       " 'mass spectrometrydriven blast blast database search protocol for identifying unknown protein sequence similarity homologous protein available database blast utilizes redundant degenerate and partially inaccurate peptide sequence data obtained novo interpretation tandem mass spectrum and ha become powerful tool functional proteomic research using computational modeling evaluated the potential blast for proteomewide identification unknown protein determined how the success rate protein identification depends the fulllength sequence identity between the queried protein and it closest homologue database also estimated phylogenetic distance between organism under study and related reference organism with completely sequenced genome that allow substantial coverage unknown proteome',\n",
       " 'cerebrospinal fluid csf potential source biomarkers for many disorder the central nervous system including alzheimer disease prior comparing csf sample between individual identify pattern diseaseassociated protein important examine variation within individual over short period time that one can better interpret potential change csf between individual well change within given individual over longer time span this study analyzed csf sample composed pair sample from six individual obtained week apart multiaffinity depletion twodimensional dige and tandem mass spectrometry were used number protein whose abundance varied between the two time point wa identified for each individual some these protein were commonly identified multiple individual more importantly despite the intraindividual variation hierarchical clustering and multidimensional scaling analysis the proteomic profile revealed that two csf sample from the same individual cluster the closest together and that the betweensubject variability much larger than the withinsubject variability among the six subject comparison between the four cognitively normal and the two very mildly demented subject also yielded some protein that have been identified previous biomarker study these result validate our method identifying difference proteomic profile csf sample and have important implication for the design csf biomarker study for and other central nervous system disorder',\n",
       " 'normal chinese liver proteome expression profile one the major part human liver proteome project before starting the study necessary examine the interindividual variation normal liver proteome and evaluate the minimal size sample for proteomic analysis this study normal liver sample from ten individual volunteer were collected and the proteome profile these sample were analyzed using difference gel electrophoresis dige combined with malditoftof the individual liver tissue lysates were labeled with cy3 and cy5 while the pooled sample wa labeled with cy2 internal standard which minimized geltogel variation after analysis the decyder software protein spot were detected the master gel the standardized abundance wa calculated for the protein spot that were matched across all ten gel the value these protein spot ranged from and the median wa approximately which demonstrated that the protein expression normal liver among different individual wa relatively stable the eight protein with value over were identified which would caveat when considering these protein potential diseaserelated marker moreover the oneway anova feature showed correlation between sample size and individual variation the result showed that when the sample size exceeded the individual variation were not significant the whole pool our result are important basis for liver protein expression profile and comparative proteomics liver disease',\n",
       " 'formalinfixed paraffinembedded ffpe tissue rich source clinically relevant material that can yield important translational biomarker discovery using proteomic analysis protocol for analyzing ffpe tissue lcmsms exist but standardization procedure and critical analysis data quality limited this study compared and characterized data obtained from ffpe tissue using two method urea insolution digestion method uisd versus commercially available qproteome ffpe tissue kit method qkit each method wa performed independently three time serial section homogenous ffpe tissue minimize preanalytical variation and analyzed with three technical replicates lcmsms data were evaluated for reproducibility and physiochemical distribution which highlighted difference the ability each method identify protein different molecular weight and isoelectric point each method replicate resulted significant number new protein identification and both method identified significantly more protein using three technical replicates compared only two uisd wa cheaper required le time and introduced significant protein modification compared the qkit method which provided more precise and higher protein yield these data highlight significant variability among method replicates and type method used despite minimizing preanalytical variability utilization only one method too few replicates both method and technical may limit the subset proteomic information obtained',\n",
       " 'dynaprot present advanced online database for dynamic access proteome and twodimensional gel the database wa designed administer complete silico proteome and link them with experimental proteomic data the manner electrophoresis gel ipgdalt the gel serve reference map gel analysis well tool for navigation the database switch between experimental and predicted data therefore all identified spot the gel are clickable and linked with summarized protein information the protein information table contain calculated characteristic which are often used proteomics such the molecular weight isoelectric point codon adaptation index grand average hydropathicity etc the design the database permit online extension gel data and protein attribute without knowledge any software language besides navigation via gel the clear graphical user interface permit quick and intuitive searching throughout complete proteome and support the search for protein with isoelectric point within range interest protein class ribosomal protein transporter the first organism implemented the database lactococcus lactis the database available wwwwzwtumdeproteomiklactis',\n",
       " 'staphylococcus aureus versatile grampositive pathogen that gain increasing importance due the rapid spreading resistance functional genomics technology can provide new insight into the adaptational network this bacterium and it response environmental challenge while functional genomics technology including proteomics have been extensively used study these phenomenon shake flask culture study bacteria from vivo setting lack behind particularly for proteomics study the major bottleneck the lack sufficient proteomic coverage for low number cell this study introduce workflow that combine pulsechase stable isotope labelling amino acid cell culture approach with high capacity cell sorting onmembrane digestion and highsensitivity detect and quantitatively monitor several hundred aureus protein from few million internalised bacteria this workflow ha been used proofofprinciple experiment reveal change level protein with function protection against oxidative damage and adaptation cell wall synthesis strain rn1hg upon internalisation human bronchial epithelial cell',\n",
       " 'embryonic stem cell escs and embryonic germ cell egcs provide exciting model for understanding the underlying mechanism that make cell pluripotent indeed such understanding would enable dedifferentiation and reprogrammation any cell type from patient needing cell therapy treatment proteome analysis ha emerged important technology for deciphering these biological process and thereby esc and egc proteome are increasingly studied nevertheless their nuclear proteome have only been poorly investigated now order investigate signaling pathway potentially involved pluripotency proteomic analysis have been performed mouse esc and egc nuclear protein nucleus from escs and egcs undifferentiated stage were purified subcellular fractionation after separation subtractive strategy subtracting culture environment contaminating spot wa applied and comparison esc day post coïtum dpcegc and dpcegc specific nuclear proteome wa performed total esc dpcegc and dpcegc spot were identified malditofms andor nanolcmsms this approach led the identification two isoforms with and without nterminal acetylation known pluripotency marker namely developmental pluripotency associated dppa5 which ha never been identified before gelms study escs and egcs furthermore demonstrated the efficiency our subtracting strategy association with nuclear subfractionation the identification new protein protein arginine nmethyltransferase prmt7 behaving protein involved pluripotency',\n",
       " '',\n",
       " 'study human cancer development cell culture model for malignant transformation can used hahn and coworkers introduced such model system and established herewith basis for research human tumorigenesis primary human fibroblast are sequentially transduced with defined genetic element htert and hrasv resulting four defined cell line whereby the last ha fully transformed phenotype order get deeper insight into the molecular biology human tumorigenesis compared the proteome these four cell line following multimethod concept the beginning assumed silac and sample fractionation with cofradic the method choice analyze the cell culture model for malignant transformation here the compared sample are combined before sample preparation thus avoiding difference sample preparation and using cofradic notably reduces sample complexity because 2dpage standard method for the separation and visualization closely related proteome decided analyze and compare the proteome these four cell line first approach differential 2dpage surprisingly discovered much more unique result with itraq and sample fractionation with scx than with the combination 2dpage and silaccofradic moreover itraq outperforms the other strategy not only number yielded result but also analysis time here present the itraq quantification result and compare them with the result 2dpage and silaccofradic found change the protein level each transition thereby ha the strongest impact the proteome detail identified regulated protein beside others these protein are involved cytoskeleton rna processing and cell cycle such cdc2 hnrnps snrnps collagen and mcm protein for example mcm protein are upregulated and collagen are downregulated due expression furthermore made the observation that protein containing the same domain have analogous regulation profile during malignant transformation for instance several protein containing lim domain are downregulated moreover this study and the defined cell culture model change could clearly matched specific step during tumorigenesis',\n",
       " '',\n",
       " 'goal the chromosomecentric human proteome project identify all human protein specie with protein annotated missing this challenging moreover proteolytic processing generates new protein specie with characteristic neon terminus that are frequently accompanied altered halflives function interaction and location enucleated and largely void internal membrane and organelle erythrocyte are simple yet proteomically challenging cell due the high hemoglobin content and wide dynamic range protein concentration that impedes protein identification using the nterminomics procedure tail identified human erythrocyte natural and neontermini and protein multiple semitryptic nterminal peptide exhibited improved mass spectrometric identification property versus the intact tryptic peptide enabling identification novel erythrocyte protein and six missing protein identified for the first time the human proteome with improved bioinformatics workflow developed new classification system and the terminus cluster score thereby described new stabilizing nend rule for processed protein terminus which discriminates novel protein specie from degradation remnant and identified protein domain hot spot susceptible cleavage strikingly the ntermini were within genomeencoded protein sequence revealing alternative translation initiation site pervasive endoproteolytic processing and stabilization protein fragment vivo the mass spectrometry proteomics data have been deposited proteomexchange with the data set identifier ltpxdgt',\n",
       " 'the butyrateproducing anaerobe fusobacterium varium integral constituent human gut microflora unlike many gut microorganism varium capable fermenting both amino acid and glucose although varium ha been implicated beneficial well pathological bacteriumhost interaction it genome ha not been sequenced obtain better understanding the metabolic process associated with amino acid fermentation varium used gelbased proteomic approach examine the change the soluble proteome accompanying the utilization eight different growth substrate glucose and dglutamate lhistidine and dlysine and and dserine using lcmsms analyze approximately the detected protein spot were able identify distinct protein while the intracellular concentration enzyme characteristic catabolic pathway for specific amino acid were selectively increased response the presence excess that amino acid the growth medium the concentration the core acetatebutyrate pathway enzyme remained relatively constant our analysis revealed high intracellular concentration glutamate mutase and betamethylaspartate ammonialyase under all growth condition underscoring the importance the methylaspartate pathway glutamate catabolism varium the presence two enzyme the hydroxyglutarate pathway glutamate degradation the proteome varium r2hydroxyglutarylcoa dehydratase and nadspecific glutamate dehydrogenase specifically when lglutamate wa the main energy source iii the presence gene the genome varium encoding each the enzyme the hydroxyglutarate pathway the presence both and dserine ammonialyases dehydratases which permit varium thrive either dserine respectively and the presence aspartatesemialdehyde dehydrogenase and dihydrodipicolinate synthase consistent with the ability varium synthesize mesodiaminopimelic acid component it peptidoglycan protein involved other cellular process including oxidationreduction reaction protein synthesis and turnover and transport were also identified',\n",
       " 'since the publication the human genome two key point have emerged first still not certain which region the genome code for protein second the number discrete proteincoding gene far fewer than the number different protein proteomics ha the potential address some these postgenomic issue the obstacle that face can overcome our effort combine proteomic and genomic data there are many challenge associated with highthroughput and highoutput proteomic technology consequently for proteomics continue it current growth rate new approach must developed ease data management and data mining initiative have been launched develop standard data format for exchanging mass spectrometry proteomic data including the proteomics standard initiative formed the human proteome organization database such swissprot and uniprot are publicly available repository for protein sequence annotated for function subcellular location and known potential posttranslational modification the availability bioinformatics solution crucial for proteomics technology fulfil their promise adding further definition the functional output the human genome the aim the oxford genome anatomy project provide framework for integrating molecular cellular phenotypic and clinical information with experimental genetic and proteomics data this perspective also discus model make the oxford genome anatomy project accessible and beneficial for academic and commercial research and development',\n",
       " 'applied proteomic technology analyze the human fetal brain such analysis could provide with important information the development the early neuronal life healthy and diseased state the protein from the cerebellum control subject were analyzed twodimensional electrophoresis and identified matrixassisted laser desorptionionizationmass spectrometry the basis peptide mass fingerprinting following ingel digestion with trypsin approximately spot excised from three twodimensional gel were analyzed which resulted the identification about protein that were the product different gene about half them are enzyme subunit and are mainly localized the cytosol and mitochondrion the most frequently identified protein the various gel were heat shock protein housekeeping enzyme such atp synthase chain protein disulfide isomerase and structural protein such tubulin chain seven gene product were identified for the first time the fetal brain the other protein had also been detected other human sample which were analyzed our laboratory most protein were represented multiple spot average about spot were detected per gene product the fetal brain database includes protein with important function and also with unknown function and represents today one the largest twodimensional database for higher eukaryotic proteome may useful tool the investigation protein change neurodegenerative disease early life',\n",
       " 'cell treated with ultraviolet radiation undergo cell cycle arrest the sphase and g1s boundary allowing dna repair occur several protein such replication protein and dnadependent protein kinase have been suggested involved uvinduced inhibition dna replication however the role these protein inhibiting dna replication remains unknown other protein may play important role modulating function these protein response uvirradiation understand the broad range protein involved this inhibition carried out systematic study identify specific protein involved uvinduced replication arrest using twodimensional gel electrophoresis and mass spectrometry unique change protein expression level for protein were observed over hour time course including calgizzarin cyclophilin and macrophage migration inhibitory factor the expression level change these protein are dynamically correlated dna replication activity suggesting involvement these protein modulating dna replication and repair activity this proteomic approach provides opportunity gain insight into the mechanism which dna replication inhibited',\n",
       " 'although only few year old the combination linear ion trap with orbitrap analyzer ha become one the standard mass spectrometer characterize protein and proteome here describe novel version this instrument family the orbitrap elite which improved three main area the ion transfer optic ha ion path that block the line sight achieve more robust operation the tandem acquisition speed the dual cell linear ion trap now exceeds most importantly the resolving power the orbitrap analyzer ha been increased twofold for the same transient length employing compact highfield orbitrap analyzer that almost double the observed frequency enhanced fourier transform algorithmincorporating phase informationfurther double the resolving power for transient for topdown experiment combine survey scan with selected ion monitoring scan the charge state the protein fragmented and with several hcd microscans despite the resolving power for sim and hcd scan the total cycle time within several second and therefore suitable for liquid chromatography tandem for bottomup proteomics combined survey scan resolving power with datadependent collisioninduced dissociation the most abundant precursor total cycle time sincreasing protein identification complex mixture about the speed the orbitrap elite furthermore allows scan mode which complementary dissociation mechanism are routinely obtained all fragmented peptide',\n",
       " 'the complexity human plasma present number challenge the efficient and reproducible proteomic analysis differential expression response disease before individual variation and diseasespecific protein biomarkers can identified from human plasma the experimental variability inherent the protein separation and detection technique must quantified report the variation found twodimensional difference gel electrophoresis dige analysis human plasma eight aliquot human plasma sample were subjected top6 highest abundant protein depletion and were subsequently analyzed triplicate for total dige sample gel spotwise standard deviation estimate indicated that fold change greater than can detected with manageable number replicates simple anova experiment with human plasma mixedeffects statistical modeling quantified the effect the dye and segregated the spotwise variance into component sample preparation geltogel difference and random error the geltogel component wa found the largest source variation followed the sample preparation step improved protocol for the depletion the top6 highabundance protein suggested which along with the use statistical modeling and future improvement gel quality and image processing can further reduce the variation and increase the efficiency dige proteomic analysis human plasma',\n",
       " 'the industrial production cooked ham from pork meat involves initial step the injection brine and prolonged meat massage these process strongly affect the quality the final product because they determine the breakage muscle cell and the release their protein content the produced dense exudate act glue the final cooked ham order exploit modern tool direct the technological process still mainly based empirical observation and traditional recipe have carried out comprehensive proteomic analysis the exudate function brine concentration temperature and length meat massage each condition wa found generate specific protein pattern peptide mass fingerprinting analysis wa applied allowing the identification protein whose presence andor quantity can defined biomarkers meat processing and potentially final product quality',\n",
       " 'labelfree quantification lfq based dataindependent acquisition workflow currently experience increasing popularity several software tool have been recently published are commercially available the present study focus the evaluation three different software package progenesis synapter and isoquant supporting ion mobility enhanced dataindependent acquisition data order benchmark the lfq performance the different tool generated two hybrid proteome sample defined quantitative composition containing tryptically digested proteome three different specie mouse yeast escherichia coli this model dataset simulates complex biological sample containing large number both unregulated background protein well and downregulated protein with exactly known ratio between sample determined the number and dynamic range quantifiable protein and analyzed the influence applied algorithm retention time alignment clustering normalization etc quantification result analysis technical reproducibility revealed median coefficient variation reported protein abundance below for mse data for progenesis and isoquant regarding accuracy lfq evaluation with synapter and isoquant yielded superior result compared progenesis addition discus reporting format and user friendliness the software package the data generated this study have been deposited the proteomexchange consortium with identifier pxd httpproteomecentralproteomexchangeorgdatasetpxd',\n",
       " 'diverse mass spectrometric instrument have been used provide data for accurate mass and retention time amt tag proteomics analysis including ion trap quadrupole timeofflight and fourier transform mass spectrometry ftms important attribute these instrument beside mass accuracy their spectral resolution fact the ability separate peak with close value likely play major role enabling specie identification and matching analysis very complex proteomics sample ftms resolution directly proportional the detection period and can therefore easily tuned took advantage this feature investigate the effect resolution specie identification and matching amt tag experiment using arabidopsis thaliana chloroplast protein extract prototypical reallife sample have compared the number detected feature the optimal mass tolerance for specie matching the number matched specie and the false discovery rate obtained various resolution setting appears that while the total number match not significantly affected reduction resolution the range investigated the confidence level identification significantly drop evidenced the estimated false discovery rate',\n",
       " '',\n",
       " 'successful strategy for the identification shell protein based proteomic analysis where soluble and insoluble fraction isolated from organic shell matrix are digested with trypsin with the aim generating peptide which are used identify novel shell protein contained database however using trypsin sole degradative agent limited the enzyme cleavage specificity and dependent upon the occurrence lysine and arginine the shell protein sequence bypass this limitation investigated the ability trifluoroacetic acid tfa lowspecificity chemical degradative agent generate cluster analyzable peptide from organic shell matrix suitable for database annotation acetic acidinsoluble fraction from haliotis tuberculata shell were processed trypsin followed tfa digestion the hydrolysate were used annotate expressed sequence tag library constructed from the mantle tissue haliotis asinina tropical abalone specie the characterization sequence with repeat motif featured some the shell matrix protein benefited from tfainduced serial cutting which can result peptide ladder series using the degradative specificity tfa and trypsin were able identify five novel shell protein this pilot study indicates that mild chemical digestion organic shell matrix combined with trypsin generates peptide suitable for proteomic analysis for better characterization mollusc shell matrix protein',\n",
       " '',\n",
       " 'multidimensional chromatography coupled tandem mass spectrometry emerging technique for the analysis proteome and rapidly being implemented many researcher for proteomic analysis this technology profile particular proteomic approach known multidimensional protein identification technology mudpit discussed mudpit biphasic microcapillary column packed with highperformance liquid chromatography grade reversed phase and strong cation exchange packing material loaded with complex peptide mixture and placed line with quaternary highperformance liquid chromatography and tandem mass spectrometer mudpit ha the capability analyze highly complex proteomic mixture such whole proteome organelle and protein complex',\n",
       " '',\n",
       " 'the plasma membrane control cell exchange both material and information with the outside environment and pmassociated protein play key role cellular regulation numerous cell surface receptor allow cell perceive and respond various signal from neighbor cell pathogen the environment large number transporter and channel protein control material uptake release quantitative proteomic analysis pmassociated protein can identify key protein involved signal transduction and cellular regulation here describe protocol for quantitative proteomic analysis protein using twodimensional difference gel electrophoresis the protocol ha been successfully employed identify new component the brassinosteroid signaling pathway and should also applicable the study other plant signal transduction pathway and regulatory mechanism',\n",
       " 'the hupo brain proteome project hupo bpp held it workshop sao paulo brazil may and the focus wa the progress the human brain proteome atlas well idea strategy and methodological aspect clinical neuroproteomics',\n",
       " 'study complex proteome brings forward higher request for the quantification method using mass spectrometry technology this paper present mass spectrometry labelfree quantification tool for complex proteome called freequant which integrated quantification with functional analysis effectively freequant consists two wellintegrated module labelfree quantification and functional analysis with biomedical knowledge freequant support labelfree quantitative analysis which make full use tandem mass spectrometry msms spectral count protein sequence length shared peptide and ion intensity adopts spectral count for quantitative analysis and build new method for shared peptide accurately evaluate abundance isoforms for protein with low abundance msms total ion count coupled with spectral count included ensure accurate protein quantification furthermore freequant support the largescale functional annotation for complex proteome mitochondrial proteome from the mouse heart the mouse liver and the human heart were used evaluate the usability and performance freequant the evaluation showed that the quantitative algorithm implemented freequant can improve accuracy quantification with better dynamic range',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'instrument parameter value for quadrupole orbitrap mass spectrometer were optimized for performing global proteomic analysis fourteen factor were evaluated for their influence datadependent acquisition with emphasis both the rate sequencing and spectral quality maximizing two individually tested response variable unique peptide and protein group the factor factor were assigned significant contrast value plt for both response variable fundamentally when optimizing parameter balance between spectral quality and duty cycle need reached order maximize proteome coverage this especially true when using datadependent approach for sequencing complex proteome for example maximum ion injection time automatic gain control setting and minimum threshold setting for triggering msms isolation and activation all heavily influence ion signal the number spectrum collected and spectral quality better ass the effect these parameter have data acquisition all msms data were parsed according ion abundance calculating the percent the agc target reached for each msms event and then compared with successful peptidespectrum match this proved effective approach for understanding the effect ion abundance successful peptidespectrum match and establishing minimum ion abundance threshold for triggering msms isolation and activation',\n",
       " '',\n",
       " 'most proteomic analysis require prefractionation and protein purification strategy achieve maximal proteome coverage especially plant which cell often have few highly abundant protein and substance like polyphenols secondary metabolite that can have significant impact proteome coverage several method have been developed reduce cellular complexity and increase protein dynamic range one approach the display the plant cell proteome single twodimensional gel other approach use fractionation strategy reduce sample complexity subset functionally related protein pathway module here describe strategy separate the proteome purified cell organelle using twodimensional gel electrophoresis 2de the proteome plant chloroplast and nonphotosynthetic plastid wa further fractionated differential protein solubilization method that fully compatible with 2de the final protein complement individual fraction comprised approximately different protein specie that can fully resolved and visualized single 2de gel',\n",
       " 'the gonad sea urchin are high value seafood product with considerable research being undertaken worldwide the development sea urchin aquaculture the best price are obtained for specific gonad attribute research ha also focused the development artificial diet that enhance gonad quality and quantity total protein ha been used measure gonad quality yet study date have applied proteomics technology diet development here use mudpit and 2de approach describe the major protein mature ovary new zealand sea urchin evechinus chloroticus this tissue which target seafood product contained protein that were identified from the recently completed sea urchin genome strongylocentrotus purpuratus with high confidence the majority these protein had general function with only related ovarian reproductive function eighteen protein were located the 2de four these were directly identified from purpuratus protein sequence combination this paper show that the genome resource purpuratus can used identify protein sea urchin from different family describes the proteome chloroticus mature ovary and provides proteomic tool for analysis gonad from other edible sea urchin',\n",
       " 'here present the swissczech proteomics server swicz which host the proteomic database summarizing information about the cell cycle the aquatic bacterium caulobacter crescentus the database provides searchable tool for easy access global protein synthesis and protein stability data examined during the crescentus cell cycle protein synthesis data collected from five different cell cycle stage were determined for each protein spot relative value the total amount smethionine incorporation protein stability pulselabeled extract were measured during chase period equivalent one cell cycle unit quantitative information for individual protein together with descriptive data such protein identity apparent molecular mass and isoelectric point were combined with information protein function genomic context and the cell cycle stage and were then assembled relational database with world wide web interface httpproteombiomedcascz which allows the database record searched and display the recovered information total protein spot were reproducibly detected twodimensional gel electropherograms which were identified mass spectroscopy the database accessible either through clickable twodimensional gel electrophoretic map mean set dedicated search engine basic characterization the experimental procedure data processing and comprehensive description the web site are presented it current state the swicz proteome database provides platform for the incorporation new data emerging from extended functional study the crescentus proteome',\n",
       " 'one the major challenge for large scale proteomics research the quality evaluation result protein identification from complex biological sample experimental setup often manual and subjective task which lack profound statistical evaluation this not feasible for highthroughput proteomic experiment which result large datasets thousand peptide and protein and their corresponding mass spectrum improve the quality reliability and comparability scientific result estimation the rate erroneously identified protein advisable moreover scientific journal increasingly stipulate that article containing considerable data should subject stringent statistical evaluation present newly developed easytouse software tool enabling quality evaluation generating composite targetdecoy database usable with all relevant protein search engine this tool when used conjunction with relevant statistical quality criterion enables reliably determine peptide and protein high quality even for nonexperienced user laboratory staff researcher without programming knowledge different strategy for building decoy database are implemented and the resulting database are characterized and compared the quality protein identification highthroughput proteomics usually measured the false positive rate fpr but shown that the false discovery rate fdr delivers more meaningful robust and comparable value',\n",
       " 'glycoconjugates play critical role regulating cellular and organismal function consequently defining the relative level these glycoconjugates under varied physiological condition important thus identifying and understanding the regulation the enzyme that process these glycoconjugates are essential step understanding the role this glycocode development and disease activitybased affinity reagent are useful tool for probing these enzyme and should facilitate the unraveling proteome one advantage activitybased affinity probe that they can simultaneously reveal multiple enzyme having similar activity these probe can also used enrich proteome interest thereby facilitating identification and cloning new carbohydrateprocessing enzyme here review the current state activitybased affinity probe for profiling carbohydrateprocessing enzyme focusing success and limitation general design feature and specific example describing profiling exoglycosidases from cell lysates',\n",
       " 'the available escherichia coli genome sequence offer opportunity further expand our understanding this bacterium the current study present rapid method for the isolation bacterial alkaline protein using acid incubation purification and protein array 2de followed protein identification using fiftyseven protein were randomly chosen which were identified database searching data the searching result showed that most these alkaline protein were involved special function within the cell suggesting that alkaline proteome ideal fraction for understanding their special function furthermore alkaline proteome were compared between the period majority live bacteria culture the period similar amount live and dead bacteria culture and the period majority dead bacteria culture six protein were identified differentially expressed target which putative transcriptional regulator and superoxide dismutase gene were cloned and expressed for antiserum preparation the antiserum were applied for the confirmation result obtained from 2de the presented data clearly reveal that alkaline proteome analysis 2de with play important role the understanding protein function within the cell and six alkaline protein are determined key one overgrowthmediated growth cycle coli',\n",
       " 'the first proteomic map database china will published bioinformation center and research center proteomics shanghai institute for biological science the chinese academy sciencesthe database consists physical layer link layer and interfacial layer java technique wa introduced construct the database system the database doe not depend special working platform with the search instrument user can easily look through the proteomic map and get concrete information protein they are interested',\n",
       " 'the hupo brain proteome project hupo bpp held it workshop boston usa september during the hupo annual word congress the focus wa the progress the human brain proteome atlas well idea strategy and methodological aspect',\n",
       " 'experimentally derived genomewide protein interaction network have been useful the elucidation functional information that not evident from examining individual protein but determination these network complex and time consuming address this problem several computational method for predicting protein network novel genome have been developed recent publication date and marcotte describes the use phylogenetic profiling for elucidating novel pathway proteome that have not been experimentally characterized this method combination with other computational method for generating proteininteraction network might help identify novel functional pathway and enhance functional annotation individual protein',\n",
       " '',\n",
       " 'reference proteome are generated increasingly sophisticated annotation pipeline part regular genome build release yet the corresponding change reference proteome content are dramatic the history the ncbicurated human proteome the total number entry ha remained roughly constant but approximately half the protein from the build are longer represented entry current release while about the same number new protein have been added for sequence identity threshold although mostly hypothetical protein are affected there are also spectacular case entry removaladdition well studied protein the change between the and recent human proteome are similar order magnitude the difference between recent human and chimpanzee proteome release application example show that the proteome fluctuation affect the interpretation about hit organellespecific massspectrometry data although proteome quality tends improve with more recent release for example the fraction protein with functional annotation ha increased over time existing evidence implies that apparently the proteome content still remains incomplete not just pertaining isoformssequence variant but also protein and their family that are clearly distinct',\n",
       " 'the human proteome organisation wa launched february result the need international proteomic forum improve the understanding human disease the initiative dealing with the brain the human proteome organisation brain proteome project chaired germany and korea order estimate the existing approach brain proteomics well establish standardized data reprocessing pipeline pilot study were initiated including both mouse and human sample data had submitted data collection center for central reprocessing and are now publicly accessible the pride database serving reference data for future analysis became clear that heterogeneity for example different analysis strategy and data format are real challenge when comparing result and when working consortium therefore standardization the organisation data management and the synergistic effect consortium collaborator are outstanding importance any big proteome analysis the following manuscript will highlight these activity and aim the human proteome organisation brain proteome project summarizing it historical timeline and it two pilot study',\n",
       " 'the statistical classification strategy have developed for magnetic resonance infrared and raman spectrum for the analysis biomedical data discussed particularly applies proteomic mass spectrum general discussion the current use pattern recognition method given with caveat and suggestion relevant for clinical applicability',\n",
       " 'proteomic investigation number different separation technique can applied fractionate whole cell proteome into more manageable fraction for subsequent analysis this work utilizing hplc and mass spectrometry for protein identification two different fractionation method were compared and contrasted determine the potential each method for the simple and reproducible fractionation bacterial proteome columnbased chromatofocusing and liquidbased isoelectric focusing both utilized gradient produce similar result term the number protein successfully identified and protein and the consistency protein identified from one experiment the next change however there wa limited overlap the protein set with the protein identified common between the set protein identified the different system addition the number protein identified and consistency those identified the reduced monetary cost experimentation and increased assay flexibility produced using isoelectric focusing wa considered order adopt system best suited for comparative proteomic project',\n",
       " 'hydrophobic membrane protein mp occupy unique niche the brain proteome research due their important physiological role therefore the extraction separation and identification mp are great interest proteomic analysis applied various proteomic technique enrich separate and analyze the human brain proteome including membrane proteome temperatureinduced phase fractionation with the nonionic surfactant triton wa used simultaneously extract separate and concentrate low abundant hydrophobic and high abundant hydrophilic protein from human brain tissue the extracted and delipidated protein were analyzed twodimensional gel electrophoresis 2de approximately spot were detected the gel insolution digestion wa performed kda spin filter tryptic peptide were separated using nanolc and analyzed using two different high performance mass spectrometer linear ion trapfourier transform and linear ion traporbitrap reveal the low abundant mp total and unique protein were identified using linear ion trapfourier transform and linear ion traporbitrap mass spectrometer respectively more than the identified protein were classified mp with significant biological function such ion channel and transporter our study establishes simple and rapid shotgun approach for the characterization the brain proteome and allows comprehensive analysis brain membrane proteome',\n",
       " 'visualize birdseye view ensemble proteome for various specie recently developed novel method mapping proteome ensemble into threedimensional vector space this study the proteome defined the entire set all protein encoded genome sequence and these protein were dealt with the level the scop fold first represented the proteome specie dimensional vector where it length represents the overall amount all the scop fold the proteome and it unit vector xsxs represents the relative composition the scop fold the proteome and the size the dimension the number all possible fold the proteome ensemble given second mapped the vector the vector based the two simple principle ysxs and the angle between and maximally correlate with the angle between and applied the mapping proteome ensemble for specie which were retrieved from the genome protein structure and function gtop database result succeeded the mapping that the property the dimensional vector were quantitatively conserved the vector particularly the angle between vector before and after the mapping highly correlated with each other correlation coefficient were this new mapping method will allow researcher intuitively interpret the visual information presented the map highly effective manner',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'chinese hamster ovary cho cell are the most commonly used mammalian host cell line for biopharmaceutical production because their ability correctly fold and posttranslationally modify recombinant protein that are compatible with human use proteomics along with other omic platform are being used understand the biology cho cell with the ultimate aim enhancing cho cell factory for more efficient production biopharmaceuticals this chapter will describe efficient protocol called filter aided sample preparation fasp for the extraction protein from cho cell for proteomic study fasp us common ultrafiltration device whereby the membrane pore are small enough allow contaminating detergent pas through while protein are too large and are retained and concentrated the filter unit this method sample preparation and protein digestion universally applicable and can easily employed any proteomics facility standard everyday laboratory reagent and equipment are used',\n",
       " 'consider the problem the construction invariant for characterization map such proteome map nmr spectral map etc that addition facilitating cataloguing such map can used for comparison map and numerical evaluation their degree similarity novel approach based the concept that the nearest neighborhood point spot map are sufficiently flexible allow one not only vary the number point used for characterization the map but also the density information their relative position put forward the method illustrated with the coomassie brilliant blue stained gel electrophoresis pattern the proteome from liver cell healthy male fisher rat and the rat treated with four peroxisome proliferators',\n",
       " 'hereby provide twoyear update the hupo human plasma proteome project hppp informed advance presented the hppp session the hupo world congress toronto september and sydney september',\n",
       " '',\n",
       " 'geoduck clam panopea generosa are increasingly important fishery and aquaculture product along the eastern pacific coast from baja california mexico alaska these longlived clam are highly fecund although sustainable hatchery production genetically diverse larva hindered the lack sexual dimorphism resulting asynchronous spawning broodstock unequal sex ratio and low number breeder the development assay gonad physiology could indicate sex and maturation stage well used ass the status natural population proteomic profile were determined for three reproductive maturation stage both male and female clam using datadependent acquisition dda gonad protein gonad proteome became increasingly divergent between male and female maturation progressed the dda data were used develop target analyzed with selected reaction monitoring srm gonad tissue well hemolymph the srm assay yielded suite indicator peptide that can used efficient assay determine geoduck gonad maturation status application srm hemolymph sample demonstrates that this procedure could effectively used ass reproductive status marine mollusk nonlethal manner',\n",
       " 'continuity with the review dealing with difference gender nonsexual organ this review collect data the proteome the sexual organ involved human reproduction under both physiological and pathological condition also collect data the tissue structure and biological fluid typical pregnancy such placenta and amniotic fluid well what may tested preimplantation embryo during medically assisted reproduction the review includes well mention all fluid and secretion connected with sex organ andor reproduction including sperm and milk exemplify two distinctive item male and female physiology',\n",
       " '',\n",
       " 'subcellular organization yielding largescale analysis researcher are now applying robust massspectrometrybased proteomics method obtain inventory biochemically isolated organelle that contain hundred protein highresolution method allow accurate protein identification and novel algorithm can distinguish genuine from copurifying component organellar proteome have been analysed bioinformatic method and integrated with other largescale data set the dynamic organelle can also studied quantitative proteomics which offer powerful method that are complementary fluorescencebased microscopy here review the emerging trend this rapidly expanding area and discus the role organellar proteomics the context functional genomics and system biology',\n",
       " 'the elucidation the subcellular distribution protein under different condition major challenge cell biology this challenge further complicated the multicompartmental and dynamic nature protein localization address this issue quantitative proteomics workflow have been developed reliably identify the protein complement whole organelle well for protein assignment subcellular location and relative protein quantification based different cell culture condition here review quantitative msbased approach that combine cellular fractionation with proteomic analysis the application these method the characterization organellar composition and the determination the dynamic nature protein complex improving our understanding protein function and dynamic',\n",
       " 'the deep proteome ha been accessible mass spectrometry for some time however the number protein identified cell the same type ha plateaued without transfer from reference proteomesdata moreover limited sequence coverage hamper the discrimination protein isoforms when using trypsin standard protease multienzyme approach appear improve sequence coverage and subsequent isoform discrimination here expanded proteome and protein sequence coverage mcf7 breast cancer cell yet unmatched depth employing workflow that address current limitation deep proteome analysis multiple stage used gelaided sample preparation gasp and combined trypsinelastase digest increase peptide orthogonality concatenated highph prefractionation and iii charge ordered parallel ion analysis chopin available orbitrap fusion lumos mass spectrometer achieve median protein sequence coverage protein group unigene id single cell line chopin allows the use both detector the orbitrap predefined precursor type that optimizes parallel ion processing leading the identification total unique peptide covering the deep proteome unprecedented detail',\n",
       " 'number factor including low stoichiometry phosphorylation ion suppression and reduced peptide backbone fragmentation interfere with precise identification protein phosphoproteomic analysis therefore enrichment phosphopeptides important process for subsequent mass spectrometric analysis here have developed simple and efficient method for phosphopeptides enrichment which employ biphasic phosphatebinding tag phostagc tip consisting overlaid phostag the resin pipet tip the improvement selectivity for phosphopeptides wa achieved using acn solution under the phosphopeptides binding condition also assessed the adequacy phostagc tip for quantitative phosphoproteomic analysis using the itraq technology after protein digestion and subsequent itraq labeling interfering substance including excess itraq reagent were directly removed phostagc tip single step applying this method phosphoproteomic analysis hela cell stimulated with tumor necrosis factor alpha wa rapidly and successfully achieved',\n",
       " '',\n",
       " 'maxquant proteomic software widely used for largescale tandem mass spectrometry data have designed and developed enhanced result reporting tool for maxquant named maxreport this tool can optimize the result maxquant and provide additional function for result interpretation maxreport can generate report table for protein nterminal modification also support isobaric labelling based relative quantification the protein peptide site level obtain overview the result maxreport performs general descriptive statistical analysis for both identification and quantification result the output result maxreport are well organized and therefore helpful for proteomic user better understand and share their data the script maxreport which freely available httpwebsdoornetbioinfomaxreport developed using python code and compatible across multiple system including window and linux',\n",
       " 'mass spectrometry msbased proteomics moving beyond the simple generation protein inventory biological sample the ability quantitatively probe complex protein mixture increasingly being used study protein structural and biophysical property proteomescale provides readout for proteomewide structural alteration folding and stability aggregation and molecular interaction all nativelike condition such cell lysates even intact cell provide overview method that yield such proteomewide structural information covering crosslinkingms limited proteolysisms cofractionationms hydroxyl radical footprintingms thermal proteome profiling and numerous approach for monitoring molecular interaction large scale method determine structural property the native proteome will drive structural system biology',\n",
       " 'there immediate need for improved method systematically and precisely quantify large set peptide complex biological sample date protein quantification biological sample ha been routinely performed triple quadrupole instrument operated selected reaction monitoring mode srm and two major challenge remain firstly the number peptide included one survey experiment need increased routinely reach several hundred and secondly the degree selectivity should improved reliably discriminate the targeted analytes from background interference high resolution and accurate mass hram analysis the recently developed qexactive mass spectrometer can potentially address these issue this instrument present unique configuration constituted orbitrap mass analyzer equipped with quadrupole mass filter the frontend for precursor ion mass selection this configuration enables new quantitative method based hram measurement including targeted analysis mode single ion monitoring and msms mode parallel reaction monitoring the ability the quadrupole select restricted range allows one overcome the dynamic range limitation associated with trapping device and the msms mode provides additional stage selectivity when applied targeted protein quantification urine sample and benchmarked with the reference srm technique the quadrupoleorbitrap instrument exhibit similar better performance term selectivity dynamic range and sensitivity this high performance further enhanced leveraging the multiplexing capability the instrument design novel acquisition method and apply them large targeted proteomic study for the first time demonstrated tryptic yeast peptide analyzed one min experiment the increased quality quadrupoleorbitrap data ha the potential improve existing protein quantification method complex sample and address the pressing demand system biology biomarker evaluation study',\n",
       " 'proteomics refers group analytical technique for high throughput protein analysis providing evidence for protein expression level subcellular localization posttranslational modification and molecular interaction such proteomics ha contributed largely our knowledge regarding molecular mechanism underlying health and disease and pinpointed potential disease biomarkers the scope this review briefly introduce the principle major proteomics technique employed biological research including novel quantitative and molecular imaging mass spectrometrybased platform few example from the application these technique biomarker discovery for kidney disease are also provided',\n",
       " 'oxidatively modified protein are characterized elevation proteinresident carbonyl 3nitrotyrosine measure protein oxidation protein bound reactive alkenals such 4hydroxy2nonenal measure lipid peroxidation oxidatively modified protein nearly always have altered structure and function redox proteomics that branch proteomics used identify oxidized protein and determine the extent and location oxidative modification the proteome interest this technique nearly always employ mass spectrometry the major platform achieve the goal identifying the target protein once identified oxidatively modified protein can placed specific molecular pathway provide insight into protein oxidation and human disease both original research and review article are included this forum redox proteomics the topic related redox proteomics range from basic chemistry sulfur radicalinduced redox modification protein the thiol secretome and inflammatory network reversible thiol oxidation proteome the role glutamine synthetase peripheral and central environment inflammation and insulin resistance bioanalytical aspect tyrosine nitrated protein protein oxidation human smoker and model thereof and alzheimer disease including article the brain ubiquitinylome and the triangle death composed oxidatively modified protein involved energy metabolism mammalian target rampamycin activation and the proteostasis network this forum redox proteomics both timely and critically important resource highlight one the key tool needed better understand protein structure and function oxidative environment health and disease antioxid redox signal',\n",
       " 'formaldehydefixed paraffinembedded ffpe tissue repository represent valuable resource for the retrospective study disease progression and response therapy however the proteomic analysis ffpe tissue ha been hampered formaldehydeinduced protein modification which reduce protein extraction efficiency and may lead protein misidentification here demonstrate the use heat augmented with high hydrostatic pressure psi novel method for the recovery intact protein from ffpe mouse liver when ffpe mouse liver wa extracted using heat and elevated pressure there wa 4fold increase protein extraction efficiency 3fold increase the extraction intact protein and fold increase the number nonredundant protein identified mass spectrometry compared matched tissue extracted with heat alone more importantly the number nonredundant protein identified the ffpe tissue wa nearly identical that matched freshfrozen tissue',\n",
       " '',\n",
       " 'quantitative proteomics employing isobaric reagent ha been established powerful tool for biological discovery current workflow often utilize dedicated quantitative spectrum improve quantitative accuracy and precision consequence this approach dramatic reduction the spectral acquisition rate which necessitates the use additional instrument time achieve comprehensive proteomic depth this work ass the performance and benefit online and realtime spectral identification quantitative multiplexed workflow realtime search rts algorithm wa implemented identify fragment spectrum within millisecond they are acquired using probabilistic score and trigger quantitative spectrum only upon confident peptide identification the rtsmssup3sup wa benchmarked against standard workflow using complex twoproteome model interference and targeted plex comparison kinase abundance profile applying the rtsmssup3sup method provided the comprehensive characterization plex proteome le acquisition time these data indicate that the rtsmssup3sup approach provides dramatic performance improvement for quantitative multiplexed experiment',\n",
       " 'over the past decade suite new massspectrometrybased proteomics method ha been developed that now enables the conformational property protein and proteinligand complex studied complex biological mixture from cell lysates intact cell highlighted here are seven the technique this new toolbox these technique include chemical crosslinking xlms hydroxyl radical footprinting hrf drug affinity responsive target stability dart limited proteolysis lip pulse proteolysis stability protein from rate oxidation sprox and thermal proteome profiling tpp the above technique all rely conventional bottomup proteomics strategy for peptide sequencing and protein identification however they have required the development unconventional proteomic data analysis strategy discussed here are the current technical challenge associated with these different data analysis strategy well the relative analytical capability the different technique the new biophysical capability that the above technique bring bear proteomic research are also highlighted the context several different application area which these technique have been used including the study protein ligand binding interaction protein target discovery study and protein interaction network analysis and the characterization biological state',\n",
       " 'conventional topn datadependent acquisition dda lcmsms analysis identifies only limited fraction all detectable precursor because the ionsampling rate contemporary mass spectrometer insufficient target each precursor complex sample topn dda preferentially target highabundance precursor with limited sampling lowabundance precursor and repeated analysis only marginally improve sample coverage due redundant precursor sampling this work advanced precursor ion selection algorithm were developed and applied the bottomup analysis hela cell lysate overcome the above deficiency precursor fragmented previous run were efficiently excluded using automatically aligned exclusion list which reduced overlap identified peptide between replicates exclusion previously fragmented highabundance peptide allowed deeper probing the hela proteome over replicate lcm run resulting the identification more peptide beyond the saturation level achievable using conventional topn dda the gain peptide identification using the developed approach translated the identification several hundred lowabundance protein group which were not detected conventional topn dda exclusion only identified peptide compared with the exclusion all previously fragmented precursor resulted increase additional peptide identification over four run suggesting the potential for further improvement the depth proteomic profiling using advanced precursor ion selection algorithm',\n",
       " '',\n",
       " 'hematuria common sign many renal and urologic pathologic condition and may affect the proteomic analysis urinary extracellular vesicle uev nanovesicles released from all cell contact with the urinary space this condition hinders uev based proteomic study aiming discover biomarkers therefore studied the effect hematuria the proteome uev and introduced possible solution reduce it misleading impact mimicked hematuria adding increasing amount blood nonaffected urine and investigated it effect uev isolation purity and proteomic composition proposed trypsin treatment able reduce the impact hematuria uev the effect the treatment were investigated evaluating the uev proteomic profile the enrichment known uev marker and assessing differential protein content msbased labelfree quantification result showed that the blood contamination increased affected both the proteome profile and the yield uev isolated from urine our treatment with trypsin wa able counteract completely these effect for lowmedium level hematuria which are most commonly encountered this promising finding could lead the reliable use hematuria sample for uev proteomic investigation',\n",
       " 'despite intense global effort new clinical andor viable biomarkers have been established overcome the limitation the prostate specific antigen the early diagnosis and prognosis prostate cancer pca the current proteomic approach pca biomarker discovery each have distinct advantage and disadvantage yet when combined hold real promise the coming year one key approach this effort the development nontargeted depletionfree and quantitative liquid chromatographyultra high resolution tandem mass spectrometry lcm pipeline for the systemswide interrogation the diverse proteome encompassed whole tissue and blood serum plasma derived quantitative proteome can decoded for their biomedical relevance with advanced bioinformatics and bibliographic mining yield promising molecular portrait that can gauge prostatic disease the serological level their functional annotation although potentially useful beyond our current level biological understanding and should not requisite for their effective use the clinical monitoring prostatic disease',\n",
       " 'recent literature that highlight the power using mass spectrometry for protein identification from preparation highly purified organelle and other large subcellular structure covered this review with emphasis technique that preserve the integrity the functional protein complex recent advance distinguishing contaminant protein from bonafide organellelocalized protein and the affinity capture protein complex are reviewed well bioinformatic strategy predict protein organellar localization and integrate proteinprotein interaction map obtained from msaffinity capture method with data obtained from other technique those development demonstrate that revolution cellular biology fueled technical advance msbased proteomic technique well underway',\n",
       " '',\n",
       " '',\n",
       " 'liquid chromatography and mass spectrometry msbased proteomics now allows very deep coverage proteome twodimensional high performance liquid chromatography hplc useful tool for the proteome analysis complex biosystem however ha drawback long running time and typically requires peptide amount the milligram range and large volume collected fraction this study introduce ultraperformance liquid chromatography uplc and eightport rotor valve highly efficient and convenient method for firstdimension separation and collection system the combination our uplcbased fractionation using basic buffer with online lcmsms provided orthogonal peptide separation and demonstrated the powerful performance for protein identification upon applying the novel method triplicate measurement human cell line observed excellent quantitative reproducibility between replicates coefficient determination irisup2supgt and more than peptide identification over the conventional stagetip approach the fractionation method described here flexible straightforward and robust and enables proteome analysis with minimal sample requirement',\n",
       " 'broad range mass spectrometer are used mass spectrometry msbased proteomics research each type instrument posse unique design data system and performance specification resulting strength and weakness for different type experiment unfortunately the native binary data format produced each type mass spectrometer also differ and are usually proprietary the diverse nontransparent nature the data structure complicates the integration new instrument into preexisting infrastructure impedes the analysis exchange comparison and publication result from different experiment and laboratory and prevents the bioinformatics community from accessing data set required for software development here introduce the mzxml format open generic xml extensible markup language representation data have also developed accompanying suite supporting program expect that this format will facilitate data management interpretation and dissemination proteomics research',\n",
       " 'most therapeutic are designed alter the activity protein from metabolic enzyme cell surface receptor connecting the function protein cellular phenotype the activity drug and clinical outcome represents key mechanistic milestone during drug development yet even for therapeutic with exquisite specificity the sequence event following target engagement can complex interconnected community structural metabolic and signalling protein modulate diverse downstream effect that manifest interindividual difference efficacy adverse effect and resistance therapy recent advance mass spectrometry proteomics have made possible decipher these complex relationship and understand how factor such genotype cell type local environment and external perturbation influence them this review explore how proteomic technology are expanding our understanding protein community and their response large and smallmolecule therapeutic',\n",
       " 'proteomics technology demonstrate enormous datagathering capability discover diseasespecific biomarkers serum plasma urine tissue and other biological sample the traditional way achieve this consists protein separation performed using twodimensional polyacrylamide gel electrophoresis 2de however the 2de approach ha it drawback with respect automation sensitivity and throughput considerable effort have been devoted the development nongelbased proteome separation technology able resolve complex protein and peptide mixture prior mass spectrometric analysis this review discus some the most recent advance multidimensional peptide separation technique compatible with analysis including gel electrophoresis isoelectric focusing capillary electrophoresis and liquid chromatography technique based future perspective and our experience special attention given the application twodimensional chromatographic separation coupled the socalled multidimensional protein identification technology approach',\n",
       " 'recent year much progress ha been made proteomic study unravel metabolic pathway and basic cellular process this especially interesting for member the archaea the third domain life archaea exhibit extraordinary feature and many their cultivable representative are adaptable extreme environment archaea harbor many unique trait besides bacterial attribute such size shape and dna structure and eukaryal characteristic like information processing sulfolobus solfataricus thermoacidophilic archaeal representative wellestablished model organism adapted lowph environment and high temperature the genome ha size mbp and it sequence ha been deciphered approximately predicted open reading frame have been identified and the genome characterized great number diverse insertion sequence element unraveling the organism metabolism and lifestyle proteomic analysis have played major role much effort ha been directed this organism and reviewed here with the help proteomics unique metabolic pathway were resolved solfataricus target for regulatory protein phosphorylation identified and cellular response upon virus infection well oxidative stress analyzed',\n",
       " 'plant growth and development are regulated hormone and the associated signalling pathway share several common step the first being the detection the signal receptor protein this typically lead conformational change the receptor thereby modifying it spectrum interaction partner next secondary signal are transmitted via rapid posttranslational cascade such targeted phosphorylation ubiquitination resulting the activationdeactivation relocalization degradation target protein these event finally give rise the signaldependent readout such change gene expression and regulation protein activity far the majority study aimed unravelling hormone signalling pathway plant relied genetic transcriptomic approach during the last decade however msdriven proteomic method became increasingly popular tool plant research they reveal the specific mechanism controlled phytohormone which for large part occur the level the proteome here provide uptodate review the growing body work these area using msbased technique with focus nonpeptide plant hormone',\n",
       " 'laser microdissection lmd wa utilized for the separation the yolk follicular wall granulosa and theca and surrounding stromal cell small white follicle swf obtained from reproductively active domestic fowl herein provide situ proteomicsbased approach studying follicular development through the use lmd and mass spectrometry this study resulted total protein identified from the three specific isolated compartment white yolk from the smallest avian follicle resulted the identification protein while isolated follicular wall and ovarian stroma yielded and protein respectively annotation highlighted the functional difference between the compartment among the three compartment examined the relative abundance vitellogenins steroidogenic enzyme antimullerian hormone transcription factor and protein involved retinoic acid receptorsretinoic acid synthesis transcription factor and cell surface receptor such egfr and their associated signaling pathway reflected known cellular function the ovary this study ha provided global proteome for swf white yolk and ovarian stroma the avian ovary that can used baseline for future study and verifies that the coupling lmd with proteomic analysis can used evaluate protein from small physiologically functional compartment complex tissue',\n",
       " 'the mapu database contains proteome organelle tissue and cell type measured mass spectrometry msbased proteomics contrast other database meant contain limited number experiment and only those with very highresolution and accuracy data mapu display the proteome organelle tissue and body fluid conversely display the occurrence protein interest all these proteome the new release address msspecific problem including ambiguous peptidetoprotein assignment and provides insight into general functional feature the protein level ranging from gene ontology classification comprehensive swissprot annotation moreover the derived proteomic data are used annotate the genome using distributed annotation service da via ensembl service mapu model for database specifically designed for highaccuracy proteomics and member the proteomexchange consortium available line httpwwwmapuproteomecom',\n",
       " 'essential first step the understanding disease and environmental perturbation the early and quantitative detection the increased level the inflammatory marker nitrotyrosine compared with it endogenous level within the tissue cellular proteome thus method that successfully address proteomewide quantitation nitrotyrosine and related oxidative modification can provide early biomarkers risk and progression disease well effective strategy for therapy multidimensional separation coupled with tandem mass spectrometry lcmsms ha recent year significantly expanded our knowledge human and mammalian model system proteome including some nascent work identification posttranslational modification this chapter discus the application lcmsms for quantitation and identification nitrotyrosinemodified protein within the context complex protein mixture presented mammalian proteome',\n",
       " 'singlecell analysis came change the way look cell population rna sequencing single cell allowed appreciate the diversity cell type the human brain unprecedented manner and it power reveal celltype specific change cell population ha just begun explored this context looking the proteome single cell promise bring functional information and contribute completing the picture the potential single cell proteome developing better understanding the intricate connection between the very diverse cell population the brain huge whereas early approach address singlecell proteome have identified hundred protein today technique combining isobaric labelling and lcm can lead the identification thousand protein this review describe method which have been used identify and quantify protein from single cell and propose that the application isobaric labeling and labelfree quantitative proteomics approach for singlecell analysis ready provide useful information for the neurobiology field',\n",
       " 'amphipols apols are newly designed and milder class detergent they have been used primarily protein structure analysis for membrane protein trapping and stabilization have recently demonstrated that apols can used alternative detergent for proteome extraction and digestion achieve onestop singletube workflow for proteomics this workflow apols are removed precipitation after protein digestion without depleting the digested peptide here took further advantage this precipitation characteristic apols concentrate protein from diluted sample contrast with tryptic peptide decrease lead the unbiased coprecipitation apols with protein including globular hydrophilic protein demonstrated that this precipitation combined effect acid precipitation and the apols protein interaction also have been able demonstrate that apolsaided protein precipitation work well diluted sample such secretome sample and provides rapid method for protein concentration',\n",
       " 'the suprachiasmatic nucleus scn act the central clock coordinate circadian oscillation mammalian behavior physiology and gene expression despite our knowledge the circadian transcriptome the scn how impact genomewide protein expression not well understood here interrogated the murine scn proteome across the circadian cycle using silacbased quantitative mass spectrometry the protein that were accurately quantified protein displayed timeofdaydependent expression profile within this timeofday proteome protein were further defined circadian based sinusoidal expression pattern with period nine circadianly expressed protein exhibited rhythm the transcript level with average time lag that exceeded substantial proportion the timeofday proteome exhibited abrupt fluctuation the anticipated lighttodark and darktolight transition and wa enriched for protein involved several key biological pathway most notably mitochondrial oxidative phosphorylation additionally predicted target mirab were enriched specific hierarchical cluster and were inversely correlated with mirab expression the scn these insight into the proteomic landscape the scn will facilitate more integrative understanding cellular control within the scn clock',\n",
       " '',\n",
       " 'human proteome project ha been proposed ensure that such project will have transformative impact biology it scope should determined wide and open discussion',\n",
       " 'gene expression analysis stem cell sc will help uncover further define signaling pathway and molecular mechanism involved the maintenance selfrenewal pluripotency andor multipotency recent year proteomic approach have produced wealth data identifying protein and mechanism involved proliferation and differentiation although many proteomics technique have been developed and improved peptide and protein separation well mass spectrometry several important issue including sample heterogeneity posttranslational modification proteinprotein interaction and highthroughput quantification hydrophobic and lowabundance protein still remain addressed and require further technical optimization this review summarizes the methodology used and the information gathered with proteome analysis sc and discus biological and technical challenge for proteomic study sc disclosure potential conflict interest found the end this article',\n",
       " 'ion mobility mass spectrometry imms allows the structural interrogation biomolecules reporting their collision cross section cc the major bottleneck for exploiting imms structural proteomics lie the lack speed which structure and model can related experimental data here present impact ion mobility projection approximation calculation tool which overcomes these twin challenge providing accurate cc time faster than alternative method this allows ass the cc space presented the entire structural proteome interrogate ensemble protein conformers and monitor molecular dynamic trajectory our data demonstrate that the cc highly informative parameter and that imms considerable practical value structural biologist',\n",
       " 'when feature selection mass spectrometry based single value problem arise from the fact that variability not only vertical but also horizontal direction also slightly differing value may correspond the same feature hence propose use the full spectrum input classifier but select small group block adjacent value instead single value only for that purpose modify the logitboost obtain version the socalled blockwise boosting procedure for classification shown that blockwise boosting ha high potential predictive proteomics',\n",
       " 'the avian eggshell membrane esm stabilized extensive crosslinkages making the identification it protein constituent technically challenging herein applied various extractionsolubilization condition followed proteomic analysis characterize the protein constituent esm derived from the unfertilized chicken egg the egg white and eggshell proteome including previous published work were determined and compared esm identify protein that are relatively highly specific esm merging the result from different extractionsolubilization condition with various proteome allowed the identification and protein the esm egg white and eggshell proteome respectively these and protein were relatively highly specific esm respectively term analysis the common protein and esm unique protein generated and significantly enriched functional group respectively different family protein that were identified esmspecific included collagen cremps histone avbds lysyl oxidaselike loxl2 and ovocalyxin ocx these protein serve foundation for the mechanically stable esm that rest upon the egg white compartment and physical barrier against pathogen invasion overall our result highlight the structural nature the esm constituent that are relevant various biomedical application such wound healing',\n",
       " 'reversedphase liquid chromatography the most commonly used separation method for shotgun proteomics nanoflow chromatography ha emerged the preferred chromatography method for it increased sensitivity and separation despite it common use there are wide range parameter and condition used across research group these parameter have effect the quality the chromatographic separation which critical maximizing the number peptide identification and minimizing ion suppression here examined the relationship between column length gradient length peptide identification and peptide peak capacity found that while longer column and gradient length generally increase peptide identification the degree improvement dependent both parameter and diminished longer column and gradient peak capacity comparison showed more linear increase with column and gradient length discus the discrepancy between these two result and some the consideration that should taken into account when deciding the chromatographic condition for proteomics experiment',\n",
       " 'mollusk one the most widely studied shell texture nacre the lustrous aragonitic layer that constitutes the internal component the shell several bivalve few gastropodsand one cephalopod the nautilus nacre contains minor organic fraction which display wide range function relation the biomineralization process here have biochemically characterized the nacre matrix the cephalopod nautilus macromphalus the acidsoluble matrix contains mixture polydisperse and discrete protein and glycoprotein which interact with the formation calcite crystal addition few bind calcium ion furthermore have used proteomic approachwhich wa applied the acetic acidsoluble and insoluble shell matrix well spot obtained after gel electrophoresis our data demonstrate that the insoluble and soluble matrix although different their bulk monosaccharide and amino acid composition contain numerous shared peptide strikingly most the obtained partial sequence are entirely new few only partly match with bivalvian nacre proteinsour finding have implication for knowledge the longterm evolution molluskan nacre matrix',\n",
       " 'virus replicate and proliferate host cell while continuously adjusting and modulating the host environment they encode wide spectrum multifunctional protein which interplay with and modify protein host cell viral genome were chronologically the first sequenced however the corresponding viral proteome the alteration host proteome upon viral infection and the dynamic nature protein such posttranslational modification enzymatic cleavage and activation destruction proteolysis remain largely unknown emerging highthroughput technique particular quantitative semiquantitative mass spectrometrybased proteomics analysis viral and cellular proteome have been applied define virus and their interaction with their host here review the major area viral proteomics including virion proteomics structural proteomics viral protein interactomics and change the host cell proteome upon viral infection',\n",
       " 'the grampositive bacterium staphylococcus aureus serious human pathogen causing wide variety disease and it increasing resistance toward all available antibiotic make it further investigation absolutely essential examined the membrane proteome exponentially growing cell aureus col because this subproteome play major role the virulence the bacterium it host general analysis membrane protein impeded their hydrophobic nature well high abundance many cytosolic protein the implementation three different technology onedimensional gellc twodimensional and membrane shaving approach combined with msms analysis enabled identification integral and peripheral membrane protein from exponentially growing cell particular the latter approach that combined membrane shaving with subsequent chymotrypsin digest integral membrane domain protein greatly facilitated the detection hydrophobic peptide derived from membranespanning segment peptide all peptide and therefore yielded almost exclusively highly hydrophobic integral membrane protein comparison the various method disclosed the onedimensional gellc and the shaving approach highly complementary technique combination them will reveal most comprehensive view membrane proteome',\n",
       " '',\n",
       " 'bottomup proteomics currently the dominant strategy for proteome analysis relies critically upon the use protease digest protein into peptide which are then identified liquid chromatographymass spectrometry lcm the choice protease ha substantial impact upon the utility the bottomup result obtained protease selection determines the nature the peptide produced which turn affect the ability infer the presence and quantity the parent protein and posttranslational modification the sample present here the software tool proteaseguru which provides silico digestion candidate protease allowing evaluation their utility for bottomup proteomic experiment this information useful for both study focused single small number protein and for analysis entire complex proteome proteaseguru provides convenient user interface valuable peptide information and data visualization enabling the comparison digestion result different protease the information provided includes data table theoretical peptide sequence and their biophysical property result summary outlining the number shared and unique peptide per protease histogram facilitating the comparison proteomewide proteolytic data proteinspecific summary and sequence coverage map example are provided it use inform analysis variantcontaining protein the human proteome well for study requiring the use multiple proteomic database such humanmouse xenograft model and microbiome metaproteomics',\n",
       " '',\n",
       " 'the capability mass spectrometrybased proteomics ha matured ten thousand peptide can measured simultaneously which ha the benefit offering system view protein expression however major challenge that with increase throughput protein quantification estimation from the native measured peptide ha become computational task limitation existing computationally driven protein quantification method that most ignore protein variation such alternate splicing the rna transcript and posttranslational modification other possible proteoforms which will affect significant fraction the proteome the consequence this assumption that statistical inference the protein level and consequently downstream analysis such network and pathway modeling have only limited power for biomarker discovery here describe bayesian proteoform quantification model bpquant1 that us statistically derived peptide signature identify peptide that are outside the dominant pattern the existence multiple overexpressed pattern improve relative protein abundance estimate researchdriven approach that utilizes the objective the experiment defined the context standard statistical hypothesis identify set peptide exhibiting similar statistical behavior relating protein this approach infers that change relative protein abundance can used surrogate for change function without necessarily taking into account the effect differential posttranslational modification processing splicing altering protein function verify the approach using dilution study from mouse plasma sample and demonstrate that bpquant achieves similar accuracy the current stateoftheart method proteoform identification with significantly better specificity bpquant available matlab® and package',\n",
       " 'proteome analysis becoming powerful tool the functional characterization organism and take broad comprehensive systematic approach understanding biology following the sequencing the phytoplasma genome the next step characterize the expressed proteome phytoplasmas acquire the verification and functional annotation all predicted gene and their protein product here describe the protocol mulberry dwarf phytoplasma purification phytoplasma protein extraction and separation sdspage ingel tryptic digestion the protein separation the digested peptide liquid chromatography and identification the peptide mass spectrometry the protocol described here also applicable the analysis other phytoplasma proteome',\n",
       " 'the analysis the expression virulence gene and the elucidation metabolic and regulatory pathway staphylococcus aureus provide with important information about the interaction between the pathogen and it host mechanism which this organism cause disease and the resistance antibiotic order investigate regulatory network aureus analyze the cytoplasmic and extracellular proteome using twodimensional gel analysis combined with matrixassisted laser ionizationtimeofflight mass spectrometry malditof gelbased proteomics extremely valuable tool microbial physiology that can combination with various visualization and quantitation software package very rapidly provide comparative and quantitative data for multisample comparison',\n",
       " 'largescale multiomic analysis allows thorough understanding different physiological pathological condition particularly cancer here extraction method simultaneously yielding dna rna and protein thereby referred triple extraction tex wa tested for it suitability unbiased systemwide proteomic investigation largely proven efficient for transcriptomic and genomic study aimed exploring tex compatibility with mass spectrometrybased proteomics and phosphoproteomics compared standard urea extraction tex suitable for the shotgun investigation proteome providing similar result ureabased protocol both the qualitative and quantitative level tex likewise compatible with the exploration phosphorylation event actually providing higher number correctly localized site than urea although the nature extracted modification appears somewhat distinct between both technique these result highlight that the presented protocol well suited for the examination the proteome and modified proteome this bladder cancer cell model efficiently other more widely used workflow for mass spectrometrybased analysis potentially applicable other mammalian cell type and tissue tex represents advantageous strategy for multiomics scarce andor heterogenous sample',\n",
       " 'several technique based stable isotope labeling are used for quantitative these include stable isotope metabolic labeling method for cell culture well live organism with the assumption that the stable isotope ha effect the proteome here investigate the isotope effect escherichia coli culture that were grown either unlabeled nlabeled medium lcesimsmsbased relative protein quantification consistent protein expression level difference and altered growth rate were observed between and nlabeled culture furthermore targeted metabolite analysis revealed altered metabolite level between and nlabeled bacteria our data demonstrate for the first time that the introduction the isotope affect protein and metabolite level coli and underline the importance implementing control for unbiased protein quantification using stable isotope labeling technique',\n",
       " 'the study snake venom proteome venomics ha been experiencing burst report however the comprehensive knowledge the dynamic range protein present within single venom the set posttranslational modification ptms well the lack comprehensive database related venom protein are among the main challenge venomics research the phenotypic plasticity snake venom proteome together with their inherent toxin proteoform diversity point out the use integrative analysis order better understand their actual complexity this regard such system venomics task should encompass the integration data from transcriptomic and proteomic study specially the venom gland proteome the identification biological ptms and the estimation artifactual proteome and peptidomes generated sample handling procedure',\n",
       " 'activitybased protein profiling abpp ha matured into standard method for the fast sensitive and selective identification enzyme activity and inhibitor proteome using natural product based probe the target many uncharacterized molecule can easily identified complex proteome and their exact function and mechanism action understood natural product and their derivative can also serve pharmaceutical lead structure that impede essential component the cell and their effect can studied biological assay since the complex regulatory process cell beyond mere transcription translation and activation imperative also identify the product the active proteomethe metabolite and binding partner individual enzyme and protein therefore method which the chemically complex metabolome can characterized are necessary series interesting approach have become available recent year that enable the global investigation enzymemetabolite pair',\n",
       " '',\n",
       " 'plant tissue usually contain high level protease and secondary metabolite that severely interfere with protein extraction separation and identification preparation highquality protein sample from plant tissue for proteomic analysis represents great challenge this article briefly describes the critical point protein separation especially secondary metabolite plant tissue and removal strategy provides updated overview three total protein extraction method and their application proteomic analysis various recalcitrant tissue',\n",
       " 'lactococcus lactis bacteria with high biotechnological potential where frequently used the amino acid production and production fermented dairy product well drug delivery system and mucosal vaccine vector the knowledge functional core proteome important extremely for both fundamental understanding cell function and for synthetic biology application this study characterized the lacits proteome from proteomic analysis four biotechnological strain lactis lactis subsp lactis ncdo lactis subsp lactis lactis subsp cremoris and lactis subsp cremoris our labelfree quantitative proteomic analysis the whole bacterial lysates from each strain resulted the characterization the lactis core proteome that wa composed protein which might contribute resistance this bacterium different stress condition well involved the probiotic characteristic lactis kegg enrichment analysis show that ribosome metabolic pathway pyruvate metabolism and microbial metabolism diverse environment were the most enriched according our quantitative proteomic analysis protein related translation process were the more abundant the core proteome which represent important step the synthetic biology addition identified subset conserved protein that are exclusive the lactis subsp cremoris lactis subsp lactis which some are related metabolic pathway exclusive regarding specific proteome ncdo detected strainspecific protein finally proteogenomics analysis allows the identification protein which were not previously annotated and the result obtained this study allowed increase our knowledge about the biology lactis which contributes the implementation strategy that make possible increase the biotechnological potential this bacterium',\n",
       " 'mass spectrometrybased proteomics ha experienced unprecedented advance comprehensive analysis protein and posttranslational modification with particular technical progress liquid chromatography coupled tandem mass spectrometry lcmsms and isobaric labeling multiplexing capacity here introduce deep proteomics profiling protocol that combine plex tandem mass tag tmt labeling with optimized lcmsms platform quantitate whole proteome and phosphoproteome the major step include protein extraction and digestion tmt labeling twodimensional liquid chromatography tiosub2submediated phosphopeptide enrichment highresolution mass spectrometry and computational data processing this protocol routinely lead confident quantification more than protein and approximately phosphosites mammalian sample quality control step are implemented for troubleshooting and evaluating experimental variation such multiplexed robust method provides powerful tool for dissecting proteomic signature the system level variety complex sample ranging from cell culture animal tissue human clinical specimen',\n",
       " 'bintroductionb biomarkers are expected improve the management cancer patient enabling early detection and prediction therapeutic response protein reflect molecular phenotype have high potential biomarkers and also are key target for intervention given the ease collection and proximity certain tumor the urinary proteome rich source biomarkers and several protein have been already implemented bareas coveredb examined the literature urine protein and proteome analysis oncology from report published during the last year generate overview the status urine protein and peptide biomarkers with emphasis their actual clinical value bexpert commentaryb few study report biomarkers that are ready implemented patient management among others bladder cancer and cholangiocarcinoma these report are based multimarker approach high number biomarkers though ha been described study with low statistical power fact several them have been consistently reported across different study the latter should the focus attention and tested properly designed confirmatory and ultimately prospective investigation expected that multimarker classifier for specific contextofuse will the preferred path toward clinical implementation',\n",
       " 'the analysis protein biological membrane form major challenge proteomics despite continuous improvement and the development more sensitive analytical method the analysis membrane protein ha always been hampered their hydrophobic property and relatively low abundance this review describe recent successful strategy that have led indepth analysis the membrane proteome facilitate membrane proteome analysis essential that biochemical enrichment procedure are combined with special analytical workflow that are all optimized cope with hydrophobic polypeptide these include technique for protein solubilization and also wellmatched development protein separation and protein digestion procedure finally discus approach target membraneprotein complex and lipidprotein interaction such approach offer unique insight into function and architecture cellular membrane',\n",
       " 'numerous application ambient mass spectrometry have been demonstrated over the past decade they promoted the emergence various microsampling technique such laser ablationdroplet capture ladc ladc consists the ablation analytes from surface and their subsequent capture solvent droplet which can then analyzed ladc thus generally performed the range using wavelength which analytes the matrix absorb this work explore the potential visible range ladc microsampling technology for largescale proteomics analysis demonstrate that biomolecule analysis using ladc are possible despite the low absorbance biomolecules this wavelength this due the preponderance indirect substratemediated ablation mechanism low laser energy which contrast with the conventional direct ablation driven sample absorption using our custom ladc system and taking advantage this substratemediated ablation mechanism were able perform largescale proteomic analysis microsampled tissue section and demonstrated the possible identification protein with relevant biological function consequently the ladc technique offer new tool for biological and clinical application',\n",
       " 'proteogenomics consists the annotation reannotation proteincoding nucleic acid sequence based the empirical observation their gene product while functional annotation predicted gene increasingly feasible given the multiplicity genome available for many branch the tree life the accurate annotation the translational start site still point contention extensive coverage the proteome including specifically the ntermini now possible thanks nextgeneration mass spectrometer able record data from thousand protein once effort increase the peptide coverage protein sequence and detect low abundance protein are important make proteomic and proteogenomic study more comprehensive this review present the panoply nterminusoriented strategy that have been developed over the last decade',\n",
       " 'despite year preclinical development biological intervention designed treat complex disease such asthma often fail phase iii clinical trial these failure suggest that current method analyze biomedical data might missing critical aspect biological complexity such the assumption that case and control come from homogeneous distribution here discus why and how method from the rapidly evolving field visual analytics can help translational team consisting biologist clinician and bioinformaticians address the challenge modeling and inferring heterogeneity the proteomic and phenotypic profile patient with complex disease because primary goal visual analytics amplify the cognitive capacity human for detecting pattern complex data begin with overview the cognitive foundation for the field visual analytics next organize the primary way which specific form visual analytics called network ha been used model and infer biological mechanism which help identify the property network that are particularly useful for the discovery and analysis proteomic heterogeneity complex disease describe one such approach called subjectprotein network and demonstrate it application two proteomic datasets this demonstration provides insight help translational team overcome theoretical practical and pedagogical hurdle for the widespread use subjectprotein network for analyzing molecular heterogeneity with the translational goal designing biomarkerbased clinical trial and accelerating the development personalized approach medicine',\n",
       " 'protease specificity profiling using proteomederived databasesearchable peptide library novel approach define the active site specificity proteolytic enzyme call pic proteomic identification protease cleavage site proteomederived peptide library are generated trypsin gluc chymotrypsin digestion biologically relevant proteome such cytosolic lysates generate three separate library that each differ from the others their cterminal amino acid residue according the protease specificity primary amine all peptide are then chemically protected that after incubation with test protease the neontermini the primeside cleavage product with exposed αamines can specifically biotinylated enriched and identified liquid chromatographytandem mass spectrometry the corresponding nonprimeside sequence are derived bioinformatically suited for all protease class except carboxyproteases and those aminoproteases and dipeptidases requiring free αamine for cleavage pic simultaneously profile the specificity prime and nonprime position and directly determines scissile peptide bond hundred cleavage site sequence single experiment this wealth sequence specificity information also allows for the investigation subsite cooperativity herein describe simplified procedure produce pic peptide library the method perform pic assay and new method data analysis',\n",
       " 'sperm motility crucial for ram fertility however difference the proteome sperm with high and low motility ram ovis aries ha yet achieved thus the aim this study wa identify and characterize ram spermatozoon protein with different abundance between high and low motility and identify proteomic marker for ram spermatozoon motility using tandem mass tag tmt protein labeling and lcmsms this study the abundance protein highmotility ram sperm wa significantly different compared with lowmotility sperm protein involved sperm motility mitochondrial activity and spermatogenesis showed higher abundance ram spermatozoon protein involved protein processing and spliceosome were more abundant ram spermatozoon conclusion phosphatidylethanolamine binding protein potential proteomic marker for ram sperm motility cctshsps are hallmark the spermatozoon ram our finding highlight the functional difference between and ejaculated spermatozoon and ha identified candidate protein interest linked sperm motility ram significance inadequate sperm motility one the most important reason for male subfertility infertility this study found that the abundance protein highmotility ram sperm wa significantly different compared with lowmotility sperm proteomic biomarkers were discovered reflect the motility variation ram spermatozoon these biomarkers may assist illuminating the molecular mechanism underlying sperm motility this study expands the potential direction for sperm quality screening and animal breeding',\n",
       " 'there are estimated million people with visual impairment worldwide whom million are blind the pathogenesis many eye disease remains poorly understood the human eye currently emerging proteome that may provide key insight into the biological pathway disease review proteomic investigation the human eye and present catalogue nonredundant protein identified human eye tissue and biofluids date highlight the need identify new biomarkers for eye disease using proteomics recent advance proteomics now allow the identification hundred thousand protein tissue and fluid characterization various ptms and simultaneous quantification multiple protein facilitate proteomic study the eye the human eye proteome project hepp wa organized september the hepp one the most recent component the biologydiseasedriven human proteome project bdhpp whose overarching goal support the broad application stateoftheart measurement protein and proteome life scientist studying the molecular mechanism biological process and human disease the large repertoire investigative proteomic tool ha great potential transform vision science and enhance understanding physiology and disease process that affect sight',\n",
       " '',\n",
       " 'while neurovascular disease such ischemic and hemorrhagic stroke are the leading cause disability the world the repertoire therapeutic intervention ha remained remarkably limited there dire need develop new diagnostic prognostic and therapeutic option the study proteomics particularly enticing for cerebrovascular disease such stroke which most likely involve multiple gene interaction resulting wide range clinical phenotype currently rapidly progressing neuroproteomic technique have been employed clinical and translational research help identify biologically relevant pathway understand cerebrovascular pathophysiology and develop novel therapeutic and diagnostics future integration proteomic with genomic transcriptomic and metabolomic study will add new perspective better understand the complexity neurovascular injury here review cerebrovascular proteomics research both preclinical animal cell culture and clinical blood urine cerebrospinal fluid microdialyates tissue study will also discus the reward challenge and future direction for the application proteomics technology the study various disease phenotype capture the dynamic range cerebrovascular injury and repair with translational targeted and discovery approach emphasize the importance complementing innovative proteomic technology with existing molecular biology model preclinical study and the need advance pharmacoproteomics directly probe clinical physiology and gauge therapeutic efficacy the bedside',\n",
       " 'the clinical fertility test available the market fail define the exact cause male infertility almost half the case and point toward crucial need developing better way infertility investigation the protein biomarkers may help toward better understanding unknown case male infertility that turn can guide find better therapeutic solution many clinical attempt have been made identify biomarkers male infertility sperm proteome but only few study have targeted seminal plasma human seminal plasma rich source protein that are essentially required for development sperm and successful fertilization this viewpoint article highlight the importance human seminal plasma proteome reproductive physiology and suggests that differential proteomics integrated with functional analysis may help searching potential biomarkers male infertility',\n",
       " 'microscale separation liquid chromatography capillary electrophoresis coupled with mass spectrometry ha become the primary tool for advanced proteomics indispensable technology for gaining understanding complex biological process recent decade significant advance have been achieved msbased proteomics however the current proteomics platform still face analytical challenge overall sensitivity towards nanoproteomics application for starting material le than 1μg total protein cellular heterogeneity tissue pathology herein review recent advance microscale separation technique and integrated sample processing strategy that improve the overall sensitivity and proteome coverage the proteomics workflow and their contribution towards nanoproteomics application',\n",
       " '',\n",
       " 'the present review highlight recent progress the technique combinatorial peptide ligand library cppl methodology that ha much offer for the detection low verylow abundance protein nanogramsml scale and below any proteome particular advance exploration the urinary plasma and tissue proteome are discussed and evaluated shown that when treating biological fluid such plasma with cplls the detection sensitivity which the control only reach ngml can enhanced high pgml with increment sensitivity three order magnitude the possibility using cplls twodimensional prefractionation any proteome also evaluated the charge axis cpll capture can implemented le than three different value and thus permitting capture proteinaceous analytes bearing net positive net negative charge respectively when capture performed the absence salt high level salt the hofmeister series one can favor the capture hydrophilic hydrophobic protein respectively this would thus genuine protocol working orthogonal separation principle charge hydrophobicity the horizon cplls expanding and it use exponentially growing expect major breakthrough biomarker discovery field that ha suffered decade failure',\n",
       " '',\n",
       " 'soil contains low amount protein but high amount interfering substance current extraction method for soil protein can not produce highquality sample suitable for proteomic analysis resolve the problem devised sequential extraction method through sequentially extracting soil citrate and sd buffer followed phenol extraction the method allows for obtaining applicable and protein profile with various agricultural soil and detecting glomalinrelated soil protein the method may valuable tool for soil proteomics',\n",
       " 'oncoproteomics the study cancerassociated protein and their interaction using proteomic technology expected play crucial role the detection and surveillance cancer well the development personalized medicine for oncology this review will outline the current status advantage and limitation oncoproteomics further will discus available strategy arising from better understanding oncoproteomics including integrated omics approach overcome current problem the discovery and validation biomarkers identification novel drug target and management personalized medicine',\n",
       " 'advance proteomics technique over the past decade closely integrated with genomic and physicochemical approach have played great role developing knowledge the biofilm lifestyle bacteria despite bacterial proteome versatility many study have demonstrated the ability proteomics approach elucidating the biofilm phenotype though these investigation have been largely used for biofilm study the last decade they represent however very low percentage proteomics work performed now such approach have offered new target for combating microbial biofilms providing comprehensive quantitative and qualitative overview their protein cell content herein summarized the state the art knowledge about biofilm physiology after one decade proteomic analysis second part highlighted missing research track for the next decade emphasizing the emergence posttranslational modification proteomic study stemming from recent advance mass spectrometrybased proteomics',\n",
       " 'the myriad protein may involved the mechanism underlying the development and maintenance neuropathic pain extremely disabling condition that originates from pathology the nervous system address the mechanism here analyzed protein and cellular network the dorsal spinal cord mediating pain processing wellestablished rat model neuropathic pain induced spinal nerve ligation snl labelingbased proteomic method together with highresolution mass spectrometry for proteome analysis were applied protein including synapsin and microtubuleassociated protein were identified differently expressed the snl group pathway analysis suggests that maladaptive change the level these protein may contribute abnormal synaptic transmission and neuronal intracellular signaling underlying the onset and development neuropathic pain',\n",
       " 'since protein carry out many functional role cell with different concentration proteomics likely more appropriate approach explain biological process and cellular event than mrna study although gene ontology provides systematic and organized vocabulary biological term for protein need more detail decide about the correct duty and annotation protein specific condition one can assume that change protein concentration related biological process that protein with negligible error therefore can obtain more information about the function protein studying these profile this study used timecourse proteomic data twenty day differentiation study embryonic stem cell escs differentiating embryoid body eb hierarchical clustering wa used cluster timeseries concentration profile protein our result demonstrate that there are eleven active process with distinct concentration profile this initial differentiation according the concentration profile protein suggest new change point equivalently new stage the course embryonic differentiation',\n",
       " 'early detection and correct diagnosis are essential for effective treatment cancer and patient survival complete sequencing the human genome and the genome other specie provides valuable tool for discerning the genetic abnormality cancer however difference between cancerous and normal cell reflect more than variation genetic sequence and abundance transcribed rna many cancer biomarkers are manifestation difference posttranscriptional splicing andor posttranslational modification thus proteomic tool are being increasingly utilised the postgenomic era for discovery new cancer biomarkers this paper will provide overview the biomarker discovery process from the proteomic profiling point view with emphasis given the principle that are involved the process including the protein identification strategy and how surface enhanced laser desorption ionisation mass spectrometry fit into the picture the aim provide resource for the experimental practitioner seeking awareness the analytical tool that are now available contemporary cancer research',\n",
       " '',\n",
       " 'the production polyhydroxyalkanoates pha bioplastics from waste surplus feedstock using mixed microbial consortium mmc and aerobic dynamic feeding adf growing field within mixed culture biotechnology this study aimed optimize 2de workflow investigate the proteome dynamic mmc synthesizing pha from fermented dairy manure mitigate the challenge posed effective 2de this complex sample matrix the bacterial biomass wa purified using accudenz gradient centrifugation agc before protein extraction the optimized 2de method yielded highquality gel suitable for quantitative comparative analysis and subsequent protein identification lcmsms the optimized 2de method could adapted other proteomic investigation involving mmc complex organic environmental matrix',\n",
       " '',\n",
       " 'important issue for molecular biology establish whether transcript level given gene can used proxy for the corresponding protein level here have developed targeted proteomics approach for set human nonsecreted protein based parallel reaction monitoring measure steadystate condition absolute protein copy number across human tissue and cell line and compared these level with the corresponding mrna level using transcriptomics the study show that the transcript and protein level not correlate well unless genespecific rnatoprotein rtp conversion factor independent the tissue type introduced thus significantly enhancing the predictability protein copy number from rna level the result show that the rtp ratio varies significantly with few hundred copy per mrna molecule for some gene several hundred thousand protein copy per mrna molecule for others conclusion our data suggest that transcriptome analysis can used tool predict the protein copy number per cell thus forming attractive link between the field genomics and proteomics',\n",
       " '',\n",
       " '',\n",
       " 'proteomics and fermentation technology have begun integrate investigate fermentation organism bioprocess development this the first shotgun proteomics study employed monitor the proteome scheffersomyces stipitis during xylose fermentation under oxygen limitation identified nonredundant protein and observed highly similar proteome from exponential early stationary phase analyzing the temporal proteome identified unique expression pattern biological process and metabolic pathway including alternative respiration salicylhydroxamic acid sham pathway activation glyoxylate cycle expression galactose enzyme and secondary zinccontaining alcohol dehydrogenase and oglycosyl hydrolases identified the expression putative highaffinity xylose sugar transporter xut1p but lowaffinity xylose transporter were absent throughout cell growth housekeeping process included oxidative phosphorylation glycolysis nonoxidative branch the pentose phosphate pathway gluconeogenesis biosynthesis amino acid and aminoacyl total rna trna protein synthesis and proteolysis fatty acid metabolism and cell division this study emphasized qualitative analysis and demonstrated that shotgun proteomics capable monitoring stipitis fermentation and identifying physiological state such nutrient deficiency',\n",
       " 'historically ha been challenging beyond epidemiology investigate the pathogenic change caused tobacco smoking the epiairway mattek corp ashland wa employed investigate the effect cigarette smoke component exposure the airliquidinterface represented particle and vapour phase component cigarette smoke proteomic study utilising itraq labelling compared expression profile the correlative histopathology revealed focal region hyperplasia hypertrophy cytolysis and necrosis identified protein with parameter two more peptide four these protein are potential marker lung injury and three are related mechanistic pathway disease',\n",
       " 'cartilage highly specialized loadbearing tissue with small number cell and high proportion extracellular matrix ecm the abundance heavily sulfated proteoglycans and poorly soluble collagenous ecm present major technical challenge 2de here report proteomic analysis mouse growth plate cartilage using novel methodology for tissue dissection and sample prefractionation have successfully resolved cartilage tissue extract 2de for the first time and identified cartilage ecm protein western blotting and msms',\n",
       " '',\n",
       " 'glutathione reductase cytosolic protein play vital role maintaining correct redox status cell however comprehensive investigation grmodulated cellular response including protein level alteration and redox regulation have yet performed this study cultured human lung adenocarcinoma line transfected with empty plko1 vector control clshcontrol and it grknockdown derivative clshδgr evaluate differential protein level alteration and redox regulation these two cell line identified spot that exhibited marked change intensity and protein showing significant change thiol reactivity response depletion several protein involved redox regulation calcium signaling cytoskeleton regulation and protein folding showed significant change expression whereas protein involved redox regulation protein folding and glycolysis displayed change thiol reactivity interestingly knockdown induces peroxiredoxin1 overexpression the airexposed tissue and high oxygen consuming tissue such cornea and liver but not the low oxygen consuming tissue such breast and uterine summary used comprehensive lung adenocarcinoma based proteomic approach for identifying grmodulated protein expression alteration and redox modification based our research this the first comprehensive proteomic and redoxproteomic analysis used investigate the role mammalian cell model',\n",
       " 'molting essential biological process occurring multiple time throughout the life cycle most ecdysozoa molting fluid accumulate and function the exuvial space during the molting process this study used liquid chromatographytandem mass spectrometry investigate the molting fluid analyze the molecular mechanism molting the silkworm bombyx mori total protein were identified molting fluid from the silkworm before pupation and eclosion including chitin metabolismrelated enzyme serine protease peptidase and protease inhibitor gene ontology analysis indicated that catalytic constitutes the most enriched function the molting fluid gene expression pattern and bioinformatic analysis suggested that numerous enzyme are involved the degradation cuticle protein and chitin proteinprotein interaction network and activity analysis showed that protease inhibitor are involved the regulation multiple pathway molting fluid additionally many immunerelated protein may involved the immune defense during molting these result provide comprehensive proteomic insight into proteolytic enzyme and protease inhibitor molting fluid and will likely improve the current understanding physiological process insect molting',\n",
       " 'proteolytic digestion time consuming and critical step bottomup proteomic analysis the most widely used protease trypsin ha high specificity and generates peptide that are considered ideally suited for bottomup lcm technology exploiting key factor affecting enzymatic activity obtained simple straightforward and rapid insolution digest protocol that performed better than the conventional overnight digestion method term amino acid coverage protein number peptide generated and peptide ion abundance prolonged digestion time such overnight digestion lead decline protein amino acid coverage and loss tryptic peptide this wa found caused complete digestion trypsin leading increased number small peptide that are not lcm detectable slowrate nontryptic digestion peptide contributing factor for loss peptide ion intensity during extended digestion time our work demonstrates that for both qualitative and quantitative bottomup proteomic study beneficial prevent trypsin digestion completion reducing treatment time from the conventional several hour few minute cleavage time',\n",
       " 'connect gridenabled pipeline ontologybased environment for proteomics spectrum management with machine learning platform for unbiased predictive analysis exploit two existing software platform msanalyzer and biodcv the emerging proteomics standard and the middleware and computing resource the egee biomed grid infrastructure the setup biodcv accessed the msanalyzer workflow web service thus providing complete grid environment for proteomics data analysis predictive classification study malditof data based this environment are presented',\n",
       " 'the identification small protein and peptide below amino acid complex biological sample hampered the dominance highermolecularweight protein the contrary the increasing knowledge about alternative short open reading frame creates need for method that allow the existence the corresponding gene product proven proteomics experiment present acetonitrilebased precipitation methodology that depletes the majority protein above kda parameter such depletion mixture composition and temperature were optimized using model protein mixture and the method wa evaluated comparison with the established differential solubility method the approach wa applied the analysis the lowmolecularweight proteome the archaea methanosarcina mazei mean lcm the data clearly show beneficial effect from reduction complexity especially term the quality msmsbased identification small protein this fast detergentfree method allowed for with minimal sample manipulation the successful identification several not yet identified short open reading frame encoded peptide mazei',\n",
       " 'proteomics commonly used approach that can provide insight into complex biological system the cochlear sensory epithelium contains receptor that transduce the mechanical energy sound into electrochemical energy processed the peripheral and central nervous system several proteomic technique have been developed study the cochlear inner ear such twodimensional difference gel electrophoresis 2ddige antibody microarray and mass spectrometry the most comprehensive and versatile tool proteomics and conjunction with separation method can provide indepth proteome biological sample separation method combined with ha the ability enrich protein sample detect low molecular weight and hydrophobic protein and identify low abundant protein reducing the proteome dynamic range different digestion strategy can applied whole lysate fractionated protein lysate enhance peptide and protein sequence coverage utilization different separation technique including strong cation exchange scx reversedphase and geleluted liquid fraction entrapment electrophoresis gelfree can applied reduce sample complexity prior analysis for protein identification',\n",
       " '',\n",
       " 'protein signaling between tissue tissue crosstalk becoming recognized fundamental biological process that incompletely understood shotgun proteomic analysis tissue and plasma explore this concept are regularly challenged high dynamic range protein abundance which limit the identification lower abundance protein this viewpoint article highlighted how focus protein contained within extracellular vesicle ev not only partially address this issue but can also reveal underappreciated complexity the circulating proteome various physiological and pathological context furthermore how quantitative proteomics can inform mediated crosstalk highlighted and the importance high coverage sensitive proteomic analysis ev identify both the optimal method isolate subtypes interest and protein that characterize them stressed',\n",
       " 'mass spectrometry msbased proteomics ha become indispensable tool for protein identification and quantification this paper common workflow are described with emphasis application msbased proteomics developmental biology progress ha been made the analysis proteome change during tissue differentiation and various genetic perturbation msbased proteomics ha been particularly useful for identifying novel protein interaction affinity purificationmass spectrometry apms many which have been subsequently functionally validated and led the discovery previously unknown mode developmental regulation quantitative proteomics approach can used study posttranslational modification ptms protein such phosphorylation reveal the dynamic intracellular signal transduction integrative approach combine quantitative msbased proteomics with other highthroughput method with the promise system level understanding developmental regulation',\n",
       " 'mammalian skeletal muscle are composed multinucleated cell termed slow fast fiber according their contractile and metabolic property here developed highsensitivity workflow characterize the proteome single fiber analysis segment the same fiber traditional and unbiased proteomics method yielded the same subtype assignment discovered novel subtypespecific feature most prominently mitochondrial specialization fiber type substrate utilization the fiber typeresolved proteome can applied variety physiological and pathological condition and illustrate the utility single cell type analysis for dissecting proteomic heterogeneity',\n",
       " 'alteration mitochondrial structure function have been described variety human disease for nearly half century the complete sequence the human mitochondrial genome ha been published the mitochondrial proteome database however still incomplete here give short review recent advance determine the complete set mitochondrial protein the main emphasis put gelbased proteomic approach identify differentially expressed mitochondrial protein neurodegenerative disease',\n",
       " '',\n",
       " 'one the main goal proteomics solve biological and molecular question regarding set identified protein order achieve this goal one ha extract and collect the existing biological data from public repository for every protein and afterward analyze and organize the collected data due the complexity this task and the huge amount data available not possible gather this information hand making necessary find automatic method data collection within proteomic context have developed protein information and knowledge extractor pike which solves this problem automatically accessing several public information system and database across the internet pike bioinformatics tool start with set identified protein listed the most common protein database accession code and retrieves all relevant and updated information from the most relevant database once the search complete pike summarizes the information for every single protein using several file format that share and exchange the information with other software tool our opinion that pike represents great step forward for information procurement and drastically reduces manual database validation for large proteomic study available httpproteocnbcsicespike',\n",
       " '',\n",
       " '',\n",
       " 'novel multifunctional reagent were applied combination with lipid probe for affinity enrichment myristoylated protein and direct detection lipidmodified tryptic peptide mass spectrometry this method enables highconfidence identification the myristoylated proteome unprecedented scale cell culture and allowed the first quantitative analysis dynamic change protein lipidation during vertebrate embryonic development',\n",
       " 'change protein conformation can affect protein function but method probe these structural change global scale cell have been lacking enable largescale analysis protein conformational change directly their biological matrix present method that couple limited proteolysis with targeted proteomics workflow using our method assessed the structural feature more than yeast protein simultaneously and detected altered conformation for protein upon change nutrient find that some branch carbon metabolism are transcriptionally regulated whereas others are regulated enzyme conformational change detect structural change aggregationprone protein and show the functional relevance one these protein the metabolic switch this approach enables probing both subtle and pronounced structural change protein large scale',\n",
       " 'paralytic shellfish toxin pst are group potent neurotoxic alkaloid produced cyanobacteria and dinoflagellate the pst biosynthesis gene cluster and several toxinrelated protein have been unveiled cyanobacteria yet little known about dinoflagellate here compared the protein profile toxinproducing dinoflagellate alexandrium catenella achkt and it nontoxic mutant achknt and characterized differentially displayed protein using combination the itraqbased proteomic approach and the transcriptomic database totally protein were identified from catenella and protein involved carbohydrate amino acid and energy metabolism were the most abundant among them protein were differentially displayed protein involved amino acid biosynthesis protein and carbohydrate metabolism and bioluminescence were more abundant achkt while protein participating photosynthesis fatty acid biosynthesis and the process occurring peroxisome displayed higher abundance achknt seven toxinrelated protein were identified but they varied insignificantly between the two strain different carbon and energy utilization strategy were potentially related the toxin producing ability and the regulation mechanism pst biosynthesis wa more complex dinoflagellate our study provides the first comprehensive dataset the dinoflagellate proteome and lay the groundwork for future proteomic study',\n",
       " 'the mzquantml data standard wa designed capture the output quantitative software proteomics support submission public repository development visualization software and pipelinemodular approach the standard designed around common core that can extended support particular type technique through the release semantic rule that are checked validation software the first release mzquantml supported four quantitative proteomics technique via four set semantic rule intensitybased ms1 label free ms1 labelbased such silac iii ms2 tagbased itraq tandem mass tag and spectral counting present update mzquantml for supporting srm technique the update includes representing the quantitative measurement and associated metadata for srm transition the mechanism for inferring peptidelevel proteinlevel quantitative value and support for both labelbased labelfree srm protocol through the creation semantic rule and controlled vocabulary term have updated the specification document for mzquantml version and the mzquantml validator ensure that consistent file are produced different exporter also report the capability for production mzquantml file from popular srm software package such skyline and anubis',\n",
       " 'frequently debated question whether technology drive biology whether biology drive the development new technology this issue discussed this manuscript account that cover approximately decade during which mass spectrometry and protein biochemistry have intersected shown that the capability the mass spectrometric method initially developed address the specific need identify protein reliably and high sensitivity soon transcended the intended task the rapid development mass spectrometric technology applied protein research ha catalyzed entirely new experimental approach and opened new type biological question experimentation culminating the field proteomics some conclusion from this case study relating technological research and the environment which carried out are also discussed',\n",
       " 'cellselective protein metabolic labeling great interest for studying cellcell communication and tissue homeostasis herein describe nitrilaseactivatable noncanonical amino acid tagging nancat strategy that exploit exogenous nitrilase enzymatically convert the nitrilesubstituted precursor their corresponding noncanonical amino acid ncaas lazidohomoalanine aha homopropargylglycine hpg living cell only cell expressing the nitrilase can generate aha hpg cellulo and metabolically incorporate them into the nascent protein subsequent clicklabeling the azide alkyneincorporated protein with fluorescent probe with affinity tag enables visualization and proteomic profiling nascent proteome respectively have demonstrated that nancat can serve versatile strategy for cellselective labeling proteome both bacterial and mammalian cell',\n",
       " 'lactobacillus plantarum widespread probiotic bacteria found many fermented food product this study the wholecell protein and secretory protein plantarum were separated twodimensional electrophoresis method total protein were identified tandem mass spectrometry including plasmidencoded hypothetical protein plp the information first highest abundance protein wa listed for the further genetic manipulation plantarum such construction highlevel expression system furthermore the first interaction map plantarum wa established bluenativesdspage technique heterodimeric complex composed maltose phosphorylase map3 and map2 and two homodimeric complex composed map3 and map2 respectively were identified the same time indicating the important role these protein these finding provided valuable information for the further proteomic research plantarum',\n",
       " '',\n",
       " 'the annual spring workshop the proteomics standard initiative psi the human proteome organization hupo wa held this year jointly with the metabolomics coordination standard metabolomics cosmos group the range existing standard mzml mzidentml mzquantml mztab traml wa reviewed and updated the light new methodology and advance technology adaptation meet the need the metabolomics community were incorporated and new data format for nmr nmrml wa presented the molecular interaction workgroup began work new version the existing xml data interchange format psimi xml will enable the capture more abstract data type such protein complex topology derived from experimental data allosteric binding and dynamic interaction further information about the work the hupopsi can found httpwwwpsidevinfo',\n",
       " 'the standard approach for proteomic data acquisition isobarictagged sample mass spectrometry datadependent acquisition this semistochastic identificationfirst paradigm generates wealth peptidelevel data without regard relative abundance introduce data acquisition concept called sequential windowed acquisition reporter mass swarm this approach performs quantitation first thereby allowing subsequent acquisition decision predicated userdefined pattern reporter ion intensity the efficacy this approach validated through experiment with both synthetic mixture escherichia coli ribosome spiked into human cell lysates known ratio and the quantitative evaluation the human proteome response the inhibition cullinbased protein ubiquitination via the small molecule mln find that swarminformed parallel reaction monitoring acquisition display effective acquisition biasing toward analytes displaying quantitative characteristic interest resulting improvement the detection differentially abundant analytes the swarm concept provides flexible platform for the further development new acquisition method',\n",
       " 'streaking severe problem when narrow range basic immobilized gradient strip are used the first dimension twodimensional electrophoresis demonstrated that this cysteinyl related streaking eliminated when focusing done the presence hydroxyethyl disulfide destreak use destreak also result map with simplified spot pattern and improved reproducibility',\n",
       " 'activated hepatic stellate cell hsc the main myofibroblast cell the liver fibrosis important characteristic the recovery not only the apoptosis activated hscs but also reversal myofibroblastlike phenotype quiescentlike phenotype understanding the change secreted protein the reversion activated hscs may provide the broader view cellular regulatory network and discover candidate marker target for therapeutic strategy this study stable isotope labeling with amino acid silac combined with linear ion trapfourier transform ion cyclotron resonance mass spectrometer ltqft wa performed vitro activated hscs and reverted hscs obtain proteomic view secretory protein total protein showed significant difference reverted hscs among these upregulated protein were mainly involved amino acid metabolism pathway and glucose metabolism pathway using genegometacore software while downregulated protein are closely associated with hscs activation such cytoskeleton remodeling chemokines and cell adhesion additionally set novel protein associated with hscs activation and reversion were validated western blotting the cell secretion and the serum including vitronectin laminin beta and ubiquitin conjugation factor e4b our study provided the valuable insight into the mechanism the reversion activated hscs and identified some potential biomarkers clinical study all data have been deposited the proteomexchange with identifier pxd httpproteomecentralproteomexchangeorgdatasetpxd',\n",
       " 'this unit review the new discipline proteomics which includes any largescale proteinbased systematic analysis the proteome defined subproteome from cell tissue entire organism proteomics originated the mid due two key enabling advance availability complete genome sequence and mass spectrometry advance that allowed high sensitivity identification protein proteome analysis can broadly categorized into three type study quantitative protein profile comparison analysis proteinprotein interaction and compositional analysis simple proteome subproteomes such organelle large protein complex the complexity different type proteome the merit targeted versus global proteome study and the advantage alternative separation and analysis technology are discussed',\n",
       " '',\n",
       " 'sparked the article from lescuyer and colleague recent issue aim here further encourage interest and discussion clinically relevant biomarker research express our view proteomics for biomarker discovery addressing multiple relevant issue including the inherent difference between biological fluid and how these difference affect current analytical approach and experimental design maximize the efficiency moving from the bench the bedside herein also include suggestion for definition the term biomarker based the use set universal characterizationvalidation requirement and illustrate several recent example successful transition benchtop proteomic study work clinical practice',\n",
       " 'saccharomyces cerevisiae and schizosaccharomyces pombe are the most commonly studied yeast model system yet comparison global proteome remodeling between these yeast specie are scarce here profile the proteome cerevisiae and pombe cultured with either glucose pyruvate the sole carbon source define common and distinctive alteration the protein landscape across specie addition develop updated streamlinedtandem mass tag sltmt strategy that substitute chemicalbased precipitation with more versatile beadbased protein aggregation method sp3 prior enzymatic digestion and tmt labeling our new workflow slsp3tmt allow for nearcomplete proteome profile single experiment for each specie the data reveal expected alteration protein abundance and difference between specie highlighted complete canonical biochemical pathway and provided insight into previously uncharacterized protein the technique used herein namely slsp3tmt can applied virtually any experiment aiming study remodeling the proteome using highthroughput comprehensive yet streamlined mass spectrometrybased strategy significance saccharomyces cerevisiae and schizosaccharomyces pombe are singlecelled eukaryote that diverged from common ancestor over period million year such that evolution ha driven fundamental difference between the two specie cellular metabolism and the regulation thereof are vital for living organism here hypothesize that large scale proteomic alteration are prevalent upon the substitution glucose with another carbon source this case pyruvate efficiently process our sample developed updated streamlinedtandem mass tag sltmt strategy with more versatile beadbased protein aggregation the data revealed expected alteration protein abundance and illustrated difference between specie highlighted complete canonical biochemical pathway and provided insight into previously uncharacterized protein',\n",
       " 'quantify protein global level from mammalian tissue method wa developed metabolically introduce stable isotope into the protein rattus norvegicus for use internal standard the longterm metabolic labeling rat with diet enriched did not result adverse health consequence the average amino acid enrichment reflected the relative turnover rate the different tissue and ranged from mpe brain mpe plasma using the nenriched liver quantitative internal standard change individual protein level response cycloheximide treatment were measured for protein these measurement revealed protein with altered protein level most protein with altered level have previously reported function involving xenobiotic metabolism and proteinfolding machinery the endoplasmic reticulum this approach powerful tool for the global quantitation protein capable measuring proteomewide change response drug and will useful for studying animal model disease',\n",
       " 'human blood plasma provides highly accessible window the proteome any individual health and disease since it inception the human proteome organization human plasma proteome project hppp ha been promoting advance the study and understanding the full protein complement human plasma and determining the abundance and modification it component review the history the hppp and the advance human plasma proteomics general including several recent achievement then present the latest build human plasma peptideatlas which yield million peptidespectrum match and distinct peptide sequence from individual experiment proteinlevel fdr globally across all experiment applying the latest human proteome project data interpretation guideline catalog protein that have least two nonnested uniquely mapping peptide nine amino acid more and additional protein with ambiguous evidence apply the same twopeptide guideline historical peptideatlas build going back and examine the progress made the past ten year plasma proteome coverage also compare the distribution protein historical peptideatlas build various rna abundance and cellular localization category then discus advance plasma proteomics based targeted mass spectrometry well affinity assay which during early target protein finally describe consideration about sample handling and study design concluding with outlook for future advance deciphering the human plasma proteome',\n",
       " 'murine brain excellent tool for studying protein expression and brain function mammal although mouse are extensively used model recapitulate various pathological condition the proteome the normal mouse brain ha not been yet reported the present study identified the total protein different part the brain cb7bl6 mouse widely used strain applying proteomic methodology the adult mouse brain wa dissected anatomically into the following region frontal cortex olfactory bulb hippocampus midbrain cerebellum hypothalamus and medulla total protein extract these region were separated twodimensional gel electrophoresis and analyzed matrixassisted laser desorption ionisation timeofflight mass spectrometry following ingel digestion with trypsin protein identification wa carried out peptide mass fingerprint thus different singlegene product were identified total expressed specifically the olfactory bulb the hippocampus the frontal cortex five the cerebellum nine the midbrain eight the hypothamamus and the medulla the majority the protein were enzyme structural protein and transporter moreover the distribution these molecule appears exhibit direct correlation with the function the brain region where they were expressed this study lead the complete characterization the normal mouse brain proteome well the protein expression profile the different brain region these result will aid addressing unmet scientific need regarding physiological and pathological brain function',\n",
       " 'over recent year analysis secretomes complete set secreted protein have been reported various organism cell type and pathology and such study are quickly gaining popularity fungi secrete enzyme can break down potential food source plant secreted protein are primarily part the cell wall proteome and human secreted protein are involved cellular immunity and communication and provide useful information for the discovery novel biomarkers such for cancer diagnosis continuous development methodology support the wide identification and quantification secreted protein given cellular state the role secreted factor also investigated the context the regulation major signaling event and connectivity map are built describe the differential expression and dynamic change secretomes bioinformatics ha become the bridge between secretome data and computational task for managing mining and retrieving information prediction can made based this information contributing the elucidation given organism physiological state and the determination the specific malfunction disease state here provide overview the available bioinformatics database and software that are used analyze the biological meaning secretome data including description the main function and limitation these tool the important challenge data analysis are mainly related the integration biological information from dissimilar source improvement database and development software will likely substantially contribute the usefulness and reliability secretome study this article part special issue entitled updated secretome',\n",
       " 'using integrative genome annotation pipeline igap for proteomewide protein structure and functional domain assignment analyzed all the protein arabidopsis thaliana threedimensional structure the level the domain are assigned fold recognition and threading based novel fold library that extends common domain classification igap being applied protein from all available proteome part comparative proteomics resource the database accessible from the web',\n",
       " 'femtogram proteomics ultrasensitive capillary zone electrophoresismass spectrometry system that based improved nanospray interface ha been developed this system used for the analysis picogram femtogram amount ecoli digest for example over protein were identified from digest tandem mass spectrometry amtsaccurate mass and time tag',\n",
       " 'proteinprotein interaction are central all cellular process understanding proteinprotein interaction therefore fundamental for many area biochemical and biomedical research and will facilitate understanding the cell processregulating machinery disease causative mechanism biomarkers drug target discovery and drug development this review summarize method for populating and analyzing the interactome highlighting their advantage and disadvantage application interactomics both the biochemical and clinical arena are presented illustrating important recent advance the field',\n",
       " 'here discus the current state research the rapidly growing field quantitative proteomics and it application system biology quantitative proteomics can successfully used for characterizing alteration protein abundance finding novel proteinprotein and proteinpeptide interaction investigating formation large macromolecular complex and elucidating temporal change organellar protein composition and phosphorylation signal transduction cascade further quantitative proteomics can directly compare activation entire signaling network response individual stimulus and discover critical difference their circuit that account for alteration cell response maturation proteomic bioinformatics application and continuous improvement proteomics and related genomics and transcriptomics technology now allows investigate cellular mechanism the integrative system level',\n",
       " 'plant tissue contain large amount secondary compound that significantly interfere with protein extraction and 2de analysis thus sample preparation crucial step prior 2de plant proteomics this tutorial highlight the guideline that need followed perform adequate total protein extraction before 2de plant proteomics briefly describe the history development and feature major sample preparation method for the 2de analysis plant tissue that trichloroacetic acidacetone precipitation and phenol extraction introduce the interfering compound plant tissue and the general guideline for tissue disruption protein precipitation and resolubilization describe detail the advantage limitation and application the trichloroacetic acidacetone precipitation and phenol extraction method enable the reader select the appropriate method for specific specie tissue cell type the current application the sample preparation method plant proteomics the literature are analyzed comparative proteomic analysis between male and female plant pistacia chinensis used example represent the sample preparation methodology 2debased proteomics finally the current limitation and future development these sample preparation method are discussed this tutorial part the international proteomics tutorial programme iptp',\n",
       " 'the human liver proteome project hlpp wa formally launched hupo the hlpp workshop wa held september during the 8th hupo world congress toronto canada the representative progress were presented from four group subsequently the workshop ended with lively discussion four topic related the project well other initiative',\n",
       " 'emerging drug resistance thwart progress chemotherapy resulting increased morbidity mortality and healthcare cost understanding the mechanism which drug resistance phenotype emerge important prolong the useful life existing drug but may also highlight pathway that play role the acquisition resistance and which may themselves present resistanceproof drug target comparative proteomic approach have demonstrated potential link drug resistance phenotype molecular change but will also prove powerful the elucidation the mechanism which drug resistance arises',\n",
       " 'evaluation the icpl isotopecoded protein labeling nonisobaric labeling technique wa performed using two different biological model two sample containing phage capsid were mixed ratio after being labeled with the light heavy version the icpl reagent the analysis this proteome demonstrated the feasibility this approach for differential quantitative proteomics and wa employed optimize the experimental parameter the icpl workflow icplmediated analysis two more complex proteome those salmonella enterica serovar typhimurium virulent strain and isogenic attenuated mutant and it comparison with the result obtained 2dpage classical approach confirmed that icpl valuable alternative other labeling technique currently use addition our result suggest that labeling the peptide level instead following the standard icpl workflow should increase both the number protein quantified and the reliability the quantification',\n",
       " 'chronic obstructive pulmonary disease copd chronic inflammatory lung disease associated with progressive obstruction airflow affecting peripheral airway the proteomic analysis highlight way identify novel biomarkers for diagnosis therapy and prognosis copd human sample for example lung tissue bronchoalveolar lavage sputum and serum have been used for copd proteomic research each with it own merit and demerit the present review aimed discussing the feasibility clinical study copd proteomics and the potential candidate for copd biomarkers detected human sample that are sensitive the progress copd diseasespecific copd and associated with the status the patient there increasing need able perform proteomic study patient with copd that describe the association with disease specificity severity progress and prognosis well monitor the efficacy therapy there urgent need establish and clarify the criterion for subject including control and data analysis and standardize the study design methodology and process this vital when designing prospective clinical study copd important clarify the source the sample the efficiency and quality when dealing with large amount candidate and the specificity biomarkers according the severity therapeutic effect progress and prognosis the disease',\n",
       " 'the increasing throughput and sharing proteomics mass spectrometry data have now yielded over onethird million public mass spectrometry run however these discovery are not continuously aggregated open and errorcontrolled manner which limit their utility facilitate the reusability these data built the massive knowledge base massivekb communitywide continuously updating knowledge base that aggregate proteomics mass spectrometry discovery into open reusable format with full provenance information for community scrutiny reusing public human data stored mass spectrometry interactive virtual environment massive the massivekb contains million precursor from protein larger than before the total and double proteome coverage million amino acid the proteome with strict libraryscale false discovery control thereby providing evidence for protein for which sufficient proteinlevel evidence wa previously missing furthermore massivekb can inform experimental design help identify and quantify new data and provides tool for community construction specialized spectral library',\n",
       " 'the soilinhabiting nematodetrapping fungus monacrosporium lysipagum capture mobile stage nematode using specialized morphological structure sticky knob that arise from mycelium study wa conducted separate the proteome lysipagum mycelium containing knob twodimensional gel resulting partial map the proteome the fungus wa grown liquid soy peptone medium supplemented with the amino acid phenylalanine and valine produce mycelium with knob protein extracted from the mycelium were separated gel electrophoresis and relatively high abundant protein were identified peptide mass fingerprinting pmf out the protein analysed pmf were identified crossspecies matching due unavailability genomic information from lysipagum this the first published report proteomic analysis nematodetrapping fungus',\n",
       " 'myeloidderived suppressor cell mdscs are heterogeneous population cell that are defined their myeloid origin immature state and ability potently suppress tcell response they regulate immune response and the population significantly increase the tumor microenvironment patient with glioma and other malignant tumor for their study mdscs are usually isolated from the spleen directly tumor from large number tumorbearing mouse although promising vivo differentiated mdsc production system have been recently developed during the last year proteomics ha emerged powerful approach analyze mdscs proteome using shotgunbased mass spectrometry providing functional information about cellular homeostasis and metabolic state global level here will revise recent proteome profiling study performed mdscs from different origin moreover will perform integrative functional analysis the protein compilation derived from these largescale proteomic study order obtain comprehensive view mdscs biology finally will also discus the potential application highthroughput proteomic approach study global proteome dynamic and posttranslational modification ptms during the differentiation process mdscs that will greatly boost the identification novel mdscspecific therapeutic target apply cancer immunotherapy',\n",
       " 'plant rapidly respond environmental fluctuation through coordinated multiscalar regulation enabling complex reaction despite their inherently sessile nature particular protein posttranslational signaling and proteinprotein interaction combine manipulate cellular response and regulate plant homeostasis with precise temporal and spatial control understanding these proteomic network are essential addressing ongoing global crisis including those food security rising global temperature and the need for renewable material and fuel technological advance mass spectrometrybased proteomics are enabling investigation unprecedented depth and are increasingly being optimized for and applied plant system this review highlight recent advance plant proteomics with emphasis spatially and temporally resolved analysis posttranslational modification and protein interaction also detail the necessity for generation comprehensive plant cell atlas while highlighting recent accomplishment within the field',\n",
       " 'the goal this work wa create model for the long persistence mycoplasma gallisepticum depleted medium and under low growth temperature followed proteomic study the model nanoforms and revertants for gallisepticum were obtained proteomic map were produced for different stage the formation nanoforms and revertants shown that protein responsible for essential cellular process glycolysis translation elongation and dnak chaperone involved the stabilization newly synthesized protein are crucial for the reversion gallisepticum vegetative form based the current data assumed that change the metabolism gallisepticum during nanoforming are not postmortal thus gallisepticum doe not transform uncultivable form but remains reversible dormant state during prolonged unfavorable condition',\n",
       " 'the adrenal gland ag are endocrine organ essential for life they undergo fetal adult developmental maturation process occurring rat during the first postnatal month the molecular modification underlying these ontogenic change are essentially unknown here report the result comparative proteomic analysis performed neonatal postnatal day versus adult postnatal day ag searching for protein with relative higher abundance each age have identified subset protein with relevant expression each developmental period using 2de and dige analysis the identified protein belong several functional category including proliferationdifferentiation cell metabolism and steroid biosynthesis study the change the proteome are correlated with change the mrna level have randomly selected several protein with differential expression and measured their relative mrna level using quantitative rtpcr cellcycle regulating protein retinoblastoma binding protein and prohibitin with contrasting effect proliferation are expressed differentially neonatal and adult progesterone metabolizing enzyme upregulated the neonatal gland might contribute the hyporesponsiveness the adrenal cortex characteristic this developmental period have also observed the adult gland marked upregulation enzyme involved nadph production thus providing the reducing power necessary for steroid hormone biosynthesis',\n",
       " 'proteomics focus the high throughput study the expression structure interaction and some extent function large number protein true understanding the functioning living cell also requires quantitative description the stoichiometry kinetics and energetics each protein complex cellular pathway classical molecular biophysical study contribute understanding these detailed property protein smaller scale than doe proteomics that individual protein are usually studied this perspective article deal with the role biophysical method the study protein the proteomic era several important physical biochemical method are discussed briefly and critiqued from the standpoint information content and data acquisition the focus conformational change and macromolecular assembly the utility dynamic and static structural data and the necessity combine experimental approach obtain full functional description the conclusion are that biophysical information protein useful adjunct standard proteomic method that data can obtained high throughput technology some instance but that hypothesisdriven experimentation may frequently required',\n",
       " '',\n",
       " '',\n",
       " 'noninvasive and easily available biological fluid urine becoming ideal sample for proteomic study recent year researcher endeavored profiling urinary proteome and discovering potential disease biomarkers however there are still many challenge the study urinary proteome for the complexity urine this article review current status urinary sample preparation including collection storage and extraction urinary protein and the overall urinary proteome analysis far which may helpful for urinary proteome analysis',\n",
       " 'owing recent advance proteomics analytical method and bioinformatics capability there growing trend toward using these capability for the development drug treat human disease including target and drug evaluation understanding mechanism drug action and biomarker discovery currently the genetic sequence many major organism are available which have helped greatly characterizing proteome model animal system and human through proteomics global profile different disease state can characterized change type and relative level well change ptms such glycosylation phosphorylation although intracellular proteomics can provide broad overview physiology cell and tissue ha been difficult quantify the low abundance protein which can important for understanding the diseased state and treatment progression for this reason there increasing interest coupling comparative proteomics method with subcellular fractionation and enrichment technique for membrane nucleus phosphoproteome glycoproteome well low abundance serum protein this review will provide example where the utilization different proteomicscoupled enrichment technique ha aided target and biomarker discovery understanding the drug targeting mechanism and mab discovery taken together these improvement will help provide better understanding the pathophysiology various disease including cancer autoimmunity inflammation cardiovascular disease and neurological condition and the design and development better medicine for treating these affliction',\n",
       " 'the depth proteomic analysis often limited the overwhelming proportion confounding background ion that compromise the identification and quantification low abundance peptide alleviate these limitation present new high field asymmetric waveform ion mobility spectrometry faims interface that can coupled the orbitrap tribrid mass spectrometer the interface provides several advantage over previous generation faims device including ease operation robustness and high ion transmission replicate lcfaimsmsms analysis ini hek protein digest showed stable ion current over extended time period with uniform peptide identification more than distinct peptide for complex tryptic digest analysis the coupling faims lcmsms enabled gain unique peptide identification compared with nonfaims experiment improvement sensitivity facilitated the identification low abundance peptide and extended the limit detection almost order magnitude the reduction chimeric msms spectrum using faims also improved the precision and the number quantifiable peptide when using isobaric labeling with tandem mass tag tmt plex reagent compared quantitative proteomic measurement for lcmsms analysis performed using synchronous precursor selection sps and lcfaimsmsms profile the temporal change protein abundance hek cell following heat shock for period faims provided fold increase the number quantifiable peptide compared with nonfaims experiment peptide from protein for faims iversusi peptide from protein with sps altogether the enhancement ion transmission and duty cycle the new faims interface extended the depth and comprehensiveness proteomic analysis and improved the precision quantitative measurement',\n",
       " 'antibody have been the workhorse reagent protein capture and quantification since their debut the rias developed yalow and berson however there are technical challenge the use antibody highly multiplexed array aimed measuring hundred even thousand protein one time describe here recently developed class synthetic proteinbinding reagent slow offrate modified aptamer discus the chemical makeup and protein binding specification slow offrate modified aptamer reagent compare them traditional aptamers and antibody briefly describe the novel proteomic assay that take advantage their unique property and provide several example their multiple application biomarker discovery and validation across range biomedical science question',\n",
       " 'the function protein defined it interaction with other protein and molecule mapping protein interaction can highlight new functionality for known protein can even define the function novel protein with the draft sequence the human genome now available possible perform highthroughput mapping proteinprotein interaction human which termed functional proteomics the development functional proteomics are particularly timely since pharmaceutical company are searching for technology that will strengthen their genomic effort and prioritize their drug discovery pipeline this article review recent development functional proteomics',\n",
       " 'the advent multiple highthroughput technology ha brought drug discovery round almost full circle from pharmacological testing compound vivo engineered molecular target assay and back integrated phenotypic screen cell and organism the past primary screen identify new pharmacological agent involved administering compound animal and monitoring pharmacologic endpoint for example antihypertensive agent were identified dosing spontaneously hypertensive rat with compound and observing whether their blood pressure dropped taking this phenomenological approach scientist were focused the final goal this example lowering blood pressure rather than developing understanding the target target the compound were impacting with the evolution rational targetbased approach scientist were able study the direct interaction compound with their intended target expecting that this would lead moreselective and safer therapeutic with the industrialization screening referred hts hundred thousand compound were screened robotdriven assay against target interest with this goal mind however unintentional outcome the migration from vivo primary screen highly targetspecific hts assay wa reduction biological context caused the separation the target from other cellular protein and process that might impact it function recognition the potential consequence this oversimplification drove the modification hts process and equipment compatible with cellular assay',\n",
       " 'aging associated with progressive loss skeletal muscular function that often lead progressive disability and loss independence although muscle aging well documented the molecular mechanism this condition still remain unclear gain greater insight into the change associated with aging skeletal muscle performed quantitative proteomic analysis young month and aged month mouse gastrocnemius muscle using mtraq stable isotope mass tag identified and quantified total peptide corresponding protein protein probability among them protein were more than fold upregulated and protein were more than fold downregulated aged muscle compared with young muscle ontological analysis revealed that differentially expressed protein belonged distinct functional group including ion homeostasis energy metabolism protein turnover and ca2 signaling identified protein included aralar1 βenolase fatty acidbinding protein 3hydroxyacylcoa dehydrogenase hadh fbox protein fbox and leucinerich repeat protein voltagedependent ltype calcium channel subunit beta1 ryanodine receptor ryr and calsequestrin ectopic expression calsequestrin c2c myoblast resulted decreased activity nuclear factor activated tcells and increased level atrogin1 and murf1 ligase suggesting that these differentially expressed protein are involved muscle aging',\n",
       " 'the male gamete sperm can fertilize egg and pas the male genetic information the offspring ha long been thought that sperm had simple protein composition effort have been made identify the sperm proteome different specie and only about protein were reported however with advanced mass spectrometry and optimized proteomics platform successfully identified human sperm protein which were testisspecific this large number identified protein indicates the complex composition and function human sperm comparison with the sperm transcriptome reveals little overlap which show the importance future study sperm the protein level interestingly many signaling pathway such the il6 insulin and tgfbeta receptor signaling pathway were found overrepresented addition found that protein were annotated target known drug three four drug studied were found affect sperm movement this indepth human sperm proteome will rich resource for further study sperm function and will provide candidate target for the development male contraceptive drug',\n",
       " 'proteomics the study cellular protein expression evolving technology platform that ha the potential identify novel protein involved key biological process the cell that may serve potential drug target while proteomics ha considerable theoretical promise individual cellstissues have the potential generate many million protein while the current analytical technology that involve the use timeconsuming two dimensional gel electrophoresis 2dige and various mass spectrometry technique are unable handle complex biological sample without multiple highresolution purification step reduce their complexity this can significantly limit the speed data generation and replication and requires the use bioinformatic algorithm reconstitute the parent proteome process that doe not always result reproducible outcome addition membrane bound protein receptor and ion channel that are the target many existing drug are not amenable study due part limitation current proteomic technique and also these being present low abundance and thus disproportionally represented proteome profile subproteomes with reduced complexity have been used generate data related specific hypothesisdriven question regarding target identification proteininteraction network and signaling pathway however progress date with the exception diagnostic proteomics the field cancer ha been exceedingly slow with inability put such study the context larger proteome limiting the value the information additionally the pathway for target validation which can more accurately described the preclinical level target confidence building remains unclear important that the ability measure and interrogate proteome match expectation avoiding repetition the disappointment and subsequent skepticism that accompanied what proved unrealistic expectation for the rapid contribution data based the genome map biomedical research',\n",
       " 'understanding gene function requires complementation gene and gene expression analysis the systematic analysis protein progress plant proteomics ha been lagging behind animal and microbial proteomics due the lack plant genome data and the problem involved successful protein extraction from plant material with the sequencing more and more plant genome this slow progress will soon overcome the moss physcomitrella patens model organism the field plant functional genomics patens the first seedless plant for which the complete genome wa sequenced genome annotation currently progress while identification protein requires knowledge all coding gene the organism under study gene annotation and functional characterization benefit greatly from the finding proteome analysis the proteome patens accessible and approach are under way increase the spectrum proteomic method applied this plant here provide protocol for the extraction protein from patens and describe the basic and still most important method proteome analysis twodimensional polyacrylamide electrophoresis protein this technique not entirely unjustifiably ha the reputation being unpredictably complicated provide detailed protocol intended reduce the reluctance that many scientist may have using this technique',\n",
       " '',\n",
       " 'plant mitochondrion perform wide range function the plant cell ranging from providing energy and metabolic intermediate via coenzyme biosynthesis and their own biogenesis retrograde signaling and programmed cell death perform these function they contain proteome different protein expressed some cell under some condition the vast majority these protein are imported many case dedicated protein import machinery recent proteomic study have identified about different protein both arabidopsis and potato mitochondrion but even for energyrelated protein the most wellstudied functional protein group mitochondrion the protein are recognized mitochondrial even one six the most widely used prediction algorithm the mitochondrial proteome contain protein representing wide range different function some protein group like energyrelated protein membrane transporter and novo fatty acid synthesis appear well covered the proteome while others like rna metabolism appear poorly covered possibly because low abundance the proteomic study have improved our understanding basic mitochondrial function have led the discovery new mitochondrial metabolic pathway and are helping towards appreciating the dynamic role the mitochondrion the response the plant cell biotic and abiotic stress',\n",
       " 'the need for better understanding cellular heterogeneity ha pushed mass spectrometry technology the analysis singlecell and singlecelltype proteome although several challenge still limit their widespread implementation among the effort toward singlecell and lowinput analysis there the adoption dataindependent acquisition method increase analytical sensitivity here revisited the use linear ion trap mass analyzer dataindependent acquisition method and demonstrate their benefit boost peptide and protein identification lowinput proteome',\n",
       " 'large scale proteomic strategy rely database interrogation thus only referenced protein can identified recently alternative protein altprots translated from nonannotated alternative open reading frame altorfs were discovered using customized database because their small size which confers them peptidelike physicochemical property they are more difficult detect using standard proteomics strategy this study tested different preparation workflow for improving the identification altprots nch human glioma cell line the highest number identified altprots wa achieved with ripa buffer boiling water extraction followed acetic acid precipitation',\n",
       " 'western blotting analysis the most popular and widely used methodology for protein detection and characterization over recent decade accordance with the advancement the technology for the acquisition signal quantitative value used present the abundance target protein complex sample thereby requiring the use specific protein internal reference that represent total protein heretofore protein encoded housekeeping gene such gapdh βtubulin and βactin have been commonly used loading control without any hesitation because their mrna expression level tend high and constant many different cell and tissue experimentally however some the housekeeping reference protein are often displayed with inconsistent expression level both homogeneous and heterogeneous tissue and term mrna level they have weak correlation the abundance protein estimate accurate reliable and reproducible protein quantification crucial define appropriate reference control for this paper explored the recently released largescale human proteomic database proteomicsdb including liquid chromatography tandemmassspectrometry data from human tissue and suggest ubiquitously and constitutivelyexpressed putative internalreference control for the quantification differential protein expression intriguingly the most commonly used known housekeeping gene were entirely excluded our newly defined candidate although the application the candidate under many different biological condition and other organism are yet empirically verified propose reliable potential loading control for analysis this paper',\n",
       " 'mass spectrometry sensitive and specific analytical technique that capable providing qualitative and quantitative data resolve the protein element biochemical pathway that are altered antibiotic here present method study antibiotic susceptibility change protein abundance exemplified klebsiella pneumoniae gramnegative pathogen that colonizes mucosal surface the human gastrointestinal and respiratory tract cultured bacteria are exposed antibiotic the total proteome collected cell pellet are converted complex peptide mixture filteraided sample preparation fasp and the peptide are further processed optimized desalting procedure mixture peptide from klebsiella pneumoniae proteome are analyzed highresolution mass spectrometry that coupled sensitive and comprehensive nanoliquid chromatography nanolc the generic method described here for the identification and quantification the proteome will provide snapshot differential protein abundance resulting from antimicrobial sensitivity which can used model directed perturbation the global system and select target specific interest for further study',\n",
       " 'ubiquitin essential highlyconserved small regulatory protein eukaryotic cell covalently modifies wide variety targeted protein the form monomer and polymer altering the conformation and binding property the protein and thus regulating proteasomal delivery protein activity and localization mass spectrometry ha emerged indispensable tool for indepth characterization protein ubiquitination ubiquitinated protein cell lysates are usually enriched affinity chromatography and subsequently analyzed mass spectrometry for identification and quantification ubiquitinconjugated amino acid residue can determined unique mass shift caused the modification moreover the complex structure polyubiquitin chain substrate can dissected bottomup and middledown mass spectrometric approach revealing potential novel function polyubiquitin linkage here review the advance and caveat these strategy emphasizing caution the validation ubiquitinated protein and the interpretation raw data',\n",
       " 'recently several massspectrometry and proteindenaturationbased proteomic method have been developed facilitate protein target discovery effort drug modeofaction study these method which include the stability protein from rate oxidation sprox pulse proteolysis chemical denaturation and protein precipitation cpp and thermal proteome profiling tpp technique have been used increasing number application recent year however while the advantage and disadvantage using these different technique have been reviewed the analytical characteristic these method have not been directly compared reported here such direct comparison using the wellstudied immunosuppressive drug cyclosporine csa and the protein yeast cell lysate also described onepot strategy that can utilized with each technique streamline data acquisition and analysis find that there are benefit utilizing all four strategy for protein target discovery including increased proteomic coverage and reduced false positive rate that approach moreover the onepot strategy described here make such experiment feasible because the fold reduction reagent cost and instrument time affords',\n",
       " 'two pea line pisum sativum with contrasted behaviour towards chilling and subsequent frost were studied proteomic approach better understand cold acclimation following chilling period the champagne line becomes tolerant frost whereas terese remains sensitive variance analysis allowed select statistically variable spot with identified protein leaf stem and root these protein were shared out protein related chilling response cold acclimation the better adaptation champagne chilling might related higher content protein involved photosynthesis and defence mechanism moreover champagne might prevent freezing damage particularly thanks higher constitutive expression housekeeping protein related terese after three day subsequent frost proteome previously chilled plant also showed significant difference compared unchilled plant out statistically variable spot leaf stem and root protein were identified these protein were related frost response frost resistance seems that champagne could resist frost with the reorientation the energy metabolism',\n",
       " 'the global human proteomics community fully engaged project that aim create better understanding human biology and it complexity and provide product from this new knowledge that will some way benefit humanity human proteomics like any other scientific enterprise need identify area direction and development both for the near future completing current research project and into the longterm for the engagement with even more complex challenge this editorial highlight and discus four important area that collectively believe require attention and demand collective response going forward these four area are provide highquality standardized sensitive specific quantitative and readily accessible protein peptide other biomarkers health disease response therapy into the approval process regulatory agency food and drug administration fda and obtaining approval from the relevant agency for their use clinical other testing setting implement standard process for collecting processing and storing human clinical sample biorepositories and enforcement measure ensure subject integrity including informed consent for the downstream use sample and registration subject identity within study database test and validate mass spectrometry technology platform that hold much promise for creating opportunity for obtaining new important knowledge level detection previously not achievable organize clinical discovery operation and activity intuitive manner meet the challenge increased interest the science provide and diminishing level centrally financed resource and infrastructure support',\n",
       " 'cancer stem cell characterization represents breakthrough cancer research despite evidence showing the existence and the role cancer stem cell osteosarcoma onset and progression little known about their specific surface phenotype address this issue carried out cytometric analysis with antibodyarray comprising membrane protein comparing the stem and differentiated cell experimental model chose the stemlike cell line 3aminobenzamideos and it parental differentiated cell line identified differentially expressed homogeneously expressed and not expressed protein the two cell line model thus defining surface protein signature specific for each them furthermore selected erk mitogenactivated protein kinase potential pathway correlated with process that characterize tumorigenic potential and stemness 3aminobenzamideos cell',\n",
       " 'the term peptidomics can defined the systematic analysis the peptide content within cell organelle tissue organism the science peptidomics usually refers the study naturally occurring peptide another meaning refers the peptidomics approach protein analysis ancient roman strategy divide impera divide and conquer reflects the essence peptidomics most effort this field spent purifying and dividing the peptidomes which consist ten hundred sometimes thousand functional peptide followed their structural and functional characterisation this chapter introduces the concept peptidomics outline the range methodology employed and describes key target the peptide group which are often sought after such study',\n",
       " 'regulation gene expression the level protein synthesis crucial element driving how the genetic landscape expressed however are still limited technology that can quantitatively capture the immediate proteomic change that allow cell respond specific stimulus here present method capture and identify nascent proteome situ across different cell type without disturbing normal growth condition using opropargylpuromycin opp cellpermeable opp rapidly label nascent elongating polypeptide which are subsequently conjugated biotinazide using click chemistry and captured with streptavidin bead followed digestion and analysis using liquid chromatographytandem mass spectrometry our technique oppmediated identification oppid allows detection widespread proteomic change within short 2hour pulse opp illustrate our technique recapitulating alteration proteomic network induced potent mammalian target rapamycin inhibitor mln addition employing oppid identify more than protein and uncover distinct protein network underlying early erythroid progenitor and differentiation state not amenable alternative approach such amino acid analog labeling present oppid method quantitatively identify nascent proteome across array biological context while preserving the subtlety directing signaling the native cellular environment',\n",
       " 'the identification the activator heat shock protein hsp atpases aha1 proteinprotein interaction ppi network will provide critical insight into the relationship aha1 with multimolecular complex and shed light onto aha1s interconnection with hspregulated biological function flagtagged aha1 wa overexpressed hela cell and isolated antiflag affinity pull down followed trypsin digestion and identification coadsorbing protein liquid chromatographytandem mass spectroscopy lcmsms probabilitybased identification aha1 ppis wa generated from the lcmsms analysis using relative quantification strategy spectral counting comparing the scbased protein level between aha1 pulldown sample and negative control aha1interacting protein were identified that were quantitatively enriched the pulldown sample over the control the identified aha1interacting protein are involved wide number intracellular bioprocesses including dna maintenance chromatin structure rna processing translation nucleocytoplasmic and vesicle transport among others the interaction the identified protein with aha1 were further confirmed western blotting demonstrating the reliability our affinitypurificationcoupled quantitative scms strategy our proteomic data suggests that aha1 may participate diverse biological pathway facilitate hsp chaperone function response stress',\n",
       " 'over the past several year large effort ha been focused improvement twodimensional gel electrophoresisbased proteomics technology and development novel approach for proteome analysis here describe the application alternative strategy for the analysis complex proteome the strategy combine isoelectric focusing immobilized gradient strip ingel ief mass spectrometry and bioinformatics protein mixture separated ingel ief and the entire strip cut into set gel section protein each gel section are digested with trypsin and the tryptic peptide are subjected liquid chromatographynanoelectrosprayquadrupole iontrap tandem mass spectrometry lcesimsms the lcesimsms data are used identify the protein through search protein sequence database using this ingel ieflcmsms strategy have identified protein from human pituitary this study demonstrates the potential the ingel ieflcmsms approach for analysis complex mammalian proteome',\n",
       " '',\n",
       " '',\n",
       " 'reproductive fluid from the female reproductive tract are gaining attention for their potential support and optimize reproductive process including gamete maturation and embryo culture vitro quantitative proteomics powerful way decipher the proteome reproductive tract fluid and identify biologically relevant protein the present review describes proteomic strategy for analysing female reproductive fluid protein addition considers the strategy for the preparation oviductal uterine and follicular fluid sample finally highlight the main result quantitative proteomic study providing insight into the biological process related reproductive biology farm animal significance assisted reproductive technology art have become vitally important for farm animal breeding and much effort going into the optimization and refinement the technique there are also attempt imitate physiological condition adding reproductive fluid individual fluid protein improve vitro procedure detailed knowledge the reproductive fluid proteome indispensable the present review summarizes the most widely used quantitative proteomic approach for the analysis fluid from the female reproductive tract and highlight the potential quantitative proteomics delineate reproductive process and identify candidate protein for art farm animal',\n",
       " 'the availability extensive genome information for many different microbe including unculturable specie mixed community from environmental sample ha enabled systemsbiology interrogation providing mean access genomic transcriptomic and proteomic information this end metaproteomics exploit the power highperformance mass spectrometry for extensive characterization the complete suite protein expressed microbial community environmental sample',\n",
       " 'drosophila melanogaster ha been used genetic model organism understand the fundamental molecular mechanism human biology including memory formation that ha been reported involving protein synthesis andor posttranslational modification this study employed proteomic platform based fluorescent 2de and malditof build standard melanogaster head proteome map for proteomeproteome comparison order facilitate the comparison interactive database ha been constructed for systematically integrating and analyzing the proteome from different condition and further implicated study human disease related melanogaster model summary the fundamental head proteomic database and bioinformatic analysis will useful for further elucidating the biological mechanism such memory formation and neurodegenerative disease',\n",
       " '',\n",
       " 'peptide cleanup essential for the removal contaminating substance that may introduced during sample preparation step bottomup proteomic workflow recent study have described benefit carboxylatemodified paramagnetic particle over traditional reversedphase method for detergent and polymer removal but challenge with reproducibility have limited the widespread implementation this approach among laboratory overcome these challenge the current study systematically evaluated key experimental parameter regarding the use carboxylatemodified paramagnetic particle and determined those that are critical for maximum performance and peptide recovery and those for which the protocol tolerant deviation these result supported the development detailed easytouse standard operating protocol termed sp2 which can applied remove detergent and polymer from peptide sample while concentrating the sample solvent that directly compatible with typical lcm workflow demonstrate that sp2 can applied phosphopeptides and glycopeptides and that the approach compatible with robotic liquid handling for automated sample processing altogether the result this study and accompanying detailed operating protocol for both manual and automated processing are expected facilitate reproducible implementation sp2 for various proteomics application and will especially benefit core shared resource facility where unknown unexpected contaminant may particularly problematic',\n",
       " 'today proteomics usually compare clinical sample use bottomup profiling with high resolution mass spectrometry where all protein product single gene are considered integral whole the same time proteomics proteoforms which considers the variety protein specie offer the potential discover valuable biomarkers proteoforms are protein specie that arise consequence genetic polymorphism alternative splicing posttranslational modification and other lessexplored molecular event the comprehensive observation proteoforms ha been exclusive privilege topdown proteomics here review the possibility bottomup approach address the microheterogeneity the human proteome special focus given shotgun proteomics and structurebased bioinformatics source hypothetical proteoforms which can potentially verified targeted mass spectrometry determine the relevance proteoforms disease',\n",
       " 'the analysis proteinprotein interaction ha the growing importance modern proteomics caused necessity indepth functional study protein which rule operate living system due each other interaction the stable dynamic protein complex this review the brief description and comparative evaluation most frequently used approach existing variety protein interactomics method are given',\n",
       " 'for the first time proteomics and biochemical variable have been employed unravel the growth strategy for the different root growth period ginseng panax ginseng may araliaceae enzymatic activity and cellular content except for starch related defence and metabolism were significantly increased the slowgrowth period but decreased the fastgrowth period proteomic characterisation twodimensional gel electrophoresis 2de showed differentially expressed spot spot were upregulated and spot were downregulated the slowgrowth period when compared the fastgrowth period the identification these spot indicated that the major group differential protein were associated with energy metabolism and defence which wa consistent with the change observed the biochemical measurement these result clearly demonstrate that ginseng store energy during it fastgrowth period promote root elongation whereas expends energy improve the synthesis secondary metabolite and stress resistance during it slowgrowth period the level many protein were changed during the conversion period from fast slow growth providing new insight into ginseng proteome evolution the proposed hypothetical model explains the interaction metabolic protein associated with the growth strategy ginseng',\n",
       " 'fueled evergrowing dna sequence information proteomicsthe large scale analysis proteinshas become one the most important discipline for characterizing gene function for building functional linkage between protein molecule and for providing insight into the mechanism biological process highthroughput mode now possible examine the expression more than protein using mass spectrometry technology coupled with various separation method highthroughput yeast twohybrid approach and analysis protein complex using affinity tag purification have yielded valuable proteinprotein interaction map largescale protein tagging and subcellular localization project have provided considerable information about protein function finally recent development protein microarray technology provide versatile tool study proteinprotein proteinnucleic acid proteinlipid enzymesubstrate and proteindrug interaction other type microarrays though not fully developed also show great potential diagnostics protein profiling and drug identification and validation this review discus highthroughput technology for proteome analysis and their application also discussed are the approach used for the integrated analysis the voluminous set data generated proteome analysis conducted global scale',\n",
       " 'describe novel method for using hexafluoroisopropanol hfip induced coacervates with variety surfactant extract protein from yeast whole cell lysate and conduct global proteomic investigation the extracted protein yeast whole cell lysates were prepared and protein were extracted using two workflow protein were extracted into the coacervate generated from the mixture hfip surfactant and cell lysate protein were extracted from cell lysate using surfactant solution and hfip wa added the supernatant generate coacervate phase with concentrated protein both initial extraction were followed modified filteraided sample preparation fasp cleanup the methodology yield significant protein concentration the coacervate phase order magnitude increase concentration and increase proteome sequence coverage membrane protein identification and identification protein from low abundance cellular subfractions total id',\n",
       " 'proteomic investigation normal and diseased brain state ha the potential reveal novel molecular therapeutic and diagnostic target for multitude pathological central nervous system condition due their unique property integral membrane protein are likely play central role the aetiology these disorder these property however have prevented comprehensive analysis this important class protein recent advance sample preparation and proteomic quantification platform specifically focused recovery and enrichment integral membrane protein are discussed',\n",
       " 'with the recent development proteomic technology complete human proteome project hpp appears feasible for the first time however there still debate how should designed and what should encompass proteomics speak the debate revolves around the central question whether genecentric proteincentric proteomics approach the most appropriate way forward this paper try shed light what these definition mean how largescale proteomics such hpp can insert into the larger omics chorus and what can reasonably expect from hpp the way ha been proposed far',\n",
       " 'advancement proteome analytical technique and the development protein database have been helping understand the physiology and subtle molecular mechanism behind biofilm formation bacteria this review highlight how the evolving proteomic approach have revealed fundamental molecular process underlying the formation and regulation bacterial biofilms based the survey research report available differential expression protein biofilms bacterial from wide range environment four important cellular process viz metabolism motility transport and stress response that contribute formation bacterial biofilms are discussed this review might answer how protein related these cellular process contribute significantly stabilizing biofilms different bacteria diverse environmental condition',\n",
       " '',\n",
       " 'proteomic genotyping detects single amino acid polymorphism infer the genotype corresponding nonsynonymous snp like any dna genotype these inference can used estimate random match probability fingermark are common source biological evidence that sample limited and highly variable source identifying dna genetically variant peptide from fingermark that contain single amino acid polymorphism are additional source identifying genetic information discover these peptide biomarkers epidermal corneocytes from subject were isolated processed digested with trypsin and applied mass spectrometry the resulting proteomic and matching exome datasets were used discover characterize and validate genetically variant peptide average genetically variant peptide were detected from each subject resulting total snp allele inference with true and false positive false discovery rate random match probability were estimated using the genotype frequency from the matching major population the genome project estimate ranged value ×sup8sup with median probability ×sup6sup furthermore the proteomicallyinferred genotype are likely compatible with the strbased random match probability estimate since the closest str locus wa from the nearest gvpinferred snp this project represents novel mode genetic information that can obtained from fingermark and ha the potential complement other method human identification including analysis ridge pattern touch dna',\n",
       " 'every year since the central and eastern european proteomic conference ceepc ha excelled representing stateoftheart proteomics and around central and eastern europe and linking international institution worldwide it mission remains contribute all approach proteomics including traditional and oftenrevisited methodology well the latest technological achievement clinical quantitative and structural proteomics with view system biology variety process the 9th ceepc wa held from june the institute bioorganic chemistry polish academy science poznań poland the scientific program stimulated exchange proteomic knowledge whilst the spectacular venue the conference allowed participant enjoy the cobblestoned historical city poznań',\n",
       " 'the main intricacy the human proteome that tremendously complex and composed from diverse and heterogeneous gene product these product are called protein specie proteoforms and are the smallest unit the proteome pursuit the comprehensive profiling the human proteome significant advance the technology called topdown mass spectrometry based proteomics have been made however the scale performance this approach still far behind the bottomup peptidecentric technique the classical twodimensional electrophoresis 2de the most powerful and convenient method for separation proteoforms remains superior method topdown proteomics here some aspect approach for establishing inventory proteoforms based 2de and mass spectrometry are discussed biological significance the systematic effort the human proteome project map the entire human proteome greatly depend currently available and emerging technique and approach here the possibility visual representation the human proteome combination virtualexperimental 2de with protein identification mass spectrometry immunologically discussed application this approach several profile gene product show it convenience informative representation the whole proteome and single gene product proteoforms protein specie this approach could very helpful the emerging global inventory all human proteoforms',\n",
       " '',\n",
       " 'tissue microarrays have become essential tool translational pathology they are used confirm result from other experimental platform such expression microarrays well primary tool explore the expression profile protein immunohistochemical analysis tissue microarrays are routinely used molecular epidemiology drug development and determining the diagnostic prognostic and predictive value new biomarkers applying traditional protein based assay well novel assay the platform tissue microarrays have gained new utility proteomic tool for both basic science well clinical investigation this article will explore the new approach that are being applied tissue microarrays characterize the human proteome and new technology that allow tissue microarrays function protein array',\n",
       " 'micrornas mirnas regulate gene expression through translational repression and rna degradation recently developed highthroughput proteomic method measure gene expression change protein level and therefore can reveal the direct effect mirnas translational repression here present web server proteomirexpress that integrates proteomic and mrna expression data together infer mirnacentered regulatory network with both type highthroughput data from the user proteomirexpress able discover not only mirna target that have decreased mrna but also subgroup target with suppressed protein whose mrna are not significantly changed with decreased mrna whose protein are not significantly changed which are usually ignored most current method furthermore both direct and indirect target mirnas can detected therefore proteomirexpress provides more comprehensive mirnacentered regulatory network used several published data ass the quality our inferred network and prove the value our server proteomirexpress available online with free access academic user',\n",
       " 'cervicalvaginal fluid cvf potential rich source biomarkers for enhancing our understanding human parturition and pathologic condition affecting pregnancy this study performed comprehensive survey the cvf proteome pregnancy utilizing multidimensional liquid chromatography 2dlc coupled with mass spectrometry and gelelectrophoresisbased protein separation and identification total unique protein were identified using multiple protein identification algorithm metabolism and immune responserelated protein are the major functional category represented the cvf proteome comparison the cvf serum and amniotic fluid proteome showed that protein are unique cvf while and cvf protein also occur serum and amniotic fluid respectively this data set provides foundation for evaluation these protein potential cvf biomarkers for noninvasive diagnosis pregnancyrelated disorder',\n",
       " 'the entire complement protein expressed the genome organism under set defined condition represents the proteome the need for rapid efficient and highthroughput technology understand the proteome organism have resulted the development protein microarrays traditional cellbased technology for the generation protein array involve laborious and protracted procedure which also pose several other technical problem these limitation have been suitably overcome cellfree expression system that carry out rapid situ synthesis protein from their corresponding dna template this article provides overview commonly used cellfree system that have been used for the generation protein microarrays with special emphasis the diverse application cellfree synthesisbased protein microarrays',\n",
       " '',\n",
       " 'massspectra based proteomic profile have received widespread attention potential tool for biomarker discovery and early disease diagnosis major dataanalytical problem involved the extremely high dimensionality number feature variable proteomic data particular when the sample size small this article review dimensionality reduction method that have been used proteomic biomarker study then focus the problem selecting the most appropriate method for specific task dataset and proposes method combination potential alternative singlemethod selection finally point out the potential novel dimension reduction technique particular those that incorporate domain knowledge through the use informative prior causal inference',\n",
       " 'redoxregulated protein play fundamentally important role not only during the defense organism against oxidative stress condition but also target cellular signaling event this realization ha spurred the development proteomic technique geared towards characterizing the redoxome protein with highly reactive cysteine residue whose thiol oxidation state control the function the protein and extension the pathway they are part will here summarize the most recent advance made the field redox proteomic analysis aimed elucidate the cellular redox network that appear control prokaryotic and eukaryotic organism',\n",
       " 'report the comparative proteomic characterization the venom bothrops atrox barnetti and pictus the venom were subjected rphplc and the resulting fraction analyzed sdspage the protein were cut from the gel digested with trypsin and identified via peptide mass fingerprint and manual sequencing selected peptide malditoftof mass spectrometry around protein were identified belonging only protein family metalloproteinases the class and piii were the most abundant protein all venom based peak area followed phospholipasesa2 disintegrins and serine proteinase and some these protein occurred several isoforms contrast cysteinerich secretory protein and lamino acid oxidase appeared only single isoforms and were found only atrox and barnetti ctype lectin were also detected all venom but low level furthermore the venom contain variable number peptide lt3 kda and nonprotein compound which were not considered this work the protein composition the investigated bothrops specie agreement with their pharmacological and pathological effect',\n",
       " 'the explosion biomedical data both the genomic and proteomic side well clinical data will require complex integration and analysis provide new molecular variable better understand the molecular basis phenotype currently much data exist silo and not analyzed framework where all data are brought bear the development biomarkers and novel functional target this beginning change network biology approach which emphasize the interaction between gene protein and metabolite provide framework for data integration such that genome proteome metabolome and other omics data can jointly analyzed understand and predict disease phenotype this review recent advance network biology approach and result are identified common theme the potential for network analysis provide multiplexed and functionally connected biomarkers for analyzing the molecular basis disease thus changing our approach analyzing and modeling genome and proteomewide data',\n",
       " 'functional genomics the grampositive model organism bacillus subtilis reveals valuable insight into basic concept cell physiology this study monitor temporal change the proteome transcriptome and extracellular metabolome subtilis caused glucose starvation for proteomic profiling combination vivo metabolic labelling and shotgun mass spectrometric analysis wa carried out for five different proteomic subfractions cytosolic integral membrane membrane surface and extracellular proteome fraction leading the identification the predicted proteome subtilis quantitative proteomic and corresponding transcriptomic data were analysed with voronoi treemaps linking functional classification and relative expression change gene product according their fate the stationary phase the obtained data comprise the first comprehensive profiling change the membrane subfraction and allow indepth analysis major physiological process including monitoring protein degradation',\n",
       " '',\n",
       " 'rpxiprp hplc separation scheme wa combined with online esiit tandem offline maldi tandem tof and applied the analysis eukaryotic subcellular proteome previous proteomic study were complemented the approval the approach eukaryotic proteome using the fission yeast schizosaccharomyces pombe the major focus wa set the analysis primary human hepatocyte microsome representing compartment high interest due it involvement xenobiotic detoxification and cholesterol homeostasis the protein identified from two donor are involved cholesterol homeostasis xenobioticlipid metabolism the identified protein belong the group membrane protein difficult investigate using gelbased proteomic approach further demonstrated the reproducibility and comparability the approach and reduced the amount sample load almost with only minor loss information about the protein identified the sample the presented study clearly demonstrates the good applicability the experimental setup the analysis subcellular proteome including large membrane fraction where only low amount sample material are available',\n",
       " 'bottomup proteomics relies the use protease and the method choice for identifying thousand protein group complex sample topdown proteomics ha been shown robust for direct analysis small protein and offer solution the peptidetoprotein inference problem inherent with bottomup approach here describe the first largescale integration genomic bottomup and topdown proteomic data for the comparative analysis patientderived mouse xenograft model basal and luminal human breast cancer whim2 and whim respectively using these wellcharacterized xenograft model established the national cancer institute clinical proteomic tumor analysis consortium compared and contrasted the performance bottomup and topdown proteomics detect cancerspecific aberration the peptide and proteoform level and measure differential expression protein and proteoforms bottomup proteomic analysis the tumor xenograft detected almost time many coding nucleotide polymorphism and peptide resulting from novel splice junction than topdown for protein the range kda where quantitation wa performed using both approach bottomup proteomics quantified protein group from peptide while topdown proteomics quantified proteoforms mapping protein example both concordant and discordant quantitation were found ratio providing unique opportunity for topdown fill missing information the two technique showed complementary performance with bottomup yielding eight time more identification kda protein xenograft proteome but failing detect difference certain posttranslational modification ptms such phosphorylation pattern change alphaendosulfine this work illustrates the potency combined bottomup and topdown proteomics approach deepen our knowledge cancer biology especially when genomic data are available',\n",
       " '',\n",
       " 'global protein analysis human gallbladder tissue vital for identification molecular regulator and effector it physiological activity here employed genomewide deep rna sequencing analysis human tissue identify the gene overrepresented the gallbladder and complemented with antibodybased immunohistochemistry human tissue characterized human gallbladder protein and identified gallbladderspecific protein with elevated expression the gallbladder compared the other analyzed tissue five gene were categorized enriched with least fivefold higher level gallbladder gene were categorized group enriched with elevated transcript level gallbladder shared with least one other tissue and gene were categorized enhanced with higher expression than the average expression other tissue explored the localization the gene within the gallbladder through celltype specific antibodybased protein profiling and the subcellular localization the gene through immunofluorescentbased profiling finally revealed the biological process and metabolic function carried out these gene through the use kegg pathway and hmr that compilation the human metabolic reaction demonstrated the result the combined analysis the transcriptomics and affinity proteomics',\n",
       " 'the significant potential tissuebased proteomic biomarker study can restricted difficulty accessing sample optimal freshfrozen form while archival formalinfixed tissue collection with attached clinical and outcome data represent valuable alternate resource the use formalin fixative which induces protein crosslinking ha generally been assumed render them unsuitable for proteomic study however this view ha been challenged recently with the publication several paper accomplishing variable degree heatinduced reversal crosslinks although still it infancy and requiring the quantitative optimisation several critical parameter formalinfixed tissue proteomics hold promise powerful tool for biomarkerdriven translational research here critically review the current status research the field highlighting challenge which need addressed for robust quantitative application protocol ensure confident high impact inference can made',\n",
       " 'basic science research hematology ha been determining the function gene product using classical approach that typically involve studying one few gene time proteomics defined the study protein property large scale provides tool globally analyze malignant hematologic cell major challenge cancer therapy the identification drug that kill tumor cell while preserving normal cell differential display via proteomics enables analysis direct well sideeffects drug molecular level proteomics also allows better understanding cell signaling pathway involved during apoptosis hematologic cell storing the information electrophoresis database enhances the efficiency proteome research malignant cell finally the work needed carried out proteomic analysis prior routine clinical adoption discussed and the necessity for multiinstitutional collaboration emphasized',\n",
       " 'proteomic technology have begun providing evidence that viable embryo posse unique protein profile some these potential protein biomarkers have been identified extracellular and could used the development noninvasive quantitative method for embryo assessment the field assisted reproductive technology would benefit from defining the human embryonic proteome and secretome thereby expanding our current knowledge embryonic cellular process',\n",
       " 'peptide mass spectrometry invaluable technique globally quantify the proteome central proteome profiling are efficient method extract protein digest protein into peptide and enrich for posttranslationally modified peptide prior mass spectrometry this chapter describe method extract protein process them into peptide and optionally enrich for phospho and acetylpeptides prior analysis mass spectrometry',\n",
       " 'the integration proteomic method virology ha facilitated significant breadth biological insight into mechanism virus replication antiviral host response and viral subversion host defense throughout the course infection these cellular mechanism rely heavily the formation temporally and spatially regulated virushost proteinprotein interaction reviewed here are proteomicbased approach that have been used characterize this dynamic virushost interplay specifically discussed are the contribution integrative mass spectrometry antibodybased affinity purification protein complex crosslinking and protein array technique for elucidating complex network virushost protein association during infection with diverse range rna and dna virus the benefit and limitation applying proteomic method virology are explored and the contribution these approach important biological discovery and inspiring new tractable avenue for the design antiviral therapeutic highlighted',\n",
       " 'thousand manmade chemical are annually released into the environment agriculture transport industry and other human activity general chemical analysis environmental sample used ass the pollution status specific ecosystem complicated the complexity the mixture and some case the very low toxicity threshold chemical present that sense proteomics approach capable detecting subtle change the level and structure individual protein within the whole proteome response the altered surroundings ha obvious application the field ecotoxicology addition identifying new protein biomarkers can also help provide insight into underlying mechanism toxicity despite being comparatively new field with number caveat proteomics application have spread from microorganism and plant invertebrate and vertebrate gradually becoming established technology used environmental research this review article highlight recent advance the field environmental proteomics mainly focusing experimental approach with potential understand toxic mode action and identify novel ecotoxicological biomarkers',\n",
       " '',\n",
       " 'isobaric tagging powerful strategy for global proteome profiling caveat isobarictagbased quantification interference which may caused coeluting peptide that are coisolated cofragmented and coanalyzed thereby confounding quantitative accuracy here present twoproteome standard that challenge the mass spectrometer measure range protein abundance ratio background potential interference the hypro standard consists tandem mass tag tmt prolabeled human peptide ratio across all channel into which spiked tmtprolabeled yeast peptide triplicate and ratio showcase the hypro standard altering the ms2 isolation window width and examining different data acquisition method hrms2 spsms3 rtsms3 our data illustrate that wider isolation width moderately increase the tmt signal the benefit which are offset decreased ratio accuracy also show that using realtime database searching rtsms3 resulted the most accurate ratio additionally the number quantified yeast protein using rtsms3 approach that hrms2 when using yeastspecific database for realtime searching short this quality control standard allows for the assessment multiple quantitative measurement within single run which can compared across instrument benchmark and track performance',\n",
       " 'stable isotope labeling coupled with mass spectrometry ha revolutionized the scope and impact protein expression study label incorporation can occur metabolically chemically and each method bear specific strength and weakness quantitative proteomics confidently identifies specific interaction between protein and other biological specie such nucleic acid and metabolite extending labelbased method phosphorylationmodified form protein enables the construction signaling network and their temporal response stimulus the integration multiple data type offer systemslevel insight coordinated biological process finally the development method applicable tissue quantification suggests the emerging role labelbased quantitative mass spectrometry translational science',\n",
       " 'blood plasma the most complex humanderived proteome containing other tissue proteome subset this proteome ha only been partially characterized due the extremely wide dynamic range the plasma protein more than ten order magnitude thus the reduction sample complexity prior mass spectrometric analysis particularly important and alternative separation methodology are required more effectively mine the lower abundant plasma protein here demonstrated novel separation approach using freeflow electrophoresis ffe separating protein and peptide solution according their prior lcmsms used the combination sequential protein and peptide separation first separating the plasma protein into specific ffe fraction tryptic digest the separated protein were generated and subsequently separated using ffe the protein separation medium wa optimized segregate albumin into specific fraction containing only few other protein optimization throughput for the protein separation reduced the separation time plasma approximately providing sufficient material for digestion and the subsequent peptide separation our approach revealed lowabundant protein lselectin ngml and vascular endothelialcadherin precursor ngml and several tissue leakage product thus providing powerful orthogonal separation step the proteomics workflow',\n",
       " 'with the human genome sequence now determined the field molecular medicine moving beyond genomics proteomics the field cancer research the key question how can oncologist best use technique proteomics basic research and clinical application the postgenomic era proteomics promise the discovery biomarkers and tumor marker for early detection and diagnosis novel proteinbased drug target for anticancer therapy and new endpoint for the assessment therapeutic efficacy and toxicity this review paper will explore key theme proteomics and their application clinical cancer research',\n",
       " 'cell alter the proteome respond environmental and developmental cue global analysis proteomic response limited value heterogeneous environment where there average cell advance sequencing protein labeling mass spectrometry and data analysis have fueled recent progress the investigation specific subpopulation cell complex system here highlight recently developed chemical tool that enable cellselective proteomic analysis complex biological system from bacterial pathogen whole animal',\n",
       " 'eye disorder and resulting visual loss are major public health problem affecting million people worldwide this context the sclera opaque thick outer coat covering more than the eye and essential maintaining the shape the eye and protecting the intraocular content against infection and the external environment despite effort undertaken decipher the scleral proteome the functional and structural picture the sclera still remain elusive recently proteomics ha arisen powerful tool that enables identification protein playing critical role health and disease therefore carried out indepth proteomic analysis the human scleral tissue using highresolution orbitrap fusion tribrid mass spectrometer identified protein using sequestht and mascot search algorithm proteome discoverer importantly the protein including radixin synaptopodin paladin netrin and kelchlike family member were identified for the first time human sclera gene ontology analysis unveiled that the majority protein were localized the cytoplasm and involved cell communication and metabolism sum this study offer the largest catalog protein identified sclera with the aim facilitating their contribution diagnostics and therapeutic innovation visual health and autoimmune disorder this study also provides valuable baseline for future investigation map the dynamic change that occur sclera various pathological condition',\n",
       " 'chronic obstructive pulmonary disease copd inflammatory disease characterized the progressive deterioration pulmonary function and irreversible airway obstruction investigation the molecular pathogenesis copd have not yet provided complete answer the mechanism that determine the onset and progression this illness therefore therapeutic choice are limited and new strategy are needed prevent manage and treat this disorder particular the application complementary approach including gel and liquid chromatography mass spectrometrybased proteomic technique sputum andor bronchoalveolar lavage may provide better understanding the proteome differentially expressed copd patient the course the disease the identification appropriate and reliable biomarkers thus essential step for the diagnostics and treatment these patient',\n",
       " 'proteomics one the pivotal nextgeneration biotechnology the current postgenomics era little known about the way which innovative proteomics science navigating the complex sociopolitical space between laboratory and society can not assumed that the trajectory between proteomics laboratory and society linear and unidirectional concerned about public accountability and hope for knowledgebased innovation funding agency and citizen increasingly expect that emerging science and technology such proteomics are effectively translated and disseminated innovation society here describe translation strategy promoted the knowledge translation and science communication literature and examine the use these strategy within the field proteomics drawing data generated from qualitative interview with proteomics scientist and ethnographic observation international proteomics conference over 5year period found that proteomics science incorporates variety strategy reach knowledge user outside the field attain the full benefit however proteomics scientist must challenge their own normative assumption and approach innovation disseminationbeyond the current paradigm relying primarily publication for one scientific peer within one fieldand embrace the value broader interdisciplinary strategy promoting the uptake their research notably the human proteome organization hupo paying increasing attention broader range strategy including targeted dissemination integrated and public outreach suggest that increasing the variety strategy employed proteomics scientist timely and would serve well the omics system science community',\n",
       " 'system biology requires comprehensive data all molecular level mass spectrometry msbased proteomics ha emerged powerful and universal method for the global measurement protein the most widespread format us liquid chromatography coupled highresolution tandem mass spectrometry msms identify and quantify peptide large scale this peptide intensity information the basic quantitative proteomic data type used quantify protein between different proteome state including the temporal variation the proteome determine the complete primary structure protein including posttranslational modification localize protein organelle and determine protein interaction here describe the principle analysis and the area biology where proteomics can make unique contribution the largescale nature proteomics data and it high accuracy pose special opportunity well challenge system biology that have been largely untapped far',\n",
       " '',\n",
       " 'conducted proteomics study order detect the proteomic method which provides the most complete characterization the protein rice milk particular compared the result obtained from lcmsms after protein precipitation with acetone tca well the result obtained from lcmsms after protein prefractionation based sdspage gelcmsms proteominer™ technology proteominerlcmsms and after peptide prefractionation based ief pieflcmsms total protein specie have been detect rice milk the physicalchemical analysis and classification the identified protein were also reported particular showed that pieflcmsms method led significant increase the proteome coverage allowing the identification total protein milk rice this study demonstrates the utility prefractionation step based pief before the shotgun proteomic analysis and offer indepth insight into the rice milk proteome',\n",
       " 'cellular redox status play key role mediating various physiological and developmental process often through modulating activity redoxsensitive protein various stress trigger overproduction reactive oxygennitrogen specie which lead oxidative modification redoxsensitive protein identification and characterization redoxsensitive protein are important step toward understanding molecular mechanism stress response here report highthroughput quantitative proteomic approach termed oxitraq for identifying protein whose thiols undergo reversible oxidative modification arabidopsis cell subjected oxidative stress this approach biotinylated thiolreactive reagent used for differential labeling reduced and oxidized thiols the biotintagged peptide are affinity purified labeled with itraq reagent and analyzed using paralleled hcdcid fragmentation mode ltqorbitrap with this approach identified cysteinecontaining peptide from protein whose thiols underwent oxidative modification arabidopsis cell following the treatment with hydrogen peroxide majority those redoxsensitive protein including several transcription factor were not identified previous redox proteomics study this approach allows identification the specific redoxregulated cysteine residue and offer effective tool for elucidation redox proteome',\n",
       " 'the two key step for analyzing proteomic data generated highresolution are database searching and postprocessing while the two step are interrelated study their combinatory effect and the optimization these procedure have not been adequately conducted here investigated the performance three popular search engine sequest mascot and amanda conjunction with five filtering approach including respective scorebased filtering groupbased approach local false discovery rate lfdr peptideprophet and percolator total eight data set from various proteome coli yeast and human produced various instrument with highaccuracy survey scan ms1 and high lowaccuracy fragment ion scan ms2 ltqorbitrap orbitrapvelos orbitrapelite qexactive orbitrapfusion and qtof were analyzed wa found combination involving percolator achieved markedly more peptide and protein identification the same fdr level than the other combination for all data set among these combination sequestpercolator and amandapercolator provided slightly better performance for data set with lowaccuracy ms2 ion trap and high accuracy ms2 orbitrap tof respectively than did other method for approach without percolator sequestgroup performs the best for data set with ms2 produced collisioninduced dissociation cid and analysis mascotlfdr give more identification for data set generated higherenergy collisional dissociation hcd and analyzed orbitrap hcdot and orbitrap fusion hcdit amandagroup excels for the qtof data set and the orbitrap velos hcdot data set therefore percolator wa not used specific combination should applied for each type data set moreover higher percentage multiplepeptide protein and lower variation protein spectral count were observed when analyzing technical replicates using percolatorassociated combination therefore percolator enhanced the reliability for both identification and quantification the analysis were performed using the specific program embedded proteome discoverer scaffold and inhouse algorithm buildsummary these result provide valuable guideline for the optimal interpretation proteomic result and the development fitforpurpose protocol under different situation',\n",
       " '',\n",
       " 'largescale proteomic method are essential for the functional characterization protein their native cellular context however proteomics ha lagged far behind genomic approach scalability standardization and cost here introduce vivo mrna display technology that convert variety proteomics application into dna sequencing problem vivoexpressed protein are coupled with their encoding messenger rna mrna via highaffinity stemloop rna binding domain interaction enabling highthroughput identification protein with high sensitivity and specificity next generation dna sequencing have generated highcoverage vivo mrna display library the isaccharomyces cerevisiaei proteome and demonstrated it potential for characterizing subcellular localization and interaction protein expressed their native cellular context vivo mrna display library promise circumvent the limitation mass spectrometrybased proteomics and leverage the exponentially improving cost and throughput dna sequencing systematically characterize native functional proteome',\n",
       " 'the present study evaluated four small molecule affinitybased probe based agaroseimmobilized benzamidine aba ophospholtyrosine ptyr 8aminohexylcamp camp 8aminohexylatp atp for their ability remove highabundant protein such serum albumin from plasma sample thereby enabling the detection mediumtolow abundant protein plasma sample mass spectrometrybased proteomics compared their performance with the most commonly used immunodepletion method the multi affinity removal system human mar targeting the top most abundant plasma protein and also the proteominer protein equalization method labelfree quantitative liquid chromatography tandem mass spectrometry lcmsms analysis the affinitybased probe demonstrated high reproducibility for lowabundant plasma protein down picomol per level compared the multi affinity removal system mar and the proteominer method and also demonstrated superior removal the majority the highabundant plasma protein the ababased affinity probe and the proteominer protein equalization method performed better compared all other method term the number analyzed protein all the tested method were highly reproducible for both highabundant plasma protein and lowabundant protein measured correlation analysis six replicate experiment conclusion our result demonstrated that smallmolecule based affinitybased probe are excellent alternative the commonly used immunedepletion method for proteomic biomarker discovery study plasma data are available via proteomexchange with identifier pxd',\n",
       " 'translational proteomics emerging subdiscipline the proteomics field the biological science translational plant proteomics aim integrate knowledge from basic science translate into field application solve issue related but not limited the recreational and economic value plant food security and safety and energy sustainability this review highlight the substantial progress reached plant proteomics during the past decade which ha paved the way for translational plant proteomics increasing proteomics knowledge plant not limited model and nonmodel plant proteogenomics crop improvement and food analysis safety and nutrition but many more potential application given the wealth information generated and some extent applied there the need for more efficient and broader channel freely disseminate the information the scientific community this article part special issue entitled translational proteomics',\n",
       " 'detailed knowledge the proteome crucial advance the biological science lowabundant protein are particular interest many biologist they include for example those protein involved signal transduction recent technological advance resulted tremendous increase protein identification sensitivity mass spectrometry however the dynamic range protein abundance still form fundamental problem that limit the detection lowabundant protein complex proteome these protein will typically escape detection shotgun experiment due the presence other protein abundance severalfold higher order magnitude therefore specific enrichment strategy are required overcome this technical limitation msbased protein discovery have searched for novel signal transduction protein more specifically kinase and calciumbinding protein and here describe different approach for enrichment these lowabundant protein from isolated chloroplast from pea and arabidopsis for subsequent proteomic analysis these approach could extended include other signal transduction protein and target different organelle',\n",
       " 'biomarker crucial tool for measuring the progress disease and the effect treatment for better clinical outcome cancer patient diagnostic predictive and prognostic biomarkers are required various clinical setting the proteome functional translation the genome considered rich source biomarkers therefore sizable time and funding have been spent proteomics develop biomarkers although significant progress ha been made technology toward comprehensive protein expression profiling and many biomarker candidate published none the reported biomarkers have proven beneficial for cancer patient the present deceleration biomarker research can attributed technical limitation additional effort are required further technical progress however there are many example demonstrating that problem biomarker research are not much with the technology but the study design the study biomarkers for early diagnosis candidate are screened and validated comparing case and control similar sample size and the low prevalence disease often ignored although reasonable take advantage multiple rather than single biomarkers when studying diverse disease mechanism the annotation individual component reported multiple biomarkers doe not often explain the variety molecular event underlying the clinical observation tissue biomarker study the heterogeneity disease tissue and pathological observation are often not considered and tissue are homogenized whole for protein extraction addition the challenge technical limitation the fundamental aspect biomarker development disease study need addressed this article part special issue entitled biomarkers proteomic challenge',\n",
       " '',\n",
       " 'neuroproteomics aimed study the molecular organisation the nervous system the protein level twodimensional electrophoresis the most frequently used technique quantitative proteomics the aim this study wa ass the experimental and biological variation this proteomic platform using mouse brain tissue mouse are the most generally used lab animal for modelling human disease investigating the effect drugcandidate treatment experimental design play crucial role quantitative proteomics hence understanding and minimizing the variable essential our result indicate that the technical variance dominantly contributes the total variance mouse brain and the genetic background ha negligible effect the total variation the result also characterise the anticipated variation using mouse brain for proteomic study hence they should useful for future experimental design other proteomics laboratory',\n",
       " '',\n",
       " 'proteomic approach are being used elucidate better discretion interaction occurring between host pathogen andor beneficial microorganism the molecular level application proteomic technique unravel pathogenicity stressrelated and antioxidant protein expressed amid plantmicrobe interaction and good information have been generated being perceived that fine regulation protein expression take place for effective pathogen recognition induction resistance and maintenance host integrity however our knowledge molecular plantmicrobe interaction still incomplete and inconsequential this review aim provide insight into numerous way used for proteomic investigation including peptideprotein identification separation and quantification during host defense response here highlight the current progress proteomics defense response elicited bacterial fungal and viral pathogen plant along with which the proteome level change induced beneficial microorganism are also discussed',\n",
       " 'the spanish slug arion vulgaris considered one the hundred most invasive specie central europe the immense and very successful adaptation and spreading vulgaris suggest that developed highly effective mechanism deal with infection and natural predator current transcriptomic and proteomic study gastropod have been restricted mainly marine and freshwater gastropod transcriptomic proteomic study vulgaris ha been carried out far and the current study the first transcriptomic database from adult specimen vulgaris reported facilitate and enable proteomics this nonmodel organism mrnaderived protein database wa constructed for protein identification gelbased proteomic approach wa used obtain the first generation comprehensive slug mantle proteome total protein were unambiguously identified protein represent novel protein with significant homology ncbi nonredundant database combined transcriptomic and proteomic analysis revealed extensive repertoire novel protein with role innate immunity including many associated pattern recognition effector protein and cytokinelike protein the number and diversity gene family encoding lectin point complex defense system probably result adaptation pathogenrich environment these result are providing fundamental and important resource for subsequent study mollusc well for putative antimicrobial compound for drug discovery and biomedical application',\n",
       " 'protein phosphorylation major mechanism that regulates many basic cellular process identification and characterization substrate for given protein kinase can lead better understanding signal transduction pathway however still difficult efficiently identify substrate for protein kinase here propose integrated proteomic approach consisting vitro dephosphorylation and phosphorylation phosphoprotein enrichment and 2ddige phosphatase treatment significantly reduced the complexity the phosphoproteome which enabled efficiently identify the substrate employed mitogenactivated protein kinase map kinase model kinase and identified novel candidate substrate for this kinase seven selected candidate were phosphorylated map kinase vitro and map kinaseactivated cell this proteomic approach can applied any protein kinase allowing global identification novel substrate',\n",
       " 'cataract major cause blindness worldwide characterized lens opacification and accompanied extensive posttranslational modification ptms various protein ptms play essential role lens opacification several ptms have been described protein isolated from relatively old human lens including phosphorylation deamidation racemization truncation acetylation and methylation overwhelming majority previous cataract proteomic study have exclusively focused crystallin protein which are the most abundant proteome component the lens investigate the proteome cataract marker this chapter focus the proteomic research the functional relevance the major ptms crystallins human cataractous lens elucidating the role these modification cataract formation ha been challenging task because they are among the most difficult ptms study analytically the proteomic status some amide present similar property normal aged and cataractous lens whereas some may undergo greater ptms cataract therefore great importance review the current proteomic research crystallins the major protein marker different type cataract elucidate the pathogenesis this major humanblinding condition',\n",
       " 'pulmonary surfactant complex mixture lipid and protein that essential for postnatal function surfactant synthesized alveolar type cell and stored multibilayer membrane specialized secretory lysosomerelated organelle lro known the lamellar body prior secretion into the alveolar airspace few protein have been identified and the mechanism regulating formation and trafficking this organelle are poorly understood lamellar body were isolated from rat lung separated into limiting membrane and core population fractionated sdspage and protein identified nanolctandem mass spectrometry total protein were identified significantly extending previous study that identified protein rat lung the lung proteome reflects the dynamic interaction this organelle with the biosynthetic secretory and endocytic pathway the type epithelial cell comparison with other lro proteome indicated that protein were detected one more other proteome confirming classification the lro remarkably the shared it protein with the melanosome but only with lamellar body from the skin the protein not detected other lro proteome subset protein wa enriched lung relative other tissue protein with lipidrelated function comprised significant proportion the unique subset consistent with the major function this organelle the organization storage and secretion surfactant lipid the lung proteome will facilitate identification molecular pathway involved biogenesis surfactant homeostasis and disease pathogenesis',\n",
       " 'the study metabolically labeled probemodified protein important area chemical proteomics isolation and purification the protein target necessary step before identification the biotinstreptavidin system widely used this process but the harsh denaturing condition also release natively biotinylated protein and nonselectively bound protein cleavable linker strategy promising approach for solving this problem though several cleavable linkers have been developed and tested efficient easily synthesized and inexpensive cleavable linker desirable addition the proteomics toolbox here describe the chemical proteomics application vicinal diol cleavable linker through easytohandle chemistry incorporate this linker into activitybased probe and biotin alkyne tag amenable for bioorthogonal ligation with these reagent background protein identification are significantly reduced relative standard onbead digestion',\n",
       " 'the wide reactivity the thiol group enables the formation number chemically and biologically distinct posttranslational modification protein within nearly all major family undergo some form cysteine modification and the modification are associated with regulatory function across many biological process however the susceptibility thiols redox shift well the labile nature most thiol modification render detection difficult analysis difficulty are compounded further complex protein mixture due the typical low abundance cysteine modification under normal physiological condition here describe method for the analysis three cysteine modification nitrosylation glutathionylation and sacylation the three method use the same organic mercuryconjugated agarose resin enrichment platform date over site protein have been identified between the three modification using this method using equivalent processing enrichment and analytical method ha enabled more comprehensive picture the redox proteome landscape',\n",
       " '',\n",
       " 'quantitative analysis ha become increasingly important the proteomics field however the large amount mass spectrometric data and the different type quantitative strategy make data analysis ever challenging here describe quantitative software tool called census analyze highthroughput mass spectrometry data from shotgun proteomics experiment efficient way census capable analyzing various stable isotope labeling experiment using silac itraq tmt addition labelingfree experiment with highresolution data census increase the quantitative accuracy minimizing the contribution interfering peak and chemical noise with small accuracy tolerance for each isotope peak census provides various scoring algorithm including leastsquares correlation weight average singleton peptide detection with discriminant analysis and probability score for each peptide furthermore census ha builtin multiple statistical filter maintain robust quality control quantitative result',\n",
       " 'used proteomicsbased approach investigate potential regulatory protein the pancreas domestic cattle bos taurus that were associated with difference pancreatic αamylase activity two group and crossbred steer year and respectively were fed high moisture cornbased diet and were ranked according their pancreatic αamylase activity steer with high medium and low αamylase activity were selected with for each activity range and for each experimental year and their proteomic profile were compared pancreatic sample from each animal were fractionated using 2dhplc and fraction detected using spectrophotometry software analysis revealed common protein fraction among the animal and statistical analysis revealed these fraction differing plt abundance between animal from the high and low pancreatic αamylase activity group five protein fraction identified after tandem mass spectrometry analysis and database search were found match protein with proteinbinding nucleotidednabinding enzymatic capability bioinformatics analysis these fraction revealed porphobilinogen deaminase dnabinding protein and putative peptidase that increased abundance with increasing αamylase activity with putative atpgtp binding protein decreasing abundance with increasing pancreatic αamylase activity change these fraction may represent adaptation the pancreas domestic cattle that are associated with difference αamylase activity',\n",
       " 'lipid peroxidation responsible for the generation chemically reactive diffusible lipidderived electrophiles ldes that covalently modify cellular protein target these protein modification modulate protein activity and macromolecular interaction and induce adaptive and toxic cell signaling protein modification induced ldes can identified and quantified affinity enrichment and liquid chromatographytandem mass spectrometry lcmsmsbased technique tagged lde analog probe with different electrophilic group can covalently captured click chemistry for lcmsms analysis thereby enabling indepth study proteome damage the protein and peptide sequence level conversely clickreactive thioldirected probe can used evaluate thiol damage caused lde difference these analytical approach permit systematic study the dynamic protein damage caused lde and mechanism which oxidative stress contribute toxicity and disease',\n",
       " 'the aim this study wa explore the molecular mechanism induced leaf senescence preventing pollination maize using proteomic method combined with other physiological method elite maize inbred line wa selected for evaluation it senescence mechanism phenotypic and chlorophyll content analysis revealed that the onset leaf senescence occurred earlier nonpollinated nonpol leaf than pollinated pol leaf leaf protein specie nonpol and pol leaf were separately extracted and their proteome were assessed using isobaric tag for relative and absolute quantitation itraq analysis total protein specie were identified which exhibited differentially altered abundance plt the identified protein specie were related diverse function including photosystems plant hormone cell death oxidative degradation and protein metabolism suggesting potential signaling cascade for ear leaf senescence induced pollination prevention addition leaf total soluble sugar and leaf starch content were remarkably higher nonpol plant than pol plant these finding suggest that induced leaf senescence might associated with nutrient remobilization our result reveal network molecular mechanism the protein level and provide some insight into the early senescence mechanism higher plant biological significance the coordination between growth and timing for senescence critical for maize production however the molecular mechanism induced leaf senescence preventing pollination maize remains further elucidated the proteomic level herein revealed some new protein specie that are involved hormone signaling glycometabolism oxidationreduction protein degradation and photosystem breakdown and other biological process that were not previously known associated with leaf senescence this the first largescale proteomics study examine induced leaf senescence maize preventing pollination',\n",
       " 'the performance serum pretreatments for biomarker discovery ha been recently assessed very poor not only for immunosubtraction turn evaluated tool unable look deep into the lowabundance proteome lap and thus incapable lead any novel biomarker discovery proteome re but also for combinatorial peptide ligand library cpll proteomics the performance both tool ha been given enabling barely detect meagre more compared control untreated serum meanwhile other study indicated the extreme effectiveness peptide library enlarge the knowledge proteome composition this contradictory situation are here reevaluating some protocol aspect and report that indeed cpll excellent tool able dig really deep into the lowabundance proteome the problem that those report underoptimized capture and elution protocol had been adopted with the protocol here reported namely abandoning the step adding salt the sample capture three different value and and most importantly eluting from cpll bead boiling sd the presence dtt can largely expand the window visibility particular here shown that common elution protocol adopted several report urea and chap barely elutes about the captured specie nevertheless the cpll bead thus treated are further eluted with boiling sdsdtt additional recovered',\n",
       " 'tectona grandis teak one the premier hardwood timber the world ranking present the top five tropical hardwood specie term worldwide plantation area characterization the protein present teak leaf will provide basis for the development new tool aimed assisting tree selection the monitoring plant propagation and the certification clonal and phenotypic identity this paper describe the extraction separation and identification leaf protein from grandis using tcaacetone protocol 2de and malditof after tcaacetone protein extraction leaf wellresolved spot were detected coomassiestained gel within the kda relative molecular mass range ranging from total spot were digested and subjected these nonredundant protein specie were successfully identified functional classification the identified protein revealed that protein involved photosynthesis protein translation and energy production were the most abundant this work the first highthroughput attempt study the grandis leaf proteome and represents stepping stone for further differential expression proteomic study related growth development biomass production and cultureassociated physiological response',\n",
       " 'clonal microbial cell not behave identical manner and form subpopulation during cultivation besides varying microenvironmental condition cell inherent feature like cell cycle dependent localization and concentration regulatory protein well epigenetic property are well accepted mechanism creating cell heterogeneity another suspected reason molecular noise the transcriptional and translational level promising tool unravel reason for cell heterogeneity the combination cell sorting and subpopulation proteomics this review summarizes recent development prokaryotic singlecell analytics and provides workflow for selection single cell low cell number mass spectrometry and proteomics evaluation this approach useful for understanding the dependency individual cell decision inherent protein profile',\n",
       " 'chemoproteomics represents new research discipline the interface medicinal chemistry biochemistry and cell biology focused studying the molecular mechanism action drug and other bioactive small molecule research strategy frequently combine phenotypic screening with subsequent target identification and aim proteomewide characterization druginduced change cellular protein expression and posttranslational modification recent year quantitative mass spectrometry ha taken center stage many these approach this review describes experimental strategy current chemical proteomics research discus recent example successful application and highlight area drug discovery where chemical proteomicsbased assay using native endogenous protein are expected have substantial impact',\n",
       " 'iemiliania huxleyii one the most abundant marine plankton and ha crucial feature the carbon cycle however proteomic analysis iemiliania huxleyii have not been done extensively this study threedimensional liquid chromatography 3dlc system consisting strong cation exchange high and lowph reversedphase liquid chromatography wa established for indepth proteomic profiling iemiliania huxleyii from tryptic proteome digest fraction were generated and analyzed using liquid chromatographytandem mass spectrometry total more than unique peptide and protein group were identified with false discovery rate the physicochemical property the identified peptide were evaluated using cluego approximately gene ontology term and pathway were defined from the identified protein group with ipivalue covering wide range biological process cellular component and molecular function many biological process associated with cosub2sub fixation photosynthesis biosynthesis and metabolic process were identified various molecular function relating protein binding and enzyme activity were also found the 3dlc strategy powerful approach for comparative proteomic study iemiliania huxleyii reveal change it protein level and related mechanism',\n",
       " '',\n",
       " 'mass spectrometry play key role relative quantitative comparison protein order understand their functional role biological system upon perturbation this review review study that examine different aspect isobaric labelingbased relative quantification for shotgun proteomic analysis particular focus different type isobaric reagent and their reaction chemistry amine carbonyl and sulfhydrylreactive various factor such ratio compression reporter ion dynamic range and others cause underestimation change relative abundance protein across sample undermining the ability the isobaric labeling approach truly quantitative these factor that affect quantification and the suggested combination experimental design and optimal data acquisition method increase the precision and accuracy the measurement will discussed finally the extended application isobaric labelingbased approach hyperplexing strategy targeted quantification and phosphopeptide analysis are also examined',\n",
       " 'modern microbial systematics requires range methodology for the comprehensive characterization classification and identification microorganism while wholegenome sequence provide the ultimate reference for defining microbial phylogeny and taxonomy selected biomarkerbased strategy continue provide the mean for the bulk microbial systematic study proteomics the study the expression gene well the structure and function the resulting protein offer indirect measure genome sequence data recent development application proteomics for analyzing microorganism have paralleled the growing microbial genome sequence database well the evolution mass spectrometry instrumentation and bioinformatics malditof which generates proteomic mass pattern for fingerprintbased characterization ha provided marked breakthrough for microbial identification however malditof limited the number target that can detected for strain characterization advanced method tandem mass spectrometry which protein and peptide generated from protein are characterized and identified using lcmsms provide the ability detect hundred thousand expressed microbial strain marker for highresolution characterization and identification model study demonstrate the application proteomicsbased analysis for bacterial specie and strainlevel detection and identification and for characterization environmentally relevant metabolically diverse bacteria proteomicsbased approach represent emerging complement traditional method characterizing microorganism enabling the elucidation the expressed biomarkers genome sequence information which can applied proteotyping application microorganism all taxonomic level',\n",
       " 'present wholeproteome phylogeny prokaryote constructed comparing feature frequency profile ffps whole proteome feature are lmers amino acid and each organism represented profile frequency all feature the selection feature length critical the ffp method and have developed procedure for identifying the optimal feature length for inferring the phylogeny prokaryote strictly speaking proteome phylogeny our ffp tree are constructed with whole proteome prokaryote unicellular eukaryote and random sequence highlight the branching order major group present simplified proteome ffp tree monophyletic class phylum with branch support our wholeproteome ffp tree archaea bacteria eukaryota and random sequence outgroup are clearly separated archaea and bacteria form sister group when rooted with random sequence iii planctomycetes which posse intracellular membrane compartment placed the basal position the bacteria domain almost all group are monophyletic prokaryote most taxonomic level but many difference the branching order major group are observed between our proteome ffp tree and tree built with other method and previously unclassified genome may assigned the most likely taxon describe notable similarity and difference between our ffp tree and those based other method grouping and phylogeny prokaryote',\n",
       " 'the biological activity protein regulated many level ranging from control transcription and translation posttranslational modification ptm proteolytic processing irreversible ptm generating novel isoforms mature protein termed proteoforms proteoform dynamic major focus current proteome research since ha been associated with many pathological condition massspectrometry msbased proteomics and ptmspecific enrichment workflow have become the method choice study proteoforms vitro and vivo here give overview currently available msbased degradomics method and outline how they can optimally applied study protease cleavage event discus the advantage and disadvantage selected approach and describe stateoftheart improvement degradomics technology introducing the concept combinatorial degradomics combination global discovery degradomics and highly sensitive targeted degradomics demonstrate how msbased degradomics further evolves powerful tool biomedical protease research',\n",
       " 'this study performed indepth characterization the male pediatric infant urinary proteome parallel proteomic analysis normal healthy adult and infant male and comparison available published data total protein group were identified these protein were identified sample from both cohort although present both cohort these common protein were significantly enriched urine from adult and protein were significantly enriched urine from infant using gene ontology found that the infantenriched specific subproteome protein had overrepresentation protein that are involved translation and transcription cellular growth and metabolic process contrast the adult enriched specific subproteome protein showed overexpression protein involved immune response and cell adhesion this study demonstrates that the nondiseased male urinary proteome quantitatively affected age ha agespecific subproteomes and identifies common subproteome with agedependent abundance variation these finding highlight the importance agematching urinary proteomics this article part special issue entitled biomarkers proteomic challenge',\n",
       " 'bovine colostrum well known for it large content bioactive component and it importance for neonatal survival unfortunately the colostrum proteome complicated wide dynamic range because few dominating protein that hamper sensitivity and proteome coverage achieved low abundant protein moreover the composition colostrum complex and the protein are located within different physical fraction that make the colostrum gain more exhaustive picture the bovine colostrum proteome and gather information protein location performed extensive preanalysis fractionation colostrum prior 2dlcmsms analysis physical and chemical property the protein and colostrum were used alone combination for the separation protein elisa wa used quantify and verify the presence protein colostrum total protein were identified the nonfractionated colostrum and seven fraction f1f7 using six different fractionation technique fractionation contributed with additional protein the fluid phase compared with different fractionation technique each resulted detection unique subset protein whey production highspeed centrifugation contributed most detection low abundant protein hence prefractionation colostrum prior 2dlcmsms analysis expanded our knowledge the presence and location low abundant protein bovine colostrum',\n",
       " 'over the last two decade technological advancement the field proteomics have advanced our understanding the complex biological system living organism technique based mass spectrometry have emerged powerful tool contextualise existing genomic information and create quantitative protein profile from plasma tissue cell line various specie proteomic approach have been used increasingly veterinary science investigate biological process responsible for growth reproduction and pathological event however the adoption proteomic approach veterinary investigator lag behind that researcher the human medical field furthermore contrast human proteomics study interpretation veterinary proteomic data difficult due the limited protein database available for many animal specie this review article examines the current use advanced proteomics technique for evaluation animal health and welfare and cover the current status clinical veterinary proteomics research including successful protein identification and data interpretation study includes description emerging tool sequential window acquisition all theoretical fragment ion mass spectrum swathms available selected mass spectrometry instrument this newly developed data acquisition technique combine advantage discovery and targeted proteomics approach and thus ha the potential advance the veterinary proteomics field enhancing identification and reproducibility proteomics data',\n",
       " 'twodimensional capillary array liquid chromatography system coupled with matrixassisted laser desorptionionization maldi mass spectrometry wa developed for highthroughput comprehensive proteomic analysis which one strong cationexchange scx capillary chromatographic column wa used the first separation dimension and parallel reversedphase liquid chromatographic rplc capillary column were used the second separation dimension novel multichannel interface wa designed and fabricated for online coupling the scx rplc column array system besides the high resolution based the combination scx and rplc separation the developed new system provided the most rapid twodimensional liquid chromatography 2dlc separation ten threeway microsplitter valve used stopandflow switch transferring scx fraction onto rplc column addition the threeway valve also acted mixing chamber rplc effluent with matrix the system enables online mixing the array effluent with matrix solution during the elution and directly depositing the analytematrix mixture maldi plate from the tenplexed channel parallel through array capillary tip with the novel system thousand peptide were well separated and deposited maldi plate only min for complex proteome sample compared with common 2dlc system the parallel 2dlc system showed about time faster analytical procedure combination with high throughput tandem time flight mass spectrometry the system wa proven very effective for proteome analysis analyzing complicated sample soluble protein extracted from liver cancer tissue which over protein were identified',\n",
       " 'mussel byssus represents fascinating class biological material with unique capacity adhere onto virtually any solid surface protein expressed byssus the byssalproducing organ foot well mantle tissue from mytilus edulis mytilus californianus are analyzed liquid chromatography tandem mass spectrometry lcmsms the mantle used control tissue pinpoint unique protein from the foot sample potentially involved byssogenesis this work represents important step towards identifying biologically important protein expressed foot well extending knowledge the byssus proteome considering the minimal proteomics data the studied specie this data also contributes more complete description edulis and californianus proteome',\n",
       " '2de one the most efficient and widely used method for resolving complex protein mixture for efficient analysis complex sample highresolution separation protein gel essential and for that purpose good sample preparation crucial this study have improvized method for preparing bacterial total cellular proteome from strategy applied earlier recalcitrant plant tissue which gave highquality resolution 2de the method involving phenol extraction followed methanolammonium acetate precipitation wa first optimized for the chemolithotrophic proteobacteria tetrathiobacter kashmirensis and pseudaminobacter salicylatoxidans kct that did not yield quality protein prep conventional trichloroacetic acidacetone precipitation method subsequently validate it general applicability the method wa evaluated against the trichloroacetic acidacetone precipitation method for two other model bacteria escherichia coli dh5α and mycobacterium smegmatis mc2 identification least four protein each from the outer membrane periplasm and cytoplasm kashmirensis maldims not only proved the efficiency the method extracting protein from the different cellular compartment but also the amenability the obtained protein spot toward maldims based identification',\n",
       " 'human cerebrospinal fluid csf sample choice the study brain disorder this biological fluid circulates the brain and the spinal cord and contains tissuespecific protein indicative health and disease condition despite it potential valid source biological marker csf remains largely understudied compared blood particular due it more invasive way samplingchallenges remain when performing proteomic analysis clinical research study stateoftheart mass spectrometry enables deep characterization the human proteome but some technical limitation are cardinal addressed such the capacity routinely analyze large cohort sample importantly tradeoff still need made between the proteome coverage depth and the number measured sample this context developed scalable automated proteomic pipeline for the analysis csf because it versatility this workflow can adapted accommodate proteome coverage andor sample throughput allows prepare and quantitatively analyze hundred thousand csf sample can also allow identification more than protein csf sample when coupled with isoelectric focusing fractionationin this chapter describe endtoend pipeline for the proteomic analysis csf the main step the sample preparation comprise spiking standard protein digestion isobaric labeling and purification these are performed well plate format enabling automation depending the targeted depth the csf proteome optional analytical step can included such the removal abundant protein and sample prefractionation liquid chromatography tandem well data processing and analysis complete the pipeline',\n",
       " 'the field proteomics continues driven improvement analytical technology notably peptide separation quantitative and informatics this study have characterized hybrid linear ion trap high field orbitrap mass spectrometer orbitrap elite for proteomic application the very high resolution available this instrument allows all peptide mass measured with subppm accuracy that turn improves protein identification database searching further confirm again that mass accuracy tandem mass spectrum valuable parameter for improving the success protein identification the new cid rapid scan type the orbitrap elite achieves similar performance higher energy collision induced dissociation fragmentation and both allow the identification hundred protein from little protein digest column the new instrument outperforms it predecessor the orbitrap velos considerable margin each metric assessed that make valuable and versatile tool for msbased proteomics',\n",
       " 'analysis the human proteome ha become increasingly sophisticated and offer invaluable potential insight into the pathophysiology human disease the increasing standardization method speed and sophistication mass spectrometric analysis availability reliable antibody and dissemination information among the scientific community ha allowed for exponential growth our knowledge base the continued effort provide molecular explanation for future medical application based biomarker discovery epitomized the outstanding effort the human proteome project whose goal generate map the human proteome however proteomic analysis underrepresented pediatric illness given the unique challenge research the pediatric population proteomic analysis represents enormous untapped potential especially the further elucidation the pathophysiology rare disease such pulmonary hypertension this article will describe the unique challenge pediatric research the importance alternative avenue such proteomics for indepth analysis pediatric pathobiology the cellular level the specific need for proteomic investigation pediatric the current status proteomics and future direction',\n",
       " 'biomedical scientist tend focus only small fraction the protein encoded the human genome despite overwhelming genetic evidence that many understudied protein are important for human disease one the best way interrogate the function protein and determine it relevance drug target using pharmacological modulator such chemical probe antibody these tool were available for most human protein should possible translate the tremendous advance genomics into greater understanding human health and disease and catalyze the creation innovative new medicine target global federation for developing and applying new technology with the goal creating chemogenomic library chemical probe andor functional antibody for the entire proteome',\n",
       " '',\n",
       " 'crisprcas9 ha proven versatile tool for the discovery essential genetic element involved various disease state crisprassisted dense mutagenesis focused therapeutically challenging protein complex allows systematically perturb proteincoding sequence iin situi and correlate them with functional readout such perturbation can mimic targeting therapeutic and serve foundation for the discovery highly specific modulators however translation such genomics data ha been challenging due the missing link for proteomics under the physiological state the cell present method based cellular thermal shift assay easily interrogate proteomic shift generated crisprassisted dense mutagenesis well case focused nurd epigenetic complex',\n",
       " 'hypha filamentous fungi undergo polar extension bifurcation and hyphal fusion form reticulating network mycelium hyphal fusion anastomosis ubiquitous process among filamentous fungi vital strategy for how fungi expand over their substrate and interact with recognise self and nonself hypha neighbouring mycelium their environment morphological and genetic characterisation anastomosis ha been studied many model fungal specie but little known the direct proteomic response two interacting fungal isolates agaricus bisporus the most widely cultivated edible mushroom crop worldwide wa used vitro model profile the proteome interacting culture the globally cultivated strain wa paired with two distinct strain commercial hybrid strain and wild isolate strain each coculture presented different interaction ranging from complete vegetative compatibility self lack interaction and antagonistic interaction these incompatible strain are the focus research into diseaseresistance commercial crop the spread intracellular pathogen namely mycoviruses limited the lack interhyphal anastomosis unique proteomic response were detected between all cocultures array cell wall modifying enzyme plus fungal growth and morphogenesis protein were found significantly altered abundance nitrogen metabolism dominated the intracellular proteome with evidence nitrogen starvation between competing noncompatible culture change key enzyme bisporus morphogenesis were observed particularly via increased abundance glucanosyltransferase competing interaction and certain chitinases vegetative compatible interaction only carbohydrateactive enzyme arsenal are expanded antagonistic interaction bisporus pathway involved carbohydrate metabolism and genetic information processing were higher interacting culture most notably during selfrecognition new insight into the differential response interacting strain bisporus will enhance our understanding potential barrier viral transmission through vegetative incompatibility our result suggest that differential proteomic response occurs between bisporus strainlevel and finding from this work may guide future proteomic investigation fungal anastomosis',\n",
       " 'bryozoan are typical modular organism they consist repetitive structural unit the zooid bryozoan colony grow zooidal budding with the distribution pattern the budding locus underlying the diversity colony form budding usually restricted the zooid the periphery the colony which form growing edge local terminal growth zone nonbudding part the colony can functionally subdivided too many specie colony consists regular often repetitive zone feeding and nonfeeding module associated with periodical degeneration and regeneration the polypide retractile tentacle crown with gut and the accompanying musculature there functional differentiation bryozoan colony but it mechanism are unknown presumably budding andor polypide recycling different colony part are induced inhibited certain determinant functional specialization effective tool their identification the comparison proteome functionally different zone here report the result proteomic analysis three bryozoan specie from the white sea which have different colony form flustrellidra hispida terminoflustra membranaceotruncata and securiflustra securifrons using differential twodimensional electrophoresis 2ddige compared proteome the growing edge and the zone consisting feeding and nonfeeding zooid these specie estimated the overall proteome variability revealed protein whose relative abundance gradually changed along the proximaldistal colony axis and suggested that they might involved the functional differentiation the colony',\n",
       " '',\n",
       " 'shotgun proteomics rapidly becoming one the most efficient and popular tool examine protein expression cell numerous laboratory now have wide array low and highperformance mass spectrometry instrumentation necessary complete proteomewide project often these laboratory have time and financial constraint that prohibit all project from being conducted highperformance stateoftheart mass spectrometer here compare shotgun proteomic result using direct lyse digest and analyse approach highperformance mass spectrometer the ltqft with the result from much lowerperformance instrument the lcqduo where for the latter various traditional protein prefractionation step and gasphase fractionation were used increase the proteome coverage our result demonstrate that shotgun proteomic analysis conducted the lowerperformance lcqduo mass spectrometer could adequately characterize phop constitutive strain salmonella typhimurium proteome prefractionation step and gasphase fractionation were included',\n",
       " '',\n",
       " 'the escherichia coli proteome wa digested with trypsin and fractionated using spe spe column seven fraction were collected and analyzed czeesimsms the separation wa performed cmlong linear polyacrylamidecoated capillary with formic acid separation buffer electrokinetic sheathflow electrospray interface wa used couple the separation capillary with orbitrapvelos operating higherenergy collisional dissociation mode each czeesimsms run lasted min and total time wa min total peptide spectrum match peptide id and protein group id were generated using mascot with false discovery rate le than the peptide level the total mass spectrometer analysis time wa le than the sample identification rate proteinsh wa more than two time higher than previous study the coli proteome and the amount sample consumed lt1 wa roughly fourfold le than previous study these result demonstrate that cze useful tool for the bottomup analysis prokaryote proteome',\n",
       " '',\n",
       " '',\n",
       " 'purification technology are one the working horse organelle proteomics study they guarantee the separation organellespecific protein from the background contamination other subcellular compartment the development method for the separation organelle wa major prerequisite for the initial detection and characterization peroxisome discrete entity the cell since then isolated peroxisomes fraction have been used numerous study order characterize organellespecific enzyme function allocate the peroxisomespecific proteome unravel the organellar membrane composition this review will give overview the fractionation method used for the isolation peroxisomes from animal plant and fungi addition classic centrifugationbased isolation method relying the different density individual organelle the review will also summarize work alternative technology like freeflowelectrophoresis flow field fractionation which are based distinct physicochemical parameter final chapter will further describe how different separation method and quantitative mass spectrometry have been used proteomics study assign the proteome',\n",
       " 'proteinprotein interaction are essential biologic process that occur inter and intracellular level gain insight into the various complex cellular function these interaction necessary ass them under physiologic condition recent advance various proteomic technology allow investigate proteinprotein interaction network living cell the combination proximitydependent labelling and chemical crosslinking will greatly enhance our understanding multiprotein complex that are difficult prepare such organellebound membrane protein this review describe our current understanding mass spectrometrybased proteomics mapping method for elucidating organellebound membrane protein complex living cell with focus proteinprotein interaction mitochondrial subcellular compartment',\n",
       " 'mass spectrometry ha been widely applied study biomolecules and one rapidly developing field the global analysis protein proteomics understanding and handling mass spectrometry data multifaceted task that requires many decision made get the most comprehensive information from experiment later chapter this book deal indepth with various aspect the process and how different tool can applied the many analytical challenge this introductory chapter intended basic introduction mass spectrometry msbased proteomics set the scene for newcomer and give pointer reference material there are many application mass spectrometry proteomics and each application associated with some analytical choice instrumental limitation and data processing step that depend the aim the study and mean conducting different aspect the proteome can explored choosing the right combination sample preparation instrumentation and data processing this chapter give outline for some these commonly used setup and some the key concept many which are explored greater depth later chapter',\n",
       " '',\n",
       " 'the complexity tumor genome are rapidly being uncovered but how they are regulated into functional proteome remains poorly understood standard proteomics workflow use database known protein but these database not capture the uniqueness the cancer transcriptome with it point mutation unusual splice variant and gene fusion oncoproteogenomics integrates mass spectrometrygenerated data with genomic information identify tumorspecific peptide linking tumorderived dna rna and protein measurement into centraldogma perspective ha the potential improve our understanding cancer biology',\n",
       " 'article assembled the second part this special issue describe some experimental and computational approach for the structural and functional characterization intrinsically disordered protein since these tool represent specialized gear for the focused analysis various aspect dark proteome they can viewed torch candle lamp lantern flashlight spotlight night vision goggles and other mean needed see darkness',\n",
       " 'posttranslational modification ptms are involved the regulation wide range biological process and affect protein structure activity and stability several hundred ptms have been described the literature but relatively few have been studied using mass spectrometry and proteomics general method for ptm characterization are developed study yeast and mammalian biology and later adopted investigate plant our point view that advantageous enrich for ptms the peptide level part quantitative proteomics strategy not only identify the ptm but also determine the functional relevance the context regulation response abiotic stress etc protein phosphorylation the only ptm that ha been studied extensively the proteome wide level plant using mass spectrometry based method review phosphoproteomics study plant and discus the redox mediated ptms snitrosylation tyrosine nitration and sglutathionylation ubiquitylation sumoylation and glycosylation including gpi anchor and the quantitative proteomics method that are used study these modification plant where appropriate contrast the method those used for mammalian ptm characterization',\n",
       " 'the reactivity amino acid residue protein contextdependent and difficult predict chemical biology can used understand the chemical modification protein help elucidate the nature the reactive proteome the resulting insight can applied pharmacoproteomics target identification and molecular pathology',\n",
       " '',\n",
       " 'maize zea may host numerous pathogenic specie that impose serious disease it ear and foliage negatively affecting the yield and the quality the maize crop considerable amount research ha been carried out elucidate mechanism maizepathogen interaction with major goal identify defenseassociated protein this review summarize interaction maize with it agriculturally important pathogen that were assessed the proteome level employing differential analysis such the comparison pathogenresistant and susceptible maize variety well change maize proteome after pathogen challenge numerous protein were identified possible candidate maize resistance describe finding various research group that used mainly mass spectrometrybased high throughput proteomic tool investigate maize interaction with fungal pathogen aspergillus flavus fusarium spp and curvularia lunata and viral agent rice blackstreaked dwarf virus and sugarcane mosaic virus',\n",
       " 'currently the most common practice human breast tissue preservation formalin fixation which ensures good quality for histopathological analysis but damage dna rna and protein impairing their usefulness for molecular analysis and biomarker investigation investigated the potential value nontoxic fixative for sparing protein preserved paraffinembedded breast biopsy specimen were fixed formalinfree fixative prior paraffin embedding and then processed for quality and quantity protein conservation similar protein pattern were observed formalinfree fixative and frozen tissue using mono and bidimensional electrophoresis well western blotting protein pattern assessed mass spectrometric analysis were found identical for frozen and formalinfreefixed tissue immunohistochemistry using various antibody showed comparable result for both tissue storage method conclusion believe that formalinfree fixative represents easytouse alternative formalin for archived tissue and for biomarker investigation since simultaneously protects both the histomorphology and the integrity macromolecule',\n",
       " 'historically plant biology study have lagged behind system biology study animal and human however there are sign positive change evidenced the rise big data plant proteomics and the availability data science tool and nextgeneration sequencing technology currently the sequence information nearly plant specie available although they are curated varying degree sophistication this ha led significant enrichment representation the corresponding plant proteome database analysis the proteome component organism offer structural functional and network scale insight moreover the development highthroughput mass spectrometric technique ha augmented our understanding protein and their expression pattern under various condition several thousand protein can now identified from single mass spectrometric analysis this expert review provide indepth analysis plant proteome database how access them and importantly the biological research and application context which each database significant their comparative strength and limitation aimed this analysis reach out young scholar embarking plant biology and proteomic research well those already established the field provide integrated critical analysis plant proteome database and bioinformatics tool this nascent field system science conclusion plant proteome research emerging and exciting frontier integrative biology scholarship and innovation our future effort must also invested integrating the available database allow for multiomics data analysis research and development',\n",
       " '',\n",
       " '',\n",
       " 'ongoing improvement instrumentation fractionation technique and enrichment procedure have dramatically increased the coverage the proteome achievable via lcmsmsbased methodology opening the call for approach quantitatively ass difference proteomewide scale stable isotope labeling amino acid cell culture silac ha emerged powerful and versatile approach for proteomewide quantitation mass spectrometry silac utilizes the cell own metabolism incorporate isotopically labeled amino acid into it proteome which can mixed with the proteome unlabeled cell and difference protein expression can easily read out comparing the abundance the labeled versus unlabeled protein silac ha been applied numerous different cell line and the technique ha been adapted for wide range experimental procedure this chapter provide detailed procedure for performing silacbased experiment for proteomewide quantitation including protocol for optimizing silac labeling also provide update the most recent development this technique',\n",
       " 'the field proteomics several approach have been developed for separating protein and analyzing their differential relative abundance one the oldest yet still widely used 2de despite the continuous advance new method which are le demanding from technical standpoint 2de still compelling and ha lot potential for improvement the overall variability which affect 2de includes biological experimental and postexperimental softwarerelated variance important highlight how much the total variability this technique due postexperimental variability which far ha been largely neglected this short review have focused this topic and explained that postexperimental variability and source error can further divided into those which are softwaredependent and those which are operatordependent discus these issue detail offering suggestion for reducing error that may affect the quality result summarizing the advantage and drawback each approach',\n",
       " 'the field proteomics shift from qualitative identification protein subfractions quantitative comparison protein from complex biological sample apparent that the number approach for quantitation can daunting for the resultoriented biologist there have been many recent review quantitative proteomic approach discussing the strength and limitation each unfortunately there are few detailed methodological description any one these quantitative approach here present detailed bioinformatics workflow for one the simplest most pervasive quantitative approachspectral counting the informatics and statistical workflow detailed here includes newly available freeware such sepro and patternlab which postprocess data according false discovery rate parameter and statistically model the data detect difference and trend',\n",
       " 'the pace genome sequencing mean that many not all newly discovered gene have either known function function that best are poorly described the classical approach detailed annotation individual gene can not deal with the flood new sequence increasingly highthroughput global approach are being used where change expression many gene are measured parallel proteomics one such approach present snapshot all protein level cell taken indirectly looking the mrna the transcriptome making the comparable measurement the protein level technically difficult and unlikely competitive with mrnabased approach correlation the abundance the mrna and cognate protein however far from simple nonetheless proteomics ha role more focused study especially the study protein complex where the direct isolation complex combined with mass spectrometry ha proved very powerful tool this approach ha been termed cell map proteomics and offer rapid route studying protein complex and higher order structure',\n",
       " 'argentina one the most relevant country latin america playing major role regional economics culture and science over the last year argentinean history ha been characterized several upward and downward phase that had major consequence the development science the country and most recently proteomics this article characterize the evolution proteomics science argentina over the last decade and half describe the proteomics publication output the country the framework the regional and international context demonstrating that argentina solidly anchored regional context showing result similar other emergent and latin american country albeit still far from the european american australian reality also provide casestudy the importance proteomics specific sector the area food science the use bacteria technological interest highlighting major achievement obtained argentinean proteomics scientist finally provide general picture the endeavor being undertaken argentinean proteomics scientist and their international collaborator promote the proteomicsbased research with the new generation scientist and phd student both argentina and other country the southern cone',\n",
       " 'recently there ha growing interest msbased analysis leishmania for biology study hostparasite interaction and drug target discovery the aim this study were analyzed protein profile the procyclic and metacyclic stage tropica and investigate their potential role metacyclogenesis molecular mechanism sequential window acquisition all theoretical fragment ion spectrum mass spectrometry swathms analysis wa used analyze protein profile each procyclic and metacyclic stage protein with fold changegt2 and plt were considered significantly differentially expressed protein deps the deps were subjected gene ontology kegg pathway and network analysis using panther and string database respectively quantitative realtime pcr six selected gene validated the proteomic data quantified total protein procyclic and metacyclic cell and differentially expressed protein downregulated metacyclic compared procyclic the basis biological process the deps were primarily involved metabolic process and cellular process addition several enriched term were identified via molecular function which among them catalytic activity and binding were disclosed top category the kegg pathway analysis indicated metabolic pathway pvalue including gene term the top pathway deps these finding bring new insight our understanding the molecular characterization metacyclogenesis and infective form tropica comparative analysis the proteome both developmental stage the tropica would help the identification protein candidate for the development new potential drug target and vaccine',\n",
       " 'mass spectrometry msbased proteomics emerging broadly effective mean for identification characterization and quantification protein that are integral component the process essential for life characterization protein the proteome and subproteome the phosphoproteome proteoglycome degradomepeptidome level provides foundation for understanding fundamental aspect biology emerging technology such ion mobility separation coupled with and microchipbasedproteome measurement combined with instrumentation and chromatographic separation technique such nanoscale reversed phase liquid chromatography and capillary electrophoresis show great promise for both broad undirected and targeted highly sensitive measurement msbased proteomics increasingly contribute our understanding the dynamic interaction and role that protein and peptide play advancing our understanding biology system wide level for wide range application including investigation microbial community bioremediation and human health',\n",
       " 'cephalopod like other protostomes lack adaptive immune system and only rely innate immune system the main immune cell are haemocytes hcts which are able respond pathogen and external attack first report based morphological observation revealed that the white body located the optic sinus cuttlefish wa the origin hcts combining transcriptomic and proteomic analysis identified several factor known involved haematopoiesis vertebrate specie cuttlefish among these factor member the jakstat signaling pathway were identified some them for the first time molluscan transcriptome and proteome immune factor such member the tollnfκb signaling pathway pattern recognition protein and receptor and member the oxidative stress response were also identified and support immune role the both transcriptome and proteome analysis revealed that the harbor intense metabolism concurrent with the haematopoietic function finally comparative analysis the and hct proteome revealed many protein common confirming previous morphological study the origin hcts cuttlefish this molecular work demonstrates that the multifunctional and provides base for haematopoiesis regulation cuttlefish',\n",
       " 'vascular proteomics providing two main type data protein that actively participate vascular pathophysiological process and novel protein candidate that can potentially serve useful clinical biomarkers although both type protein can identified similar proteomic strategy and method important clearly distinguish biomarkers from mediator disease particular protein group protein may participate pathogenic process but not serve effective biomarker alternatively useful biomarker may not mediate pathogenic pathway associated with disease creactive protein date there are clear successful example which discovery proteomics ha led novel useful clinical biomarker cardiovascular disease nevertheless new source biomarkers are being explored secretomes circulating cell exosomes and microparticles increasing number novel protein involved atherogenesis are constantly described and new technology and analytical strategy quantitative proteomics are being developed access low abundant protein therefore this presage new era discovery and further step the practical application diagnosis prognosis and early action medical treatment cardiovascular disease',\n",
       " 'the honey from chestnut acacia sunflower eucalyptus and orange wa analysed for it proteome content order see any plant protein present would allow the proteotyping these different variety since the total protein content turned out minute each honey type were diluted and then added with proteominer enhance the visibility the proteinaceous material all band visible the sdspage profile each type honey were eluted digested and identified mass spectrometry ltqxl instrument turned out that all protein identified except one the enzyme glyceraldehyde3phosphate dehydrogenase from mesembryanthemum crystallinum were not plant origin but belonged the apis mellifera proteome among the total protein identified eight but only seven basic constituent all type honey five belonged the family major royal jelly protein and were also the most abundant one any type honey together with αglucosidase and defensin1 thus appears that honey ha proteome resembling the royal jelly proteome but with considerably fewer specie except that it protein concentration lower three four order magnitude compared royal jelly attempt identifying additional plant pollen nectar protein via peptidome analysis were unsuccessful',\n",
       " 'the growing availability genome sequence and advancement highthroughput omics and analytical chemistry technology have reinvigorated the study biodegradation process aromatic hydrocarbon catabolism particular proteomics approach globally have identified and quantified bacterial enzyme responsible for aromatic hydrocarbon metabolism which significantly assist determining strategy for the biodegradation aromatic hydrocarbon with application environmental bioremediation',\n",
       " 'protein homeostasis dysfunction ha been implicated the development and progression aging related human pathology there need for the establishment quantitative method evaluate global protein homoeostasis function the ubiquitin proteasome system play key role regulating protein homeostasis applied quantitative proteomic method evaluate the sensitivity sitespecific ubiquitylation event marker for protein homeostasis dysfunction here demonstrate that the ubmodified proteome can exceed the sensitivity engineered fluorescent reporter marker for proteasome dysfunction and can provide unique signature for distinct proteome challenge which not possible with engineered reporter demonstrate that combining ubproteomics with subcellular fractionation can effectively separate degradative and regulatory ubiquitylation event distinct protein population using recently developed potent inhibitor the critical protein homeostasis factor pvcp demonstrate that distinct insult protein homeostasis function can elicit robust and largely unique alteration the ubmodified proteome taken together demonstrate that proteomic approach monitor the ubmodified proteome can used evaluate global protein homeostasis and can used monitor distinct functional outcome for spatially separated protein population',\n",
       " 'proteomics experiment commonly aim estimate and detect differential abundance across all expressed protein within this experimental design some the most challenging measurement are small fold change for lower abundance protein while bottomup proteomics method are approaching comprehensive coverage even complex eukaryotic proteome failing reliably quantify lower abundance protein can limit the precision and reach experiment much le than the identifiedlet alone totalproteome here test the ability two common method tandem mass tagging tmt method and labelfree quantitation method lfq achieve comprehensive quantitative coverage benchmarking their capacity measure different level change and fold across entire data set both method achieved comparably accurate estimate for all 3foldchanges however the tmt method detected change that reached statistical significance three time more often due higher precision and fewer missing value these finding highlight the importance refining proteome quantitation method bring the number usefully quantified protein into closer agreement with the number total quantified protein',\n",
       " 'within mammalian organism the interaction among cell both short and long distance mediated soluble factor released cell into the extracellular environment the secreted protein may involve extracellular matrix protein proteinase growth factor protein hormone immunoregulatory cytokine chemokines other bioactive molecule that have direct impact target cell phenotype stem cell mesenchymal adipose neural and embryonic origin fibroblast feeder cell well primary isolates astrocyte endothelial and muscle cell have recently become target intensive secretome profiling with the search for protein regulating cell survival proliferation differentiation inflammatory response recent advance and challenge the stem cell and primary cell secretome analysis together with the most relevant result are discussed this review',\n",
       " 'platelet are the fundamental player primary hemostasis but are also involved several pathological condition the remarkable advance proteomic methodology have allowed better understanding the basic physiological pathway underlying platelet biology addition recent platelet proteomics focused disease condition helping elucidate the molecular mechanism complex andor unknown human disorder and find novel biomarkers for early diagnosis and drug target the most common and innovative proteomic technique both gelbased and gelfree used platelet proteomics will reviewed here particular focus will given study that used subproteomic strategy analyze specific platelet condition resting activated compartment membrane granule and microparticles fraction phosphoproteome glycoproteome the thousand platelet protein and interaction discovered far these different powerful proteomic approach represent precious source information for both basic science and clinical application the field platelet biology',\n",
       " 'synopsis the annual progress report for the center for structure membrane protein specialized center the protein structure initiative',\n",
       " 'lactobacillus acidophilus ncfm ncfm welldocumented probiotic bacterium isolated from human gut detailed gelbased ncfm proteomics addressed the socalled alkaline range protein were identified the spot picked from the coomassie brilliant blue stained gel using malditofms the unique gene product among the protein identification were assigned different functional category and evaluated considering calculated distribution abundance well grand average hydrophobicity value none the very few available lactic acid bacteria proteome reference map included the range the present report such data the proteome ncfm fundamentally complement current knowledge protein profile limited the acid and neutral range',\n",
       " 'the etiology type diabetes t1d remains elusive enzymatically isolated and cultured eic islet can not fully reflect the natural protein composition and disease process vivo islet because the stress from isolation procedure order study islet protein composition condition close the natural environment performed proteomic analysis eic islet and laser capture microdissected lcm human islet and acinar tissue from freshfrozen pancreas section three cadaveric donor and protein were identified from islet equivalent ieq lcm islet and acinar tissue respectively the proteomic profile lcm islet were reproducible within and among cadaveric donor the endocrine hormone were only detected lcm islet whereas catalytic enzyme were significantly enriched acinar tissue furthermore high overlap protein and similar function distribution were found between lcm and eic islet proteome except that eic islet had more acinar contaminant and stressrelated signal transducer activity protein the comparison among lcm islet lcm acinar tissue and eic islet proteome indicates that lcm combined with proteomic method enables accurate and unbiased profiling islet proteome from frozen pancreata this pave the way for proteomic study human islet during the progression t1d',\n",
       " 'citrullination posttranslational modification implicated various human disease including rheumatoid arthritis alzheimers disease multiple sclerosis and cancer due relatively low concentration citrullinated protein the total proteome confident identification citrullinated proteome challenging mass spectrometry msbased proteomic analysis from these msbased analysis feature that characterize citrullination such immonium ion ims and neutral loss nls called diagnostic ion have been reported however there ha been lack systematic approach comprehensively search for diagnostic ion and statistical method for the identification citrullinated proteome based these diagnostic ion here present systematic approach identify diagnostic ims internal ion ints and nls for citrullination from tandem mass msms spectrum diagnostic ints mainly consisted internal fragment ion for and tripeptides that contained two and three amino acid with least one citrullinated arginine respectively statistical logistic regression model wa built for confident assessment citrullinated peptide that database search identified true positive and prediction citrullinated peptide that database search failed identify false negative using the diagnostic ims ints and nls application our model complex global proteome data set demonstrated the increased accuracy the identification citrullinated peptide thereby enhancing the size and functional interpretation citrullinated proteome',\n",
       " 'regenerative tissue such the skin epidermis the intestinal mucosa the hematopoietic system are organized hierarchical manner with stem cell building the top this hierarchy somatic stem cell harbor the highest selfrenewal activity and generate series multipotent progenitor which differentiate into lineage committed progenitor and subsequently mature cell this report applied indepth quantitative proteomic approach analyze and compare the full proteome vivo isolated and facssorted population highly enriched for either multipotent hematopoietic stemprogenitor cell hspcs linnegsca1ckit myeloid committed precursor linnegsca1ckit employing stable isotope dimethyl labeling and highresolution mass spectrometry more than protein were quantified from biological triplicate experiment subjected rigorous statistical evaluation protein were found differentially expressed between multipotent and myeloid committed cell the differential protein content these cell population point distinct structural organization the cytoskeleton including remodeling activity addition found marked difference the expression metabolic enzyme including clear shift specific protein isoforms the glycolytic pathway protein involved translation showed collective higher expression myeloid progenitor indicating increased translational activity strikingly the data uncover unique signature related immune defense mechanism centering the rigi and type1 interferon response system which are installed multipotent progenitor but not evident myeloid committed cell this suggests that specific and far unrecognized mechanism protect these immature cell before they mature conclusion this study indicates that the transition hematopoietic stemprogenitors toward myeloid commitment accompanied profound change processing cellular resource adding novel insight into the molecular mechanism the interface between multipotency and lineage commitment',\n",
       " 'detection protein which may potential biomarkers disorder represents big step forward understanding the molecular mechanism that underlie pathological process this context proteomics play the important role opening path for the identification molecular signature that can potentially assist early diagnosis several clinical disturbance aim this report provide overview the wide variety proteomic strategy that have been applied the investigation chronic obstructive pulmonary disease copd severe disorder that cause irreversible damage the lung and for which there cure yet the result this area published over the past decade show that proteomics indeed ha the ability monitoring alteration expression profile protein from fluidstissues patient affected copd and healthy control however these data also suggest that proteomics while being attractive tool for the identification novel pathological mediator copd remains technique mainly generated and developed research laboratory great effort dedicated the validation these biological signature will result the proof their clinical utility',\n",
       " 'identifying and analyzing component complex essential understand the activity and organization the cell moreover provides additional information the possible function protein involved these complex two bioinformatics approach are usually used for this purpose the first based the identification clustering algorithm full densely connected subgraphs proteinprotein interaction network derived from experimental source that might represent complex the second approach consists the integration genomic and proteomic data using bayesian network decision tree this approach based the hypothesis that protein involved complex usually share common property',\n",
       " 'report technique selectively and continuously label the proteome individual cell type coculture named cell typespecific labeling using amino acid precursor ctap through transgenic expression exogenous amino acid biosynthesis enzyme vertebrate cell overcome their dependence supplemented essential amino acid and can selectively labeled through metabolic incorporation amino acid produced from heavy isotopelabeled precursor when testing ctap several human and mouse cell line could differentially label the proteome distinct cell population coculture and determine the relative expression protein quantitative mass spectrometry addition using ctap identified the cell origin extracellular protein secreted from cell coculture believe that this method which allows linking protein their cell source will useful study cellcell communication and potentially for discovery biomarkers',\n",
       " 'targeted analysis dataindependent acquisition dia data powerful mass spectrometric approach for comprehensive reproducible and precise proteome quantitation requires spectral library which contains for all considered peptide precursor ion empirically determined fragment ion intensity and their predicted retention time rts however are not comparable absolute scale especially heterogeneous measurement are combined here present method for highprecision prediction which significantly improves the quality targeted dia analysis compared silico prediction and the state the art indexed retention time irt normalization approach describe highprecision normalized algorithm which implemented the spectronaut software furthermore investigate the influence nine different experimental factor such chromatographic mobile and stationary phase irt precision summary show that using targeted analysis dia data with highprecision irt significantly increase sensitivity and data quality the irt value are generally transferable across wide range experimental condition best result however are achieved library generation and analytical measurement are performed the same system',\n",
       " 'timely classification and identification bacteria vital importance many area public health present mass spectrometry msbased proteomics approach for bacterial classification this method bacterial proteome database derived from all potential protein coding open reading frame orfs found fully sequenced bacterial genome amino acid sequence tryptic peptide obtained lcesi msms analysis the digest bacterial cell extract are assigned individual bacterial proteome the database phylogenetic profile these peptide are used create matrix sequencetobacterium assignment these matrix viewed specific assignment bitmap are analyzed using statistical tool reveal the relatedness between test bacterial sample and the microorganism database shown that sufficient amount sequence information obtained from the msms experiment bacterial sample can classified strain level using this proteomics method leading it positive identification',\n",
       " '',\n",
       " 'the last decade proteomic analysis ha become integral tool for investigation tumor biology complementing the genetic analysis the idea proteomics characterize protein evaluation their expression function and interaction proteomics may also provide information about posttranslational modification protein and evaluate their value specific disease biomarkers the major purpose clinical proteomics study improve diagnostic procedure including the precise evaluation biological feature tumor cell and understand the molecular pathogenesis cancer invent novel therapeutic strategy and target this review briefly describes the latest report proteomic study nsclc contains summary the method used detect proteomic marker different type biological material and their clinical application diagnostic prognostic and predictive biomarkers compiled the basis the most recent literature and our own experience',\n",
       " 'plant are sessile organism that need respond environmental change quickly and efficiently they can accomplish this triggering specialized signaling pathway often mediated protein phosphorylation and dephosphorylation phosphorylation fast response that can switch off myriad biological pathway and process proteomics and are the main tool employed the study protein phosphorylation advance the technology allow simultaneous identification and quantification thousand phosphopeptides and protein that are essential understanding the sophisticated biological system and regulation this review summarize the advance phosphopeptide enrichment and quantitation for phosphorylation site mapping and new data acquisition method database and informatics interpretation biological insight and crosstalk with other ptms well future direction and challenge the field phosphoproteomics',\n",
       " 'hepesglutamic acid buffermediated organic solvent protection effect hopefixation ha been introduced alternative formalin fixation clinical sample beyond preservation morphological structure for histology hopefixation wa demonstrated compatible with recent method for rna and dna sequencing however the suitability hopefixed material for the inspection proteome mass spectrometry far remained undefined this particular interest since protein constitute prime resource for drug research and can give valuable insight into the activity status signaling pathway this study extracted protein from human lung tissue and tested hopetreated and snapfrozen tissue comparatively proteome and phosphoproteome analysis high confident data from accurate mass spectrometry allowed the identification protein and phosphorylation site hopefixation did not hinder the representative extraction protein and investigating their biochemical property covered subcellular localization and cellular process revealed bias caused the type fixation conclusion proteome well phosphoproteome data hope lung sample were qualitatively equivalent result obtained from snapfrozen tissue thus hopetreated tissue match clinical demand both histology and retrospective proteome analysis patient sample proteomics',\n",
       " 'protein identified multiple separate band gel reflects variation the molecular weight caused alternative splicing endoproteolytic cleavage ptms such glycosylation ubiquitination characterize such protein distribution over the band defined entity called island the band region including the band the same protein identified sequentially quantified the island distribution using new variable called iscore previously described park proteomics analyzed human brain tissue using multidimensional msms separation method here the new method island analysis wa applied the previous proteome data the soluble and membrane protein fraction human brain tissue were reanalyzed using the island distribution the proteome the soluble fraction exhibited more variation island position than that the membrane fraction through the island analysis identified protein modification and protein complex over the gel band',\n",
       " 'staphylococcus aureus grampositive commensal bacterium that regarded major threat for modern health care system this relates both the ability aureus overcome antibiotic therapy developing highlevel resistance against multiple antibiotic and this bacterium extensive arsenal virulence factor understanding the mechanism resistance and functional study stress and starvation response are the main goal proteomics staphylococcal research this review highlight recent advance gelbased and gelfree proteomics analysis aureus and pinpoint the importance locationspecific proteomics study targeting the cytosol the membrane the cell surface and the extracellular milieu combination with integrated global proteome study emerging hot topic staphylococcal proteomics are discussed with special focus vivo proteomics membrane vesicle biofilm formation and the acquisition absolute proteome data for system biological modeling approach',\n",
       " 'virus constantly adapt and modulate the host environment during replication and propagation both dna and rna virus encode multifunctional protein that interact with and modify host cell protein while viral genome were the first complete sequence known the corresponding proteome are only now elucidated with some surprising result even more daunting the task globally monitor the impact viral infection the proteome the host cell and many technical hurdle must still overcome order facilitate robust and reproducible measurement further complicating the picture the dynamic nature protein including posttranslational modification enzymatic cleavage and activation destruction proteolytic event nevertheless several promising study have been published using highthroughput method directly measuring protein abundance particularly quantitative semiquantitative mass spectrometrybased analysis viral and cellular proteome are now being used characterize virus and their host interaction addition the full set interaction between viral and host protein the interactome beginning emerge with often unexpected interaction that need carefully validated this review will discus two major area viral proteomics first virion proteomics such the protein characterization viral particle and second proteoviromics including the viral protein interactomics and the quantitative analysis host cell proteome during viral infection',\n",
       " 'the term proteoform describes all combination change protein elucidated through intact mass proteomics paleoproteomic study have begun using digestionfree and topdown technique access information from ancient and historical remains however discus protein change that uniquely occur archaeological and paleontological proteome the result diagenesis physical and chemical change imparted burial novel term needed that both address issue combinatorics and distinguishes diageneticspecific alteration significance the term diagenetiform provides the opportunity communicate clearly the set diagenetic change found preserved protein the diagenetiform nomenclature will allow for topdown paleoproteomic study accurately describe the total change detected ancient protein',\n",
       " 'biological experiment are most often performed with immortalized cell line because they are readily available and can expanded without limitation however cell line may differ from the vivo situation important aspect here introduce straightforward methodology compare cell line their cognate primary cell and derive comparative functional phenotype used silac stable isotope labeling amino acid cell culture for quantitative mass spectrometrybased comparison the hepatoma cell line hepa with primary hepatocytes the resulting quantitative proteome protein had asymmetric distribution with many protein downregulated the cell line bioinformatic analysis the quantitative proteomics phenotype revealed that hepa cell were deficient mitochondrion reflecting rearrangement metabolic pathway drastically upregulate cell cycleassociated function and largely shut down drug metabolizing enzyme characteristic for the liver this quantitative knowledge change provides important basis adapt cell line more closely resemble physiological condition',\n",
       " 'comprehensive characterization proteome defines fundamental goal proteomics order maximize proteome coverage for complex protein mixture identify many protein possible various different fractionation experiment are typically performed and the individual fraction are subjected mass spectrometric analysis the resulting data are integrated into large and heterogeneous datasets proteome coverage prediction refers the task extrapolating the number protein discovery future measurement conditioned sequence already performed measurement proteome coverage prediction early stage enables experimentalists design and plan efficient proteomics study date there doe not exist any method that reliably predicts proteome coverage from integrated datasets present generalized hierarchical pitmanyor process model that explicitly capture the redundancy within integrated datasets the accuracy our approach for proteome coverage prediction assessed applying integrated proteomics dataset for the bacterium interrogans the proposed procedure outperforms hoc extrapolation method and prediction method designed for nonintegrated datasets furthermore the maximally achievable proteome coverage estimated for the experimental setup underlying the interrogans dataset discus the implication our result for determining rational stop criterion and their influence the design efficient and reliable proteomics study',\n",
       " 'understanding the molecular mechanism endogenous and environmental metabolite crucial for basic biology and drug discovery with the genome proteome and metabolome many organism being readily available researcher now have the opportunity dissect how key metabolite regulate complex cellular pathway vivo nonetheless characterizing the specific and functional protein target key metabolite associated with specific cellular phenotype remains major challenge innovation chemical biology are now poised address this fundamental limitation physiology and disease this review highlight recent advance chemoproteomics for targeted and proteomewide analysis metaboliteprotein interaction that have enabled the discovery unpredicted metaboliteprotein interaction and facilitated the development new small molecule therapeutic',\n",
       " 'singlecell proteomics represents field extremely sensitive proteomic analysis owing the minute amount yet complex protein single cell without amplification potential nucleic acid singlecell mass spectrometry analysis demand special instrumentation running with optimized parameter maximize the sensitivity and throughput for comprehensive proteomic discovery facilitate such analysis here investigated two factor critical peptide sequencing and protein detection shotgun proteomics precursor ion isolation window and maximum precursor ion injection time itmax ultrahighfield quadrupole orbitrap qexactive counterintuitive the frequently used proteomic parameter for bulk sample our experimental data and subsequent modeling suggested universally optimal for sample quantity ranging from and samplequantity dependent itmax more than for 1ng sample compared with the benchmark condition and itmax our optimization generated increase the detected protein group for 1ng sample the additionally identified protein allowed deeper penetration proteome for better revealing crucial cellular function such signaling and cell adhesion hope this effort can prompt singlecell and trace proteomic analysis and enable rational selection parameter',\n",
       " '',\n",
       " 'the present review deal with the use combinatorial ligand library composed hexapeptides the capture and concentration the lowabundance proteome this method first reported compared with other current methodology aimed exploring the deep proteome such depletion highabundance protein especially serum and cerebrospinal fluid individual combined antibody against the most abundant specie narrow 1phunit ipgs zoom gel and prefractionation with multicompartment electrolyzers with offgel electrophoresis the physicochemical property the hexapeptide library are also explored namely assessing the proper length the bait and the behavior shorter oligopeptides down capture elicited single amino acid number example the use this library given such the analysis biological fluid human serum urine bile cerebrospinal fluid and cell lysates platelet red blood cell all case wa possible detected from three five time many protein compared control untreated sample perhaps the most spectacular result were obtained with the erythrocyte proteome where protein could identified the minority proteome representing only the total cell lysate another interesting area application regard the concentration and detection trace impurity contaminating rdna protein meant for human consumption several host protein never reported before could revealed for the first time other nonhuman sample are currently under investigation such eggwhite where le than unique gene product could identified egg yolk with unique specie and latex from hevea brasiliensis anticipated that the ligand library could most useful tool for detecting biomarkers for different pathology and for drug treatment future outlook one could envision the synthesis specialized library able capture given class protein phospho and glycoprotein additionally the library could used association with other technique currently vogue such zoom gel offgels and the like',\n",
       " 'protein are involved all task life and their characterization essential understand the underlying mechanism biological process present method called differential visual proteomics geared study proteomewide structural change protein and proteincomplexes between disturbed and undisturbed cell between two cell population implement this method the cell are lysed and the lysate prepared lossless manner for singleparticle electron microscopy the sample are subsequently imaged the individual particle are computationally extracted from the image and pooled together while keeping track which particle originated from which specimen the extracted particle are then aligned and classified final quantitative analysis the particle class found identifies the particle structure that differ between positive and negative control sample the algorithm and graphical user interface developed perform the analysis and visualize the result were tested with simulated and experimental data the result are presented and the potential and limitation the current implementation are discussed envisage the method tool for the untargeted profiling the structural change the proteome singlecells response disturbing force',\n",
       " '',\n",
       " 'measuring the synthesis new protein the context much greater number preexisting protein can difficult overcome this obstacle bioorthogonal noncanonical amino acid tagging boncat can combined with stable isotope labeling amino acid cell culture silac for comparative proteomic analysis novo protein synthesis bonlac the present study show that alkyne resinbased isolation lazidohomoalanine ahalabeled protein using azidealkyne cycloaddition minimizes contamination from preexisting protein using this approach isolated and identified boncatlabeled protein the nascent proteome isolated boncat wa very similar the steadystate proteome although transcription factor were highly enriched boncat about the methionine residue were replaced aha our boncat sample which allowed for identification methioninecontaining peptide there wa bias against lowmethionine protein boncat the proteome level when applied the bonlac approach screen for brainderived neurotrophic factor bdnfinduced protein synthesis protein were found significantly changed after bdnf stimulation our study demonstrated that the newly synthesized proteome even after short period stimulation can efficiently isolated boncat and analyzed depth that similar that the steadystate proteome',\n",
       " 'this study the concept sizeselective proteolysis wa first described using the mesoporous silicabased trypsin nanoreactor for analysis complex protein sample lowmw protein were preferentially digested for identification while highmw protein were excluded from digestion',\n",
       " '',\n",
       " 'progress msbased method for veterinary research and diagnostics lagging behind compared the human research and proteome data domestic animal still not well represented open source data repository this particularly true for the equine specie here present first equine peptideatlas encompassing highresolution tandem analysis sample representing selection equine tissue and body fluid from healthy and diseased animal the raw data were processed through the transproteomic pipeline yield high quality identification protein and peptide the current release comprises distinct peptide representing canonical protein observed false discovery rate the peptide level and the protein level data from the equine peptideatlas are available for experimental planning validation new datasets and proteomic data mining resource the advantage the equine peptideatlas are demonstrated example mining the content for information potential and wellknown equine acute phase protein which have extensive general interest the veterinary clinic the extracted information will support further analysis and emphasizes the value the equine peptideatlas resource for the design targeted quantitative proteomic study',\n",
       " 'bacterial pathogen were rapidly identified shotgun proteomics using novel easy implement database search strategy peptide sequence data from nanolcmsms wa searched against database represented concatenated proteome completed genome sequence select bacterial specie including bsl3 select agent were used demonstrate this method',\n",
       " 'one the major goal the chromosomecentric human proteome project chpp catalog and annotate myriad heterogeneous proteoforms produced thousand gene achieve detailed and personalized understanding into proteome suggest using customized rnaseq library potential proteoforms which includes aberrant variant specific certain biological sample twodimensional electrophoresis coupled with highperformance liquid chromatography allowed downgrade the difficulty biological mixing following shotgun mass spectrometry benchmark the proposed pipeline examined heterogeneity the hepg2 hepatoblastoma cell line proteome data are available via proteomexchange with identifier pxd',\n",
       " 'the temporal regulation protein abundance and posttranslational modification key feature cell division recently analysed gene expression and protein abundance change during interphase under minimally perturbed condition here show that using specific intracellular immunolabelling protocol facs separation interphase and mitotic cell including mitotic subphases can combined with proteomic analysis mass spectrometry using this primmus proteomic analysis intracellular immunolabelled cell subset approach now compare protein abundance and phosphorylation change interphase and mitotic fraction from asynchronously growing human cell identify set phosphorylation site increased during termed early riser this set includes phosphorylation tpx2 which show important for tpx2 function and mitotic progression further use primmus provide the first proteomewide analysis protein abundance remodeling between prophase prometaphase and anaphase',\n",
       " 'membrane protein play critical role many biological function and are often the molecular target for drug discovery however their analysis present special challenge largely due their highly hydrophobic nature present surfactantaided shotgun proteomics approach for membrane proteome analysis this approach membrane protein were solubilized and digested the presence sd followed newly developed autooffline liquid chromatographymatrixassisted laser desorption ionization lcmaldi tandem analysis because high tolerance maldi sd onedimensional separation can combined with maldi for direct analysis protein digest containing sd without the need for extensive sample cleanup addition the heated droplet interface used lcmaldi can work with high flow separation allowing relatively large amount protein digest used for lcmaldi which facilitates the detection low abundance protein the proteome identification result obtained lcmaldi are compared the gel electrophoresisms method well the shotgun proteomics method using lcelectrospray ionization demonstrated that while lcmaldi provides more extensive proteome coverage compared the other two method these three method are complementary each other and combination these method should provide more comprehensive membrane proteome analysis',\n",
       " 'sample multiplexing using isobaric tagging powerful strategy for proteomewide protein quantification one major caveat isobaric tagging ratio compression that result from the isolation fragmentation and quantification coeluting nearisobaric peptide phenomenon typically referred ion interference robust platform ensure quality control optimize parameter and enable comparison across sample essential new instrumentation and analytical method evolve here introduce tkoiqc integrated platform consisting the triple knockout tko yeast digest standard and automated webbased database search and protein profile visualization application highlight two new tko standard based the tmtpro reagent tkopro9 and tkopro well updated tko viewing tool tvt tkoiqc greatly facilitates the comparison instrument performance with straightforward and streamlined workflow',\n",
       " 'cancer cell are characterized higher level intracellular reactive oxygen specie ro due metabolic aberration ro are widely accepted second messenger triggering pivotal signaling pathway involved the process cell metabolism cell cycle apoptosis and autophagy however the underlying cellular mechanism remain largely unknown recently accumulating evidence ha demonstrated that ro initiate redox signaling through direct oxidative modification the cysteine key redoxsensitive protein termed redox sensor uncovering the functional change underlying redox regulation redox sensor urgently required and the role different redox sensor distinct disease state still remains identified assist this redox proteomics ha been developed for the highthroughput screening redox sensor which will benefit the development novel therapeutic strategy for cancer treatment highlighted here are recent advance redox proteomics approach and their application identifying redox sensor involved tumor development',\n",
       " '',\n",
       " 'teeth have been focus interest for many centuriesdue medical problem with them they are the hardest part the human body and are composed three mineralized partsenamel dentin and cementum together with the soft pulp however saliva also ha significant impact tooth quality proteomic research human teeth now accelerating and includes all part the tooth some methodological problem still need overcome this research fieldmainly connected with calcified tissue this review will provide overview the current state research with focus the individual part the tooth and pellicle layer well saliva these proteomic result can help not only stomatology term early diagnosis identifying risk factor and systematic control',\n",
       " 'plastid are essential organelle present virtually all cell plant and green algae the proteome plastid and particular chloroplast have received significant amount attention recent year various fractionation and mass spectrometry technique have been applied catalogue the chloroplast proteome and it membrane compartment neural network and hidden markov model combination with experimentally derived filter were used try predict the chloroplast subproteomes some the many proteinprotein interaction well posttranslational modification have been characterized nevertheless our understanding the chloroplast proteome and it dynamic very incomplete rapid improvement and widescale implementation and new tool for comparative proteomics will undoubtedly accelerate this understanding the near future proteomics study often generate large amount data and these data are only meaningful they can easily accessed via the worldwideweb and connected other type biological information the plastid proteome data base ppdb httpwwwppdbtccornelledu and other web resource are discussed this review will briefly summarize recent experimental and theoretical effort attempt translate these data into the function the chloroplast and outline expectation and possibility for comparative chloroplast proteomics',\n",
       " 'amino acidtagging strategy are widespread proteomics because the central role mass spectrometry detection technique protein science the term mass tagging wa coined describe the attachment label which serf analysis andor add analytical value the measurement these socalled mass tag can used for separation enrichment detection and quantitation peptide and protein this context cysteine frequent target for modification because the thiol function can react specifically nucleophilic substitution addition furthermore cysteine present natural modification biological importance and low occurrence the proteome that justify the development strategy specifically target them peptide protein the present review the masstagging method directed cysteine residue are comprehensively discussed and the advantage and drawback these strategy are addressed some concrete application are given underline the relevance cysteinetagging technique for msbased proteomics',\n",
       " 'the molecular side signaling still ha many secret reveal this editorial guide describes area particular interest with regard understanding cellular regulation the level individual molecule and macromolecular complex advance technology and the accumulation proteomic and genomic data reveal previously unknown component signaling network the spatial organization macromolecular complex and how information flow through the network produce specific cellular response',\n",
       " 'one method improving the protein profiling complex mammalian proteome the use prefractionation followed application narrow range two dimensional gel the success this strategy relies sample solubilization poor solubilization ha been associated with missing protein fraction and diffuse streaked andor trailing protein spot this study sought optimize the solubilization prefractionated human cancer cell sample using isoelectric focusing ief rehydration buffer containing variety commercially available reducing agent detergent chaotropes and carrier ampholytes the solubilized protein were resolved gel and compared among five tested ief rehydration buffer those containing cholamidopropyldimethylamino1propane sulfonate chap and dithiothreitol dtt provided superior resolution while that containing nonidet did not significantly affect protein resolution and the tributyl phosphine tbpcontaining buffer yielded consistently poor result addition found that buffer containing typically high urea and ampholyte level generated sharper gel using these optimized condition wa able apply gel analysis successfully fractionated protein from human breast cancer tissue mcf7 across range',\n",
       " 'the barnacle balanus amphitrite one the primary model organism for rockyshore ecology study and biofouling research this barnacle specie ha complex life cycle during which the swimming nauplius molt six time and transforms into cyprid stage cyprids must attach surface metamorphose into juvenile barnacle clarify the overall profile protein expression during larval development and metamorphosis 2de wa used compare the proteome the nauplius the swimming cyprid the attached cyprid and the metamorphosed cyprid the proteome the swimming cyprid wa distinctly different from that other life stage and had about spot the proteome the attached and metamorphosed cyprids were similar with respect major protein but had significantly lower number spot compared that swimming larval stage obviously synthesis most protein from swimming cyprids wa switched off after attachment and metamorphosis our advanced analysis malditoftof msms allowed identify the protein that were differentially and abundantly expressed the swimming cyprid these protein included signal transduction protein adenylate cyclase and calmodulin and juvenile hormone binding protein summary for the first time have analyzed the global protein expression pattern fouling marine invertebrate larva during metamorphosis our study provides new insight into the mechanism barnacle larval metamorphosis and also provides foundation for exploring novel target for antifouling treatment',\n",
       " '',\n",
       " '',\n",
       " 'understanding signaling pathway neuroscience requires highresolution map the underlying protein network proximitydependent biotinylation with engineered enzyme combination with mass spectrometrybased quantitative proteomics ha emerged powerful method dissect molecular interaction and the localization endogenous protein recent application neuroscience have provided insight into the composition subsynaptic structure including the synaptic cleft and inhibitory postsynaptic density here compare the different enzyme and smallmolecule probe for proximity labeling the context cultured neuron and tissue review existing study and provide technical suggestion for the vivo application proximity labeling',\n",
       " 'protein regulate many cellular function and analyzing the presence and abundance protein biological sample are central focus proteomics the discovery and validation biomarkers pathway and drug target for various disease can accomplished using mass spectrometrybased proteomics however with masslimited sample like tumor biopsy can challenging obtain sufficient amount protein generate highquality mass spectrometric data technique developed for macroscale quantity recover sufficient amount protein from milligram quantity starting material but sample loss become crippling with these technique when only microgram amount material are available combat this challenge proteomicists have developed microscale technique that are compatible with decreased sample size lower and still enable excellent proteome coverage extraction contaminant removal protein quantitation and sample handling technique for the microgram protein range are reviewed here with emphasis liquid chromatography and bottomup mass spectrometrycompatible technique also range biological specimen including mammalian tissue and model cell culture system are discussed',\n",
       " 'currently there are few predictive biomarkers key biomonitoring specie such oyster that can detect heavy metal pollution coastal waterway several attribute make oyster superior other organism for positive biomonitoring heavy metal pollution particular they are filter feeder with high capacity for bioaccumulation this study used two proteomics approach namely labelfree shotgun proteomics based sdspage gel separation and gas phase fractionation investigate the heavy metal stress response sydney rock oyster protein sample were prepared from haemolymph oyster exposed μgl pbcl2 cucl2 zncl2 for day closed aquarium peptide were identified using bivalvia protein sequence database due the unavailability complete oyster genome sequence statistical analysis revealed potential biomarker protein well several protein biosynthetic pathway greatly impacted metal stress these have the potential incorporated into bioassay for prevention and monitoring heavy metal pollution australian oyster bed the study confirms that proteomic analysis biomonitoring specie promising approach for assessing the effect environmental pollution and our experiment have provided insight into the molecular mechanism underlying oyster stress response',\n",
       " 'the proteomic profile hypothalamus key organ cns explored here employing two widely used technique hplcesiion trap and hplcesiquadrupoletof strong cation exchange used for the fractionation peptide and protein search engine mascot employed for data query one hundred and thirty six protein with peptide were identified hplcesiion trap while protein with peptide were characterized hplcesiquadrupoletof among the total protein identified both experiment protein were common both set condition the rest the protein were identified distinctly both strategy unique protein were found using the quadrupoletof while were found with the hplcesiion trap moreover these protein were classified into group based their function performed the body result presented here identified some important signal and cellular defense protein inevitable for survival stressed condition additionally also shown that any single strategy not reliable for good result due loss data depending sensitivity the instrument used',\n",
       " 'root hair are unicellular extension specialized epidermis cell under limiting condition they significantly increase the water and nutrient uptake capacity plant enlarging their root surface thus far little known about the initiation and growth root hair the monocot model specie maize gain first insight into the protein composition these specialized cell the most abundant protein maize root hair attached fourdayold primary root the inbred line were identified combining 1de with nanolcmsms shotgun proteomic experiment among the identified protein homologues protein have been previously associated with root hair formation and development other specie comparison the root hair reference proteome the monocot specie maize with the previously published root hair proteome the dicot specie soybean revealed conserved but also unique protein function root hair these two major group flowering plant',\n",
       " 'nonsmall cell lung cancer nsclc account for lung cancer and subdivided into two major histological subtypes adenocarcinoma adc and squamous cell carcinoma scc there unmet need further subdivide nsclc according distinctive molecular feature that may associated with responsiveness therapy four primary tumorderived xenograft proteome twoeach adc and scc were quantitatively compared using supersilac labeling approach together with ultrahighresolution protein highly differentially expressed the two subtypes were identified including that were validated independent cohort nsclc primary tumorderived xenograft tumor whose proteome were quantified alternative labelfree shotgun methodology the protein signature contains metabolism enzyme including phosphoglycerate dehydrogenase which more highly expressed scc well comprehensive set cytokeratins and other component the epithelial barrier which therefore distinctly different between adc and scc these result demonstrate the utility the supersilac method for the characterization primary tissue and compatibility with datasets derived from different msbased platform the validation proteome signature nsclc subtypes support the further development and application msbased quantitative proteomics basis for precision classification and treatment tumor all data have been deposited the proteomexchange with identifier pxd httpproteomecentralproteomexchangeorgdatasetpxd',\n",
       " 'native electrophoresis used for the separation individual protein protein complex and supercomplexes stable and labile proteinprotein interaction can identified depending detergent and buffer condition native gel are immediately applicable for ingel detection fluorescentlabeled protein and for ingel catalytic activity assay native gel and blot are used determine native mass and oligomeric state membrane protein protein extract from native gel are used for generation antibody for proteomic work and for advanced structural investigation separation subunit protein complex sdspage mostly used for immunological and proteomic study following the discussion these general feature specific application native electrophoresis technique various research field are highlighted immunological and receptor study biogenesis and assembly membrane protein complex protein import into organelle dynamic proteasomes proteome and subproteome investigation the identification and quantification mitochondrial alteration apoptosis carcinogenesis and neurodegenerative disorder like parkinson disease alzheimers disease and the vast variety mitochondrial encephalomyopathies',\n",
       " '',\n",
       " 'most digestive malignancy have asymptomatic course often progressing poor outcome stage surgical resection usually represents the only potentially curative option but prior assumption the malignant nature the lesion mandatory avoid exposing patient unnecessary risk unfortunately currently available diagnostic tool lack accuracy many case consequently more reliable marker are needed improve detection malignant lesion this challenging context fluid surrounding digestive malignancy represent valuable source for the search new potential biomarkers and proteomic tool offer the opportunity achieve this goal the new field proximal fluid proteomics thus emerging the arena digestive cancer biomarker discovery the present review the stateoftheart proteomic investigation aimed identifying new cancer biomarkers fluid surrounding gastrointestinal malignancy summarized comprehensive catalog proteomic study which potential cancer biomarkers from gastrointestinal fluid have been identified and assessed for their diagnostic performance also provided this article part special issue entitled biomarkers proteomic challenge',\n",
       " 'protein degradation fundamental biological process which essential for the maintenance and regulation normal cellular function human and animal protein can degraded number mechanism the ubiquitinproteasome system autophagy and intracellular protease the advance contemporary protein analysis mean that proteomics increasingly being used explore these key pathway and mean monitoring protein degradation the dysfunction protein degradative pathway ha been associated with the development number important disease including cancer muscle wasting disorder and neurodegenerative disease this review will focus the role proteomics study cellular degradative process and how these strategy are being applied understand the molecular basis disease arising from disturbance protein degradation',\n",
       " 'effective protein extraction essential especially producing wellresolved proteome gel wellresolved placental cotyledon proteome with good reproducibility have allowed researcher study the protein underlying the physiology and pathophysiology pregnancy the aim this study determine the best protein extraction protocol for the extraction protein from placental cotyledon tissue for twodimensional gel electrophoresis 2dge based widely used protein extraction strategy different extraction methodology were carefully selected which included one chemical extraction two mechanical extraction coupled protein precipitation and nine chemical extraction coupled protein precipitation extracted protein were resolved onedimensional gel electrophoresis and 2dge then wa compared with set criterion extraction efficacy protein resolution reproducibility and recovery efficiency our result revealed that better profile wa obtained chemical extraction comparison mechanical extraction further compared chemical extraction coupled protein precipitation methodology where the dnaselithium chloridedense sucrose homogenization coupled dichloromethanemethanol precipitation dnaselicldshdmpe method showed good protein extraction efficiency this however wa carried out with the best protein resolution and proteome reproducibility 2dgels dnaselicldshdmpe wa efficient the extraction protein from placental cotyledon tissue addition this methodology could hypothetically allow the protein extraction any tissue that contains highly abundant lipid and glycogen',\n",
       " 'integrated multidimensional nanoflow liquid chromatography platform with the combination protein and peptide separation via online digestion immobilized enzymatic reactor wa established and successfully applied for proteome analysis this platform protein were first separated weak anion and weak cation mixedbed microcolumn under series salt step online digested trypsin immobilized enzymatic reactor digest trapped and desalted precolumn separated nanoreversed phase liquid chromatography and finally identified electrospray ionizationmsms evaluate the performance such platform escherichia coli whole cell lysate protein were analyzed compared with the result obtained shotgun approach the identified protein number wa increased with the consumed time decreased from also compared with integrate platform based microhplc and the required sample amount wa decreased these result demonstrated that such integrated approach would attractive alternative commonly applied approach for proteome research',\n",
       " 'the present report try ass the possibility performing capture proteome via combinatorial peptide ligand library cpll twodimensional mode via orthogonal complementarity the capture phase that aim serum protein are captured physiological either low ionic strength nacl high concentration lyotropic salt the hofmeister series ammonium sulphate favouring hydrophobic interaction indeed such mechanism seems operative since the captured protein are common the two capture mode are specific only the ionic interaction mode and are found only the hydrophobically driven interaction additional bonus loss protein specie from the initial sample one the major drawback cplls are diminished about and are found only the ionic capture whereas the hydrophobically engendered capture lossfree',\n",
       " 'plant root system form complex network with the surrounding soil environment and are controlled both internal and external factor better understand the function root tip soybean during germination three proteomic technique were used analyze the protein profile root tip cell protein were extracted from the root tip 4dayold soybean seedling and analyzed using twodimensional gel electrophoresisbased proteomics sdsgel based proteomics and gelfree proteomics technique total and protein were identified root tip using the gelbased sd gelbased and gelfree proteomic technique respectively the protein identified gelbased proteomic analysis were predominantly localized the cytoplasm whereas nuclearlocalized protein were most commonly identified the sd gelbased and gelfree proteomics technique the protein identified the sd gelbased proteomic analysis were protein synthesisrelated protein furthermore protein identified using the 2dgel based proteomic technique shifted between acidic and basic isoelectric point and protein heat shock protein and aaatype atpase displayed two different molecular weight the same isoelectric point taken together these result suggest that number protein related protein synthesis and modification are activated the root tip soybean seedling during germination',\n",
       " 'the present study aimed the evaluation saliva sample pretreatment particular the sample clearance usually performed centrifugation the contribution salivary proteome and peptidome using ingel and offgel approach large content salivary protein wa detected the pellet fraction that usually discarded addition chaotropicdetergent treatment combination with sonication before the centrifugation step resulted salivary complex disruption and consequently the extraction high amount protein based this data suggest the use ureadetergent with sonication standard saliva sample pretreatment procedure also described procedure extract salivary peptide which can performed even after saliva sample treatment with chaotropicdetergents overall reported for the first time the contribution the pellet fraction the whole saliva proteome itraq analysis highlighted higher number different peptide well distinct quantity each protein class when after sample treatment with urea and sonication acetone precipitation followed solubilization with acetonitrilehcl wa performed',\n",
       " 'utilized setup based extensive prefractionation proteolytic peptide and nanoflow reversedphase lcmsms identify the subproteome human follicular fluid this indepth screen specific protein were identified the majority which are involved coagulation and immuneresponse pathway our aim define set protein marker which could predict oocyte quality',\n",
       " 'proteomic analysis extracellular vesicle ev from biological fluid powerful approach discover potential biomarkers for human disease including cancer secreted biological fluid are originated from the affected tissue order investigate significant molecule related the pathogenesis bladder cancer ev were isolated from patient urine which wa analyzed mass spectrometry based proteomics comparison the proteome the whole urine proteome demonstrated increased number protein identification comparative analysis urinary from control subject and bladder cancer patient identified total protein statistical analysis provided protein significantly increased bladder cancer urine including protein for which expression level varied cancer stage pvalue while urine represents valuable noninvasive specimen for biomarker discovery urologic cancer there high degree intra and interindividual variability urine sample the enrichment urinary demonstrated it capability and applicability providing focused identification biologically relevant protein urological disease',\n",
       " 'the incomplete surgery section invasive nonfunctional pituitary adenoma nfpas carry the increased risk complication and requires adjuvant radiotherapy and medication necessary clarify the molecular mechanism and marker invasiveness guide the management nfpa patient the study aimed proteomic variation invasive and noninvasive nfpas and sought the protein marker for invasive nfpas invasive and noninvasive nfpa tissue were analyzed each tissue with 2de and pdquest software twentyfour highresolution 2de gel were quantitatively compared determine differentially expressed protein deps between invasive and noninvasive nfpas approximately protein spot were detected each 2de map and differential spot upregulated and downregulated were identified among those differential spot deps upregulated and downregulated were characterized with peptide mass fingerprint and msms geneontology and ingenuity pathway analysis those deps revealed pathway network including mitochondrial dysfunction oxidative stress mitogenactivated protein kinase signaling abnormality trrxr activation proteolysis abnormality ketogenesis and ketolysis cyclindependent kinase signaling abnormality and amyloid processing that were significantly associated with invasive characteristic invasive nfpa those data demonstrate that proteomic variation exist between invasive and noninvasive nfpas 2debased comparative proteomics effective approach identify proteomic variation and pathway network variation those finding will serve basis understand the molecular mechanism invasive nfpas and discover protein marker effectively manage patient with invasive nfpas',\n",
       " 'this study cytoplasmic membrane protein pneumoniae strain atcc bba were effectively separated from cell wall extracellular protein sodium carbonate precipitation scp and ultracentrifugation forty seven protein were analyzed cytoplasmic membrane protein from the protein identified the shotgun proteomic method using sdspagelcmsms abc transporter for metabolite such metal oligopeptides phosphate sugar and amino acid and membrane protein involved phosphotransferase system were identified the predominant and abundant cytoplasmic membrane protein that would essential for nutrient uptake antibiotic resistance and virulence mechanism our result support that gelbased shotgun proteomics combined with sodium carbonate precipitation and ultracentrifugation effective method for analysis cytoplasmic membrane protein pneumoniae',\n",
       " 'current direction autism spectrum disorder asd research may require moving beyond genetic analysis alone based the complexity the disorder heterogeneity and convergence genetic alteration the cellularfunctional level mass spectrometry ha been increasingly used study cns disorder including asds proteomic research using directed understanding endogenous protein change that occur asd this review focus how ha been used study asds with particular focus proteomic analysis other neurodevelopmental disorder have been investigated using including fragile syndrome fxs and smithlemliopitz syndrome slos genetic syndrome highly associated with asd comorbidity',\n",
       " 'exploring bone proteome important and challenging task for understanding the mechanism physiologicalpathological process bone tissue however classical method protein extraction for soft tissue and cell are not applicable for bone tissue therefore method development efficient protein extraction critical for bone proteome analysis found this study that the protein extraction efficiency wa improved significantly when bone tissue wa demineralized hydrochloric acid hcl sequential protein extraction method wa developed for largescale proteome analysis bone tissue the bone tissue wa first demineralized hcl solution and then extracted using three different lysis buffer large amount acid soluble protein also presented the hcl solution besides collection protein the extracted lysis buffer the protein the demineralized hcl solution were also collected for proteome analysis automated 2dlcmsms analysis the collected protein fraction resulted the identification unique peptide which matched unique protein the identified protein revealed broad diversity the protein identity and function more than bonespecific protein and potential protein biomarkers previously reported were observed this study wa demonstrated that the developed extraction method protein bone tissue which wa also the first largescale proteomic study bone wa very efficient for comprehensive analysis bone proteome and might helpful for clarifying the mechanism bone disease',\n",
       " '',\n",
       " 'the current indepth proteomics make use long chromatography gradient get access more peptide for protein identification resulting covering many mammalian gene product day mass spectrometer running time here report fast sequencing fastseq workflow the use dual reverse phase high performance liquid chromatography mass spectrometry hplcms with short gradient achieve the same proteome coverage day adapted this workflow quantitative version fast quantification fastquan that wa compatible largescale protein quantification subjected two identical sample the fastquan workflow which allowed systematically evaluate different parameter that impact the sensitivity and accuracy the workflow using the statistic significant test unraveled the existence substantial falsely quantified differential protein and estimated correlation false quantification rate and parameter that are applied labelfree quantification optimized the setting parameter that may substantially minimize the rate falsely quantified differential protein and further applied them real biological process with improved efficiency and throughput expect that the fastseqfastquan workflow allowing pair wise comparison two proteome day may make available the mass and impact biomedical research positive way',\n",
       " 'snake venom are extremely rich source pharmacologically active protein with considerable clinical and medical potential date this potential ha not been fully explored mainly because our incomplete knowledge the venom proteome and the pharmacological property it component particular those devoid enzymatic activity this review summarizes the latest achievement the determination snake venom proteome based primarily the development new strategy and technique detailed knowledge the venom toxin composition and biological property the protein constituent should provide the scaffold for the design new more effective drug for the treatment the hemostatic system and heart disorder inflammation cancer and consequence snake bite well new tool for clinical diagnostic and assay hemostatic parameter',\n",
       " 'recent year the array mass spectrometry application address question molecular and cellular biology ha greatly expanded and continues grow modern mass spectrometer allow for identification characterization well quantification protein composition and their modification complex biological sample prior analysis any biological sample need properly prepared for the experiment here present protocol that combine preseparation protein gel electrophoresis followed analysis situ digested protein product tandem mass spectrometry msms all step the sample preparation are explained detail and the procedure compatible with downstream analysis any mass spectrometer available with minor adjustment the protocol can used with gel well the protocol provided can applied analyze specific protein particular interest well entire proteome silaclabeled protein sample are mixed prior gel separation the protein content the sample can furthermore accurately quantified',\n",
       " 'vesicle formation integral part the physiological erythrocyte aging process recent biophysical and immunochemical data have suggested that vesicle originate the extrusion membrane patch that during aging have become damaged and simultaneously enriched removal signal thereby vesiculation may serve postpone the untimely removal functional cell first step toward the identification the underlying mechanism isolated erythrocytederived vesicle from plasma fluorescenceactivated cell sorting analyzed their proteome mass spectrometry and compared this with the membrane proteome erythrocyte that were separated according cell age the presence band and actin the vesicle together with the absence almost all other integral membrane and cytoskeletal protein and the specific agingassociated alteration band aggregation and degradation shown proteomics well immunochemistry all suggest that the erythrocyte aging process harbor specific band 3centered mechanism for vesicle generation the agerelated recruitment plasma protein protein the ubiquitinproteasome system and small protein the erythrocyte membrane support the hypothesis that modification band andor degradation initiate vesiculation and the subsequent recognition and fast removal vesicle the immune system this article part special issue entitled integrated omics',\n",
       " 'ha become methodofchoice for proteome analysis generating large data set which reflect proteomescale proteinprotein interaction and ptm network however while rapid growth largescale proteomics data can observed the sound biological interpretation these result clearly lag behind therefore combined effort bioinformaticians and biologist have been made develop strategy and application help experimentalists perform this crucial task this review present overview currently available analytical strategy and tool extract biologically relevant information from large protein list moreover also present current research publication making use these tool example how the presented strategy may incorporated into proteomic workflow emphasis placed the analysis gene ontology term interaction network biological pathway and ptms addition topic including domain analysis and text mining are reviewed the context computational analysis proteomic result expect that these type analysis will significantly contribute deeper understanding the role individual protein protein network and pathway complex system',\n",
       " 'comparative proteomic approach using two dimensional gel electrophoresis and mass spectrometry ha been developed compare and elucidate the difference among the cellular proteome four closely related isogenic sin and clausii strain during both exponential and stationary phase growth image analysis the electropherograms reveals high degree concordance among the four proteome some protein result however differently expressed the protein spot exhibiting high different expression level were identified massspectrometry analysis alcohol dehydrogenase adha abc isoform aldehyde dehydrogenase dhas abc isoform and flagellinprotein clausii ksmk the different expression level the two dehydrogenases were confirmed quantitative rtpcr and dehydrogenases enzymatic activity the different pattern protein expression can considered cell proteome signature the different strain',\n",
       " 'mass spectrometrybased approach have enabled important breakthrough quantitative proteomics the last decade this development reflected the better quantitative assessment protein level well understand posttranslational modification and protein complex and network nowadays the focus quantitative proteomics shifted from the relative determination protein differential expression between two more cellular state absolute quantity determination required for morethorough characterization biological model and comprehension the proteome dynamism well for the search and validation novel protein biomarkers however the physicochemical environment the analyte specie affect strongly the ionization efficiency most mass spectrometry type which thereby require the use specially designed standardization approach provide absolute quantification most common such approach nowadays include the use stable isotopelabeled peptide standard isotopologues the target proteotypic peptide expected after tryptic digestion the target protein use stable isotopelabeled protein standard compensate for sample preparation sample loss and proteolysis step iii isobaric reagent which after fragmentation the msms analysis provide final detectable mass shift can used tag both analyte and standard sample labelfree approach which the absolute quantitative data are not obtained through the use any kind labeling but from computational normalization the raw data and adequate standard elemental mass spectrometrybased workflow able provide directly absolute quantification peptidesproteins that contain icpdetectable element critical insight from the analytical chemistry perspective the different standardization approach and their combination used far for absolute quantitative msbased molecular and elemental proteomics provided this review',\n",
       " 'with the advent proteomics technology possible simultaneously demonstrate the expression hundred protein the information offered proteomics provides contextbased understanding cellular protein network and ha been proven valuable approach neuroscience study the mouse hippocampus ha been major target analysis the search for molecular correlate neuronal information storage although human and rat hippocampal sample have been successfully subjected proteomic profiling elaborate analysis providing the fundamental experimental basis for proteinexpression study the mouse hippocampus ha been carried out yet this led construct master map generated from the individual hippocampal proteome five different mouse strain proteomic approach based upon 2de coupled malditoftof ha been chosen attempt establish comprehensive reference database protein expressed the mouse hippocampus individual protein represented spot displaying various functional state the respective gene product were identified proteomic profiling the hippocampus brain region with pivotal role for neuronal information processing and storage may provide insight into the characteristic protein serving this highly sophisticated function',\n",
       " 'recently the rapid development and application mass spectrometry msbased technology have markedly improved the comprehensive proteomic characterization global proteome and protein posttranslational modification ptms however the current conventional approach for global proteomic analysis often carried out separately from ptm analysis our study developed integrated workflow for multiplex analysis global glyco and phosphoproteomics using breast cancer patientderived xenograft pdx tumor sample our approach included the following step trypsindigested tumor sample were enriched for phosphopeptides through immobilized metal ion affinity chromatography imac followed enrichment glycopeptides through mixed anion exchange max method and then the flowthrough peptide were analyzed for global proteomics our workflow demonstrated increased identification peptide and associated protein global proteome compared those using the peptide without ptm depletion addition global proteome the workflow identified phosphopeptides and glycopeptides from the ptm enrichment also found subset glycans with unique distribution profile the imac flowthrough compared those enriched directly using the max method our integrated workflow provided effective platform for simultaneous global proteomic and ptm analysis biospecimens',\n",
       " 'from plant research biomedicine proteome analysis play critical role many area biological inquiry steady improvement mass spectrometer technology ha transformed the speed and depth proteome analysis proteome simple organism can now sequenced near completion just over hour comparable coverage mammalian proteome however still requires hour even day analysis here ask why current technology fails achieve comprehensive and rapid analysis the more complex mammalian proteome propose that further advancement technology alone are unlikely solve this problem and suggest that concomitant improvement peptide separation technology will critical',\n",
       " 'osteoarthritis characterized loss extracellular matrix which driven catabolic cytokine proteomic analysis the cartilage secretome enables the global study secreted protein these are important class molecule with role numerous pathological mechanism although cartilage study have identified profile secreted protein quantitative proteomics technique have been implemented that would enable further biological question addressed overcome this limitation used the secretome from human cartilage explants stimulated with il1β and compared protein released into the medium using labelfree lcmsmsbased strategy employed qconcat technology quantify specific protein using selected reaction monitoring total protein were identified nine were differentially expressed il1 stimulation selected protein candidate were quantified absolute amount using qconcat these finding confirmed significant reduction timp1 the secretome following il1β stimulation labelfree and qconcat analysis produced equivocal result indicating effect cytokine stimulation aggrecan cartilage oligomeric matrix protein fibromodulin matrix metalloproteinases and plasminogen release this study enabled comparative protein profiling and absolute quantification protein involved molecular pathway pertinent understanding the pathogenesis',\n",
       " 'female mosquito hostseeking and preference well several other important behavior are largely driven olfaction specie the afrotropical anopheles gambiae complex display divergent hostpreference that are associated with significant difference their vectorial capacity for human malaria olfactory sensitivity begin with signal transduction and activation peripheral sensory neuron that populate the antenna maxillary palps and other appendage have used shotgun proteomics characterize the profile soluble protein antenna and maxillary palps three different specie coluzzii arabiensis and quadriannulatus that display remarkable difference anthropophilic behavior this analysis revealed interspecific difference the abundance several protein that comprise cuticular component glutathione stransferase and odorant binding protein the latter which known directly involved odor recognition',\n",
       " 'sharing proteomic data with the biomedical community through unified proteomic resource especially the context individual protein challenging prospect have developed community portal designated human proteinpedia httpwwwhumanproteinpediaorg for sharing both unpublished and published human proteomic data through the use distributed annotation system designed specifically for this purpose this system allows laboratory contribute and maintain protein annotation which are also mapped the corresponding protein through the human protein reference database hprd httpwwwhprdorg thus possible visualize data pertaining experimentally validated posttranslational modification ptms protein isoforms proteinprotein interaction ppis tissue expression expression cell line subcellular localization and enzyme substrate the context individual protein with enthusiastic participation the proteomics community the past month have witnessed data contribution from more than lab around the world including distinct experiment million peptide million msms spectrum protein expression annotation ptms ppis and subcellular localization annotation human proteinpedia should serve integrated platform store integrate and disseminate such proteomic data and inching towards evolving into unified human proteomics resource',\n",
       " 'plant tissue protein are present low amount but very large number this peculiar situation many complex foreign component render protein extraction and purification very difficult the last several year interesting technology have been described improve the technical situation the point that some methodology allow reaching very lowabundance protein and minor allergen among enrichment method the one documented this report based combinatorial peptide ligand library cplls that emerged the last decade contributing largely improve the knowledge plant proteomics the aim this review describe how this technology allows detecting lowabundance protein from various plant tissue and report the dynamic the proteome component response environmental change and biotic attack typical documented example with the description their scientific interest are reported the described technical approach and selected application are considered one the most advanced approach for plant proteomics investigation with possibility not only enlarge the knowledge plant proteome but also discover novel allergen well plant biomarkers subsequent stressful situation',\n",
       " '',\n",
       " 'this review article describes analytical technique based the phosphatebinding tag molecule phostag which alkoxidebridged dinuclear metal complex with bispyridin2ylmethylaminopropan2olate for studying the protein phosphorylome the dinuclear zincii complex form stable complex with phosphate monoester dianion aqueous solution under condition neutral using series functional phostag derivative our group ha developed novel technique that are useful study kinomics and phosphoproteomics among the derivative series biotinylated phostag derivative have been used molecular tool application such western blotting for comprehensive detection phosphorylated protein and highly sensitive peptide microarraybased technique for the detection kinase activity biological sample the review also give outline phosphate affinity electrophoresis which immobilized phostag molecule general polyacrylamide gel are used separate protein and detect difference their phosphorylation status this technique permit quantitative analysis multiple phosphorylation status individual cellular protein and their timedependent change conventional mass spectrometrybased shotgun technique used phosphoproteomics detect the phosphorylation modification protein peptide fragment whereas the phostag electrophoresis technique permit the direct analysis the phosphorylation status fulllength protein the technique therefore provides greater understanding the detailed property particular protein involved specific physiological and pathological event this article part special issue entitled medical proteomics',\n",
       " 'the last decade there ha been dramatic progress separation technique mass spectrometry and bioinformatics and this progress ha significantly improved the technique protein analysis however the analysis lowabundance protein still challenging because the limited performance the method choice compared the complexity and the vast dynamic range biological sample since this issue big obstacle most proteomics investigation great interest ha been paid recently various technique such multidimensional analysis specific peptide selection highabundance protein depletion ligand library treatment address this challenge therefore here the author review recent nano liquid chromatography coupled with tandem mass spectrometrybased study the deep proteome mainly focusing their method and perspective',\n",
       " 'the human proteome project hpp aim deciphering the complete map the human proteome the past few year significant effort the hpp team have been dedicated the experimental detection the missing protein which lack reliable mass spectrometry evidence their existence this endeavor depth analysis shotgun experiment might represent valuable resource select biological matrix design validation experiment this work used all the proteomic experiment from the nci cell line and applied integrative approach based the result obtained from comet mascot omssa and xtandem this workflow benefit from the complementarity these search engine increase the proteome coverage five missing protein chpp guideline compliant were identified although further validation needed moreover missing protein were detected with only one unique peptide and their functional analysis supported their participation cellular pathway wa also proposed other study finally performed combined analysis the gene expression level and the proteomic identification from the common cell line between the nci and the ccle project suggest alternative for further validation missing protein observation',\n",
       " 'driven improvement speed and resolution mass spectrometer the field proteomics which involves the largescale detection and analysis protein cell tissue and organism continues expand scale and complexity there resulting growth datasets both raw file and processed peptide and protein identification msbased proteomics technology also used increasingly measure additional protein property affecting cellular function and disease mechanism including posttranslational modification proteinprotein interaction subcellular and tissue distribution consequently biologist and clinician need innovative tool conveniently analyse visualize and explore such large complex proteomics data and integrate with genomics and other related largescale datasets have created the encyclopedia proteome dynamic epd meet this need httpspeptrackercomepd the epd combine polyglot persistent database and webapplication that provides open access integrated proteomics data for protein from published study human cell and model organism designed provide userfriendly interface featuring graphical navigation with interactive visualization that facilitate powerful data exploration intuitive manner the epd offer flexible and scalable ecosystem integrate proteomics data with genomics information rna expression and other related largescale datasets',\n",
       " 'selective enrichment specific peptide effective way identify low abundance protein fractionation peptide prior mass spectrometry another widely used approach reduce sample complexity order improve proteome coveragein this study designed multistage digestion strategy generate peptide with different trypsin cleavage kinetics wa found that each the collected peptide fraction yielded many new protein identification compared the control group due the reduced complexity the overlapping peptide identified between adjacent fraction were very low indicating that each fraction had different set peptide the multistage digestion strategy separate tryptic peptide with different cleavage kinetics while rplc separate peptide with different hydrophobicity these two separation strategy were highly orthogonal and showed effective multidimensional separation improve proteome coverage',\n",
       " 'hplcchipms novel nanoflow analytical technology conducted microfabricated chip that allows for highly efficient hplc separation and superior sensitive detection complex proteomic mixture this possible through onchip preconcentration and separation with fluidic connection made automatically leaktight fashion minimum precolumn and postcolumn peak dispersion and uncompromised ease use result compound eluting band only few nanoliters the chip fabricated out bioinert polyimidecontaining channel and integrated chip structure such electrospray emitter column and frits manufactured laser ablation technology meanwhile variety hplcchips differing design and stationary phase are commercially available which provide comprehensive solution for application proteomics glycomics biomarker and pharmaceutical discovery the hplcchip can also easily integrated into multidimensional separation workflow where different orthogonal separation technique are combined solve highly complex separation problem this chapter describe detail the methodological chip usage and functionality and it application the elucidation the protein profile human nucleolus',\n",
       " 'this viewpoint article the importance renal tissue proteomics health and disease explored the analysis the urinary proteome and the potential clinical application these finding are progressing however additional benefit would gained from detailed parallel exploration the proteome the renal parenchyma both model and clinical sample with this aim will briefly summarize the existing literature compare the finding and propose future task special emphasis placed the importance studying specific cellular compartment and cell type within the kidney recent technical advance are also discussed anticipated that the combination such technology especially proteomic analysis material extracted laser capture microdissection from paraffin embedded tissue direct mass spectrometrical tissue imaging will revolutionize the field',\n",
       " '',\n",
       " 'the online reversedphase liquid chromatography rplc contributes lot for the large scale mass spectrometry based protein identification proteomics retention time important evidence can used distinguish the false positivetrue positive peptide identification because the nonlinear concentration curve organic phase the whole range run time and the interaction among peptide the sequence based prediction peptide ha low accuracy and difficult generalize practice and thus le effective the validation peptide identification serial and parallel support vector machine spsvm method wa proposed characterize the nonlinear effect organic phase concentration and the interaction among peptide the spsvm contains support vector regression svr only for model training named psvr and svm model named csvm 1svr ssvr and nsvr for the prediction after distinguishing the peptide chromatographic behavior csvm 1svr and ssvr were used predict the peptide specifically improve the accuracy then the peptide wa normalized nsvr characterize the peptide interaction the prediction accuracy wa improved significantly applying this method the processing the complex sample dataset the coefficient the determination between predictive and experimental rts reach the prediction error range wa le than the total run time for more than case and le than the total run time for more than case the performance this model reach the best known far more important the spsvm method provides framework take into account the interaction among peptide chromatographic separation and it performance can improved further introducing new data processing and experiment strategy',\n",
       " 'myriad and posttranslational modification occur protein and ctermini resulting extra layer proteome complexity and additional source protein regulation here review and cterminal modification and the contemporary positional proteomics technique used isolate protein terminal peptide from complex protein mixture and characterize their diversity and occurrence biological system furthermore these degradomics strategiesoften referred and cterminomicsrepresent dedicated highthroughput technique study proteolysis dynamic living system over the past decade terminomics study have provided indispensable information the functional state individual protein cell type tissue and biological process and delivered fundamental new data for the human proteome project including high confidence identification many socalled missing protein which had not been identified traditional proteomics analysis',\n",
       " 'the ability medicine effectively treat and cure cancer directly dependent their capability detect cancer their earliest stage the advent proteomics ha brought with the hope discovering novel biomarkers the early phase tumorigenesis that can used diagnose disease predict susceptibility and monitor progression this discipline incorporates technology that can applied complex biosystems such serum and tissue order characterize the content and change the proteome induced physiological change benign pathologic these tool include 2de 2ddige icat protein array mudpit and mass spectrometry including selditof the application these tool ha assisted uncover molecular mechanism associated with cancer the global level and may lead new diagnostic test and improvement therapeutic this review these approach are evaluated the context their contribution cancer biomarker discovery particular attention paid the promising contribution the proteinchipselditof platform revolutionary approach proteomic pattern analysis that can applied the bedside for discovering protein profile that distinguish disease and diseasefree state with high sensitivity and specificity understanding the basic concept and tool used will illustrate how best apply these technology for patient benefit for the early cancer detection and improved patient care',\n",
       " 'the analysis blood provides depth chemical information physiological state organism hemolymph blood the fluid the open circulatory system drosophila melanogaster that the medium for molecule regulating wide variety physiological activity and signaling between tissue adult drosophila typically le than 3mm length and consequence the available volume hemolymph usually le than from individual fly proteomic analysis volumelimited hemolymph great challenge for both sample handling and subsequent mass spectrometry characterization this chemically diverse biological fluid with wide dynamic range protein concentration le abundant protein particular could easily lost during sample preparation missed current mass spectrometry method this article describes simple and customized rplc column and iex column prefractionate volumelimited hemolymph without excessive dilution stepgradient elution method were developed and optimized enhance the identification novel protein from individual fruit fly hemolymph sample fraction from each step gradient wa analyzed agilent nanorplc chip column and then characterized high mass resolution and high mass accuracy orbitrap mass spectrometry result both rplc protein and iex fractionation approach protein identified more protein than unfractionated control approach with higher protein score empai value and coverage furthermore significant number novel protein were revealed both rplc and iex fractionation method which were missed unfractionated control the demonstration this method establishes mean deepen proteomic analysis this commonly used important biological model system',\n",
       " 'proteomics involves largescale comprehensive study specific proteome which have been widely used the field biomarker discovery drug development disease diagnosis and therapy comprehensive proteomics involve two more proteomics approach that are confirmatory complementary andor synergistic obstructive sleep apnea osa sleep disorder which cause respiratory cessation due upper airway collapse here describe comprehensive based labelfree quantitative proteomic analysis the osa induced rat atrium homogenate and matched control using dimensional sd page page and dimensional sd page page separation the protein enzymatic digestion and analysis nanoliquid chromatography tandemmass spectrometry lcmsms the outcome from the 1dpage and 2dpage study not only identified dysregulated protein due osa but also confirmed and complemented each other',\n",
       " 'the cellular thermal shift assay cetsa biophysical technique allowing direct study ligand binding protein cell and tissue the proteomewide implementation cetsa with mass spectrometry detection mscetsa ha now been successfully applied discover target for orphan clinical drug and hit from phenotypic screen identify offtargets and explain polypharmacology and drug toxicity highly sensitive multidimensional mscetsa implementation can now also access binding physiological ligand protein such metabolite nucleic acid and other protein mscetsa can thereby provide comprehensive information modulation protein interaction state cellular process including downstream effect drug and transition between different physiological cell state such horizontal information ligandmodulation cell largely orthogonal vertical information the level different protein and therefore open novel opportunity understand operational aspect cellular proteome',\n",
       " 'reversible methylation protein regulates the majority cellular process including signal transduction mrna splicing transcriptional control dna repair and protein translocation fundamental understanding these biological process the molecular level requires comprehensive characterization the methylated protein methylation often substoichiometric and only very limited number methylated protein and site have been confidently identified date although the intrinsically basichydrophilic methylated peptide can enriched the hydrophilic interaction liquid chromatography hilic other hydrophilic peptide can coelute during the enrichment process and suppress the detection methylated peptide addition the modified arg and lys residue can not efficiently cleaved trypsin the most commonly used enzyme shotgun proteomics overcome these caveat develop novel deglycoassisted methylation site identification domain strategy which enables straightforward fast and reproducible analysis protein methylation proteomewide manner combining multidimensional fractionation and multiprotease digestion our method enabled the identification methylated form protein including new methylation form cell combining this technique with stable isotope labeling quantitative proteomics and rna interference determined the differential regulation several putative methylated site that are related the protein arginine nmethyltransferase prmt3 collectively our integrated proteomics workflow for comprehensive mapping methylation site enables better understanding protein methylation while providing rapid and effective approach for global protein methylation analysis biomedical research',\n",
       " 'activitybased protein profiling abpp utilizes active sitedirected chemical probe monitor the functional state enzyme directly native biological system identification the specific site probe labeling enzyme remains major challenge abpp experiment this protocol describe advanced abpp platform that utilizes tandem orthogonal proteolysis top strategy coupled with mass spectrometric analysis simultaneously identify probelabeled protein together with their exact site probe modification elucidation probe modification site reveals fundamental insight into the molecular basis specific probeprotein interaction the topabpp method can applied any type proteomic sample including those derived from vitro vivo labeling experiment and compatible with variety chemical probe structure completion the entire protocol including chemical synthesis key reagent requires approximately day',\n",
       " 'the coupling highperformance mass spectrometry instrumentation with highly efficient chromatographic and electrophoretic separation ha enabled rapid qualitative and quantitative analysis thousand protein from minute sample biological material here review recent progress the development and application mass spectrometrybased technique for the qualitative and quantitative analysis global proteome sample derived from whole cell tissue organism technique such multidimensional peptide and protein separation coupled with mass spectrometry accurate mass measurement peptide from global proteome digest and mass spectrometric characterization intact protein hold great promise for characterization highly complex protein mixture advance chemical tagging and isotope labeling technique have enabled quantitative analysis proteome and highly specific isolation strategy have been developed aimed selective analysis posttranslationally modified protein',\n",
       " 'clinical proteomics emerging field that deal with the use proteomic technology for medical application with major objective identifying protein involved pathological process and potential biomarkers this field already gaining momentum consequently clinical proteomics data are being generated rapid pace although mechanism sharing such data with the biomedical community lag far behind most these data are either provided supplementary information through journal web site directly made available the author through their own web resource integration these data within single resource that display information the context individual protein likely enhance the use proteomic data biomedical research human proteinpedia one such portal that unifies human proteomic data under single banner the goal this resource ultimately capture and integrate all proteomic data obtained from individual study normal and diseased tissue anticipate that harnessing these data will help prioritize experiment related protein target and also permit metaanalysis uncover molecular signature disease finally encourage all biomedical investigator maximize dissemination their valuable proteomic data rest the community active participation existing repository such human proteinpedia',\n",
       " 'combinatorial peptide ligand library cplls have been adopted for investigating the proteome mango peel and pulp well their peptidome content the latter captured with resin the aim this study wa not only perform the deepest investigation far the mango proteome but also ass the potential presence allergen and peptide endowed with biological activity the protein peel and pulp have been captured under both native and denaturing extraction technique total unique protein specie have been identified the peel the pulp via capture with cplls different value and',\n",
       " '',\n",
       " 'proteome are characterized large proteinabundance difference celltype and timedependent expression pattern and posttranslational modification all which carry biological information that not accessible genomics transcriptomics here present massspectrometrybased draft the human proteome and public highperformance inmemory database for realtime analysis terabyte big data called proteomicsdb the information assembled from human tissue cell line and body fluid enabled estimation the size the proteincoding genome and identified organspecific protein and large number translated lincrnas long intergenic noncoding rna analysis messenger rna and proteinexpression profile human tissue revealed conserved control protein abundance and integration drugsensitivity data enabled the identification protein predicting resistance sensitivity the proteome profile also hold considerable promise for analysing the composition and stoichiometry protein complex proteomicsdb thus enables navigation proteome provides biological insight and foster the development proteomic technology',\n",
       " '',\n",
       " '',\n",
       " 'hydroxyprolinecontaining protein other than collagen are rare and difficult identify only such protein have been found the human proteome biochemical method despite the fact that the list includes example biological importance such hypoxiainducible factor and the ribosomal protein rps both which have significant biological function the posttranslational modification comparison multinotch search software globalptmdiscovery conventional proteomic database search software gave ninefold improvement correctly identifying noncollagen peptide containing hydroxyproline manual interpretation msms spectrum refined this list discover unique peptide representing unique hydroxyproline site protein which only pro rps had been reported previously uniprot eight the site were found conserved proline nine specie examined ranging from human yeast these include site and protein disulfideisomerase pdia1 and site and protein disulfideisomerase pdia4 the apparent occupancy these site ranged from suggesting structural and possibly functional role these ptms fifteen the site most likely contain 4irihydroxyproline pro serpin pro aspartylasparaginyl βhydroxylase pro neutral αglucosidase pro hypoxia upregulated protein pro protein ergic pro protein casc4 pro bromodomaincontaining protein pro bromodomaincontaining protein pro nucleolar rna helicase pro pdia1 pro pdia1 pro pdia3 pro pdia4 pro pdia4 and pro pdia4 the remaining site could either 4irihydroxyproline 3isihydroxyproline recommendation are made improve automated interpretation proteomic data improve future proteomic research whose goal mine more the remaining dark matter the proteome',\n",
       " 'modern ion trap mass spectrometer are capable collecting tandem msms scan per second theory providing acquisition speed that can sample every eluting peptide precursor presented the system practice however the precursor sampling capacity enabled these ultrafast acquisition rate often underutilized due host reason long injection time and wide analyzer mass range one often overlooked reason for this underutilization that the instrument exhaust all the peptide feature identifies suitable for msms fragmentation highly abundant feature can prevent annotation lower abundance precursor ion that occupy similar masstocharge space which ultimately inhibits the acquisition msms event here present advanced peak determination apd algorithm that us iterative approach annotate densely populated region increase the number peptide sampled during datadependent lcmsms analysis the apd algorithm enables nearly full utilization the sampling capacity quadrupoleorbitraplinear ion trap system which yield increase unique peptide identification from whole cell hela lysates approximately min lcmsms analysis the apd algorithm maintains improved peptide and protein identification across several mode proteomic data acquisition including varying gradient length different degree prefractionation peptide derived from multiple protease and phosphoproteomic analysis additionally the use apd increase the number peptide characterized per protein providing improved protein quantification all the apd algorithm increase the number detectable peptide feature which maximizes utilization the high msms capacity and significantly improves sampling depth and identification proteomic experiment',\n",
       " '',\n",
       " 'multiplexing strategy are the forefront massspectrometrybased proteomics with spsms3 method becoming increasingly commonplace known caveat isobaric multiplexing interference resulting from coisolated and cofragmented ion that not originate from the selected precursor interest the triple knockout tko standard wa designed benchmark data collection strategy minimize interference however limitation it widespread use ha been the lack automated analysis platform present tko visualization tool tvt the tvt viewer allows for automated webbased database searching the tko standard returning traditional figure merit such peptide and protein count scanspecific ion accumulation time well the tkospecific metric the ifi interferencefree index moreover the tvt viewer allows for plotting two tko standard ass protocol optimization compare instrument measure degradation instrument performance over time showcase the tvt viewer probing the selection stationary phase resin ms2 isolation window width and number synchronous precursor selection sps ion for spsms3 analysis using the tvt viewer will allow the proteomics community search and compare tko result optimize userspecific data collection workflow',\n",
       " 'ultrahigh pressure liquid chromatography uhplc system combined with stateoftheart mass spectrometer have pushed the limit deep proteome sequencing new height making possible identify thousand protein single lcm experiment within few hour the recently released easyspray technology allows one implement nanouhplc with straightforwardness this work initially characterized the easyspray containing column containing lt2 particle and found that the system allowed protein identified minute then asked the question whether fast and sensitive online scxrp uhplcmsms workflow could compete with long gradient analysis using total analysis time versus proteome coverage and sample usage benchmark parameter the lcm strategy consisted the easyspray system that had been augmented the addition scx column the conversion wa made facile since additional valve were required and the use component containing viper fitting benchmarked the system using human cell lysate digest the scxrp uhplcmsms workflow allowed the identification almost unique peptide and protein total analysis time hour the same system uhplcmsms workflow plateaued only peptide and unique protein and required approx hour analysis time furthermore the workflow could continue increase the proteome coverage with longer analysis time fact with hour analysis identified unique peptide and protein report here that with this fast online scxrp uhplcmsms workflow the proteome coverage can substantially extended without significantly compromising the analysis time and sample usage',\n",
       " 'bile body fluid produced the liver and drained biliary duct into the duodenum ha two major function first contains bile acid which are critical for the digestion fat and second excretory pathway for many endogenous and exogenous compound proteomic analysis bile particularly difficult since this fluid contains high concentration various substance that strongly interfere with protein separation and identification technique furthermore owing it deep location the body bile must collected surgical endoscopic procedure however wa speculated for other body fluid bile appears promising sample for the discovery disease biomarkers leaking from proximal tissue the liver pancreas biliary tree the interest clinical proteomics wa demonstrated two study that identified bile potential biomarkers for two deadly and difficult diagnose neoplasm pancreatic cancer and cholangiocarcinoma',\n",
       " '',\n",
       " '',\n",
       " 'the european proteomics association eupa scientific congress new horizon and application for proteomics hosted the british society for proteome research bspr glasgow scotland july crosslinkingmass spectrometry ended decade method development and entered the era application this year european proteomics association meeting the train ha started moving with successful application this tool multiple pioneering laboratory addressing biological and structural problem proteomics the other side see ever increasing data volume leading question how store the data mountain publically use and convert into testable hypothesis the european proteomics association meeting ha been complementary the american society for mass spectrometry meeting many way also thanks it more manageable size and the vision the organizer inviting some europe best emerging mind',\n",
       " 'advance mass spectrometry technology highthroughput proteomics and genome annotation have resulted significant increase our molecular understanding sperm composition using improved separation and detection method and updated genome annotation reanalysis the drosophila melanogaster sperm proteome dmsp ha resulted the identification sperm protein comparative analysis with our previous proteomic dataset revealed new protein and updated sperm proteome containing total protein termed the dmspii this expanded dataset includes additional protein with predicted sperm function and confirms previous finding concerning the genomic organization sperm locus bioinformatic and protein network analysis demonstrated high quality and reproducibility proteome coverage across three replicate mass spectrometry run the use wholecell proteomics conjunction with characterized phenotype functional annotation and pathway information ha advanced our system level understanding sperm proteome functional network',\n",
       " 'legume are the major source food and fodder with strong commercial relevance and are essential component agricultural ecosystem owing their ability carry out endosymbiotic nitrogen fixation recent year legume have become one the major choice plant research the legume proteomics currently represented more than reference map and equal number stressresponsive proteome among the legume the protein database most proteomic study have been accomplished two model legume soybean and barrel medic this review highlight recent contribution the field legume proteomics comprehend the defence and regulatory mechanism during development and adaptation climatic change here attempted provide concise overview the progress legume proteomics and discus future development three broad perspective proteome organstissues subcellular compartment and iii spatiotemporal change response stress such data mining may aid discovering potential biomarkers for plant growth general apart from essential component involved stress tolerance the prospect integrating proteome data with genome information from legume will provide exciting opportunity for plant biologist achieve longterm goal crop improvement and sustainable agriculture',\n",
       " 'global mean temperature are expected rise degree accompanied increase frequency and amplitude extreme temperature event greater climatic extreme and expanded range cultivation will expose rice increasing stress the future understanding gene expression disparate thermal regime important for the engineering cultivar with tolerance nonoptimal temperature our study investigated the proteomic response rice cell suspension culture sudden temperature change cell culture grown degree were subjected 3day exposure degree for lowtemperature stress and degree for hightemperature stress quantitative labelfree shotgun proteomic analysis wa performed biological triplicate each treatment over protein were expressed one more temperature treatment and these more than were found responsive either the temperature extreme these temperatureresponsive protein included more than protein which were uniquely expressed either degree our study also identified novel stressresponse protein and observed that switching between the classical and the alternative pathway sucrose metabolism occurs response extreme temperature',\n",
       " 'existing the heart cellular energy metabolism the mitochondrion uniquely positioned have major impact human disease process example mitochondrial impact human pathology abound and include etiology ranging from inborn error metabolism the site activity variety toxic compound this review the unique aspect the mechanism related the mitochondrial proteome are discussed along with overview the literature related mitochondrial proteomic exploration the review includes discussion potential area for exploration and advantage applying proteomic technique the study mitochondrion',\n",
       " 'this letter analyzes two largescale proteomics study published the same issue nature the time the release both study were portrayed draft map the human proteome and great advance the field with the initial publication the human genome these paper have broad appeal and will doubt lead great deal further analysis the scientific community however were intrigued the number proteincoding gene detected the two study number that far exceeded what ha been reported for the multinational human proteome project effort carried out simple quality test the data using the olfactory receptor family highquality proteomics experiment that doe not specifically analyze nasal tissue should not expect detect many peptide for olfactory receptor neither the study carried out experiment nasal tissue yet found peptide evidence for more than olfactory receptor the two study these result suggest that the two study are substantially overestimating the number protein coding gene they identify conclude that the experimental data from these two study should used with caution',\n",
       " 'major depressive disorder mdd common despite numerous available treatment many individual fail improve clinically diagnosis mdd continues commonly accomplished via behavioral rather than biological method biomarkers may provide objective diagnosis mdd and could include measurement gene protein and pattern brain activity proteomic analysis and validation biomarkers le explored than other area biomarker research mdd mass spectrometry comprehensive unbiased mean proteomic analysis which can complemented directed protein measurement such western blotting prior study have focused analysis several human biomaterials mdd including human postmortem brain cerebrospinal fluid csf blood component and urine further study utilizing and proteomic analysis mdd may help solidify and establish biomarkers for use diagnosis identification new treatment target and understanding the disorder biomarker biomarker signature that facilitates convenient and inexpensive predictive test for depression treatment response highly desirable',\n",
       " 'pulmonary fibrosis arises consequence aberrant remodeling and defective repair mechanism within the lung this destructive process the cause much the morbidity and mortality many pulmonary disorder unfortunately therapeutic option are limited significant advancement the management patient with pulmonary fibrosis would the identification biomarkers for diagnosis prognosis and prediction patient response therapy bronchoalveolar lavage ideal tissue target for the discovery these potential biomarkers pulmonary fibrosis integrative approach using both gel and mass spectrometrybased proteomic workflow will allow full coverage this complex proteome thereby unlocking this potential information clinical tool aid diagnosis and guide treatment for individual patient with pulmonary fibrosis',\n",
       " 'protein biomarker discovery and validation current omics era are vital for healthcare professional improve diagnosis detect cancer early stage identify the likelihood cancer recurrence stratify stage with differential survival outcome and monitor therapeutic response the success such biomarkers would have huge impact how improve the diagnosis and treatment patient and alleviate the financial burden healthcare system the past the genomics community mostly through largescale deep genomic sequencing technology ha been steadily improving our understanding the molecular basis disease with number biomarker panel already authorized the food and drug administration fda for clinical use mammaprint two recently cleared device using nextgeneration sequencing platform detect dna change the cystic fibrosis transmembrane conductance regulator cftr gene clinical proteomics the other hand albeit it ability delineate the functional unit cell more likely driving the phenotypic difference disease protein and proteinprotein interaction network and signaling pathway underlying the disease stagger make significant impact with only average protein biomarkers per year approved the fda over the past year this statistic itself raise the concern that major roadblock have been impeding efficient transition protein marker candidate biomarker development despite major technological advance proteomics recent year',\n",
       " '',\n",
       " '',\n",
       " 'new mapping approach have greatly expanded our view the cellular landscape proteinmetabolite interaction these method either identify protein interacting with selected metabolite vice versa reviewing recent development found that comprehensive mapping the proteinmetabolite interaction space can achieved eventually using existing method amongst which proteomics technique ass cell wide protein property change response metabolite treatment currently offer the highest potential since expect major advance mapping proteinmetabolite interaction the near future the challenge shift the identification interaction functionality for which currently only few specialized method are available',\n",
       " 'laser capture microdissection lcm powerful tool that enables the isolation specific cell type from tissue section overcoming the problem tissue heterogeneity and contamination combined the lcm with isotopecoded affinity tag icat technology and twodimensional liquid chromatography investigate the qualitative and quantitative proteome hepatocellular carcinoma hcc the effect three different histochemical stain tissue section have been compared and toluidine blue stain wa proved the most suitable stain for lcm followed proteomic analysis the solubilized protein from microdissected hcc and nonhcc hepatocytes were qualitatively and quantitatively analyzed with twodimensional liquid chromatography tandem mass spectrometry 2dlcmsms alone coupled with cleavable isotopecoded affinity tag cicat labeling technology total protein were qualitatively identified and protein were unambiguously quantified these result showed that the clinical proteomic method using lcm coupled with icat and 2dlcmsms can carry out not only largescale but also accurate qualitative and quantitative analysis',\n",
       " '',\n",
       " 'plasma membrane the primary determinant freezing tolerance plant because it central role freezethaw cycle change the plasma membrane protein have been one the major research area plant cold acclimation obtain comprehensive profile the plasma membrane proteome and their change during the cold acclimation process plasma membrane purification method using dextranpolyethylene glycol two polymer system and mass spectrometrybased shotgun proteomics method using nanolcmsms for the plasma membrane protein are described the proteomic result obtained are further applied labelfree protein semiquantification',\n",
       " '',\n",
       " 'chromatofocusing ha many potential application the field proteomics such for the isolation and removal major sample component facilitate the analysis lowabundance component and for sample prefractionation prior subsequent separation using sdspage narrowpirange 2dpage additional chromatography step however the chromatofocusing technique that are most commonly used employ propriety polyampholyte elution buffer and highly specialized column packing both which limit the use chromatofocusing practice expand the range application for this technique this chapter considers chromatofocusing method which employ common ionexchange column packing and elution buffer which are simple mixture readily available buffering specie particular interest the use chromatofocusing with multistep gradient for the fractionation protein mixture into narrowpirange fraction the crosscontamination characteristic these fraction using sdspage are also assessed',\n",
       " 'proteomics potentially powerful technology for elucidating brain function and neurodegenerative disease far the brain proteome ha generally been analyzed twodimensional gel electrophoresis which usually lead the complete absence membrane protein describe proteomic approach for profiling plasma membrane protein from mouse brain the procedure consists novel method for extraction and fractionation membrane onmembrane digestion diagonal separation peptide and highsensitivity analysis advanced breaking with the classical plasma membrane fractionation approach membrane are isolated without cell compartment isolation stepwise depletion nonmembrane molecule from entire tissue homogenate highsalt carbonate and urea wash followed treatment the membrane with sublytic concentration digitonin plasma membrane further enriched density gradient fractionation and protein digested onmembrane endoproteinase lysc released peptide are separated fraction digested trypsin and analyzed lcmsms single experiment the developed technology enabled identification protein from mouse brain cortex further development and miniaturization allowed analysis hippocampus revealing protein more that the identified protein are membrane protein including several class ion channel and neurotransmitter receptor our work now allows indepth study brain membrane proteome such mouse model neurological disease',\n",
       " 'improvement speed and mass accuracy mass spectrometer revolutionized proteomics with highthroughput proteomics enabling the profiling complete proteome and thousand posttranslational modification site the limit highthroughput proteomics are constantly pushed new frontier and mass spectrometrybased proteomics may eventually permit the analysis protein expression profile le than day increased data acquisition speed ha led dramatic increase the total number tandem mass spectrometry msms spectrum such that million msms spectrum are now acquired given set analysis many these spectrum are insufficiently validated instead statistical tool are commonly used estimate falsepositive falsediscovery rate for these data set many laboratory may not realize the cost associated with using these widely available but minimally validated data set the cost associated with use these data can include missed opportunity for biological insight the pollution database with increasing number falsepositive identification and time spent biologist investigating false lead resulting lack faith proteomics data improved strategy for data validation need implemented along with change the culture highthroughput proteomics linking proteomics closer biology',\n",
       " 'the diversity biological sample and dynamic range analytes being analyzed can prove analytical challenge and particularly prevalent proteomic study maximizing the peak capacity the workflow employed can extend the dynamic range and increase identification rate the focus this chapter present mean achieving this for various analytical technique such liquid chromatography mass spectrometry and ion mobility combination these method can used part data independent acquisition strategy thereby limiting issue such chimericy when analyzing region extreme analyte density',\n",
       " 'highthroughput technique are indispensable for aiding basic and translational research among them recent advance proteomics technique have allowed biomedical researcher characterize the proteome multiple organism this remarkable advancement have been well complemented proteome bioinformatics method and tool proteome bioinformatics refers the study and application informatics the field proteomics this chapter provides overview computational strategy method and technique reported this book for bioinformatics analysis protein data outline many bioinformatics tool database and proteomic technique described each the chapter given here',\n",
       " '',\n",
       " 'proteomics increasingly being used understand enzyme expression and regulatory mechanism involved the accumulation storage reserve crop with sequenced genome during the past six year considerable progress ha been made characterize proteome both mature and developing seed particularly oilseed plant which accumulate principally oil and protein storage reserve this review summarizes the emerging proteomics data with emphasis seed filling soy rapeseed castor and arabidopsis each these oilseed were analyzed using very similar proteomic strategy these parallel study provide comprehensive view sourcesink relationship specifically sucrose assimilation into organic acid intermediate for novo amino acid and fatty acid synthesis map these biochemical process for seed maturation and illustrate the difference and similarity among the four oilseed for example while the four oilseed appear capable producing cytosolic phosphoenolpyruvate the principal carbon intermediate soybean and castor also express malic enzyme and malate dehydrogenase together capable producing malate that ha been previously proposed the major intermediate for fatty acid synthesis castor discus these and other difference the context intermediary metabolism for developing oilseed',\n",
       " 'this study wa designed collect and perform proteomic analysis expired air condensate newborn receiving respiratory support the department resuscitation and intensive care the proteomic composition expired air condensate wa evaluated newborn various stage development and with different abnormality',\n",
       " '',\n",
       " 'given the diverse range transcriptional and posttranscriptional mechanism gene regulation the estimate the human proteome likely subject scientific surprise the field proteomics ha gained momentum worldwide this regard the establishment the human proteome draft using highresolution mass spectrometry tissue microarrays and immunohistochemistry three independent research group laboratory pandey kuster and uhlen accelerated the pace proteomics research the chromosome centric human proteome project chpp ha taken initiative towards the completion the human proteome project hpp understand the proteomics correlate common complex human disease and biological diversity not mention persontoperson and population difference response drug nutrition vaccine and other health intervention and hostenvironment interaction although highresolution msbased and antibody microarray approach have shown enormous promise are still unable map the whole human proteome due the presence numerous missing protein december the indian institute technology bombay mumbai the 6th annual meeting the proteomics society india psi and the international proteomics conference wa held part this interdisciplinary summit panel discussion session the quest the human proteome and missing protein wa organized eminent scientist the field proteomics and system biology including akhilesh pandey gilbert omenn mark baker and robert mortiz shed light different aspect the human proteome draft and missing protein importantly the possible reason for the missing protein shotgun workflow were identified and debated expert low tissue expression lack enzymatic digestion site protein lost during extraction among other contributing factor capture the missing protein the expert collective view wa study the wider tissue range with multiple digesting enzyme and follow targeted proteomics workflow particular the innovation trajectory from the proteomics laboratory novel proteomics diagnostics and therapeutic society will also need new conceptual frame for translation science and innovation strategy proteomics these will embody both technical well rigorous social science and humanity consideration understand the correlate the proteome from cell society',\n",
       " 'zymoseptoria tritici cause septoria tritici blotch disease wheat obtain comprehensive protein dataset this fungal pathogen proteome tritici growing nutrientlimiting and rich medium and vivo late stage wheat infection were fractionated gel strong cation exchange scx chromatography and analyzed lcmsms total and tritici protein were confidently identified from these condition respectively these vitro and planta protein and were predicted contain signal peptide respectively functional classification analysis revealed the protein were involved the various cellular activity comparison three distinct protein expression profile demonstrates the elevated carbohydrate lipid and secondary metabolism transport protein processing and energy production specifically the host environment contrast the enhancement signaling defense replication transcription and cell division vitro the data provide useful target towards better understanding the molecular basis tritici growth development stress response and pathogenicity',\n",
       " 'better peptide separation required for bottomup proteomics for further improving the proteome coverage the twodimensional liquid chromatography 2dlc system only explore difference among peptide their hydrophobicity reversedphase and charge strong cationanion exchange scxsax alternative separation technique with different separation mechanism are required further improve the separation capillary zone electrophoresis cze attractive alternative because ha high efficiency for separation biomolecules and separate analytes based their sizetocharge ratio complementary with however the low loading capacity and narrow separation window cze limit it wide application for largescale proteomics this manuscript present automated czemass spectrometry system for solving those issue the czems system can approach least halfamicroliter loading capacity with good robustness and reproducibility can routinely use over the available sample the sample vial for analysis and can generate min separation window and high peak capacity for complex proteome analysis the result represent the highest peak capacity and the widest separation window cze for peptide separation with microliterscale loading capacity the first time that czems approach both the microliterscale loading capacity and over 2hour separation window for analysis complex sample the automated czems system dramatically reduces the gap between czems and rplcms term loading capacity separation window and peak capacity truly open the door for largescale bottomup proteomics using czems',\n",
       " 'progress proteomics research ha led demand for powerful analytical tool with high separation efficiency and sensitivity for confident identification and quantification protein posttranslational modification and protein complex expressed cell and tissue this demand ha significantly increased interest capillary electrophoresismass spectrometry cems the past few year this review provides highlight recent advance cems for proteomics research including short introduction topdown mass spectrometry and native mass spectrometry native well detailed overview method both the potential and limitation these method for the analysis protein and peptide synthetic and biological sample and the challenge method are discussed along with perspective about the future direction cems wiley periodical inc mass spec rev',\n",
       " 'protein quantification proteomewide scale important aim enabling insight into fundamental cellular biology and serving constrain experiment and theoretical model while proteomewide quantification not yet fully routine many datasets approaching proteomewide coverage are becoming available through biophysical and technique data this type can accessed via variety source including publication supplement and online data repository however access the data still fragmentary and comparison across experiment and organism are not straightforward here describe recent update our database resource paxdb protein abundance across organism paxdb focus protein abundance information proteomewide scope irrespective the underlying measurement technique quantification data reprocessed unified and qualityscored and then integrated build metaresource paxdb also allows evolutionary comparison through precomputed gene orthology relation recently have expanded the scope the database include cellline sample and more systematically scan the literature for suitable datasets report that significant fraction published experiment can not readily accessed andor parsed for quantitative information requiring additional step and effort the current update brings paxdb datasets organism with semi quantitative abundance information covering more than protein',\n",
       " 'insufficient sensitivity method for detection protein single molecule level doe not yet allow obtaining the whole image human proteome but further need least know the proteome size how many different protein specie compose this proteome this the task that could least partially realized the method described this article the approach used our study based detection protein spot 2de after staining protein dye with various sensitivity the different protein spot contain different protein specie counting the spot open way for estimation number protein specie the function representing the dependence the number protein spot sensitivity lod protein dye wa generated and extrapolation this function curve theoretical point the maximum sensitivity detection single smallest polypeptide allowed counting the number different molecule polypeptide specie the concentration level single polypeptide per proteome using this approach wa estimated that the minimal number protein specie for model object escherichia coli and pirococcus furiosus are and respectively expect single human cell hepg2 contain minimum protein specie',\n",
       " 'comprehensive quantitative information abundance protein and their posttranslational modification ptms can potentially provide novel biological insight into disease pathogenesis and therapeutic intervention herein introduce quantitative strategy utilizing isobaric stable isotopelabeling technique combined with twodimensional liquid chromatographytandem mass spectrometry 2dlcmsms for largescale deep quantitative proteome profiling biological sample clinical specimen such tumor tissue the workflow includes isobaric labeling tryptic peptide for multiplexed and accurate quantitative analysis basic reversedphase fractionation and concatenation for reduced sample complexity and nanolc coupled high resolution and high mass accuracy analysis for high confidence identification and quantification protein this proteomic analysis strategy ha been successfully applied for indepth quantitative proteomic analysis tumor sample and can also used for integrated proteome and ptm characterization well comprehensive quantitative proteomic analysis across sample from large clinical cohort',\n",
       " '',\n",
       " 'global identification protein ctermini highly challenging due their low abundance conventional shotgun proteomics several enrichment strategy have been developed facilitate the detection cterminal peptide one major issue previous approach the limited cterminome coverage herein integrated lysarginase digestion chemical acetylation neonterminus and aionaided peptide matching into polyallylaminebased cterminomics termed laacter this strategy leveraged lysarginase protease with cleavage specificity nterminal lys and arg residue cover previously unidentifiable cterminome and employed chemical acetylation and aionaided peptide matching efficiently boost peptide identification triplicate laacter identified total ctermini from proteome cell which expanded the coverage more unique ctermini compared with the parallel experiment using the original workflow compared with the largest human cterminome data set containing ctermini laacter not only achieved comparable profiling depth but also yielded previously unidentified ctermini silac stable isotope labeling with amino acid cell culturebased quantitative study for identification gluccleaved product laacter quantified more cterminal peptide than the original workflow using laacter and the original workflow performed global analysis for the cterminal sequence cell the original and processed ctermini displayed distinct sequence pattern implying the cend rule that regulates protein stability could more complex than just amino acid motif conclusion reason laacter could powerful proteomic tool for indepth cterminomics and would benefit better functional understanding protein ctermini',\n",
       " 'mass spectrometrybased proteomics continuously growing field marked technological and methodological improvement cancer proteomics aimed pursuing goal such accurate diagnosis patient stratification and biomarker discovery relying the richness information quantitative proteome profile translating these highdimensional data into biological finding clinical importance necessitates the use robust and powerful computational tool and method this chapter provide detailed description standard analysis step for clinical proteomics dataset performed perseus software for functional analysis largescale quantitative omics data',\n",
       " 'avoid the specific problem concerning intrinsic membrane protein proteome analysis alternative strategy described that complementary previous investigation using polyacrylamide gel electrophoresis page technique the strategy involves obtaining purified preparation the membrane from chlorobium tepidum washing with nabr which removed membraneassociated soluble protein and membraneassociated organelle separation membrane protein complex using bluenative polyacrylamide gel electrophoresis bnpage after solubilization with ndodecylbetadmaltoside ddm combination the with tricinesdspage highthroughput mass spectrometric analysis after gel band excision ingel digestion and maldi target spotting and protein identification from mixture tryptic peptide peptide mass fingerprinting using this approach identified different protein which have not been previously reported using page technique membrane protein with transmembrane helix were found and this procedure proved efficient with protein within wide range about the identified membrane protein belong various functional category like energy metabolism transport signal transduction and protein translocation while for the others function not yet known indicating the potential the method for the elucidation the membrane proteome general',\n",
       " 'toxicoproteomics us the discovery potential proteomics toxicology research applying global protein measurement technology biofluids and tissue after host exposure injurious agent toxicoproteomic study thus far have focused protein profiling major organ and biofluids such liver and blood preclinical specie exposed model toxicant the slow pace discovery for new biomarkers toxicity signature and mechanistic insight partially due the limited proteome coverage derived from analysis native organ tissue and body fluid traditional proteomic platform improved toxicoproteomic analysis would result combining higher data density lcmsms platform with stable isotope labelled peptide and parallel use complementary platform study design that remove abundant protein from biofluids enrich subcellular structure and include cell specific isolation from heterogeneous tissue would greatly increase differential expression capability leveraging resource from immunology cell biology and nutrition research community toxicoproteomics could make particular contribution three interrelated area advance mechanistic insight and biomarker development the plasma proteome and circulating microparticles the adductome and idiosyncratic toxicity',\n",
       " 'recent analysis prokaryotic nεlysineacetylated protein highlight the posttranslational regulation broad spectrum cellular protein however the exact role acetylation remains unclear due lack acetylated proteome data prokaryote here present the nεlysineacetylated proteome grampositive thermophilic geobacillus kaustophilus affinity enrichment using acetyllysinespecific antibody followed lcmsms analysis revealed acetylated peptide representing protein these acetylated protein include not only common orthologs from mesophilic bacillus counterpart but also unique kaustophilus protein indicating that lysine acetylation pronounced thermophilic bacteria these data complement current knowledge the bacterial acetylproteome and provide expanded platform for better understanding the function acetylation cellular metabolism',\n",
       " 'normalized spectral index quantification wa recently presented accurate method labelfree quantitation which improved spectral counting incorporating the intensity peptide msms fragment ion into the calculation protein abundance present sinq tool implementing this method within the framework existing analysis software our freely available central proteomics facility pipeline cpfp demonstrate using data set protein standard acquired variety mass spectrometer that sinq can rapidly provide useful estimate the absolute quantity protein present mediumcomplexity sample addition relative quantitation standard protein spiked into complex lysate background and run without prefractionation produce accurate result amount above 1fmol column compare quantitation performance various precursor intensity and identificationbased method including the normalized spectral abundance factor nsaf exponentially modified protein abundance index empai maxquant and progenesis lcm anticipate that the sinq tool will useful asset for core facility and individual laboratory that wish produce quantitative data but lack the necessary manpower routinely support more complicated software workflow sinq freely available obtain and use part the central proteomics facility pipeline which released under opensource license',\n",
       " 'one the most important early development the field proteomics wa the advent automated data acquisition routine that allowed highthroughput unattended data acquisition during hplc introduction peptide mixture tandem mass spectrometer prior this data acquisition wa order magnitude le efficient being based entirely list predetermined ion generated prior hplcms experiment this process known generically datadependent analysis empowered the development shotgun proteomics where hundred thousand peptide sequence are matched per experiment their most popular implementation the most abundant ionized specie from every precursor ion scan each moment chromatographic time are successively selected for isolation activation and tandem mass analysis while extremely powerful this strategy ha one primary limitation that detectable dynamic range restricted topdown manner the peptide that ionize the best circumvent the serial nature the datadependent process and increase detectable dynamic range the concept multiplexed and dataindependent acquisition dia have emerged multiplexeddata acquisition based more efficient coselection and codissociation multiple precursor ion parallel the data from which subsequently deconvoluted provide polypeptide sequence for each individual precursor ion dia ha similar goal but there realtime ion selection based prior precursor ion scan instead predefined range are interrogated either fragmenting all ion entering the mass spectrometer every single point chromatographic time dividing the range into smaller range for isolation and fragmentation these approach aim fully utilize the capability mass spectrometer maximize tandem acquisition time and address the need expand the detectable dynamic range lower the limit detection and improve the overall confidence peptide identification and relative protein quantification measurement this review cover all aspect multiplexed and dataindependent tandem mass spectrometry proteomics from experimental implementation advance software for data interpretation',\n",
       " 'mitochondrial proteomics ha significantly developed since the first plant mitochondrial proteome were published many study have added our knowledge the protein component that make plant mitochondrion wide range specie here present two common and one emerging quantitative proteomic technique that can used study the abundance mitochondrial protein for this publication have described the method approach determine the amount contamination mitochondrial isolation contrast historical approach that involved the use use antibody specific marker protein the measurement activity marker enzyme however these approach could easily adapted carry out control versus treatment study',\n",
       " 'plasma and serum are rich source information regarding individual health state and protein test inform medical decision making despite major investment few new biomarkers have reached the clinic mass spectrometry msbased proteomics now allows highly specific and quantitative readout the plasma proteome here employ plasma proteome profiling define quality marker panel ass plasma sample and the likelihood that suggested biomarkers are instead artifact related sample handling and processing acquire deep reference proteome erythrocyte platelet plasma and whole blood individual protein and compare serum and plasma proteome based spikein experiment determine sample qualityassociated protein many which have been reported biomarker candidate revealed comprehensive literature survey provide sample preparation guideline and online resource wwwplasmaproteomeprofilingorg ass overall samplerelated bias clinical study and prevent costly missassignment biomarker candidate',\n",
       " 'recent year mass spectrometry ha moved more than ever before into the front line proteincentered research after being established the qualitative level the more challenging question quantification protein and peptide using mass spectrometry ha become focus for further development this chapter discus and review the strategy and problem the method currently use for the quantitative analysis peptide protein and finally proteome mass spectrometry the common theme the difference and the potential pitfall the main approach are presented order provide survey the emerging field quantitative mass spectrometrybased proteomics',\n",
       " 'evaluate the implementation various denaturant and their efficacy bottomup membrane proteomic method using lcm analysis microsome isolated from tomato root were treated with mscompatible surfactant rapigest surfactant from water and pps silent surfactant from protein discovery chaotropic reagent guanidine hydrochloride and organic solvent methanol peptide were analyzed triplicate sample and technical replicates dataindependent lcmse analysis overall unique peptide matching unique protein were detected with the order denaturant method efficacy being rapigest surfactant pps silent surfactant guanidine hydrochloride and methanol using bioinformatic analysis protein were determined integral membrane protein when normalizing the data percentage the overall number peptide and protein identified for each method the order for integral membrane protein identification efficacy wa methanol guanidine hydrochloride rapigest surfactant and pps silent surfactant interestingly only the protein were identified all four method with the silent surfactant having the greatest overlap gravy analysis the protein and peptide level indicated that methanol and guanidine hydrochloride promoted detection hydrophobic protein and peptide respectively however trypsin activity the presence each denaturant wa determined major factor contributing peptide identification lcmse these result reveal the complementary nature each denaturant method which can used integrated approach provide more effective bottomup analysis membrane proteome than can achieved using only single denaturant',\n",
       " 'proceed proteomic analysis erythrocyte the redeared turtle trachemys scripta method for obtaining turtle erythrocyte ghost teg wa first developed after hypotonic lysis using special buffer forcing the erythrocyte through the syringe with nshaped needle applying low speed homogenizing and differential centrifugation highly purified teg fraction were obtained the isolated teg protein were treated with ingel digestion separated sdspage insolution digestion followed hplc predissociation and then identified nanoesilc msms technique total teg protein wa identified validated and categorized among these protein tubulins and cellsurfacelocated ftype atp synthase revealed important information into the super tolerance trachemys scripta anoxia and low temperature exposure altogether this study not only provided method isolate high quality teg and dataset comprehensive characterization teg protein but also provides tool for proteomic research all nucleated red blood cell and thus opened new research field for exploring the mechanism super tolerance turtle harsh environment',\n",
       " '',\n",
       " 'stable isotope labeling widely used encode and quantify protein massspectrometrybased proteomics compared metabolic labeling with stable isotope labeling amino acid cell culture silac and chemical labeling stable isotope dimethyl labeling and find that they have comparable accuracy and quantitative dynamic range unfractionated proteome analysis and affinity pulldown experiment analyzing silac and dimethyllabeled sample together single liquid chromatographymass spectrometric analysis minimizes difference under analytical condition allowing comparison quantitative error introduced during sample processing find that silac more reproducible than dimethyl labeling because protein from metabolically labeled population can combined before proteolytic digestion silac particularly suited study with extensive sample processing such fractionation and enrichment peptide with posttranslational modification compared both method pulldown experiment using kinase inhibitor dasatinib and tagged grb2sh2 protein affinity bait describe stagetip dimethyllabeling protocol that applied insolution and ingel protein digest comparing the impact postdigest isotopic labeling quantitative accuracy demonstrate how specific experimental design can benefit most from metabolic labeling approach like silac and situation where chemical labeling stable isotopedimethyl labeling can practical alternative',\n",
       " 'protein kinase are the second largest human protein family but term research interest both basic and applied they are surely the most popular over the past decade many technique and approach for studying the kinome have been described and the pace development ever increasing presently molecular biologist can approach the kinome from many different angle what kinase are active during specific cell state interest become activated response specific stimulus what are the effect controlling the activation status individual kinase what substrate are targeted particular kinase either general under particular condition and what kinase responsible for targeting specific phosphorylation site interest these are some the more commonly asked question during any kinasecentric research project and different strategy have been devised for answering such query this review outline the most promising these approach particularly those with capacity for highthroughput study this article part special issue entitled from protein structure clinical application',\n",
       " '',\n",
       " '',\n",
       " '',\n",
       " 'here combine standard twodimensional difference gel electrophoresis dige protocol with subsequent poststaining gel with phosphospecific fluorescent proq diamond dye the combination these two method for fluorescence detection protein allows quantitative detection phosphoprotein 2degels established this protocol within functional proteomics experiment mammary epithelial cell eph4 were stimulated culture epidermal growth factor egf endosomal fraction prepared after subcellular fractionation and phosphorylated protein successfully detected endosomes for instance endo cytokeratin known phosphoprotein and differentiation marker inducible mapk signaling wa identified matrixassisted laser desorptionionizationtime flightmass spectrometry malditofms with this protocol all step combined proteome and phosphoproteome profiling experiment are significantly simplified and accelerated taking full advantage both method term specificity sensitivity and accuracy quantification',\n",
       " 'one the major difficulty mining low abundance biomarkers from serum plasma due the fact that small number protein such albumin alpha2macroglobulin transferrin and immunoglobulin may represent much the total serum protein the large quantity these protein make difficult identify low abundance protein serum using traditional 2dimensional electrophoresis recently used combination multidimensional liquid chromatography and gel electrophoresis coupled matrixassisted laser desorptionionizationquadrupoletime flight and ion trap liquid chromatographytandem mass spectrometry identify protein marker serum alzheimers disease insulin resistancetype2 diabetes ird2 and congestive heart failure chf patient identified protein that exhibit higher level control serum and protein that exhibit higher level disease serum for example haptoglobin and hemoglobin are elevated serum ird2 and chf patient the level several other protein including fibrinogen and it fragment alpha 2macroglobulin transthyretin proplatelet basic protein protease inhibitor clade and well protein involved the classical complement pathway such complement and inhibitor were found differ between ird2 and control serum the serum level protein such the kda subunit vitronectin alpha 1acid glycoprotein apolipoprotein fragment factor and histidinerich glycoprotein were observed different between and control the difference observed these biomarker candidate were confirmed western blot and the enzymelinked immunosorbent assay the biological meaning the proteomic change the disease state and the potential use these change diagnostic tool for therapeutic intervention will discussed',\n",
       " 'the article summarizes the achievement the pilot phase the russian part the international project human proteome and identifies the direction for further work the study the human chromosome proteome the framework the project main phase the pilot phase the project wa focused the detection least one protein for each chromosome proteincoding gene three type the biological material the application mass spectrometric detection protein the method multiple reaction monitoring mrm and genecentric approach made possible detect master form protein for which the quantitative assessment the protein content wa obtained least one type the biological material the task the main phase the project measure the proteome size healthy individual taking into account the modified protein form providing for both the bioinformatics prediction the quantity protein type and the selective experimental measurement single proteoforms since the range protein concentration corresponding the normal physiological state have not been identified the work the main phase the project focused the study clinically healthy individual the absence these data complicates significantly the interpretation the patient blood proteomic profile and prevents creating diagnostic test the long term prospect implementation the project envisages development diagnostic test system based multiple reaction monitoring mrm for quantitative measurement the protein form associated with some disease development such test system will allow predicting the extent risk different disease diagnosing disease it early stage and monitoring the effectiveness the treatment',\n",
       " 'aspergillus are important genus filamentous fungi that contribute multibillion dollar industry since many fungal genome sequencing were recently completed would advantageous profile their proteome better understand the fungal cell factory here review proteomic data generated for the aspergillus recent year thus far combined total cell surface secreted and intracellular protein have been identified based different study aspergillus proteomics summary proteome map highlighting identified protein major metabolic pathway presented',\n",
       " 'small molecule metabolite play important role regulating protein function which are acted through either covalent nonenzymatic posttranslational modification noncovalent binding interaction chemical proteomic strategy can help delineate global landscape cellular proteinmetabolite interaction and provide molecular insight about their mechanism action this review summarized the recent progress development and application chemoproteomic strategy profile proteinmetabolite interaction',\n",
       " 'mass spectrometry coupled liquid chromatography lcm and lcmsms commonly used analyze the protein content biological sample large scale study enabling quantitation and identification protein and peptide using wide range experimental protocol algorithm and statistical model analyze the data currently difficult compare the plethora algorithm for these task far curated benchmark data exists for peptide identification algorithm but data that represents ground truth for the evaluation lcm data limited hence there have been attempt simulate such data controlled fashion evaluate and compare algorithm present mssimulator simulation software for lcm and lcmsms experiment starting from list protein from fasta file the simulation will perform insilico digestion retention time prediction ionization filtering and raw signal simulation including msms while providing many option change the property the resulting data like elution profile shape resolution and sampling rate several protocol for silac itraq mse are available addition the usual labelfree approach making mssimulator the most comprehensive simulator for lcm and lcmsms data',\n",
       " 'automated method for the reverseengineering complex regulatory network are paving the way for the inference mechanistic comprehensive model directly from experimental data these novel method can infer not only the relation and parameter the known molecule defined their input datasets but also unknown component and pathway identified necessary the automated algorithm identifying the molecular nature these unknown component crucial step for making testable prediction and experimentally validating the model yet specific and efficient tool exist aid this process this end present here mocha molecular characterization tool optimized for the search unknown protein and their pathway from given set known interacting protein mocha us the comprehensive dataset proteinprotein interaction provided the string database which currently includes more than billion interaction from over organism mocha highly optimized performing typical search within second demonstrate the use mocha with the characterization unknown component from reverseengineered model from the literature mocha useful for working network model hand downstream step model inference engine workflow and represents valuable and efficient tool for the characterization unknown pathway using known data from thousand organism mocha and it source code are freely available online under the gplv3 license',\n",
       " 'effective proteome analysis are based interplay between resolution and detection had been claimed that resolution wa the main factor limiting the use twodimensional gel electrophoresis improved protein detection now indicates that this unlikely the case using highly refined protocol the rat brain proteome wa extracted resolved and detected order overcome the stain saturation threshold high abundance protein specie were excised from the gel following standard imaging gel were then imaged again using longer exposure time enabling detection lower abundance le intensely stained protein specie this resulted significant enhancement the detection resolved protein and slightly modified digestion protocol enabled effective identification standard mass spectrometric method the data indicate that the resolution required for comprehensive proteome analysis already available can ass multiple sample parallel and preserve critical information concerning posttranslational modification further optimization staining and detection method promise additional improvement this economical widely accessible and effective topdown approach proteome analysis',\n",
       " '',\n",
       " 'present protease specificity profiling based quantitative proteomics combination with proteomederived peptide library peptide library are generated endoproteolytic digestion proteome without chemical modification primary amine before exposure protease under investigation after incubation with test protease treated and control library are differentially isotopelabeled using costeffective reductive dimethylation upon analysis liquid chromatographytandem mass spectrometry cleavage product the test protease appear semispecific peptide that are enriched for the corresponding isotope label validate our workflow with two protease with wellcharacterized specificity profile trypsin and caspase3 provide the first specificity profile protease encoded human endogenous retrovirus and for chlamydial proteaselike activity factor cpaf for cpaf also highlight the structural basis negative subsite cooperativity between subsites and for disintegrin and metalloproteinase with thrombospondin motif adamts and show canonical preference profile including glutamate and glycine total report nearly cleavage site for seven protease our protocol fast avoids enrichment synthesis step and enables probing for lysine selectivity well subsite cooperativity due it simplicity anticipate usability most proteomic laboratory',\n",
       " 'protein phosphorylation constitutes major type posttranslational modification mobilizing high number gene involved many crucial cell process and largely participates the feature the proteome for several biological and technical reason the characterization protein phosphorylation requires combination distinct procedure this review special emphasis given analytical strategy connected with the determination the presence phosphorylation site and their localization mass spectrometry the feasibility selected combination analytical approach for diverse objective phosphoproteomic research discussed',\n",
       " 'the anabaena pcc ferric uptake regulator fura control iron homeostasis and appears implicated broad regulatory network whereas failure eliminate wildtype copy fura gene from the polyploid genome suggest essential function the present study comparatively analyzed the proteome furaoverexpressing strain and it parental wildtype conjunction with subsequent semiquantitative rtpcr and electrophoretic mobility shift assay order identify direct transcriptional target and unravel new biological role fura the result such approach drove find novel direct target belonging different functional category including photosynthesis energy metabolism oxidative stress defence redox regulation signal transduction mechanism dna replication thiamine biosynthesis and heterocyst differentiation two peroxiredoxins and the thioredoxin reductase exhibited the most significant change both mrna level and protein abundance under fura overexpression background indicating connection between iron metabolism redox signalling and oxidative stress defence fura box consensus sequence wa illdefined the result suggest that particular dna structure rather than defined sequence govern fura regulation it target gene overall the result provide new insight into the fura regulon anabaena pcc',\n",
       " 'proteomics highly sophisticated way study the protein profile various biological tissue fluid ha hitherto had relatively limited role ophthalmic science the few proteomic study that have been performed liquid chromatography electrophoresis gel separation and mass spectrometry have been utilized investigate the proteome several different eye structure and fluid from both human and animal model ophthalmic proteomic study have far attempted identify protein unique the eye investigate protein change due the onset various disease and identify protein that could act marker disease proteomics ha the potential improve the way which eye disease diagnosed and potentially even treated identifying novel pathogenic pathway that may susceptible therapeutic manipulation the aim this review give overview the current and potential application proteomic science ophthalmic research',\n",
       " 'the continued evolution modern mass spectrometry instrumentation and associated method represents critical component effort decipher the molecular mechanism which underlie normal physiology and understand how dysregulation biological pathway contributes human disease the increasing scale these experiment combined with the technological diversity mass spectrometer present several challenge for communitywide data access analysis and distribution here detail redesigned version multiplierz our python software library which leverage our common application programming interface mzapi for analysis and distribution proteomic data new feature include support for wider range native mass spectrometry file type interface additional database search engine compatibility with new reporting format and highlevel tool perform postsearch proteomic analysis gui desktop environment mzdesktop provides access multiplierz functionality through user friendly interface multiplierz available for download from httpsgithubcomblaisproteomicsmultiplierz and mzdesktop available for download from httpssourceforgenetprojectsmultiplierz',\n",
       " 'human biofluids especially blood plasma serum hold great potential the source candidate biomarkers for various disease however the enormous dynamic range protein concentration biofluids represents significant analytical challenge for detecting promising lowabundance protein over the last decade various immunoaffinity chromatographic method have been developed and routinely applied for separating lowabundance protein from the high and moderateabundance protein thus enabling much more effective detection lowabundance protein herein review the advance immunoaffinity separation method and their contribution the proteomic application human biofluids the limitation and future perspective immunoaffinity separation method are also discussed',\n",
       " 'the past decade mass spectrometry study skeletal muscle have become common this tissue the abundance several contractile protein significantly limit the depth the panoramic proteome analysis the use isobaric label allows improving assessment the change the protein content while analyzing sample single run here present the result comparative study various method for the fractionation skeletal muscle peptide labeled with isobaric label itraq sample from vastus lateralis eight young male were collected with needle biopsy after digestion into peptide and labeling the preparation were carried out according three different protocol peptide purification hplcmsms peptide purification isoelectric focusing hplcmsms high reversephase fractionation hplcmsms fractionation labeled peptide high reversephase wa the optimal strategy for increasing the depth the proteome analysis this approach addition contractile and mitochondrial protein allowed detect variety regulatory molecule including the nucleic acid binding the protein chaperone receptor and transcription factor',\n",
       " 'elucidating the profile extracellular integral membrane protein live cell vital for uncovering diagnostic disease biomarkers therapeutic agent and drug receptor candidate exploring the realm these protein ha proved intricate task mainly due their hydrophobic nature and low abundance furthermore the level purity achieved classical method purification and cell fractionation insufficient these restriction pose major limitation for gel electrophoresis chromatographybased separation technique the preferred methodology for highthroughput analysis mass spectrometry ha alleviated most the difficulty the identification protein general however the achilles heel still the isolation and separation membrane protein order circumvent these limitation highthroughput platform ha been devised whereby protease are applied whole intact living cell the resulting peptide fragment are then analysed liquid chromatology followed tandem lcmsms technology provide detailed profile protein exposed the surface the plasma membrane this kind protein trimming offer the advantage that prior manipulation fractionation the cell required contaminating protein are remarkably reduced and the procedure adequate for highthroughput purpose this method referred proceed proteome cell exposed extracellular domain compatible with isotope labelling technique which facilitate comparative protein expression study the methodology extendable all cell type including yeast and bacteria finally the advantage and the limitation proceed are discussed view other current technology',\n",
       " 'comprehensive and accurate evaluation data quality and falsepositive biomarker discovery critical direct the method developmentoptimization for quantitative proteomics which nonetheless remains challenging largely due the high complexity and unique feature proteomic data here describe experimental null method address this need because the method experimentally measure the null distribution either technical biological replicates using the same proteomic sample the same procedure and the same batch the casevscontol experiment correctly reflects the collective effect technical variability variationbias sample preparation lcm analysis and data processing and projectspecific feature characteristic the proteome and biological variation the performance quantitative analysis show proof concept employed the method ass the quantitative accuracy and precision and the ability quantify subtle ratio change between group using different experimental and dataprocessing approach and various cellular and tissue proteome wa found that choice quantitative feature sample size experimental design dataprocessing strategy and quality chromatographic separation can profoundly affect quantitative precision and accuracy labelfree quantification the method wa also demonstrated practical tool determine the optimal experimental parameter and rational ratio cutoff for reliable protein quantification specific proteomic experiment for example identify the necessary number technicalbiological replicates per group that affords sufficient power for discovery furthermore assessed the ability method estimate level falsepositives the discovery altered protein using two concocted sample set mimicking proteomic profiling using technical and biological replicates respectively where the truepositivesnegatives are known and span wide concentration range wa observed that the method correctly reflects the null distribution proteomic system and accurately measure false altered protein discovery rate fadr summary the method provides straightforward practical and accurate alternative statisticsbased approach for the development and evaluation proteomic experiment and can universally adapted various type quantitative technique',\n",
       " 'nanoflow liquid chromatography tandem mass spectrometry nanoflow lcmsms the mainstay proteome research because it excellent sensitivity but often come the expense robustness here show that microflow lcmsms using 1×mm column show excellent reproducibility chromatographic retention time coefficient variation and protein quantification using data from sample human cell line tissue and body fluid deep proteome analysis identifies protein and peptide and sample multiplexing using tandem mass tag increase throughput proteome the system identifies phosphopeptides and proteinprotein proteindrug interaction experiment can analyzed min per sample show that the same column can used analyze sample without apparent loss performance this study demonstrates that microflow lcmsms suitable for broad range proteomic application',\n",
       " 'proteomics ha emerged the next great scientific challenge the postgenome era but even the most basic form proteomics proteome profiling identifying all the protein expressed given sample ha proven demanding task the proteome present unique analytical challenge including significant molecular diversity extremely wide concentration range and tendency adsorb solid surface microfluidics ha been touted being useful tool for developing new method solve complex analytical challenge and such seems natural fit for application proteome profiling this review summarize the recent progress the field microfluidics four key area related this application chemical processing sample preconcentration and cleanup chemical separation and interface with mass spectrometry identify the bright spot and challenge for the marriage microfluidics and proteomics and speculate the outlook for progress',\n",
       " 'over the past several year fungal infection have shown increasing incidence the susceptible population and caused high mortality rate parallel multiresistant fungi are emerging human infection therefore the identification new potential antifungal target priority the first task this study wa analyse the protein domain and domain architecture content the fungal proteome corresponding specie available uniprotkb uniprot knowledgebase january the resulting list core and exclusive domain and domain architecture provided this paper delineates the different level fungal taxonomic classification phylum subphylum order genus and specie the analysis highlighted aspergillus the most diverse genus term exclusive domain content addition also investigated which domain could considered promiscuous the different organism application this analysis explored three different way detect potential target for antifungal drug first compared the domain and domain architecture content the human and fungal proteome and identified those domain and domain architecture only present fungi secondly looked for information regarding fungal pathway public repository where protein containing promiscuous domain could involved three pathway were identified result lovastatin biosynthesis xylan degradation and biosynthesis siroheme finally classified subset the studied fungi five group depending their occurrence clinical sample then looked for exclusive domain the group that were more relevant clinically and determined which them had the potential bind small molecule overall this study provides comprehensive analysis the available fungal proteome and show three approach that can used first step the detection new antifungal target',\n",
       " 'enhanced filter aided sample preparation efasp incorporates plastic passivation and digestionenhancing surfactant into the traditional fasp workflow reduce sample loss and increase hydrophobic protein representation qualitative and quantitative proteomics experiment resulting protein digest are free contaminant and can analyzed directly lcm',\n",
       " 'recent advance highthroughput proteome analysis ha enabled genomewide and proteomewide analysis association between single nucleotide polymorphism and protein expression level protein quantitative trait locus pqtl study using cerebrospinal fluid csf and dna sample may provide valuable insight into the genetic basis and molecular mechanism regulating protein expression the central nervous system this chapter describe stepbystep procedure csf collection and pqtl analysis using plink and software',\n",
       " 'pathogen are known release their environment large range toxin and other virulence factor their pathogenicity relies this arsenal exoproteins and their orchestrated release upon changing environmental condition exoproteomics aim describing and quantifying the protein found outside the cell thus take advantage the most recent methodology nextgeneration proteomics this approach ha been applied with great success variety pathogen increasing the fundamental knowledge pathogenicity this chapter describe how the exoproteome should prepared and handled for highthroughput identification exoproteins and their quantitation labelfree shotgun proteomics also mentioned some bioinformatics tool for extracting information such toxin similarity search',\n",
       " 'analysis blood protein hold critical promise for depth understanding physiological state protein content hemolymph from drosophila melanogaster particular analytical interest because the insect open circulatory system involves chemical signaling through the hemolymph the challenge working with this sample however the nanoliter volume solution available for analysis this study developed novel hyphenated agilent nanohplc chip columnms method obtain proteomic information from individual fruit fly hemolymph using lowvolume sample collection technique established previously the total amount individual drosophila hemolymph protein determined around based upon bradford assay with bsa hemolymph sample around were collected from single fly and digested using customized microscale digestion protocol mass spectral analysis show total protein were identified from the hemolymph individual fly these finding novel protein have been identified for the first time with evidence the translation level detection protein wellknown the literature speaks the method validity and demonstrates the ability reproducibly analyze volumelimited sample from individual fruit fly for protein content this nanoscale analysis method will facilitate future study drosophila and lead more complete understanding the physiology the fly model system',\n",
       " 'absolute protein concentration determination becoming increasingly important number field including diagnostics biomarker discovery and system biology modeling the recently introduced quantification concatamer methodology provides novel approach performing such determination and ha been applied both microbial and mammalian system while number software tool exist for performing analysis quantitative data generated related methodology such silac there currently analysis package dedicated the quantification concatamer approach furthermore most tool that are currently available the field quantitative proteomics not manage storage and dissemination such data set',\n",
       " 'here the author reason that the complexity medical problem and proteome science might tackled effectively with deep learning technology however deployment for proteomics data requires the acquisition data set from large number sample based the success medical imaging classification proteome data from thousand sample are arguably the minimal input for contemporary proteomics turning highthroughput thanks the rapid progress sample preparation and liquid chromatography mass spectrometry method particular dataindependent acquisition now enables the generation hundred thousand quantitative proteome map from clinical specimen clinical cohort with only limited sample amount clinical cohort upheaval the design largescale clinical proteomics study might required generate proteomic big data and deploy tackle complex medical problem',\n",
       " ...]"
      ]
     },
     "execution_count": 129,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "preprocessed_sentences = [preprocess_sentence(sentence, translator) for sentence in sentences]\n",
    "preprocessed_sentences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 130,
   "id": "9e0acbe8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(10000, 45423)"
      ]
     },
     "execution_count": 130,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from sklearn.feature_extraction.text import CountVectorizer\n",
    "\n",
    "count_vectorizer = CountVectorizer(ngram_range=(1,1),\n",
    "                           stop_words='english')\n",
    "\n",
    "count_data = count_vectorizer.fit_transform(preprocessed_sentences)\n",
    " \n",
    "cv_dataframe=pd.DataFrame(count_data.toarray(), columns=count_vectorizer.get_feature_names())\n",
    "cv_dataframe.shape"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "20f6f71d",
   "metadata": {},
   "source": [
    "### Probabilistic Threshing - Dictionary Size\n",
    "    1. Chemical Compound Lists"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "id": "d973ebc0",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>14βdglucosidic</th>\n",
       "      <th>1a1</th>\n",
       "      <th>1a1b</th>\n",
       "      <th>1a1blight</th>\n",
       "      <th>1acid</th>\n",
       "      <th>1alpha</th>\n",
       "      <th>1alpha1</th>\n",
       "      <th>1alpha2beta4beta7beta7hydroxidi2thienylacetyloxydimethyl3oxa9azoniatricyclononane</th>\n",
       "      <th>1amino</th>\n",
       "      <th>1antichymotrypsin</th>\n",
       "      <th>...</th>\n",
       "      <th>μlml</th>\n",
       "      <th>μm</th>\n",
       "      <th>μmdiameter</th>\n",
       "      <th>μmol</th>\n",
       "      <th>μmolg</th>\n",
       "      <th>μmsup2sup</th>\n",
       "      <th>μmthick</th>\n",
       "      <th>οphenanthroline</th>\n",
       "      <th>ρlt</th>\n",
       "      <th>σsupssup</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9995</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9996</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9997</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9998</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9999</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>10000 rows × 45423 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      14βdglucosidic  1a1  1a1b  1a1blight  1acid  1alpha  1alpha1  \\\n",
       "0                  0    0     0          0      0       0        0   \n",
       "1                  0    0     0          0      0       0        0   \n",
       "2                  0    0     0          0      0       0        0   \n",
       "3                  0    0     0          0      0       0        0   \n",
       "4                  0    0     0          0      0       0        0   \n",
       "...              ...  ...   ...        ...    ...     ...      ...   \n",
       "9995               0    0     0          0      0       0        0   \n",
       "9996               0    0     0          0      0       0        0   \n",
       "9997               0    0     0          0      0       0        0   \n",
       "9998               0    0     0          0      0       0        0   \n",
       "9999               0    0     0          0      0       0        0   \n",
       "\n",
       "      1alpha2beta4beta7beta7hydroxidi2thienylacetyloxydimethyl3oxa9azoniatricyclononane  \\\n",
       "0                                                     0                                   \n",
       "1                                                     0                                   \n",
       "2                                                     0                                   \n",
       "3                                                     0                                   \n",
       "4                                                     0                                   \n",
       "...                                                 ...                                   \n",
       "9995                                                  0                                   \n",
       "9996                                                  0                                   \n",
       "9997                                                  0                                   \n",
       "9998                                                  0                                   \n",
       "9999                                                  0                                   \n",
       "\n",
       "      1amino  1antichymotrypsin  ...  μlml  μm  μmdiameter  μmol  μmolg  \\\n",
       "0          0                  0  ...     0   0           0     0      0   \n",
       "1          0                  0  ...     0   0           0     0      0   \n",
       "2          0                  0  ...     0   0           0     0      0   \n",
       "3          0                  0  ...     0   0           0     0      0   \n",
       "4          0                  0  ...     0   0           0     0      0   \n",
       "...      ...                ...  ...   ...  ..         ...   ...    ...   \n",
       "9995       0                  0  ...     0   0           0     0      0   \n",
       "9996       0                  0  ...     0   0           0     0      0   \n",
       "9997       0                  0  ...     0   0           0     0      0   \n",
       "9998       0                  0  ...     0   0           0     0      0   \n",
       "9999       0                  0  ...     0   0           0     0      0   \n",
       "\n",
       "      μmsup2sup  μmthick  οphenanthroline  ρlt  σsupssup  \n",
       "0             0        0                0    0         0  \n",
       "1             0        0                0    0         0  \n",
       "2             0        0                0    0         0  \n",
       "3             0        0                0    0         0  \n",
       "4             0        0                0    0         0  \n",
       "...         ...      ...              ...  ...       ...  \n",
       "9995          0        0                0    0         0  \n",
       "9996          0        0                0    0         0  \n",
       "9997          0        0                0    0         0  \n",
       "9998          0        0                0    0         0  \n",
       "9999          0        0                0    0         0  \n",
       "\n",
       "[10000 rows x 45423 columns]"
      ]
     },
     "execution_count": 131,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cv_dataframe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "id": "cdc80455",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.9984647513374282"
      ]
     },
     "execution_count": 132,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "calculate_sparsity(cv_dataframe.to_numpy())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "id": "755463e2",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(10000, 45970)"
      ]
     },
     "execution_count": 125,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tf_idf_vec_smooth = TfidfVectorizer(use_idf=True,  \n",
    "                        smooth_idf=True,  \n",
    "                        ngram_range=(1,1), stop_words='english')\n",
    "\n",
    "X_unigram = tf_idf_vec_smooth.fit_transform(preprocessed_sentences)\n",
    "cv_dataframe_tfidf=pd.DataFrame(X_unigram.toarray(), columns=count_vectorizer.get_feature_names())\n",
    "cv_dataframe_tfidf.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 136,
   "id": "25d51a4f",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 10000/10000 [00:00<00:00, 1428237.14it/s]\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "['<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000596\">Amino Acids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000345\">Affinity Labels</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000906\">Antibodies</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020033\">Protein Isoforms</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029742\">Caenorhabditis elegans Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D006023\">Glycoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014764\">Viral Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.-</RegistryNumber><NameOfSubstance UI=\"D010447\">Peptide Hydrolases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000596\">Amino Acids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014764\">Viral Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019151\">Peptide Library</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009419\">Nerve Tissue Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D045424\">Complex Mixtures</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019151\">Peptide Library</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005782\">Gels</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.-</RegistryNumber><NameOfSubstance UI=\"D010447\">Peptide Hydrolases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005456\">Fluorescent Dyes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D004268\">DNA-Binding Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007554\">Isotopes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029968\">Escherichia coli Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008055\">Lipids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>N762921K75</RegistryNumber><NameOfSubstance UI=\"D009584\">Nitrogen</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029701\">Saccharomyces cerevisiae Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007328\">Insulin</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000097\">Acetonitriles</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>451W47IQ8X</RegistryNumber><NameOfSubstance UI=\"D012965\">Sodium Chloride</NameOfSubstance></Chemical><Chemical><RegistryNumber>Z072SB282N</RegistryNumber><NameOfSubstance UI=\"C032159\">acetonitrile</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D060365\">Chloroplast Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002021\">Buffers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015800\">Protozoan Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012333\">RNA, Messenger</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>1HG84L3525</RegistryNumber><NameOfSubstance UI=\"D005557\">Formaldehyde</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000906\">Antibodies</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D052157\">Aptamers, Nucleotide</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D061905\">Single-Domain Antibodies</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020418\">Algal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014688\">Venoms</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012997\">Solvents</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.-</RegistryNumber><NameOfSubstance UI=\"D010447\">Peptide Hydrolases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012471\">Salivary Proteins and Peptides</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002556\">Cerebrospinal Fluid Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D017293\">Protein S</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D018841\">HSP90 Heat-Shock Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000900\">Anti-Bacterial Agents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000970\">Antineoplastic Agents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000596\">Amino Acids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012997\">Solvents</NameOfSubstance></Chemical><Chemical><RegistryNumber>1HG84L3525</RegistryNumber><NameOfSubstance UI=\"D005557\">Formaldehyde</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>63231-63-0</RegistryNumber><NameOfSubstance UI=\"D012313\">RNA</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005456\">Fluorescent Dyes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007554\">Isotopes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029681\">Arabidopsis Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008024\">Ligands</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009842\">Oligopeptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019151\">Peptide Library</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000906\">Antibodies</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000941\">Antigens</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029721\">Drosophila Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019151\">Peptide Library</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012471\">Salivary Proteins and Peptides</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D013111\">Spider Venoms</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>K848JZ4886</RegistryNumber><NameOfSubstance UI=\"D003545\">Cysteine</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000941\">Antigens</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012910\">Snake Venoms</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D045424\">Complex Mixtures</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D004798\">Enzymes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007211\">Indoles</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>47165-04-8</RegistryNumber><NameOfSubstance UI=\"C007293\">DAPI</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>9035-51-2</RegistryNumber><NameOfSubstance UI=\"D003577\">Cytochrome P-450 Enzyme System</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019151\">Peptide Library</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>33X04XA5AT</RegistryNumber><NameOfSubstance UI=\"D019344\">Lactic Acid</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>68238-35-7</RegistryNumber><NameOfSubstance UI=\"D007633\">Keratins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D060829\">Arthropod Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D013111\">Spider Venoms</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009124\">Muscle Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012333\">RNA, Messenger</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029701\">Saccharomyces cerevisiae Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029701\">Saccharomyces cerevisiae Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D003432\">Cross-Linking Reagents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005404\">Fixatives</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D013388\">Succinimides</NameOfSubstance></Chemical><Chemical><RegistryNumber>10450-60-9</RegistryNumber><NameOfSubstance UI=\"D010504\">Periodic Acid</NameOfSubstance></Chemical><Chemical><RegistryNumber>62069-75-4</RegistryNumber><NameOfSubstance UI=\"C018052\">disuccinimidyl tartarate</NameOfSubstance></Chemical><Chemical><RegistryNumber>B45A1BUM4Q</RegistryNumber><NameOfSubstance UI=\"C009288\">metaperiodate</NameOfSubstance></Chemical><Chemical><RegistryNumber>EVY5D0U6SH</RegistryNumber><NameOfSubstance UI=\"C011240\">dithiobis(succinimidylpropionate)</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002556\">Cerebrospinal Fluid Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020033\">Protein Isoforms</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>63231-63-0</RegistryNumber><NameOfSubstance UI=\"D012313\">RNA</NameOfSubstance></Chemical><Chemical><RegistryNumber>9007-49-2</RegistryNumber><NameOfSubstance UI=\"D004247\">DNA</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D004800\">Enzymes, Immobilized</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007202\">Indicators and Reagents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008055\">Lipids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000345\">Affinity Labels</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005404\">Fixatives</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014020\">Tissue Extracts</NameOfSubstance></Chemical><Chemical><RegistryNumber>1HG84L3525</RegistryNumber><NameOfSubstance UI=\"D005557\">Formaldehyde</NameOfSubstance></Chemical><Chemical><RegistryNumber>K3Z4F929H6</RegistryNumber><NameOfSubstance UI=\"D008239\">Lysine</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000596\">Amino Acids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009363\">Neoplasm Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002413\">Cations, Divalent</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019216\">Metals, Heavy</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012910\">Snake Venoms</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D013501\">Surface-Active Agents</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007554\">Isotopes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D060749\">alpha-2-HS-Glycoprotein</NameOfSubstance></Chemical><Chemical><RegistryNumber>9007-34-5</RegistryNumber><NameOfSubstance UI=\"D003094\">Collagen</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011994\">Recombinant Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012997\">Solvents</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010936\">Plant Extracts</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000596\">Amino Acids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015335\">Molecular Probes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.21.-</RegistryNumber><NameOfSubstance UI=\"D012697\">Serine Endopeptidases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010103\">Oxygen Isotopes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.22.-</RegistryNumber><NameOfSubstance UI=\"D020169\">Caspases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029681\">Arabidopsis Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D060749\">alpha-2-HS-Glycoprotein</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029681\">Arabidopsis Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009842\">Oligopeptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019151\">Peptide Library</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000596\">Amino Acids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D034481\">Heterogeneous-Nuclear Ribonucleoprotein Group A-B</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D034741\">RNA, Small Interfering</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>K848JZ4886</RegistryNumber><NameOfSubstance UI=\"D003545\">Cysteine</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000911\">Antibodies, Monoclonal</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010851\">Picrates</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>1HG84L3525</RegistryNumber><NameOfSubstance UI=\"D005557\">Formaldehyde</NameOfSubstance></Chemical><Chemical><RegistryNumber>37330-50-0</RegistryNumber><NameOfSubstance UI=\"C026239\">Bouin\\'s solution</NameOfSubstance></Chemical><Chemical><RegistryNumber>Q40Q9N063P</RegistryNumber><NameOfSubstance UI=\"D019342\">Acetic Acid</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D003902\">Detergents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020025\">Peroxisome Proliferators</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>79956-01-7</RegistryNumber><NameOfSubstance UI=\"D048630\">Polyketide Synthases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000180\">Acrylic Resins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002457\">Cell Extracts</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"C016680\">polyacrylamide gels</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009587\">Nitrogen Isotopes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029968\">Escherichia coli Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012394\">Rosaniline Dyes</NameOfSubstance></Chemical><Chemical><RegistryNumber>M1ZRX790SI</RegistryNumber><NameOfSubstance UI=\"C004692\">coomassie Brilliant Blue</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000143\">Acids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007477\">Ions</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002556\">Cerebrospinal Fluid Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000970\">Antineoplastic Agents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009944\">Organoplatinum Compounds</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>04ZR38536J</RegistryNumber><NameOfSubstance UI=\"D000077150\">Oxaliplatin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.21.4</RegistryNumber><NameOfSubstance UI=\"D014357\">Trypsin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012333\">RNA, Messenger</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000970\">Antineoplastic Agents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002018\">Bufanolides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005353\">Fibronectins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>U549S98QLW</RegistryNumber><NameOfSubstance UI=\"C022777\">bufalin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D016326\">Extracellular Matrix Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.1.3.1</RegistryNumber><NameOfSubstance UI=\"D000469\">Alkaline Phosphatase</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009842\">Oligopeptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010748\">Phosphopeptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D003064\">Cnidarian Venoms</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008387\">Marine Toxins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019843\">Archaeal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002556\">Cerebrospinal Fluid Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>1HG84L3525</RegistryNumber><NameOfSubstance UI=\"D005557\">Formaldehyde</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 2.1.1.319</RegistryNumber><NameOfSubstance UI=\"D011484\">Protein-Arginine N-Methyltransferases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>1HG84L3525</RegistryNumber><NameOfSubstance UI=\"D005557\">Formaldehyde</NameOfSubstance></Chemical><Chemical><RegistryNumber>8002-74-2</RegistryNumber><NameOfSubstance UI=\"D010232\">Paraffin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000596\">Amino Acids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012492\">Salts</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"C017082\">brine</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.21.4</RegistryNumber><NameOfSubstance UI=\"D014357\">Trypsin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029701\">Saccharomyces cerevisiae Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009419\">Nerve Tissue Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D024101\">Mitochondrial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029701\">Saccharomyces cerevisiae Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007554\">Isotopes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000345\">Affinity Labels</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D004798\">Enzymes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011134\">Polysaccharides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029968\">Escherichia coli Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009419\">Nerve Tissue Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029968\">Escherichia coli Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009419\">Nerve Tissue Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011994\">Recombinant Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029701\">Saccharomyces cerevisiae Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010748\">Phosphopeptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014409\">Tumor Necrosis Factor-alpha</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D066329\">Protein Aggregates</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029701\">Saccharomyces cerevisiae Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>1HG84L3525</RegistryNumber><NameOfSubstance UI=\"D005557\">Formaldehyde</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D003432\">Cross-Linking Reagents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008024\">Ligands</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002241\">Carbohydrates</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.21.4</RegistryNumber><NameOfSubstance UI=\"D014357\">Trypsin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000911\">Antibodies, Monoclonal</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D054852\">Small Molecule Libraries</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019843\">Archaeal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010937\">Plant Growth Regulators</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D030161\">Avian Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>3604-79-3</RegistryNumber><NameOfSubstance UI=\"C002744\">3-nitrotyrosine</NameOfSubstance></Chemical><Chemical><RegistryNumber>42HK56048U</RegistryNumber><NameOfSubstance UI=\"D014443\">Tyrosine</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011108\">Polymers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012996\">Solutions</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D013501\">Surface-Active Agents</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D035683\">MicroRNAs</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D024101\">Mitochondrial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005782\">Gels</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D004527\">Egg Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"C015753\">chorion proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029742\">Caenorhabditis elegans Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>H0G9379FGK</RegistryNumber><NameOfSubstance UI=\"D002119\">Calcium Carbonate</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014764\">Viral Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.-</RegistryNumber><NameOfSubstance UI=\"D010447\">Peptide Hydrolases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009587\">Nitrogen Isotopes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012910\">Snake Venoms</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001688\">Biological Products</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D045424\">Complex Mixtures</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008055\">Lipids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019151\">Peptide Library</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.-</RegistryNumber><NameOfSubstance UI=\"D010447\">Peptide Hydrolases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005136\">Eye Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019151\">Peptide Library</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029701\">Saccharomyces cerevisiae Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009419\">Nerve Tissue Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D006023\">Glycoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012987\">Soil</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009363\">Neoplasm Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009363\">Neoplasm Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008372\">Manure</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002247\">Carbon Isotopes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>AR09D82C7G</RegistryNumber><NameOfSubstance UI=\"D003903\">Deuterium</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>A1TA934AKO</RegistryNumber><NameOfSubstance UI=\"D014994\">Xylose</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D016326\">Extracellular Matrix Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 1.8.1.7</RegistryNumber><NameOfSubstance UI=\"D005980\">Glutathione Reductase</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019476\">Insect Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011480\">Protease Inhibitors</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.-</RegistryNumber><NameOfSubstance UI=\"D010447\">Peptide Hydrolases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.21.4</RegistryNumber><NameOfSubstance UI=\"D014357\">Trypsin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000097\">Acetonitriles</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019843\">Archaeal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>Z072SB282N</RegistryNumber><NameOfSubstance UI=\"C032159\">acetonitrile</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D024101\">Mitochondrial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D006023\">Glycoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007202\">Indicators and Reagents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008055\">Lipids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000682\">Amyloid</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005635\">Fructosediphosphates</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011328\">Prions</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.21.4</RegistryNumber><NameOfSubstance UI=\"D014357\">Trypsin</NameOfSubstance></Chemical><Chemical><RegistryNumber>M7522JYX1H</RegistryNumber><NameOfSubstance UI=\"C029063\">fructose-1,6-diphosphate</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008387\">Marine Toxins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000480\">Alkynes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"C506098\">azidohomoalanine</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"C551041\">homopropargylglycine</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.5.4.-</RegistryNumber><NameOfSubstance UI=\"D000619\">Aminohydrolases</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.5.5.1</RegistryNumber><NameOfSubstance UI=\"C021867\">nitrilase</NameOfSubstance></Chemical><Chemical><RegistryNumber>OF5P57N2ZX</RegistryNumber><NameOfSubstance UI=\"D000409\">Alanine</NameOfSubstance></Chemical><Chemical><RegistryNumber>TE7660XO1C</RegistryNumber><NameOfSubstance UI=\"D005998\">Glycine</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D004220\">Disulfides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007554\">Isotopes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D046912\">Multiprotein Complexes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>7440-44-0</RegistryNumber><NameOfSubstance UI=\"D002244\">Carbon</NameOfSubstance></Chemical><Chemical><RegistryNumber>8558G7RUTR</RegistryNumber><NameOfSubstance UI=\"D019289\">Pyruvic Acid</NameOfSubstance></Chemical><Chemical><RegistryNumber>IY9XDZ35W2</RegistryNumber><NameOfSubstance UI=\"D005947\">Glucose</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009587\">Nitrogen Isotopes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029681\">Arabidopsis Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010636\">Phenols</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>1364PS73AF</RegistryNumber><NameOfSubstance UI=\"D000096\">Acetone</NameOfSubstance></Chemical><Chemical><RegistryNumber>Y4S76JWI15</RegistryNumber><NameOfSubstance UI=\"D000432\">Methanol</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000900\">Anti-Bacterial Agents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014344\">Trypanocidal Agents</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002247\">Carbon Isotopes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009587\">Nitrogen Isotopes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000906\">Antibodies</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D052157\">Aptamers, Nucleotide</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008024\">Ligands</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002155\">Calsequestrin</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D050778\">NFATC Transcription Factors</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D004364\">Pharmaceutical Preparations</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000900\">Anti-Bacterial Agents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>83HN0GTJ6D</RegistryNumber><NameOfSubstance UI=\"D016572\">Cyclosporine</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019151\">Peptide Library</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"C570448\">O-propargyl-puromycin</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>4A6ZS6Q2CL</RegistryNumber><NameOfSubstance UI=\"D011691\">Puromycin</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 2.7.1.1</RegistryNumber><NameOfSubstance UI=\"D058570\">TOR Serine-Threonine Kinases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"C515652\">AHSA1 protein, human</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007976\">Leupeptins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D018832\">Molecular Chaperones</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>RF1P63GW3K</RegistryNumber><NameOfSubstance UI=\"C072553\">benzyloxycarbonylleucyl-leucyl-leucine aldehyde</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007202\">Indicators and Reagents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011994\">Recombinant Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029721\">Drosophila Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D003902\">Detergents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011108\">Polymers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020005\">Propanols</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D013501\">Surface-Active Agents</NameOfSubstance></Chemical><Chemical><RegistryNumber>3D632GYQ50</RegistryNumber><NameOfSubstance UI=\"C001337\">hexafluoroisopropanol</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D035683\">MicroRNAs</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012333\">RNA, Messenger</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>9007-49-2</RegistryNumber><NameOfSubstance UI=\"D004247\">DNA</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D013434\">Sulfenic Acids</NameOfSubstance></Chemical><Chemical><RegistryNumber>6SO6U10H04</RegistryNumber><NameOfSubstance UI=\"D001710\">Biotin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D003435\">Crotalid Venoms</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>IY9XDZ35W2</RegistryNumber><NameOfSubstance UI=\"D005947\">Glucose</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012333\">RNA, Messenger</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029702\">Schizosaccharomyces pombe Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009363\">Neoplasm Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005404\">Fixatives</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>1HG84L3525</RegistryNumber><NameOfSubstance UI=\"D005557\">Formaldehyde</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014764\">Viral Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012709\">Serum Albumin</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.21.4</RegistryNumber><NameOfSubstance UI=\"D014357\">Trypsin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009363\">Neoplasm Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D013438\">Sulfhydryl Compounds</NameOfSubstance></Chemical><Chemical><RegistryNumber>K848JZ4886</RegistryNumber><NameOfSubstance UI=\"D003545\">Cysteine</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D004271\">DNA, Fungal</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012333\">RNA, Messenger</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029701\">Saccharomyces cerevisiae Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009363\">Neoplasm Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012333\">RNA, Messenger</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 2.7.11.24</RegistryNumber><NameOfSubstance UI=\"D048051\">p38 Mitogen-Activated Protein Kinases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D003459\">Crystallins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011663\">Pulmonary Surfactants</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D003432\">Cross-Linking Reagents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005960\">Glucosides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011744\">Pyrimidinones</NameOfSubstance></Chemical><Chemical><RegistryNumber>6SO6U10H04</RegistryNumber><NameOfSubstance UI=\"D001710\">Biotin</NameOfSubstance></Chemical><Chemical><RegistryNumber>COL14PJW3X</RegistryNumber><NameOfSubstance UI=\"C009661\">vicine</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.-</RegistryNumber><NameOfSubstance UI=\"D002403\">Cathepsins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D013438\">Sulfhydryl Compounds</NameOfSubstance></Chemical><Chemical><RegistryNumber>9012-36-6</RegistryNumber><NameOfSubstance UI=\"D012685\">Sepharose</NameOfSubstance></Chemical><Chemical><RegistryNumber>K848JZ4886</RegistryNumber><NameOfSubstance UI=\"D003545\">Cysteine</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.2.1.1</RegistryNumber><NameOfSubstance UI=\"D055292\">Pancreatic alpha-Amylases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000588\">Amines</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010748\">Phosphopeptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D013438\">Sulfhydryl Compounds</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.-</RegistryNumber><NameOfSubstance UI=\"D010447\">Peptide Hydrolases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>1364PS73AF</RegistryNumber><NameOfSubstance UI=\"D000096\">Acetone</NameOfSubstance></Chemical><Chemical><RegistryNumber>339NCG44TV</RegistryNumber><NameOfSubstance UI=\"D019800\">Phenol</NameOfSubstance></Chemical><Chemical><RegistryNumber>5V2JDO056X</RegistryNumber><NameOfSubstance UI=\"D014238\">Trichloroacetic Acid</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002556\">Cerebrospinal Fluid Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.21.4</RegistryNumber><NameOfSubstance UI=\"D014357\">Trypsin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D003432\">Cross-Linking Reagents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D064267\">Intrinsically Disordered Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000596\">Amino Acids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D003619\">Dansyl Compounds</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000596\">Amino Acids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D003470\">Culture Media</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D046912\">Multiprotein Complexes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015800\">Protozoan Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D006841\">Hydrocarbons, Aromatic</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D018797\">Cell Cycle Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D061988\">Proteasome Inhibitors</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014452\">Ubiquitins</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.6.1.-</RegistryNumber><NameOfSubstance UI=\"D000251\">Adenosine Triphosphatases</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.6.4.6</RegistryNumber><NameOfSubstance UI=\"C000614178\">VCP protein, human</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.6.4.6</RegistryNumber><NameOfSubstance UI=\"D000074405\">Valosin Containing Protein</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D017077\">Culture Media, Conditioned</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D006023\">Glycoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.21.5</RegistryNumber><NameOfSubstance UI=\"D013917\">Thrombin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D046912\">Multiprotein Complexes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012333\">RNA, Messenger</NameOfSubstance></Chemical><Chemical><RegistryNumber>K3Z4F929H6</RegistryNumber><NameOfSubstance UI=\"D008239\">Lysine</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010748\">Phosphopeptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002021\">Buffers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>3KX376GY7L</RegistryNumber><NameOfSubstance UI=\"D018698\">Glutamic Acid</NameOfSubstance></Chemical><Chemical><RegistryNumber>RWW266YE9I</RegistryNumber><NameOfSubstance UI=\"D006531\">HEPES</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014764\">Viral Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>63231-63-0</RegistryNumber><NameOfSubstance UI=\"D012313\">RNA</NameOfSubstance></Chemical><Chemical><RegistryNumber>9007-49-2</RegistryNumber><NameOfSubstance UI=\"D004247\">DNA</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015335\">Molecular Probes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008024\">Ligands</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019151\">Peptide Library</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010748\">Phosphopeptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019208\">Brain-Derived Neurotrophic Factor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"C506098\">azidohomoalanine</NameOfSubstance></Chemical><Chemical><RegistryNumber>OF5P57N2ZX</RegistryNumber><NameOfSubstance UI=\"D000409\">Alanine</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D004800\">Enzymes, Immobilized</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>27432CM55Q</RegistryNumber><NameOfSubstance UI=\"D012710\">Serum Albumin, Bovine</NameOfSubstance></Chemical><Chemical><RegistryNumber>7631-86-9</RegistryNumber><NameOfSubstance UI=\"D012822\">Silicon Dioxide</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.21.4</RegistryNumber><NameOfSubstance UI=\"D014357\">Trypsin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D017382\">Reactive Oxygen Species</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>K848JZ4886</RegistryNumber><NameOfSubstance UI=\"D003545\">Cysteine</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002021\">Buffers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>059QF0KO0R</RegistryNumber><NameOfSubstance UI=\"D014867\">Water</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011720\">Pyrazoles</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011725\">Pyridines</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019216\">Metals, Heavy</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009419\">Nerve Tissue Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D024101\">Mitochondrial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009363\">Neoplasm Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011257\">Pregnancy Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029968\">Escherichia coli Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019151\">Peptide Library</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D030262\">Soybean Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012471\">Salivary Proteins and Peptides</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005404\">Fixatives</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>1HG84L3525</RegistryNumber><NameOfSubstance UI=\"D005557\">Formaldehyde</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D003517\">Cyclopentanes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011743\">Pyrimidines</NameOfSubstance></Chemical><Chemical><RegistryNumber>S3AZD8D215</RegistryNumber><NameOfSubstance UI=\"C539933\">pevonedistat</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012910\">Snake Venoms</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007202\">Indicators and Reagents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D006020\">Glycopeptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010748\">Phosphopeptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.21.4</RegistryNumber><NameOfSubstance UI=\"D014357\">Trypsin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019476\">Insect Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D004798\">Enzymes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019151\">Peptide Library</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015800\">Protozoan Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D037521\">Virulence Factors</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"C514153\">1,3-bis(bis(pyridin-2-ylmethyl)amino)propan-2-ol</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011725\">Pyridines</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.21.4</RegistryNumber><NameOfSubstance UI=\"D014357\">Trypsin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009687\">Nuclear Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019476\">Insect Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008024\">Ligands</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015335\">Molecular Probes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>6SO6U10H04</RegistryNumber><NameOfSubstance UI=\"D001710\">Biotin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009363\">Neoplasm Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000485\">Allergens</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019151\">Peptide Library</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029701\">Saccharomyces cerevisiae Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D012333\">RNA, Messenger</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009419\">Nerve Tissue Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D013438\">Sulfhydryl Compounds</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 5.3.4.1</RegistryNumber><NameOfSubstance UI=\"D019704\">Protein Disulfide-Isomerases</NameOfSubstance></Chemical><Chemical><RegistryNumber>RMB44WO89X</RegistryNumber><NameOfSubstance UI=\"D006909\">Hydroxyproline</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011498\">Protein Precursors</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029721\">Drosophila Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005227\">Fatty Acids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>57-50-1</RegistryNumber><NameOfSubstance UI=\"D013395\">Sucrose</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D024101\">Mitochondrial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014157\">Transcription Factors</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D009363\">Neoplasm Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029681\">Arabidopsis Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.21.4</RegistryNumber><NameOfSubstance UI=\"D014357\">Trypsin</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002021\">Buffers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007475\">Ion Exchange Resins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010938\">Plant Oils</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.-</RegistryNumber><NameOfSubstance UI=\"D045726\">Metalloproteases</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.-</RegistryNumber><NameOfSubstance UI=\"C512490\">ulilysin, Methanosarcina acetivorans</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002232\">Carbocyanines</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005456\">Fluorescent Dyes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D024101\">Mitochondrial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D007202\">Indicators and Reagents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D013501\">Surface-Active Agents</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.21.4</RegistryNumber><NameOfSubstance UI=\"D014357\">Trypsin</NameOfSubstance></Chemical><Chemical><RegistryNumber>JU58VJ6Y3B</RegistryNumber><NameOfSubstance UI=\"D019791\">Guanidine</NameOfSubstance></Chemical><Chemical><RegistryNumber>Y4S76JWI15</RegistryNumber><NameOfSubstance UI=\"D000432\">Methanol</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000596\">Amino Acids</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 2.7.-</RegistryNumber><NameOfSubstance UI=\"D011494\">Protein Kinases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.6.5.2</RegistryNumber><NameOfSubstance UI=\"D020761\">rab1 GTP-Binding Proteins</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005456\">Fluorescent Dyes</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010446\">Peptide Fragments</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D014408\">Biomarkers, Tumor</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001798\">Blood Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002352\">Carrier Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D019151\">Peptide Library</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>EC 3.4.-</RegistryNumber><NameOfSubstance UI=\"D010447\">Peptide Hydrolases</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010750\">Phosphoproteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010940\">Plant Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>E1UOL152H7</RegistryNumber><NameOfSubstance UI=\"D007501\">Iron</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005136\">Eye Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D008565\">Membrane Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D015415\">Biomarkers</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D010455\">Peptides</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D000935\">Antifungal Agents</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D011506\">Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D002556\">Cerebrospinal Fluid Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D001426\">Bacterial Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D021382\">Protein Sorting Signals</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D037521\">Virulence Factors</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D029721\">Drosophila Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D005656\">Fungal Proteins</NameOfSubstance></Chemical><Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " '<Chemical><RegistryNumber>0</RegistryNumber><NameOfSubstance UI=\"D020543\">Proteome</NameOfSubstance></Chemical>',\n",
       " ...]"
      ]
     },
     "execution_count": 136,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "chemical_list_data = []\n",
    "\n",
    "count = 0\n",
    "for pmid in tqdm(json_data.keys()):\n",
    "    chemicals = json_data[pmid][\"medline\"]\n",
    "    chemical_list_data.append(chemicals)\n",
    "    if \"Proteome\" in chemicals:\n",
    "        count += 1\n",
    "chemical_list_data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "id": "75a88183",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0h</th>\n",
       "      <th>11</th>\n",
       "      <th>14</th>\n",
       "      <th>14βdglucosidic</th>\n",
       "      <th>1a</th>\n",
       "      <th>1a1</th>\n",
       "      <th>1a1b</th>\n",
       "      <th>1a1blight</th>\n",
       "      <th>1acid</th>\n",
       "      <th>1alpha</th>\n",
       "      <th>...</th>\n",
       "      <th>μmol</th>\n",
       "      <th>μmolg</th>\n",
       "      <th>μmsup2sup</th>\n",
       "      <th>μmthick</th>\n",
       "      <th>οphenanthroline</th>\n",
       "      <th>ππ</th>\n",
       "      <th>ρ0</th>\n",
       "      <th>ρlt</th>\n",
       "      <th>σsupssup</th>\n",
       "      <th>χ²</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 45970 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    0h   11   14  14βdglucosidic   1a  1a1  1a1b  1a1blight  1acid  1alpha  \\\n",
       "0  0.0  0.0  0.0             0.0  0.0  0.0   0.0        0.0    0.0     0.0   \n",
       "1  0.0  0.0  0.0             0.0  0.0  0.0   0.0        0.0    0.0     0.0   \n",
       "2  0.0  0.0  0.0             0.0  0.0  0.0   0.0        0.0    0.0     0.0   \n",
       "3  0.0  0.0  0.0             0.0  0.0  0.0   0.0        0.0    0.0     0.0   \n",
       "4  0.0  0.0  0.0             0.0  0.0  0.0   0.0        0.0    0.0     0.0   \n",
       "\n",
       "   ...  μmol  μmolg  μmsup2sup  μmthick  οphenanthroline   ππ   ρ0  ρlt  \\\n",
       "0  ...   0.0    0.0        0.0      0.0              0.0  0.0  0.0  0.0   \n",
       "1  ...   0.0    0.0        0.0      0.0              0.0  0.0  0.0  0.0   \n",
       "2  ...   0.0    0.0        0.0      0.0              0.0  0.0  0.0  0.0   \n",
       "3  ...   0.0    0.0        0.0      0.0              0.0  0.0  0.0  0.0   \n",
       "4  ...   0.0    0.0        0.0      0.0              0.0  0.0  0.0  0.0   \n",
       "\n",
       "   σsupssup   χ²  \n",
       "0       0.0  0.0  \n",
       "1       0.0  0.0  \n",
       "2       0.0  0.0  \n",
       "3       0.0  0.0  \n",
       "4       0.0  0.0  \n",
       "\n",
       "[5 rows x 45970 columns]"
      ]
     },
     "execution_count": 127,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cv_dataframe_tfidf.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4ed3b672",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
